<!DOCTYPE html>
<html>
<head>

     

        <link rel="stylesheet" href="http://example.org/css/styles.css">
        <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
        
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css" integrity="sha256-+N4/V/SbAFiW1MPBCXnfnP9QSN3+Keu+NlB+0ev/YKQ=" crossorigin="anonymous" />

    
    
    

    <script src="http://example.org/js/bundle.min.js"></script>
</head>
<body class="font-sans antialiased bg-grey-lightest">

<style>
    .fancy-btn-blue {
        background-image: linear-gradient(148deg, #5DBDFF 0%, #1C43D9 100%);
        box-shadow: 2px 2px 5px 0 rgba(0, 0, 0, 0.22);
        border-radius: 2rem;
        margin-top: 0.75rem !important;
        padding: 0.5rem 0.75rem 0.5rem 0.75rem !important;
    }

    .fancy-btn-blue:hover {
        box-shadow: 4px 4px 10px 0 rgba(0, 0, 0, 0.3);
        margin-top: 0.6rem !important;
        padding: 0.65rem 0.9rem 0.65rem 0.9rem;
    }

     
    .btn-arrow {
        display: block;
        background: url('/images/arrow-right.svg');
        background-repeat: no-repeat;
        background-size: 15px 11px;
        background-position: 0px 4px;
        margin-left: 6px;
        padding-left: 26px;

        -webkit-transition: background-position .05s ease-in;
        -moz-transition: background-position .05s ease-in;
        -o-transition: background-position .05s ease-in;
        transition: background-position .05s ease-in;
    }

    .btn-arrow:hover {
        background-position: 3px 4px;
    }

     
    .social-bar {
        position: fixed;
        left: 20px;
        top: 50%;
        margin-top: -98px;
        width: 196px;
    }

    .social-bar a {
        display: block;
        width: 39px;
        height: 39px;
        opacity: 0.4;
        padding: 0;
        margin: 0;
    }

    .social-bar a:hover {
        opacity: 0.6;
    }

    .social-bar ul {
        display: block;
        width: 39px;
    }

    .social-bar li {
        display: block !important;
        padding: 0;
        margin: 5px 0;
        width: 39px;
        height: 39px;
    }

    .facebook {
        background: url('/images/facebook.svg');
        background-size: 39px 39px;
    }

    .instagram {
        background: url('/images/instagram.svg');
        background-size: 39px 39px;
    }

    .twitter {
        background: url('/images/twitter.svg');
        background-size: 39px 39px;
    }

    .youtube {
        background: url('/images/youtube.svg');
        background-size: 39px 39px;
    }

     

    nav {
        position: absolute;
        top: 85px;
        right: 50px;
    }

    .sticky-header nav {
        top: 15px;
        right: 50px;
    }

    header ul, .sticky-header ul {
        list-style: none;
        margin: 0;
        padding-left: 0;
    }

    header li, .sticky-header li {
        color: #fff;
        display: block;
        float: left;
        padding: 1rem;
        position: relative;
        text-decoration: none;
        transition-duration: 0.5s;
    }

    header li:hover,
    header li:focus-within,
    .sticky-header li:hover,
    .sticky-header li:focus-within {
        cursor: pointer;
    }

    header li:focus-within a,
    .sticky-header li:focus-within a {
        outline: none;
    }

    header ul li,
    .sticky-header ul li {
        width: 140px;
        font-size: 0.8rem;
        margin: 0.1rem;
    }

    header ul li ul,
    .sticky-header ul li ul {
        visibility: hidden;
        opacity: 0;
        display: none;

         
        width: 140px;
         
        transition: all 0.5s ease;
        margin-top: 1rem;
        left: 0;
    }

    header ul li:hover>ul,
    header ul li:focus-within>ul,
    header ul li ul:hover,
    header ul li ul:focus, 
    .sticky-header ul li:hover>ul,
    .sticky-header ul li:focus-within>ul,
    .sticky-header ul li ul:hover,
    .sticky-header ul li ul:focus 
    {
        visibility: visible;
        opacity: 1;
        display: block;
    }

    header ul li ul li,
    .sticky-header ul li ul li {
        clear: both;
        font-size: 0.7rem;
        margin: -1rem;
        text-shadow: none;
        width: 140px;
         
    }

    header ul li ul li a,
    .sticky-header ul li ul li a {
        display: block;
        color: #737373;
        margin-left: -0.6rem;
        margin-right: -0.6rem;
        padding: 0.6rem;
        border: 1px solid transparent;
    }

    header ul li ul li:last-child,
    .sticky-header ul li ul li:last-child {
        margin-bottom: -1.6rem;
    }

    header ul li ul li a:hover,
    .sticky-header ul li ul li a:hover {
        background: rgba(255, 255, 255, 0.4);
        border-radius: 6px;
        border: 1px solid rgba(0, 0, 0, 0.1);
    }

    .dropdown-item {
        text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.30);
        border-radius: 6px;
    }

    #mobile-navigation {
        position: fixed;
        right: 0 !important;
        top: 0;
        margin: 0;
        background: rgba(255, 255, 255, 0.8);
        color: black !important;
        box-shadow: -10px 0 -20px rgba(2,10,50,1);
        height: 100vh;
        z-index: 9001;

        transition: all 0.1s ease-in-out;
        overflow: scroll;
    }

    #mobile-navigation ul {
        display: block;
        width: 200px;
    }

    #mobile-navigation li {
        display: block !important;
        padding: 1rem;
        margin: 5px 0;
        height: 39px;
        font-size: 10px;
        width: 100%;
        color: black !important;
    }

    #mobile-navigation li a {
        display: block;
        color: #737373;
        margin-left: -0.6rem;
        margin-right: -0.6rem;
        padding: 0.6rem;
        border: 1px solid transparent;
    }

    #mobile-navigation li a:hover {
        background: rgba(255, 255, 255, 0.4);
        border-radius: 6px;
        border: 1px solid rgba(0, 0, 0, 0.1);
    }

    .has-dropdown::after {
        display: inline-block;
        content: ' ';
        margin-left: 0.3rem;
        background: url('/images/dropdown.svg');
        text-shadow: 0px 0px 0px transparent;
        width: 12px;
        height: 9px;
    }

    .dropdown-item:hover,
    .dropdown-item:focus,
    .dropdown-item:focus-within {
        background-color: rgba(255, 255, 255, 0.8);
        box-shadow: 7px 7px 25px 0 rgba(0, 0, 0, 0.22);
        width: 140px;
        color: #2D4459;
    }

    .sticky-header .dropdown-item:hover,
    .sticky-header .dropdown-item:focus,
    .sticky-header .dropdown-item:focus-within {
        background-color: rgba(255, 255, 255, 0.95);
    }

    .menu-dropdown {
        background: transparent;
        -moz-transition: all 0.2s ease-in;
        -webkit-transition: all 0.2s ease-in;
        -o-transition: all 0.2s ease-in;
        transition: all 0.2s ease-in;
        border-radius: 6px;
    }

    .menu-dropdown:hover {
         
        background: rgba(255, 255, 255, 0.6);
        box-shadow: 7px 7px 25px 0 rgba(0, 0, 0, 0.22);
    }

    .menu-item {
        border: 1px transparent;
        color: #737373;
    }

    .menu-item:hover {
         
        box-shadow: 2px 2px 4px 0 rgba(0, 0, 0, 0.05);
        background: rgba(0, 0, 0, 0.1);
        border-radius: 4px;
    }
</style>

<header style="background: url('/images/background-blue.svg') no-repeat; min-height: 1157px; background-size: 100vw">
    <div class="mx- pt-6 w-full">

        
        <a href="http://example.org/">
        <div id="logo" class="inline-block align-middle ml-6 mt-6">
            <svg width="180px" height="123px" viewBox="0 0 210 143" version="1.1" xmlns="http://www.w3.org/2000/svg"
                xmlns:xlink="http://www.w3.org/1999/xlink">
                <defs>
                    <filter x="-15.4%" y="-24.0%" width="130.9%" height="148.0%" filterUnits="objectBoundingBox"
                        id="filter-1">
                        <feOffset dx="5" dy="5" in="SourceAlpha" result="shadowOffsetOuter1"></feOffset>
                        <feGaussianBlur stdDeviation="5" in="shadowOffsetOuter1" result="shadowBlurOuter1">
                        </feGaussianBlur>
                        <feColorMatrix values="0 0 0 0 0   0 0 0 0 0   0 0 0 0 0  0 0 0 0.205747378 0" type="matrix"
                            in="shadowBlurOuter1" result="shadowMatrixOuter1"></feColorMatrix>
                        <feMerge>
                            <feMergeNode in="shadowMatrixOuter1"></feMergeNode>
                            <feMergeNode in="SourceGraphic"></feMergeNode>
                        </feMerge>
                    </filter>
                    <linearGradient x1="0%" y1="49.1585099%" x2="101.999998%" y2="50.8583199%" id="linearGradient-2">
                        <stop stop-color="#FAAC39" offset="0%"></stop>
                        <stop stop-color="#E3209C" offset="100%"></stop>
                    </linearGradient>
                    <linearGradient x1="106.481982%" y1="58.6047602%" x2="-7.35198796%" y2="43.2132964%"
                        id="linearGradient-3">
                        <stop stop-color="#1D5279" offset="0%"></stop>
                        <stop stop-color="#0F2B45" offset="100%"></stop>
                    </linearGradient>
                    <path
                        d="M177,72 L170.46578,77.3333333 L139,77.3333333 L139,72 L177,72 Z M166.381892,80.6666667 L163.205247,83.2594945 L171.016083,86 L139,86 L139,80.6666667 L166.381892,80.6666667 Z"
                        id="path-4"></path>
                    <linearGradient x1="99.1374639%" y1="50%" x2="0%" y2="50%" id="linearGradient-5">
                        <stop stop-color="#FFFFFF" offset="0%"></stop>
                        <stop stop-color="#FFFFFF" offset="40.0299012%"></stop>
                        <stop stop-color="#B5F829" offset="44.8825793%"></stop>
                        <stop stop-color="#15F2F3" offset="55.5045952%"></stop>
                        <stop stop-color="#FFFFFF" offset="60.6309638%"></stop>
                        <stop stop-color="#FFFFFF" offset="100%"></stop>
                    </linearGradient>
                    <linearGradient x1="106.481982%" y1="51.4314786%" x2="-7.35198796%" y2="48.8709714%"
                        id="linearGradient-6">
                        <stop stop-color="#1D5279" offset="0%"></stop>
                        <stop stop-color="#0F2B45" offset="100%"></stop>
                    </linearGradient>
                    <rect id="path-7" x="0" y="8.66666667" width="35.4920635" height="5.33333333"></rect>
                    <linearGradient x1="106.481982%" y1="51.3779701%" x2="-7.35198796%" y2="48.9131743%"
                        id="linearGradient-8">
                        <stop stop-color="#1D5279" offset="0%"></stop>
                        <stop stop-color="#0F2B45" offset="100%"></stop>
                    </linearGradient>
                    <rect id="path-9" x="49.8253968" y="0" width="36.1746032" height="5.33333333"></rect>
                    <linearGradient x1="0%" y1="0%" x2="97.6494292%" y2="96.6920827%" id="linearGradient-10">
                        <stop stop-color="#0FE6F1" offset="0%"></stop>
                        <stop stop-color="#CFF968" offset="100%"></stop>
                    </linearGradient>
                    <linearGradient x1="2.44820263%" y1="3.97792628%" x2="123.107123%" y2="82.6286156%"
                        id="linearGradient-11">
                        <stop stop-color="#DA1AE6" offset="0%"></stop>
                        <stop stop-color="#140CF7" offset="100%"></stop>
                    </linearGradient>
                    <linearGradient x1="34.40034%" y1="27.1369695%" x2="95.5986745%" y2="100.573377%"
                        id="linearGradient-12">
                        <stop stop-color="#FFCC23" offset="0%"></stop>
                        <stop stop-color="#E3209C" offset="94.1067558%"></stop>
                        <stop stop-color="#E115A3" offset="100%"></stop>
                    </linearGradient>
                </defs>
                <g id="Website" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                    <g id="Full-HD" transform="translate(-53.000000, -33.000000)">
                        <g id="Logo-(Dark-BG)" filter="url(#filter-1)" transform="translate(58.000000, 38.000000)">
                            <path
                                d="M5.37577552,59.3125 C4.42555247,59.3125 3.62537665,58.9802117 2.97522404,58.315625 C2.32507143,57.6510383 2,56.8330978 2,55.8617788 L2,22.9649038 C2,21.9935849 2.32507143,21.162864 2.97522404,20.4727163 C3.62537665,19.7825687 4.42555247,19.4375 5.37577552,19.4375 C6.3760103,19.4375 7.20119162,19.7697883 7.85134423,20.434375 C8.50149684,21.0989617 8.82656827,21.9424628 8.82656827,22.9649038 L8.82656827,55.8617788 C8.82656827,56.8330978 8.48899409,57.6510383 7.81383561,58.315625 C7.13867713,58.9802117 6.32599856,59.3125 5.37577552,59.3125 Z M45.8192778,60 C40.5855493,60 35.9362388,58.8382196 31.871207,56.514624 C27.8061751,54.1910283 24.6558227,50.9482567 22.4200552,46.7862117 C20.1842877,42.6241667 19.0664207,37.8876774 19.0664207,32.5766017 C19.0664207,27.1633898 20.2350998,22.3630664 22.5724931,18.1754875 C24.9098864,13.9879085 28.0983479,10.7451368 32.1379733,8.44707521 C36.1775987,6.14901358 40.6617697,5 45.5906209,5 C49.5540269,5 53.0219552,5.48513907 55.9945097,6.45543175 C58.9670643,7.42572444 61.8252469,8.7790071 64.5691434,10.5153203 C64.9756466,10.7195925 65.3313315,11.0770634 65.6362089,11.5877437 C65.9410863,12.0984241 66.0935227,12.6601641 66.0935227,13.2729805 C66.0935227,14.1411371 65.8013529,14.8816125 65.2170046,15.494429 C64.6326563,16.1072454 63.8577712,16.413649 62.8923262,16.413649 C62.3333843,16.413649 61.7998568,16.2859808 61.2917279,16.0306407 C58.9035217,14.6007357 56.5153513,13.528323 54.1271451,12.8133705 C51.7389388,12.098418 49.1475199,11.7409471 46.3528105,11.7409471 C42.4910303,11.7409471 39.010399,12.5580234 35.9108122,14.1922006 C32.8112254,15.8263777 30.3849459,18.2137726 28.6319009,21.3544568 C26.8788559,24.4951411 26.0023466,28.2358186 26.0023466,32.5766017 C26.0023466,36.5599085 26.8280438,40.1218506 28.479463,43.2625348 C30.1308822,46.403219 32.5190526,48.8799817 35.6440458,50.6928969 C38.769039,52.5058122 42.4656219,53.4122563 46.7339053,53.4122563 C51.5103178,53.4122563 55.9563797,52.2887763 60.0722244,50.0417827 L60.0722244,36.1768802 L47.496095,36.1768802 C46.6322757,36.1768802 45.8954998,35.8704766 45.285745,35.2576602 C44.6759902,34.6448437 44.3711174,33.9043683 44.3711174,33.0362117 C44.3711174,32.1680551 44.6759902,31.4275797 45.285745,30.8147632 C45.8954998,30.2019468 46.6322757,29.8955432 47.496095,29.8955432 L62.5112313,29.8955432 C63.5274893,29.8955432 64.3912956,30.2530141 65.1026762,30.9679666 C65.8140567,31.6829191 66.1697417,32.5510627 66.1697417,33.5724234 L66.1697417,50.9610028 C66.1697417,52.0334315 65.902978,52.9909429 65.3694426,53.8335655 C64.8359071,54.6761881 64.1372403,55.3017621 63.273421,55.7103064 C57.5823764,58.5701164 51.7643868,60 45.8192778,60 Z M83.2780431,59.3125 C82.3054733,59.3125 81.4864795,58.9795879 80.821037,58.3137536 C80.1555945,57.6479193 79.8228782,56.8284433 79.8228782,55.8553009 L79.8228782,9.22152579 C79.8228782,8.24838339 80.1555945,7.41610303 80.821037,6.72465974 C81.4864795,6.03321646 82.3054733,5.6875 83.2780431,5.6875 L114.067402,5.6875 C114.988784,5.6875 115.769387,6.00760783 116.409236,6.64783309 C117.049084,7.28805836 117.369004,8.06912146 117.369004,8.99104585 C117.369004,9.91297023 117.049084,10.681229 116.409236,11.2958453 C115.769387,11.9104615 114.988784,12.217765 114.067402,12.217765 L86.8099895,12.217765 L86.8099895,29.1964542 L111.30327,29.1964542 C112.224652,29.1964542 113.005255,29.516562 113.645104,30.1567872 C114.284952,30.7970125 114.604872,31.5780756 114.604872,32.5 C114.604872,33.4219244 114.284952,34.1901832 113.645104,34.8047994 C113.005255,35.4194157 112.224652,35.7267192 111.30327,35.7267192 L86.8099895,35.7267192 L86.8099895,52.782235 L114.067402,52.782235 C114.988784,52.782235 115.769387,53.1023428 116.409236,53.7425681 C117.049084,54.3827933 117.369004,55.1638564 117.369004,56.0857808 C117.369004,57.0077052 117.049084,57.775964 116.409236,58.3905802 C115.769387,59.0051965 114.988784,59.3125 114.067402,59.3125 L83.2780431,59.3125 Z M131.71364,60 C130.75037,60 129.939207,59.6675822 129.280127,59.0027366 C128.621048,58.337891 128.291513,57.5196317 128.291513,56.5479343 L128.291513,10.7505297 C128.291513,9.36969649 128.773141,8.18066353 129.736411,7.18339513 C130.699681,6.18612672 131.865727,5.6875 133.234585,5.6875 L135.744145,5.6875 C136.758113,5.6875 137.683346,5.99434721 138.51987,6.60805085 C139.356394,7.22175448 139.97744,7.98887252 140.383028,8.90942797 L157.645756,52.3287429 L174.908485,8.90942797 C175.314073,7.98887252 175.935119,7.22175448 176.771643,6.60805085 C177.608167,5.99434721 178.5334,5.6875 179.547368,5.6875 L182.056928,5.6875 C183.425785,5.6875 184.591832,6.18612672 185.555102,7.18339513 C186.518372,8.18066353 187,9.36969649 187,10.7505297 L187,56.5479343 C187,57.5196317 186.670465,58.337891 186.011386,59.0027366 C185.352306,59.6675822 184.541143,60 183.577873,60 C182.614603,60 181.790766,59.6675822 181.106337,59.0027366 C180.421908,58.337891 180.079699,57.5196317 180.079699,56.5479343 L180.079699,12.9751942 L162.512781,56.778072 C162.107194,57.7497695 161.460799,58.5296728 160.573576,59.1178054 C159.686354,59.7059381 158.710423,60 157.645756,60 C156.581089,60 155.605159,59.7059381 154.717937,59.1178054 C153.830714,58.5296728 153.184319,57.7497695 152.778732,56.778072 L135.211814,12.9751942 L135.211814,56.5479343 C135.211814,57.5196317 134.869605,58.337891 134.185176,59.0027366 C133.500747,59.6675822 132.67691,60 131.71364,60 Z"
                                id="iGEM-Copy" fill="#FFFFFF"></path>
                            <path
                                d="M3.50770833,123 C3.08331423,123 2.72593509,122.855143 2.43556018,122.565425 C2.14518528,122.275707 2,121.919136 2,121.495702 L2,101.204394 C2,100.780959 2.14518528,100.418817 2.43556018,100.117956 C2.72593509,99.817095 3.08331423,99.6666667 3.50770833,99.6666667 L16.9430648,99.6666667 C17.3451223,99.6666667 17.6857493,99.8059521 17.964956,100.084527 C18.2441626,100.363102 18.3837638,100.702959 18.3837638,101.104107 C18.3837638,101.505255 18.2441626,101.83954 17.964956,102.106972 C17.6857493,102.374404 17.3451223,102.508118 16.9430648,102.508118 L5.04892129,102.508118 L5.04892129,109.895893 L15.7368981,109.895893 C16.1389557,109.895893 16.4795827,110.035178 16.7587893,110.313754 C17.0379959,110.592329 17.1775972,110.932185 17.1775972,111.333333 C17.1775972,111.734481 17.0379959,112.068767 16.7587893,112.336199 C16.4795827,112.603631 16.1389557,112.737345 15.7368981,112.737345 L5.04892129,112.737345 L5.04892129,120.158548 L16.9430648,120.158548 C17.3451223,120.158548 17.6857493,120.297834 17.964956,120.576409 C18.2441626,120.854984 18.3837638,121.19484 18.3837638,121.595989 C18.3837638,121.997137 18.2441626,122.331422 17.964956,122.598854 C17.6857493,122.866286 17.3451223,123 16.9430648,123 L3.50770833,123 Z M25.3530694,123 C24.9285789,123 24.5711185,122.856173 24.2806776,122.568516 C23.9902367,122.280858 23.8450185,121.926824 23.8450185,121.506401 L23.8450185,101.193457 C23.8450185,100.773034 23.9902367,100.413468 24.2806776,100.114746 C24.5711185,99.8160251 24.9285789,99.6666667 25.3530694,99.6666667 L33.1614223,99.6666667 C36.199881,99.6666667 38.4786908,100.286227 39.9979202,101.525367 C41.5171495,102.764508 42.2767528,104.390854 42.2767528,106.404457 C42.2767528,107.732107 41.9025364,108.94357 41.1540926,110.038881 C40.4056487,111.134192 39.3165117,111.925239 37.8866488,112.412044 C39.4952445,112.832466 40.5955522,113.822657 41.1876048,115.382646 C41.7796574,116.942635 42.0756793,118.474941 42.0756793,119.979611 L42.0756793,121.506401 C42.0756793,121.926824 41.930461,122.280858 41.6400201,122.568516 C41.3495792,122.856173 40.9921189,123 40.5676283,123 C40.0984545,123 39.7242382,122.861705 39.4449681,122.585111 C39.165698,122.308518 39.0260651,121.948951 39.0260651,121.506401 L39.0260651,120.510669 C39.0260651,119.404294 38.9255294,118.380912 38.7244549,117.440493 C38.5233804,116.500074 38.043043,115.659242 37.2834284,114.917971 C36.5238137,114.176699 35.3732382,113.806069 33.8316672,113.806069 L26.8946327,113.806069 L26.8946327,121.506401 C26.8946327,121.926824 26.7438291,122.280858 26.4422174,122.568516 C26.1406057,122.856173 25.77756,123 25.3530694,123 Z M32.6193847,111 C34.5338015,111 36.0530687,110.613302 37.1772321,109.839895 C38.3013954,109.066488 38.8634686,107.974635 38.8634686,106.564304 C38.8634686,105.199468 38.3793065,104.15311 37.4109678,103.425197 C36.4426291,102.697284 34.7897999,102.333333 32.4524306,102.333333 L26.5756458,102.333333 L26.5756458,111 L32.6193847,111 Z M49.8698387,122.333333 C49.2833002,122.333333 48.7813661,122.128657 48.3640213,121.719298 C47.9466766,121.309939 47.7380074,120.81761 47.7380074,120.242295 L47.7380074,100.52679 C47.7380074,100.106367 47.8846398,99.7468009 48.1779091,99.4480797 C48.4711784,99.1493584 48.8321197,99 49.2607441,99 C49.6893684,99 50.0559495,99.1493584 50.3604983,99.4480797 C50.6650472,99.7468009 50.8173193,100.106367 50.8173193,100.52679 L50.8173193,119.312945 L61.8825391,119.312945 C62.3111634,119.312945 62.6777445,119.462303 62.9822934,119.761024 C63.2868422,120.059745 63.4391144,120.419312 63.4391144,120.839734 C63.4391144,121.260157 63.2868422,121.614192 62.9822934,121.901849 C62.6777445,122.189507 62.3111634,122.333333 61.8825391,122.333333 L49.8698387,122.333333 Z M67.0087967,123 C66.5579169,123 66.1915825,122.853109 65.9097826,122.559322 C65.6279827,122.265535 65.4870849,121.903957 65.4870849,121.474576 C65.4870849,121.293784 65.543444,121.067798 65.656164,120.79661 L74.2791976,100.389831 C74.4595495,99.9830488 74.7131656,99.6497188 75.0400535,99.3898305 C75.3669413,99.1299422 75.7670912,99 76.240515,99 L76.5786732,99 C77.052097,99 77.4522469,99.1299422 77.7791347,99.3898305 C78.1060226,99.6497188 78.3596387,99.9830488 78.5399906,100.389831 L87.1630242,120.79661 C87.2757442,121.067798 87.3321033,121.293784 87.3321033,121.474576 C87.3321033,121.903957 87.1912055,122.265535 86.9094056,122.559322 C86.6276057,122.853109 86.2612713,123 85.8103915,123 C85.4947756,123 85.2073441,122.915255 84.9480881,122.745763 C84.6888322,122.57627 84.4915753,122.344634 84.3563113,122.050847 L82.1582832,116.762712 L70.660905,116.762712 L68.4628769,122.050847 C68.3276129,122.344634 68.1303559,122.57627 67.8711,122.745763 C67.6118441,122.915255 67.3244126,123 67.0087967,123 Z M71.6309963,113.666667 L81.1881919,113.666667 L76.4095941,103 L71.6309963,113.666667 Z M93.6744224,123 C93.2342616,123 92.8636055,122.853109 92.5624428,122.559322 C92.2612802,122.265535 92.1107011,121.903957 92.1107011,121.474576 L92.1107011,101.237288 C92.1107011,100.627116 92.3307782,100.101697 92.770939,99.6610169 C93.2110998,99.2203368 93.743918,99 94.3694097,99 L94.8211514,99 C95.5624748,99 96.1995401,99.3389797 96.7323663,100.016949 L110.110871,118.864407 L110.110871,100.559322 C110.110871,100.129941 110.26145,99.7627134 110.562612,99.4576271 C110.863775,99.1525408 111.234431,99 111.674592,99 C112.114753,99 112.491201,99.1525408 112.803946,99.4576271 C113.116692,99.7627134 113.273063,100.129941 113.273063,100.559322 L113.273063,120.79661 C113.273063,121.406783 113.047194,121.926552 112.59545,122.355932 C112.143706,122.785313 111.605096,123 110.979605,123 L110.562612,123 C110.191951,123 109.844461,122.915255 109.520132,122.745763 C109.195803,122.57627 108.929393,122.355934 108.720896,122.084746 L95.2728931,103.135593 L95.2728931,121.474576 C95.2728931,121.903957 95.1165225,122.265535 94.8037767,122.559322 C94.4910309,122.853109 94.1145832,123 93.6744224,123 Z M130.07106,123 C127.719675,123 125.630854,122.493041 123.804536,121.479109 C121.978217,120.465176 120.562841,119.050148 119.558366,117.233983 C118.553891,115.417818 118.051661,113.350987 118.051661,111.033426 C118.051661,108.671297 118.576719,106.576611 119.626852,104.749304 C120.676986,102.921996 122.109483,101.506969 123.924387,100.504178 C125.739291,99.5013877 127.753919,99 129.96833,99 C131.748991,99 133.307046,99.211697 134.642541,99.6350975 C135.978037,100.058498 137.262148,100.649021 138.494913,101.406685 C138.677545,101.495822 138.837345,101.651809 138.974319,101.874652 C139.111293,102.097494 139.179779,102.342617 139.179779,102.610028 C139.179779,102.98886 139.048514,103.311976 138.785981,103.579387 C138.523448,103.846798 138.175311,103.980501 137.74156,103.980501 C137.490441,103.980501 137.250741,103.924792 137.022451,103.81337 C135.949488,103.189412 134.876542,102.72145 133.80358,102.409471 C132.730618,102.097491 131.566357,101.941504 130.310763,101.941504 C128.57576,101.941504 127.011999,102.298047 125.619431,103.011142 C124.226863,103.724238 123.136795,104.76601 122.349195,106.13649 C121.561595,107.506971 121.167801,109.139266 121.167801,111.033426 C121.167801,112.771596 121.538767,114.325898 122.280709,115.696379 C123.02265,117.066859 124.095597,118.147628 125.499579,118.938719 C126.903562,119.729809 128.564345,120.125348 130.48198,120.125348 C132.627904,120.125348 134.62541,119.635102 136.474558,118.654596 L136.474558,112.604457 L130.824413,112.604457 C130.43632,112.604457 130.105305,112.470753 129.831357,112.203343 C129.557409,111.935932 129.420437,111.612815 129.420437,111.233983 C129.420437,110.855151 129.557409,110.532035 129.831357,110.264624 C130.105305,109.997213 130.43632,109.86351 130.824413,109.86351 L137.570344,109.86351 C138.026923,109.86351 138.41501,110.019497 138.734616,110.331476 C139.054222,110.643456 139.214022,111.022282 139.214022,111.467967 L139.214022,119.05571 C139.214022,119.523679 139.094172,119.941502 138.854467,120.309192 C138.614763,120.676882 138.300869,120.94986 137.912777,121.128134 C135.35593,122.376051 132.742051,123 130.07106,123 Z M146.182985,122.333333 C145.758591,122.333333 145.401212,122.188476 145.110837,121.898758 C144.820462,121.60904 144.675277,121.252469 144.675277,120.829035 L144.675277,100.537727 C144.675277,100.114293 144.820462,99.7521505 145.110837,99.4512894 C145.401212,99.1504283 145.758591,99 146.182985,99 L159.618342,99 C160.020399,99 160.361026,99.1392855 160.640233,99.4178606 C160.919439,99.6964356 161.059041,100.036292 161.059041,100.43744 C161.059041,100.838588 160.919439,101.172874 160.640233,101.440306 C160.361026,101.707738 160.020399,101.841452 159.618342,101.841452 L147.724198,101.841452 L147.724198,109.229226 L158.412175,109.229226 C158.814232,109.229226 159.154859,109.368512 159.434066,109.647087 C159.713273,109.925662 159.852874,110.265519 159.852874,110.666667 C159.852874,111.067815 159.713273,111.4021 159.434066,111.669532 C159.154859,111.936964 158.814232,112.070678 158.412175,112.070678 L147.724198,112.070678 L147.724198,119.491882 L159.618342,119.491882 C160.020399,119.491882 160.361026,119.631167 160.640233,119.909742 C160.919439,120.188317 161.059041,120.528174 161.059041,120.929322 C161.059041,121.33047 160.919439,121.664755 160.640233,121.932187 C160.361026,122.199619 160.020399,122.333333 159.618342,122.333333 L146.182985,122.333333 Z M167.40136,123 C166.961199,123 166.590543,122.853109 166.28938,122.559322 C165.988217,122.265535 165.837638,121.903957 165.837638,121.474576 L165.837638,101.237288 C165.837638,100.627116 166.057715,100.101697 166.497876,99.6610169 C166.938037,99.2203368 167.470855,99 168.096347,99 L168.548089,99 C169.289412,99 169.926477,99.3389797 170.459304,100.016949 L183.837808,118.864407 L183.837808,100.559322 C183.837808,100.129941 183.988387,99.7627134 184.28955,99.4576271 C184.590712,99.1525408 184.961369,99 185.401529,99 C185.84169,99 186.218138,99.1525408 186.530884,99.4576271 C186.843629,99.7627134 187,100.129941 187,100.559322 L187,120.79661 C187,121.406783 186.774131,121.926552 186.322387,122.355932 C185.870643,122.785313 185.332034,123 184.706542,123 L184.28955,123 C183.918888,123 183.571398,122.915255 183.247069,122.745763 C182.92274,122.57627 182.656331,122.355934 182.447834,122.084746 L168.99983,103.135593 L168.99983,121.474576 C168.99983,121.903957 168.84346,122.265535 168.530714,122.559322 C168.217968,122.853109 167.84152,123 167.40136,123 Z"
                                id="ERLANGEN-Copy" fill="url(#linearGradient-2)"></path>
                            <path
                                d="M24.5167671,72 L56,72 L56,77.3333333 L26.9213619,77.3333333 L24.5167671,72 Z M28.4242336,80.6666667 L56,80.6666667 L56,86 L24,86 L28.8746507,81.6656807 L28.4242336,80.6666667 Z"
                                id="Combined-Shape" fill="#FFFFFF" fill-rule="nonzero"></path>
                            <g id="Combined-Shape" fill-rule="nonzero">
                                <use fill="url(#linearGradient-3)" xlink:href="#path-4"></use>
                                <use fill="#FFFFFF" xlink:href="#path-4"></use>
                            </g>
                            <g id="Group-2" transform="translate(54.000000, 72.000000)">
                                <polygon id="Shape" fill="url(#linearGradient-5)" fill-rule="nonzero"
                                    points="0 0 40.2698413 0 46.4126984 8.66666667 49.8253968 8.66666667 86 8.66666667 86 14 45.047619 14 38.9047619 5.33333333 0 5.33333333">
                                </polygon>
                                <g id="Group"></g>
                                <g id="Rectangle" fill-rule="nonzero">
                                    <use fill="url(#linearGradient-6)" xlink:href="#path-7"></use>
                                    <use fill="#FFFFFF" xlink:href="#path-7"></use>
                                </g>
                                <g id="Rectangle" fill-rule="nonzero">
                                    <use fill="url(#linearGradient-8)" xlink:href="#path-9"></use>
                                    <use fill="#FFFFFF" xlink:href="#path-9"></use>
                                </g>
                            </g>
                            <g id="Green-Circle" fill="url(#linearGradient-10)" fill-rule="nonzero">
                                <path
                                    d="M7,14 C3.13400675,14 0,10.8659932 0,7 C0,3.13400675 3.13400675,0 7,0 C10.8659932,0 14,3.13400675 14,7 C14,10.8659932 10.8659932,14 7,14 Z M7,10 C8.65685425,10 10,8.65685425 10,7 C10,5.34314575 8.65685425,4 7,4 C5.34314575,4 4,5.34314575 4,7 C4,8.65685425 5.34314575,10 7,10 Z"
                                    id="Oval-2"></path>
                            </g>
                            <g id="Purple-Pentagon" transform="translate(1.000000, 67.000000)"
                                fill="url(#linearGradient-11)" fill-rule="nonzero">
                                <path
                                    d="M17.7702964,0.433333333 L23.6674501,13.6785575 L12.8928141,23.3800834 L0.336569067,16.1307319 L3.3510189,1.94886047 L17.7702964,0.433333333 Z M15.350853,4.66666667 L6.77722851,5.56779091 L4.98485294,14.000255 L12.4507284,18.3106802 L18.8572687,12.5422054 L15.350853,4.66666667 Z"
                                    id="Polygon-2"></path>
                            </g>
                            <g id="Yellow-Triangle" transform="translate(168.000000, 67.000000)"
                                fill="url(#linearGradient-12)" fill-rule="nonzero">
                                <path
                                    d="M18.494355,0.24487464 L22.4769585,23.5626719 L0.209090909,15.5887166 L18.494355,0.24487464 Z M15.5996385,8.08741097 L8.15454545,14.1780421 L17.3236674,17.4614355 L15.5996385,8.08741097 Z"
                                    id="Combined-Shape"></path>
                            </g>
                        </g>
                    </g>
                </g>
            </svg>
        </div>
        </a>

        <nav class="hidden xl:inline-block align-middle" role="navigation">

            <ul>
                <li class="dropdown-item">
                    <a href="#" class="has-dropdown" aria-haspopup="true">Team</a>
                    <ul class="dropdown w-32" aria-label="submenu">
                        <li><a href="http://example.org/team-members/ ">Team Members</a></li>
                        <li><a href="http://example.org/collaborations/">Collaborations</a></li>
                        <li><a href="http://example.org/attributions/">Attributions</a></li>
                    </ul>
                </li>

                <li class="dropdown-item">
                    <a href="#" class="has-dropdown" aria-haspopup="true">Project</a>
                    <ul class="dropdown" aria-label="submenu">
                        <li><a href="http://example.org/project/description/">Description</a></li>
                        <li><a href="http://example.org/project/design/">Design</a></li>
                        <li><a href="http://example.org/project/experiments/">Experiments</a></li>
                        <li><a href="http://example.org/labjournal-entries/">Notebook</a></li>
                        <li><a href="http://example.org/project/contribution/">Contribution</a></li>
                        <li><a href="http://example.org/project/results/">Results</a></li>
                        
                    </ul>
                </li>

                <li class="dropdown-item">
                    <a href="#" class="has-dropdown" aria-haspopup="true">Parts</a>
                    <ul class="dropdown" aria-label="submenu">
                        <li><a href="http://example.org/parts/overview/">Parts Overview</a></li>
                        <li><a href="http://example.org/parts/basic/">Basic Parts</a></li>
                        <li><a href="http://example.org/parts/composite/">Composite Parts</a></li>
                        <li><a href="http://example.org/parts/collection/">Parts Collection</a></li>
                    </ul>
                </li>

                <li class="dropdown-item"><a href="http://example.org/safety/">Safety</a></li>

                <li class="dropdown-item"><a href="http://example.org/human-practices/human-practices/">Human Practices</a></li>

                

                <li class="dropdown-item">
                    <a href="#" class="has-dropdown" aria-haspopup="true">Awards</a>
                    <ul class="dropdown" aria-label="submenu">
                        
                        
                        
                        <li><a href="">Model</a></li>
                        
                        
                    </ul>
                </li>


                <li class="mx-8 inline-block fancy-btn-blue">
                    <a href="https://igem.org/2019_Judging_Form?team=FAU_Erlangen" target="_blank" class="btn-arrow inline-block text-white" style="width: 150px !important">Judging Form</a>
                </li>

            </ul>
        </nav>

        <script>
        function toggleMobileNavigation() {
            var isClosed = $("#mobile-navigation").hasClass("navigation-hidden");

            if (isClosed) {
                $("#mobile-navigation").removeClass('navigation-hidden');

                

            } else {
                $("#mobile-navigation").addClass('navigation-hidden');
                
                
            }
        }
        </script>

        <a id="toggleBtn" class="block lg:hidden absolute top-0 text-white w-3 h-3 p-5 right-0 top-0 m-5" onclick="toggleMobileNavigation()" style="text-shadow: 2px 2px 10px rgba(0,0,0,0.3); z-index: 9010;"><i class="fas fa-bars"></i></a>

        <div id="mobile-navigation" class="block xl:hidden navigation-hidden float-right mr-8">
                <ul>
                    <li><a href="http://example.org/team-members/">Team Members</a></li>
                    <li><a href="http://example.org/collaborations/">Collaborations</a></li>
                    <li><a href="http://example.org/attributions/">Attributions</a></li>
                    <li><a href="http://example.org/safety/">Safety</a></li>
                    <li><a href="http://example.org/human-practices/human-practices/">Human Practices</a></li>
                    <li><a href="https://2019.igem.org/Team:FAU_Erlangen/Model">Model</a></li>

                    <li class="inline-block font-bold pt-12 uppercase text-gray-900">Project</li>
                    <li><a href="http://example.org/project/description/">Description</a></li>
                    <li><a href="http://example.org/project/design/">Design</a></li>
                    <li><a href="http://example.org/project/experiments/">Experiments</a></li>
                    <li><a href="http://example.org/labjournal-entries/">Notebook</a></li>
                    <li><a href="http://example.org/project/contribution/">Contribution</a></li>
                    <li><a href="http://example.org/project/results/">Results</a></li>

                    <li class="inline-block font-bold pt-12 uppercase text-gray-900">Parts</li>
                    <li><a href="http://example.org/parts/overview/">Parts Overview</a></li>
                    <li><a href="http://example.org/parts/basic/">Basic Parts</a></li>
                    <li><a href="http://example.org/parts/composite/">Composite Parts</a></li>
                    <li><a href="http://example.org/parts/collection/">Parts Collection</a></li>
        
                    <li>
                        <a href="https://igem.org/2019_Judging_Form?team=FAU_Erlangen">Judging Form</a>
                    </li>
        
                </ul>
        </div>

        <div class="social-bar xl:block lg:hidden">
            <ul>
                
                <li><a class="facebook" href="https://facebook.com/igemErlangen" target="_blank"></a></li>
                
                
                <li><a class="twitter" target="_blank" href="https://www.twitter.com/iGEM_Erlangen"></a></li>
                
                
                <li><a class="instagram" target="_blank" href="https://www.instagram.com/igem.erlangen"></a></li>
                
                
                <li><a class="youtube" target="_blank" href="https://www.youtube.com/channel/UCxT11JmhRH62aBn-HZxGhQA"></a></li>
                
            </ul>
        </div>

        
        <h1 class="text-white text-5xl z-10" style="display: block; position: absolute; left: 50%; top: 50%; width: 450px; height: 60px; vertical-align: middle; text-align: center; margin-top: -30px; margin-left: -225px;">Notebook</h1>
        

        
</header>

<div class="sm:hidden xs:hidden md:hidden lg:hidden sticky-header sticky-off sticky top-0 z-50 shadow-xl" style="background: url('/images/sticky-background-blue.svg'); height: 80px;">

        <a href="https://2019.igem.org/Team:FAU_Erlangen">
           <div id="logo" class="hidden xl:inline-block mt-2 ml-12">
            <svg height="65px" viewBox="0 0 210 143" version="1.1" xmlns="http://www.w3.org/2000/svg"
                xmlns:xlink="http://www.w3.org/1999/xlink">
                <defs>
                    <filter x="-15.4%" y="-24.0%" width="130.9%" height="148.0%" filterUnits="objectBoundingBox"
                        id="filter-1">
                        <feOffset dx="5" dy="5" in="SourceAlpha" result="shadowOffsetOuter1"></feOffset>
                        <feGaussianBlur stdDeviation="5" in="shadowOffsetOuter1" result="shadowBlurOuter1">
                        </feGaussianBlur>
                        <feColorMatrix values="0 0 0 0 0   0 0 0 0 0   0 0 0 0 0  0 0 0 0.205747378 0" type="matrix"
                            in="shadowBlurOuter1" result="shadowMatrixOuter1"></feColorMatrix>
                        <feMerge>
                            <feMergeNode in="shadowMatrixOuter1"></feMergeNode>
                            <feMergeNode in="SourceGraphic"></feMergeNode>
                        </feMerge>
                    </filter>
                    <linearGradient x1="0%" y1="49.1585099%" x2="101.999998%" y2="50.8583199%" id="linearGradient-2">
                        <stop stop-color="#FAAC39" offset="0%"></stop>
                        <stop stop-color="#E3209C" offset="100%"></stop>
                    </linearGradient>
                    <linearGradient x1="106.481982%" y1="58.6047602%" x2="-7.35198796%" y2="43.2132964%"
                        id="linearGradient-3">
                        <stop stop-color="#1D5279" offset="0%"></stop>
                        <stop stop-color="#0F2B45" offset="100%"></stop>
                    </linearGradient>
                    <path
                        d="M177,72 L170.46578,77.3333333 L139,77.3333333 L139,72 L177,72 Z M166.381892,80.6666667 L163.205247,83.2594945 L171.016083,86 L139,86 L139,80.6666667 L166.381892,80.6666667 Z"
                        id="path-4"></path>
                    <linearGradient x1="99.1374639%" y1="50%" x2="0%" y2="50%" id="linearGradient-5">
                        <stop stop-color="#FFFFFF" offset="0%"></stop>
                        <stop stop-color="#FFFFFF" offset="40.0299012%"></stop>
                        <stop stop-color="#B5F829" offset="44.8825793%"></stop>
                        <stop stop-color="#15F2F3" offset="55.5045952%"></stop>
                        <stop stop-color="#FFFFFF" offset="60.6309638%"></stop>
                        <stop stop-color="#FFFFFF" offset="100%"></stop>
                    </linearGradient>
                    <linearGradient x1="106.481982%" y1="51.4314786%" x2="-7.35198796%" y2="48.8709714%"
                        id="linearGradient-6">
                        <stop stop-color="#1D5279" offset="0%"></stop>
                        <stop stop-color="#0F2B45" offset="100%"></stop>
                    </linearGradient>
                    <rect id="path-7" x="0" y="8.66666667" width="35.4920635" height="5.33333333"></rect>
                    <linearGradient x1="106.481982%" y1="51.3779701%" x2="-7.35198796%" y2="48.9131743%"
                        id="linearGradient-8">
                        <stop stop-color="#1D5279" offset="0%"></stop>
                        <stop stop-color="#0F2B45" offset="100%"></stop>
                    </linearGradient>
                    <rect id="path-9" x="49.8253968" y="0" width="36.1746032" height="5.33333333"></rect>
                    <linearGradient x1="0%" y1="0%" x2="97.6494292%" y2="96.6920827%" id="linearGradient-10">
                        <stop stop-color="#0FE6F1" offset="0%"></stop>
                        <stop stop-color="#CFF968" offset="100%"></stop>
                    </linearGradient>
                    <linearGradient x1="2.44820263%" y1="3.97792628%" x2="123.107123%" y2="82.6286156%"
                        id="linearGradient-11">
                        <stop stop-color="#DA1AE6" offset="0%"></stop>
                        <stop stop-color="#140CF7" offset="100%"></stop>
                    </linearGradient>
                    <linearGradient x1="34.40034%" y1="27.1369695%" x2="95.5986745%" y2="100.573377%"
                        id="linearGradient-12">
                        <stop stop-color="#FFCC23" offset="0%"></stop>
                        <stop stop-color="#E3209C" offset="94.1067558%"></stop>
                        <stop stop-color="#E115A3" offset="100%"></stop>
                    </linearGradient>
                </defs>
                <g id="Website" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                    <g id="Full-HD" transform="translate(-53.000000, -33.000000)">
                        <g id="Logo-(Dark-BG)" filter="url(#filter-1)" transform="translate(58.000000, 38.000000)">
                            <path
                                d="M5.37577552,59.3125 C4.42555247,59.3125 3.62537665,58.9802117 2.97522404,58.315625 C2.32507143,57.6510383 2,56.8330978 2,55.8617788 L2,22.9649038 C2,21.9935849 2.32507143,21.162864 2.97522404,20.4727163 C3.62537665,19.7825687 4.42555247,19.4375 5.37577552,19.4375 C6.3760103,19.4375 7.20119162,19.7697883 7.85134423,20.434375 C8.50149684,21.0989617 8.82656827,21.9424628 8.82656827,22.9649038 L8.82656827,55.8617788 C8.82656827,56.8330978 8.48899409,57.6510383 7.81383561,58.315625 C7.13867713,58.9802117 6.32599856,59.3125 5.37577552,59.3125 Z M45.8192778,60 C40.5855493,60 35.9362388,58.8382196 31.871207,56.514624 C27.8061751,54.1910283 24.6558227,50.9482567 22.4200552,46.7862117 C20.1842877,42.6241667 19.0664207,37.8876774 19.0664207,32.5766017 C19.0664207,27.1633898 20.2350998,22.3630664 22.5724931,18.1754875 C24.9098864,13.9879085 28.0983479,10.7451368 32.1379733,8.44707521 C36.1775987,6.14901358 40.6617697,5 45.5906209,5 C49.5540269,5 53.0219552,5.48513907 55.9945097,6.45543175 C58.9670643,7.42572444 61.8252469,8.7790071 64.5691434,10.5153203 C64.9756466,10.7195925 65.3313315,11.0770634 65.6362089,11.5877437 C65.9410863,12.0984241 66.0935227,12.6601641 66.0935227,13.2729805 C66.0935227,14.1411371 65.8013529,14.8816125 65.2170046,15.494429 C64.6326563,16.1072454 63.8577712,16.413649 62.8923262,16.413649 C62.3333843,16.413649 61.7998568,16.2859808 61.2917279,16.0306407 C58.9035217,14.6007357 56.5153513,13.528323 54.1271451,12.8133705 C51.7389388,12.098418 49.1475199,11.7409471 46.3528105,11.7409471 C42.4910303,11.7409471 39.010399,12.5580234 35.9108122,14.1922006 C32.8112254,15.8263777 30.3849459,18.2137726 28.6319009,21.3544568 C26.8788559,24.4951411 26.0023466,28.2358186 26.0023466,32.5766017 C26.0023466,36.5599085 26.8280438,40.1218506 28.479463,43.2625348 C30.1308822,46.403219 32.5190526,48.8799817 35.6440458,50.6928969 C38.769039,52.5058122 42.4656219,53.4122563 46.7339053,53.4122563 C51.5103178,53.4122563 55.9563797,52.2887763 60.0722244,50.0417827 L60.0722244,36.1768802 L47.496095,36.1768802 C46.6322757,36.1768802 45.8954998,35.8704766 45.285745,35.2576602 C44.6759902,34.6448437 44.3711174,33.9043683 44.3711174,33.0362117 C44.3711174,32.1680551 44.6759902,31.4275797 45.285745,30.8147632 C45.8954998,30.2019468 46.6322757,29.8955432 47.496095,29.8955432 L62.5112313,29.8955432 C63.5274893,29.8955432 64.3912956,30.2530141 65.1026762,30.9679666 C65.8140567,31.6829191 66.1697417,32.5510627 66.1697417,33.5724234 L66.1697417,50.9610028 C66.1697417,52.0334315 65.902978,52.9909429 65.3694426,53.8335655 C64.8359071,54.6761881 64.1372403,55.3017621 63.273421,55.7103064 C57.5823764,58.5701164 51.7643868,60 45.8192778,60 Z M83.2780431,59.3125 C82.3054733,59.3125 81.4864795,58.9795879 80.821037,58.3137536 C80.1555945,57.6479193 79.8228782,56.8284433 79.8228782,55.8553009 L79.8228782,9.22152579 C79.8228782,8.24838339 80.1555945,7.41610303 80.821037,6.72465974 C81.4864795,6.03321646 82.3054733,5.6875 83.2780431,5.6875 L114.067402,5.6875 C114.988784,5.6875 115.769387,6.00760783 116.409236,6.64783309 C117.049084,7.28805836 117.369004,8.06912146 117.369004,8.99104585 C117.369004,9.91297023 117.049084,10.681229 116.409236,11.2958453 C115.769387,11.9104615 114.988784,12.217765 114.067402,12.217765 L86.8099895,12.217765 L86.8099895,29.1964542 L111.30327,29.1964542 C112.224652,29.1964542 113.005255,29.516562 113.645104,30.1567872 C114.284952,30.7970125 114.604872,31.5780756 114.604872,32.5 C114.604872,33.4219244 114.284952,34.1901832 113.645104,34.8047994 C113.005255,35.4194157 112.224652,35.7267192 111.30327,35.7267192 L86.8099895,35.7267192 L86.8099895,52.782235 L114.067402,52.782235 C114.988784,52.782235 115.769387,53.1023428 116.409236,53.7425681 C117.049084,54.3827933 117.369004,55.1638564 117.369004,56.0857808 C117.369004,57.0077052 117.049084,57.775964 116.409236,58.3905802 C115.769387,59.0051965 114.988784,59.3125 114.067402,59.3125 L83.2780431,59.3125 Z M131.71364,60 C130.75037,60 129.939207,59.6675822 129.280127,59.0027366 C128.621048,58.337891 128.291513,57.5196317 128.291513,56.5479343 L128.291513,10.7505297 C128.291513,9.36969649 128.773141,8.18066353 129.736411,7.18339513 C130.699681,6.18612672 131.865727,5.6875 133.234585,5.6875 L135.744145,5.6875 C136.758113,5.6875 137.683346,5.99434721 138.51987,6.60805085 C139.356394,7.22175448 139.97744,7.98887252 140.383028,8.90942797 L157.645756,52.3287429 L174.908485,8.90942797 C175.314073,7.98887252 175.935119,7.22175448 176.771643,6.60805085 C177.608167,5.99434721 178.5334,5.6875 179.547368,5.6875 L182.056928,5.6875 C183.425785,5.6875 184.591832,6.18612672 185.555102,7.18339513 C186.518372,8.18066353 187,9.36969649 187,10.7505297 L187,56.5479343 C187,57.5196317 186.670465,58.337891 186.011386,59.0027366 C185.352306,59.6675822 184.541143,60 183.577873,60 C182.614603,60 181.790766,59.6675822 181.106337,59.0027366 C180.421908,58.337891 180.079699,57.5196317 180.079699,56.5479343 L180.079699,12.9751942 L162.512781,56.778072 C162.107194,57.7497695 161.460799,58.5296728 160.573576,59.1178054 C159.686354,59.7059381 158.710423,60 157.645756,60 C156.581089,60 155.605159,59.7059381 154.717937,59.1178054 C153.830714,58.5296728 153.184319,57.7497695 152.778732,56.778072 L135.211814,12.9751942 L135.211814,56.5479343 C135.211814,57.5196317 134.869605,58.337891 134.185176,59.0027366 C133.500747,59.6675822 132.67691,60 131.71364,60 Z"
                                id="iGEM-Copy" fill="#FFFFFF"></path>
                            <path
                                d="M3.50770833,123 C3.08331423,123 2.72593509,122.855143 2.43556018,122.565425 C2.14518528,122.275707 2,121.919136 2,121.495702 L2,101.204394 C2,100.780959 2.14518528,100.418817 2.43556018,100.117956 C2.72593509,99.817095 3.08331423,99.6666667 3.50770833,99.6666667 L16.9430648,99.6666667 C17.3451223,99.6666667 17.6857493,99.8059521 17.964956,100.084527 C18.2441626,100.363102 18.3837638,100.702959 18.3837638,101.104107 C18.3837638,101.505255 18.2441626,101.83954 17.964956,102.106972 C17.6857493,102.374404 17.3451223,102.508118 16.9430648,102.508118 L5.04892129,102.508118 L5.04892129,109.895893 L15.7368981,109.895893 C16.1389557,109.895893 16.4795827,110.035178 16.7587893,110.313754 C17.0379959,110.592329 17.1775972,110.932185 17.1775972,111.333333 C17.1775972,111.734481 17.0379959,112.068767 16.7587893,112.336199 C16.4795827,112.603631 16.1389557,112.737345 15.7368981,112.737345 L5.04892129,112.737345 L5.04892129,120.158548 L16.9430648,120.158548 C17.3451223,120.158548 17.6857493,120.297834 17.964956,120.576409 C18.2441626,120.854984 18.3837638,121.19484 18.3837638,121.595989 C18.3837638,121.997137 18.2441626,122.331422 17.964956,122.598854 C17.6857493,122.866286 17.3451223,123 16.9430648,123 L3.50770833,123 Z M25.3530694,123 C24.9285789,123 24.5711185,122.856173 24.2806776,122.568516 C23.9902367,122.280858 23.8450185,121.926824 23.8450185,121.506401 L23.8450185,101.193457 C23.8450185,100.773034 23.9902367,100.413468 24.2806776,100.114746 C24.5711185,99.8160251 24.9285789,99.6666667 25.3530694,99.6666667 L33.1614223,99.6666667 C36.199881,99.6666667 38.4786908,100.286227 39.9979202,101.525367 C41.5171495,102.764508 42.2767528,104.390854 42.2767528,106.404457 C42.2767528,107.732107 41.9025364,108.94357 41.1540926,110.038881 C40.4056487,111.134192 39.3165117,111.925239 37.8866488,112.412044 C39.4952445,112.832466 40.5955522,113.822657 41.1876048,115.382646 C41.7796574,116.942635 42.0756793,118.474941 42.0756793,119.979611 L42.0756793,121.506401 C42.0756793,121.926824 41.930461,122.280858 41.6400201,122.568516 C41.3495792,122.856173 40.9921189,123 40.5676283,123 C40.0984545,123 39.7242382,122.861705 39.4449681,122.585111 C39.165698,122.308518 39.0260651,121.948951 39.0260651,121.506401 L39.0260651,120.510669 C39.0260651,119.404294 38.9255294,118.380912 38.7244549,117.440493 C38.5233804,116.500074 38.043043,115.659242 37.2834284,114.917971 C36.5238137,114.176699 35.3732382,113.806069 33.8316672,113.806069 L26.8946327,113.806069 L26.8946327,121.506401 C26.8946327,121.926824 26.7438291,122.280858 26.4422174,122.568516 C26.1406057,122.856173 25.77756,123 25.3530694,123 Z M32.6193847,111 C34.5338015,111 36.0530687,110.613302 37.1772321,109.839895 C38.3013954,109.066488 38.8634686,107.974635 38.8634686,106.564304 C38.8634686,105.199468 38.3793065,104.15311 37.4109678,103.425197 C36.4426291,102.697284 34.7897999,102.333333 32.4524306,102.333333 L26.5756458,102.333333 L26.5756458,111 L32.6193847,111 Z M49.8698387,122.333333 C49.2833002,122.333333 48.7813661,122.128657 48.3640213,121.719298 C47.9466766,121.309939 47.7380074,120.81761 47.7380074,120.242295 L47.7380074,100.52679 C47.7380074,100.106367 47.8846398,99.7468009 48.1779091,99.4480797 C48.4711784,99.1493584 48.8321197,99 49.2607441,99 C49.6893684,99 50.0559495,99.1493584 50.3604983,99.4480797 C50.6650472,99.7468009 50.8173193,100.106367 50.8173193,100.52679 L50.8173193,119.312945 L61.8825391,119.312945 C62.3111634,119.312945 62.6777445,119.462303 62.9822934,119.761024 C63.2868422,120.059745 63.4391144,120.419312 63.4391144,120.839734 C63.4391144,121.260157 63.2868422,121.614192 62.9822934,121.901849 C62.6777445,122.189507 62.3111634,122.333333 61.8825391,122.333333 L49.8698387,122.333333 Z M67.0087967,123 C66.5579169,123 66.1915825,122.853109 65.9097826,122.559322 C65.6279827,122.265535 65.4870849,121.903957 65.4870849,121.474576 C65.4870849,121.293784 65.543444,121.067798 65.656164,120.79661 L74.2791976,100.389831 C74.4595495,99.9830488 74.7131656,99.6497188 75.0400535,99.3898305 C75.3669413,99.1299422 75.7670912,99 76.240515,99 L76.5786732,99 C77.052097,99 77.4522469,99.1299422 77.7791347,99.3898305 C78.1060226,99.6497188 78.3596387,99.9830488 78.5399906,100.389831 L87.1630242,120.79661 C87.2757442,121.067798 87.3321033,121.293784 87.3321033,121.474576 C87.3321033,121.903957 87.1912055,122.265535 86.9094056,122.559322 C86.6276057,122.853109 86.2612713,123 85.8103915,123 C85.4947756,123 85.2073441,122.915255 84.9480881,122.745763 C84.6888322,122.57627 84.4915753,122.344634 84.3563113,122.050847 L82.1582832,116.762712 L70.660905,116.762712 L68.4628769,122.050847 C68.3276129,122.344634 68.1303559,122.57627 67.8711,122.745763 C67.6118441,122.915255 67.3244126,123 67.0087967,123 Z M71.6309963,113.666667 L81.1881919,113.666667 L76.4095941,103 L71.6309963,113.666667 Z M93.6744224,123 C93.2342616,123 92.8636055,122.853109 92.5624428,122.559322 C92.2612802,122.265535 92.1107011,121.903957 92.1107011,121.474576 L92.1107011,101.237288 C92.1107011,100.627116 92.3307782,100.101697 92.770939,99.6610169 C93.2110998,99.2203368 93.743918,99 94.3694097,99 L94.8211514,99 C95.5624748,99 96.1995401,99.3389797 96.7323663,100.016949 L110.110871,118.864407 L110.110871,100.559322 C110.110871,100.129941 110.26145,99.7627134 110.562612,99.4576271 C110.863775,99.1525408 111.234431,99 111.674592,99 C112.114753,99 112.491201,99.1525408 112.803946,99.4576271 C113.116692,99.7627134 113.273063,100.129941 113.273063,100.559322 L113.273063,120.79661 C113.273063,121.406783 113.047194,121.926552 112.59545,122.355932 C112.143706,122.785313 111.605096,123 110.979605,123 L110.562612,123 C110.191951,123 109.844461,122.915255 109.520132,122.745763 C109.195803,122.57627 108.929393,122.355934 108.720896,122.084746 L95.2728931,103.135593 L95.2728931,121.474576 C95.2728931,121.903957 95.1165225,122.265535 94.8037767,122.559322 C94.4910309,122.853109 94.1145832,123 93.6744224,123 Z M130.07106,123 C127.719675,123 125.630854,122.493041 123.804536,121.479109 C121.978217,120.465176 120.562841,119.050148 119.558366,117.233983 C118.553891,115.417818 118.051661,113.350987 118.051661,111.033426 C118.051661,108.671297 118.576719,106.576611 119.626852,104.749304 C120.676986,102.921996 122.109483,101.506969 123.924387,100.504178 C125.739291,99.5013877 127.753919,99 129.96833,99 C131.748991,99 133.307046,99.211697 134.642541,99.6350975 C135.978037,100.058498 137.262148,100.649021 138.494913,101.406685 C138.677545,101.495822 138.837345,101.651809 138.974319,101.874652 C139.111293,102.097494 139.179779,102.342617 139.179779,102.610028 C139.179779,102.98886 139.048514,103.311976 138.785981,103.579387 C138.523448,103.846798 138.175311,103.980501 137.74156,103.980501 C137.490441,103.980501 137.250741,103.924792 137.022451,103.81337 C135.949488,103.189412 134.876542,102.72145 133.80358,102.409471 C132.730618,102.097491 131.566357,101.941504 130.310763,101.941504 C128.57576,101.941504 127.011999,102.298047 125.619431,103.011142 C124.226863,103.724238 123.136795,104.76601 122.349195,106.13649 C121.561595,107.506971 121.167801,109.139266 121.167801,111.033426 C121.167801,112.771596 121.538767,114.325898 122.280709,115.696379 C123.02265,117.066859 124.095597,118.147628 125.499579,118.938719 C126.903562,119.729809 128.564345,120.125348 130.48198,120.125348 C132.627904,120.125348 134.62541,119.635102 136.474558,118.654596 L136.474558,112.604457 L130.824413,112.604457 C130.43632,112.604457 130.105305,112.470753 129.831357,112.203343 C129.557409,111.935932 129.420437,111.612815 129.420437,111.233983 C129.420437,110.855151 129.557409,110.532035 129.831357,110.264624 C130.105305,109.997213 130.43632,109.86351 130.824413,109.86351 L137.570344,109.86351 C138.026923,109.86351 138.41501,110.019497 138.734616,110.331476 C139.054222,110.643456 139.214022,111.022282 139.214022,111.467967 L139.214022,119.05571 C139.214022,119.523679 139.094172,119.941502 138.854467,120.309192 C138.614763,120.676882 138.300869,120.94986 137.912777,121.128134 C135.35593,122.376051 132.742051,123 130.07106,123 Z M146.182985,122.333333 C145.758591,122.333333 145.401212,122.188476 145.110837,121.898758 C144.820462,121.60904 144.675277,121.252469 144.675277,120.829035 L144.675277,100.537727 C144.675277,100.114293 144.820462,99.7521505 145.110837,99.4512894 C145.401212,99.1504283 145.758591,99 146.182985,99 L159.618342,99 C160.020399,99 160.361026,99.1392855 160.640233,99.4178606 C160.919439,99.6964356 161.059041,100.036292 161.059041,100.43744 C161.059041,100.838588 160.919439,101.172874 160.640233,101.440306 C160.361026,101.707738 160.020399,101.841452 159.618342,101.841452 L147.724198,101.841452 L147.724198,109.229226 L158.412175,109.229226 C158.814232,109.229226 159.154859,109.368512 159.434066,109.647087 C159.713273,109.925662 159.852874,110.265519 159.852874,110.666667 C159.852874,111.067815 159.713273,111.4021 159.434066,111.669532 C159.154859,111.936964 158.814232,112.070678 158.412175,112.070678 L147.724198,112.070678 L147.724198,119.491882 L159.618342,119.491882 C160.020399,119.491882 160.361026,119.631167 160.640233,119.909742 C160.919439,120.188317 161.059041,120.528174 161.059041,120.929322 C161.059041,121.33047 160.919439,121.664755 160.640233,121.932187 C160.361026,122.199619 160.020399,122.333333 159.618342,122.333333 L146.182985,122.333333 Z M167.40136,123 C166.961199,123 166.590543,122.853109 166.28938,122.559322 C165.988217,122.265535 165.837638,121.903957 165.837638,121.474576 L165.837638,101.237288 C165.837638,100.627116 166.057715,100.101697 166.497876,99.6610169 C166.938037,99.2203368 167.470855,99 168.096347,99 L168.548089,99 C169.289412,99 169.926477,99.3389797 170.459304,100.016949 L183.837808,118.864407 L183.837808,100.559322 C183.837808,100.129941 183.988387,99.7627134 184.28955,99.4576271 C184.590712,99.1525408 184.961369,99 185.401529,99 C185.84169,99 186.218138,99.1525408 186.530884,99.4576271 C186.843629,99.7627134 187,100.129941 187,100.559322 L187,120.79661 C187,121.406783 186.774131,121.926552 186.322387,122.355932 C185.870643,122.785313 185.332034,123 184.706542,123 L184.28955,123 C183.918888,123 183.571398,122.915255 183.247069,122.745763 C182.92274,122.57627 182.656331,122.355934 182.447834,122.084746 L168.99983,103.135593 L168.99983,121.474576 C168.99983,121.903957 168.84346,122.265535 168.530714,122.559322 C168.217968,122.853109 167.84152,123 167.40136,123 Z"
                                id="ERLANGEN-Copy" fill="url(#linearGradient-2)"></path>
                            <path
                                d="M24.5167671,72 L56,72 L56,77.3333333 L26.9213619,77.3333333 L24.5167671,72 Z M28.4242336,80.6666667 L56,80.6666667 L56,86 L24,86 L28.8746507,81.6656807 L28.4242336,80.6666667 Z"
                                id="Combined-Shape" fill="#FFFFFF" fill-rule="nonzero"></path>
                            <g id="Combined-Shape" fill-rule="nonzero">
                                <use fill="url(#linearGradient-3)" xlink:href="#path-4"></use>
                                <use fill="#FFFFFF" xlink:href="#path-4"></use>
                            </g>
                            <g id="Group-2" transform="translate(54.000000, 72.000000)">
                                <polygon id="Shape" fill="url(#linearGradient-5)" fill-rule="nonzero"
                                    points="0 0 40.2698413 0 46.4126984 8.66666667 49.8253968 8.66666667 86 8.66666667 86 14 45.047619 14 38.9047619 5.33333333 0 5.33333333">
                                </polygon>
                                <g id="Group"></g>
                                <g id="Rectangle" fill-rule="nonzero">
                                    <use fill="url(#linearGradient-6)" xlink:href="#path-7"></use>
                                    <use fill="#FFFFFF" xlink:href="#path-7"></use>
                                </g>
                                <g id="Rectangle" fill-rule="nonzero">
                                    <use fill="url(#linearGradient-8)" xlink:href="#path-9"></use>
                                    <use fill="#FFFFFF" xlink:href="#path-9"></use>
                                </g>
                            </g>
                            <g id="Green-Circle" fill="url(#linearGradient-10)" fill-rule="nonzero">
                                <path
                                    d="M7,14 C3.13400675,14 0,10.8659932 0,7 C0,3.13400675 3.13400675,0 7,0 C10.8659932,0 14,3.13400675 14,7 C14,10.8659932 10.8659932,14 7,14 Z M7,10 C8.65685425,10 10,8.65685425 10,7 C10,5.34314575 8.65685425,4 7,4 C5.34314575,4 4,5.34314575 4,7 C4,8.65685425 5.34314575,10 7,10 Z"
                                    id="Oval-2"></path>
                            </g>
                            <g id="Purple-Pentagon" transform="translate(1.000000, 67.000000)"
                                fill="url(#linearGradient-11)" fill-rule="nonzero">
                                <path
                                    d="M17.7702964,0.433333333 L23.6674501,13.6785575 L12.8928141,23.3800834 L0.336569067,16.1307319 L3.3510189,1.94886047 L17.7702964,0.433333333 Z M15.350853,4.66666667 L6.77722851,5.56779091 L4.98485294,14.000255 L12.4507284,18.3106802 L18.8572687,12.5422054 L15.350853,4.66666667 Z"
                                    id="Polygon-2"></path>
                            </g>
                            <g id="Yellow-Triangle" transform="translate(168.000000, 67.000000)"
                                fill="url(#linearGradient-12)" fill-rule="nonzero">
                                <path
                                    d="M18.494355,0.24487464 L22.4769585,23.5626719 L0.209090909,15.5887166 L18.494355,0.24487464 Z M15.5996385,8.08741097 L8.15454545,14.1780421 L17.3236674,17.4614355 L15.5996385,8.08741097 Z"
                                    id="Combined-Shape"></path>
                            </g>
                        </g>
                    </g>
                </g>
            </svg>
        
        </div>       
        </a>
        
        <nav class="md:hidden lg:inline-block align-middle" role="navigation">
            <ul>
                <li class="dropdown-item w-32">
                    <a href="#" class="has-dropdown" aria-haspopup="true">Team</a>
                    <ul class="dropdown w-32" aria-label="submenu">
                        <li><a href="http://example.org/team-members/">Team Members</a></li>
                        <li><a href="http://example.org/collaborations/">Collaborations</a></li>
                        <li><a href="http://example.org/attributions/">Attributions</a></li>
                    </ul>
                </li>

                <li class="dropdown-item">
                    <a href="#" class="has-dropdown" aria-haspopup="true">Project</a>
                    <ul class="dropdown" aria-label="submenu">
                        <li><a href="http://example.org/project/description/">Description</a></li>
                        <li><a href="http://example.org/project/design/">Design</a></li>
                        <li><a href="http://example.org/project/experiments/">Experiments</a></li>
                        <li><a href="http://example.org/labjournal-entries/">Notebook</a></li>
                        <li><a href="http://example.org/project/contribution/">Contribution</a></li>
                        <li><a href="http://example.org/project/results/">Results</a></li>
                    </ul>
                </li>

                <li class="dropdown-item">
                    <a href="#" class="has-dropdown" aria-haspopup="true">Parts</a>
                    <ul class="dropdown" aria-label="submenu">
                        <li><a href="http://example.org/parts/overview/">Parts Overview</a></li>
                        <li><a href="http://example.org/parts/basic/">Basic Parts</a></li>
                        <li><a href="http://example.org/parts/composite/">Composite Parts</a></li>
                        <li><a href="http://example.org/parts/collection/">Parts Collection</a></li>
                    </ul>
                </li>

                <li class="dropdown-item"><a href="">Safety</a></li>

                <li class="dropdown-item">
                    <a href="http://example.org/human-practices/human-practices/">Human Practices</a>
                </li>

                <li class="dropdown-item">
                    <a href="#" class="has-dropdown" aria-haspopup="true">Awards</a>
                    <ul class="dropdown" aria-label="submenu">
                        
                        
                        
                        <li><a href="http://example.org/awards/model/">Model</a></li>
                        
                        
                    </ul>
                </li>


                <li class="ml-12 inline-block fancy-btn-blue">
                    <a href="https://igem.org/2019_Judging_Form?team=FAU_Erlangen" class="btn-arrow inline-block text-white">Judging Form</a></li>

            </ul>

        </nav>


</div>

<div id="main" class="bg-white z-50" style="margin-top: -100px;">

<div class="container mx-auto my-10 min-h-screen">
    <h1 class="text-blue">Notebook</h1>

    <p>
        
    </p>

    

    
    
    
    

    

        
        

        
            
            
        

        

            

            <h2 class="my-10 text-3xl">May</h2>
            

            <div class="accordion flex flex-col items-center justify-center">
            
        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-1" class="hidden">
      <label for="panel-1" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              21
              
          </p>
          <p class="inline-block">Generation of pACYC184 plasmid</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Generation of pACYC184 plasmid</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>bacteria from -80°C: HVS11 pEcoE, HVS11 pEcoF</li>
<li>inoculation of both samples in 5 mL LB medium with Chloramphenicol</li>
<li>overnight culture at 37°C with 200 rpm</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-2" class="hidden">
      <label for="panel-2" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              22
              
          </p>
          <p class="inline-block">Generation of pACYC184 plasmid</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Generation of pACYC184 plasmid</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>no turbidity in the ON culture visible</li>
<li>new bacteria from -80°C: HVS11 pEcoE, HVS11 pEcoF, HVS11 pEcoD</li>
<li>inoculation of the three samples in 5 mL LB medium with Tetracyclin (2 µg/mL)</li>
<li>ON culture at 37°C with 200 rpm</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-3" class="hidden">
      <label for="panel-3" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              23
              
          </p>
          <p class="inline-block">Generation of pACYC184 plasmid</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Generation of pACYC184 plasmid</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Zymo research Miniprep - Classic (Cat.No. D4015, Lot.No. 202653)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p>Turbidity in the ON culture visible</p>

<p><em>Miniprep:</em></p>

<ol>
<li>Centrifugate the ON culture at 4000 rpm at room temperature</li>
<li>Discard the supernatant</li>
<li>Add 200 µL P1 buffer</li>
<li>Add 200 µL P2 buffer and mix by inverting two times</li>
<li>Add 400 µL P3 buffer and mix by inverting three times</li>
<li>Centrifugate the samples for 2 min in a microcentrifuge</li>
<li>Place the column in a collection tube and transfer the supernatant</li>
<li>Centrifugate for 30s</li>
<li>Discard the flow-through and return of the column to the collection tube</li>
<li>Add 200 µL Endo-Wash buffer and centrifugate for 30 s</li>
<li>Add 400 µL Plasmid Wash buffer and centrifugate for 1 min</li>
<li>Place the column in a 1.5 mL tube, add 30 µL DNA Elution Buffer and centrifugate for 30 s</li>
</ol>

<p>Storage of the purified plasmid DNA at 4°C</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-4" class="hidden">
      <label for="panel-4" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              27
              
          </p>
          <p class="inline-block">Generation of pACYC184 plasmid</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Generation of pACYC184 plasmid</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p>inoculation of 6 clones in 5 mL LB medium with Chloramphenicol for the ON culture</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-5" class="hidden">
      <label for="panel-5" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              27
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Generation of pACYC184 plasmid</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Generation of pACYC184 plasmid</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p>Nanodrop measurement of the samples from 2019-05-23
  * pEcoD: 46.1 ng/µL
  * pEcoE: 70.9 ng/µL
  * pEcoF: 27.6 ng/µL</p>

<p>restriction enzyme digest:</p>

<table>
<thead>
<tr>
<th>setpoint</th>
<th>EcoD</th>
<th>EcoE</th>
<th>EcoF</th>
<th></th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>500 ng</td>
<td>10.85 µL</td>
<td>7.05 µL</td>
<td>17.5 µL</td>
</tr>

<tr>
<td>H20</td>
<td>20 µL</td>
<td>6.65 µL</td>
<td>10.45 µL</td>
<td>0 µL</td>
</tr>

<tr>
<td>CutSmart Buffer</td>
<td>2 µL</td>
<td>2 µl</td>
<td>2 µL</td>
<td>2 µL</td>
</tr>

<tr>
<td>EcoRI</td>
<td>0.5 µL</td>
<td>0.5 µ</td>
<td>0.5 µL</td>
<td>0.5 µL</td>
</tr>
</tbody>
</table>

<p>Digestion 1h at 37°C and 350 rpm, afterwards adding of 4 µL BPB</p>

<p>agarose gel 1%:
1x TBE      |60 mL
agarose     |0.6 g
ethidiumbromid  |2 µL</p>

<ul>
<li>load the gel with 20 µL of the samples and 3 µL Gene Ruler Mix</li>
<li>run the gel at 130 V for 1 h</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-6" class="hidden">
      <label for="panel-6" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              28
              
          </p>
          <p class="inline-block">Generation of pACYC184 plasmid</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Generation of pACYC184 plasmid</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><em>Miniprep:</em></p>

<ol>
<li>Centrifugate the ON culture at 4000 rpm at room temperature</li>
<li>Discard the supernatant</li>
<li>Add 200 µL P1 buffer</li>
<li>Add 200 µL P2 buffer and mix by inverting two times</li>
<li>Add 400 µL P3 buffer and mix by inverting three times</li>
<li>Centrifugate the samples for 2 min in a microcentrifuge</li>
<li>Place the column in a collection tube and transfer the supernatant</li>
<li>Centrifugate for 30s</li>
<li>Discard the flow-through and return of the column to the collection tube</li>
<li>Add 200 µL Endo-Wash buffer and centrifugate for 30 s</li>
<li>Add 400 µL Plasmid Wash buffer and centrifugate for 1 min</li>
<li>Place the column in a 1.5 mL tube, add 30 µL DNA Elution Buffer and centrifugate for 30 s</li>
</ol>

<p>store the purified DNA at 4°C</p>

<p><em>DNA concentration:</em>
measurement at the NanoDrop</p>

<table>
<thead>
<tr>
<th>Clone</th>
<th>Clone 1</th>
<th>Clone 2</th>
<th>Clone 3</th>
<th>Clone 4</th>
<th>Clone 5</th>
<th>Clone 6</th>
</tr>
</thead>

<tbody>
<tr>
<td>concentration [ng/µL]</td>
<td>30.6</td>
<td>21.3</td>
<td>26.1</td>
<td>23.6</td>
<td>28.1</td>
<td>25.4</td>
</tr>
</tbody>
</table>

<p><em>Digestion:</em></p>

<p>digestion for 1.5 h at 37°C</p>

<table>
<thead>
<tr>
<th>Clone</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
</tr>
</thead>

<tbody>
<tr>
<td>Buffer</td>
<td>2 µL</td>
<td>2 µL</td>
<td>2 µL</td>
<td>2 µL</td>
<td>2 µL</td>
<td>2 µL</td>
</tr>

<tr>
<td>Enzyme (EcoRV)</td>
<td>0.5 µL</td>
<td>0.5 µL</td>
<td>0.5 µL</td>
<td>0.5 µL</td>
<td>0.5 µL</td>
<td>0.5 µL</td>
</tr>

<tr>
<td>DNA</td>
<td>16.3 µL</td>
<td>17.5 µL</td>
<td>17.5 µL</td>
<td>17.5 µL</td>
<td>17.5 µL</td>
<td>17.5 µL</td>
</tr>

<tr>
<td>H20</td>
<td>1.2</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
</tbody>
</table>

<ul>
<li>load 20 µL of each sample and 3.5 µL of the marker on a 1 % agarose gel</li>
<li>run 1.5 h at 130 V</li>
</ul>

<p><em>DNA extraction from agarose gel:</em></p>

<ol>
<li>add 200 µL NTI per 100 mg gel piece</li>
<li>incubate for 5 min at 50°C</li>
<li>place the column in a collection tube and transfer the sample</li>
<li>centrifugate at 11000 rpm for 30 s</li>
<li>add 700 µL NT3 twice adn centrifugate each time at 11000 rpm for 30 s</li>
<li>dry the silica membrane for 1 min at 11000 rpm</li>
<li>place the column on a sterile 1.5 mL reaction tube</li>
<li>add 20 µL NE buffer and centrifugate for 1 min at 11000 rpm</li>
</ol>

<p><em>DNA concentration</em>:
measurement at the NanoDrop
* clone 1: 11.2 ng/µL
* clone 2: 22.3 ng/µl</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-7" class="hidden">
      <label for="panel-7" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              28
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Generation of pACYC184 plasmid</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Generation of pACYC184 plasmid</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>DNA extraction from agarose gel</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>run the gel from 2019-05-27 for 45 min at 130 V</li>
<li>cut out the vector DNA from  the gel</li>
<li>place the gel pieces into reaction tubes</li>
</ul>

<p><em>DNA extraction from agarose gel:</em></p>

<ol>
<li>add 200 µL NTI per 100 mg gel piece</li>
<li>incubate for 5 min at 50°C</li>
<li>place the column in a collection tube and transfer the sample</li>
<li>centrifugate at 11000 rpm for 30 s</li>
<li>add 700 µL NT3 twice adn centrifugate each time at 11000 rpm for 30 s</li>
<li>dry the silica membrane for 1 min at 11000 rpm</li>
<li>place the column on a sterile 1.5 mL reaction tube</li>
<li>add 20 µL NE buffer and centrifugate for 1 min at 11000 rpm</li>
</ol>

<p><em>Ligation:</em>
DNA             |9 µL
Quick Ligation buffer   |10 µL
Quick Ligase        |1 µL</p>

<p>Incubate for 10 min at room temperature</p>

<p><em>Transformation into competent DH10B e.coli via heat shock:</em></p>

<ol>
<li>thaw competent bacteria on ice</li>
<li>add 10 µL of the previously ligated DNA, storage of the rest at -20°C</li>
<li>incubate 30 min on ice</li>
<li>incubate 1 min at 42°C</li>
<li>incubate 2 min on ice</li>
<li>add 500 µL Soc medium</li>
<li>30 min on 37°C</li>
</ol>

<p>plate the bacteria on Cam plates, incubate at 37°C over night</p>

<p><em>Control gel for the ligation:</em></p>

<ul>
<li>loading of 2 µL of each sample with 2 µL running buffer</li>
<li>run the gel for 1 h at 130 V</li>
<li>for each sample only one band visible</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-8" class="hidden">
      <label for="panel-8" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              29
              
          </p>
          <p class="inline-block">Generation of pACYC184 plasmid</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Generation of pACYC184 plasmid</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>some colonies visible on the plates</li>
<li>store at 4°C</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

            
            </div>
            

            <h2 class="my-10 text-3xl">June</h2>
            

            <div class="accordion flex flex-col items-center justify-center">
            
        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-9" class="hidden">
      <label for="panel-9" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              13
              
          </p>
          <p class="inline-block">Generation of more open pACYC184 with EcoRV</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Generation of more open pACYC184 with EcoRV</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Miniprep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<table>
<thead>
<tr>
<th>Substance</th>
<th>concentration</th>
<th></th>
<th></th>
</tr>
</thead>

<tbody>
<tr>
<td>Cut Smart buffer</td>
<td>2 µL</td>
<td>2 µg (= 1 µl)</td>
<td>1 µg (= µl)</td>
</tr>

<tr>
<td>DNA</td>
<td>7,98 µL</td>
<td></td>
<td></td>
</tr>

<tr>
<td>Enzyme</td>
<td>1,5 µL</td>
<td></td>
<td></td>
</tr>

<tr>
<td>Water</td>
<td>44,52 µL</td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

<p>Incoulation of 200 µL sample 2 from the 27.05.2019</p>

<p><strong>Miniprep</strong></p>

<ul>
<li>Adding 10 mL Equilibration buffer at the collum</li>
<li>Centrifugation of overnightculture for 10 min at 4000 x g and removement of all the medium</li>
<li>Resuspension of the pellet in 4 mL resuspension buffer (R3)</li>
<li>Adding of 4 mL Lysis buffer and mixing by capping the tube for 5 times</li>
<li>Incubation for 5 min at RT</li>
<li>Adding 4 mL precipitation buffer (N3) and inverting till it is homogenous</li>
<li>Centrifugation for 10 min at 5000 x g at RT</li>
<li>Loadng supernatant on the collum, drain by gravity</li>
<li>Adding of 10 mL washing buffer to the collum twice</li>
<li>Placing collum on a 15 mL tube and adding of 5 mL elution buffer (E4) and letting it drain by gravity</li>
<li>Adding 3,5 mL isopropanol and mixing it well</li>
<li>Centrifugation for 30 min at 5000 x g at RT</li>
<li>Discarding of the supernatant</li>
<li>Adding 3 mL 70% ethanol</li>
<li>Centrifugation for 5 min, 5000 x g at RT</li>
<li>Remove supernatant</li>
<li>Airdry pellet for 10 min</li>
<li>Resuspension of pellet in 200 µL</li>
<li>Storage at -20°C (still in virology lab)

<br /></li>
</ul>

<p><strong>Nanodrop Measurement</strong></p>

<p>188,8 ng/ µL</p>

<p><strong>Digest</strong></p>

<p>500 ng for 0,5 µL enzyme</p>

<p>Digest of following components at 37°C foe 1 h:</p>

<p><strong>Digest</strong></p>

<p>500 ng for 0,5 µL enzyme</p>

<p>Digest of following components at 37°C foe 1 h:</p>

<p>*see table above</p>

<p><strong>Gel</strong></p>

<p>Mixing of 60 µL digest with 12 µL loading buffer</p>

<p>Loading a agarose gel</p>

<p>130 V for 1 h</p>

<p><strong>Gel extraction</strong></p>

<p>Cutting the digest out of the gel.</p>

<ol>
<li><p>Adding 200 µL /per 100 mg gel piece of NTI.</p></li>

<li><p>5 min 50°C</p></li>

<li><p>Placing of the collum in a collection tube and loading of the sample.</p></li>

<li><p>Centrifugation at 11000 rpm for 30 s.</p></li>

<li><p>Adding twice 700 µL NT3 and centrifugation each time at 11000 rpm for 30 s.</p></li>

<li><p>Drying of the silica membrane for 1 min at 11000 rpm.</p></li>

<li><p>Placing collumn on a 1,5 mL sterile Eppi.</p></li>

<li><p>Adding 20 µL NE buffer and centrifugation for 1 min at 11000 rpm.</p></li>

<li><p>Storage at -20°C.</p></li>
</ol>

<p><strong>Nanodrop</strong></p>

<p>Different results with a wide range</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-10" class="hidden">
      <label for="panel-10" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              17
              
          </p>
          <p class="inline-block">Concentration of EcoRV opend pACYC184</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Concentration of EcoRV opend pACYC184</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Nanodrop</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>Nanodrop measurement: concentration still varies</p>

<p><strong>Concentration determination with agarose gel</strong></p>

<p>Loading of the sample (2µL and 2µL marker on a agarose gel)</p>

<p>sample: 0,5 µL loading buffer on 2,5 µL digest</p>

<p>Running gel for 1 h at 130 V</p>

<p>Marker 4000 bp ~ 50 ng/µL</p>

<p><img src="/labjournal-entries/images/img_1300.jpg" alt="&lt;agarose gel of EcoRV opend pACYC184&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-11" class="hidden">
      <label for="panel-11" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              20
              
          </p>
          <p class="inline-block">Material and Devices</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Material and Devices</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Material check</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Material and Devices</strong></p>

<ul>
<li>E.coli are in the -80°C Refrigerator in our lab - Box 12, reserved for iGEM

<ul>
<li>BL21 = Blank/no name, Star = S, Tuner = T</li>
</ul></li>
<li>4°C Refrigerator is in our lab - top shelf</li>
<li>PCR machine is on the other site of the kitchen</li>
<li>Material for different buffers is on the left side next of the Winkler lab</li>
<li>Winkler lab is on the opposite site of our lab</li>
<li>Elektroporator in Jäck AG - Contact Felix Pfister, iGEM for more information</li>
<li>Shaker Incubator in our Lab - Connect to water flow - Temperature adjustable<br /></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-12" class="hidden">
      <label for="panel-12" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              24
              
          </p>
          <p class="inline-block">Production of LB medium</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Production of LB medium</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>LB-Medium Production Protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Felix Fister</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Autoclave machine in room 02.052</li>
<li>Big incubation shaker device in 02.053</li>
<li>Create medium and label it &mdash; &ldquo;Content, Name-iGEM, room of our lab, date&rdquo; e.g LB-medium, DS-iGEM, 02.081, 24.06.19</li>
<li>Created 4L LB-Medium

<br /></li>
</ul>

<p><strong>Calculations and Results</strong></p>

<p>20g of LB-Broth dissolved in 1L milli-Q H2O</p>

<p><strong>Materials used</strong></p>

<ul>
<li>80g LB-Broth</li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Medium-bottle do not close 100%</li>
<li>not 100% contamination proof -use fast</li>
<li>Our water shaker has the wrong size. Winkler is ordering new sizes for our shaker

<ul>
<li>Look for right size. Send product link to prof. winkler. He is going to order them for us</li>
</ul></li>
<li>Get right flask holder for our shaker?

<br />

<br />

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Cleaning ladies place the autoclaved medium into our lab. Store at 8°C. Should be finished around 2pm.</li>
<li>Adding of antibiotics for selection after transformation.</li>
<li>Use for cultivation and transformation</li>
<li>Tape and pens are necessary
<br /></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-13" class="hidden">
      <label for="panel-13" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              25
              
          </p>
          <p class="inline-block">E.coli Cultivation</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>E.coli Cultivation</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Cultivation with LB-medium</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Stored LB-medium in fridge</li>
<li>Used 3 E.coli strains: Tuner, BL21 Star, BL21</li>
<li>Pipetting under bunsen burner in Winkler lab</li>
<li>For each strain: 10 mL LB-medium combined with 1 sterile pipet tip E.coli in 12 ml not fully closed reaction tubes</li>
<li>Cultivation in top incubation shaker (left side, 3 12 mL reaction tubes) in room 02.053 at 37°C with 130 rpm</li>
<li>Autoclave 100 mL flasks for tomorrow 26.06.19</li>
<li><strong>If we are using stuff from the Winkler lab, bring it back</strong></li>
<li><strong>If we are using workspace in the Winkler lab - clean the bench with 70% Ethanol</strong>

<br />

<br /></li>
</ul>

<p><strong>Protocol:</strong></p>

<ol>
<li>Unfreeze E.coli</li>
<li>Centrifuge tube with Mini-Centrifuge &ndash;Solution at the bottom of the reaction tube</li>
</ol>

<p><strong>Next Steps are performed next to the bunsen burner</strong></p>

<ol>
<li><p>Fill 10 ml LB-medium into 12 ml reaction tube under</p></li>

<li><p>Dip 1 sterile pipet tip into E.coli reaction tube</p></li>

<li><p>Throw pipet tip into 12 ml reaction tube and close it</p></li>

<li><p>Cultivation in incubation shaker for 20 hours at 37°C with 130 rpm</p></li>
</ol>

<ul>
<li>Store remaining E.coli at -80°C

<br /></li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Incubation flasks are needed for further cultivation. Autoclave 100 mL flasks for tomorrow 26.06.19</li>
<li>Need lab briefing from some of winklers lab

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>OD measurement &ndash;further cultivation</li>
<li>Cultivation in 100mL flasks with the incubation shaker in our lab</li>
<li>Freeze 1mL from each E.coli strain at -80°C</li>
<li>Wait for autoclaved flasks. Use them to cultivate our bacteria in our shaker
&hellip;</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-14" class="hidden">
      <label for="panel-14" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              26
              
          </p>
          <p class="inline-block">E. coli cultivation</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>E. coli cultivation</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Cultivation with LB-medium</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lisa Vogg, Martin Borgmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Stored LB-medium in fridge</li>
<li>Used 3 E.coli strains: Tuner, BL21 Star, BL21</li>
<li>Pipetting under bunsen burner in Winkler lab</li>
<li>For each strain: 155 mL LB-medium combined the 10 ml of the previous bacteria culture (25.06.19)</li>
<li>Cultivation in top incubation shaker in room 02.053 at 37°C with 130 rpm</li>
<li>stuff flask holder with paper towels to keep the flasks in place</li>
<li>autoclaved flasks from the previous day not yet usable

<br />

<br /></li>
</ul>

<p><strong>Protocol:</strong></p>

<p>(Performed next to a bunsen burner)</p>

<ol>
<li>Fill flask with desired volume of LB medium</li>
<li>sterilize tip of the pipet, the lid of the bottle with medium and the edge of the flask in the flame to prevent contamination</li>
<li>add content of the previous culture tubes to the new flasks</li>
<li>Cultivation in incubation shaker for 20 hours at 37°C with 130 rpm

<br /></li>
</ol>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>briefing for the lab desperately needed!</li>
<li>Which kuvettes for OD measurement?</li>
<li>flasks from the previous day not yet usable

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>measure OD</li>
<li>make cells competent</li>

<li><p>freeze a batch</p>

<ul>
<li>3x 25 ml pipette</li>
</ul></li>

<li><p>gloves, paper towels, 70% ethanol</p></li>

<li><p>490 ml LB medium</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-15" class="hidden">
      <label for="panel-15" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              28
              
          </p>
          <p class="inline-block">Overnight Cultivation of E.coli strains</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Overnight Cultivation of E.coli strains</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>E.coli Cultivation</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Leon Altmann, Lisa Vogg, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Measure OD(600nm) (1:10 dilution) after 2 days incubation in incubation shaker of the E.coli strains BL21, BL21 Star, BL21 Tuner</li>
<li>OD of 0.71 =7.1

<br /></li>
</ul>

<p><strong>Protocol:</strong></p>

<p><strong>Inoculate overnight culture:</strong></p>

<ol>
<li>inoculate 100 ml LB-medium with 100µl of the E.coli cultures</li>
<li>Fill water shaker with water</li>
<li>Place into water shaker at 37°C and 120 RPM</li>
<li>Incubate overnight: Start 5pm &ndash;19h incubation time

<br /></li>
</ol>

<p><strong>Materials used</strong></p>

<ul>
<li>OD cuvette</li>
<li>25 ml plastic tip</li>
<li>3 yellow tips

<br />

<br /></li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li><strong>NEB Kit was stored at -20°C: competent E.colis should be stored at -80°C, other lab material should be stored at 4°C (DNA Ladder, buffer, &hellip;) &ndash;Damage is possible</strong>

<br />

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Pour agar plates for heat shock test: Test transformation by heat shock with NEB control vector PUC19</li>
<li>Pour agar plates for first transformations: Agar plates with chloramphenicol</li>
<li>freeze E.coli from overnight culture</li>
<li>Heat shock test
<br /></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-16" class="hidden">
      <label for="panel-16" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              29
              
          </p>
          <p class="inline-block">Pour agar plates</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Pour agar plates</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Leon Altmann, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Pour agar plates:

<ul>
<li>3 plates with ampicillin resistance</li>
<li>21 plates with chloramphenicol resistance</li>
<li>3 OD measurements of overnight culture. Each around 45 minutes apart
<strong>Materials used</strong>

<br /></li>
</ul></li>
</ul>

<p>agar for 700 mL agar plates</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

            
            </div>
            

            <h2 class="my-10 text-3xl">July</h2>
            

            <div class="accordion flex flex-col items-center justify-center">
            
        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-17" class="hidden">
      <label for="panel-17" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
          </p>
          <p class="inline-block">Preparation Sequences</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Preparation Sequences</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr, Ruben Caspary, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>Resuspension of the IDT sequences in 20 µL TE buffer.</p>

<p>Resulting concentration 50 ng/ µL.</p>

<p>Vortexing and incubation for 15 min at 50°C.</p>

<p>Vortexing and storage at -20°C.</p>

<p>Biotwist sequences and primer still not resuspended!</p>

<p>10 µL Filtertip-Spitzen</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-18" class="hidden">
      <label for="panel-18" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Growth kinetics</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Growth kinetics</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>OD</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Freezing of E.coli</strong></p>

<p><strong>Goal</strong>: storing  the 3 E.coli strains BL21, BL21 Tuner and BL21 Star to have them ready for cultivation for transformation</p>

<p>-Spinned for 10 minutes at 2,500 rpm</p>

<p>-Discarded supernatant without disturbing the pellet, while leaving ~1ml for every sample</p>

<p>-Resuspend the pellet in the remaining supernatant.</p>

<p>-Added 900μl cell culture and 300μl 99% glycerol, to a final glycerol concentration of ~25%. 6.</p>

<p>-Store in -80 °C freezer</p>

<p><strong>Growth kinetics</strong></p>

<p><strong>Goal</strong>: get  <strong>Growth kinetics</strong>  of the 3 E.coli strains BL21, BL21 Tuner and BL21 Star to estimate the optimal time point for the transformation!</p>

<p>-<strong>Measuring OD600  of over night culture (RT)</strong></p>

<p>OD600 = 1 equals 8*108  cells/mL</p>

<table>
<thead>
<tr>
<th>Stem</th>
<th>OD</th>
<th>Cell count</th>
</tr>
</thead>

<tbody>
<tr>
<td>B21:</td>
<td>1 µg (=0,5 µl)</td>
<td>&ndash;5,5*108 cells / mL</td>
</tr>

<tr>
<td>B21 Tuner:</td>
<td>OD600 (1) = 0,1966 and OD600 (2) = 0,1402 (double determination!) -average: 0,1684</td>
<td>&ndash;1,35*109  cells / mL</td>
</tr>

<tr>
<td>B21 Star:</td>
<td>OD600 = 0,0779</td>
<td>&ndash;6,29*108 cells / mL</td>
</tr>
</tbody>
</table>

<p>-<strong>Seeding with same cell coun</strong>t -106 cells / mL in 25 mL LB medium &ndash;2,5*107  cells</p>

<table>
<thead>
<tr>
<th>Stem</th>
<th>Calculation</th>
<th></th>
</tr>
</thead>

<tbody>
<tr>
<td>B21:</td>
<td>5,5*108 cells / mL -2,5*107 cells /  5,5*108 cells / mL = 0,045 mL</td>
<td></td>
</tr>

<tr>
<td>B21 Tuner:</td>
<td>1,35*109  cells / mL - 2,5*107 cells /  1,35*109  cells / mL = 0,019 mL</td>
<td></td>
</tr>

<tr>
<td>B21 Star:</td>
<td>6,29*108 cells / mL - 2,5*107 cells /  6,29*108 cells / mL = 0,040 mL</td>
<td></td>
</tr>
</tbody>
</table>

<table>
<thead>
<tr>
<th>Time in h</th>
<th>OD</th>
<th>dilution factor</th>
<th>real OD</th>
<th>µ(t) in 1/h</th>
<th></th>
<th>OD</th>
<th>dilution factor</th>
<th>real OD</th>
<th>µ(t) in 1/h</th>
<th></th>
<th>OD</th>
<th>dilution factor</th>
<th>real OD</th>
<th>µ(t) in 1/h</th>
</tr>
</thead>

<tbody>
<tr>
<td><em>Stem:</em></td>
<td>BL12</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>BL12 Tuner</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>BL12 Star</td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>0</td>
<td>0,0192</td>
<td>1</td>
<td>0,0192</td>
<td></td>
<td></td>
<td>0,0262</td>
<td>1</td>
<td>0,0262</td>
<td></td>
<td></td>
<td>0,0199</td>
<td>1</td>
<td>0,0199</td>
<td>.</td>
</tr>

<tr>
<td>1</td>
<td>0,0324</td>
<td>1</td>
<td>0,0324</td>
<td>0,523</td>
<td></td>
<td>0,0237</td>
<td>1</td>
<td>0,0237</td>
<td>-0,100</td>
<td></td>
<td>0,0283</td>
<td>1</td>
<td>0,0283</td>
<td>0,352</td>
</tr>

<tr>
<td>2</td>
<td>0,0674</td>
<td>1</td>
<td>0,0674</td>
<td>0,732</td>
<td></td>
<td>0,027</td>
<td>1</td>
<td>0,027</td>
<td>0,130</td>
<td></td>
<td>0,0557</td>
<td>1</td>
<td>0,0557</td>
<td>0,677</td>
</tr>

<tr>
<td>2,5</td>
<td>0,1265</td>
<td>1</td>
<td>0,1265</td>
<td>1,259</td>
<td></td>
<td>0,0405</td>
<td>1</td>
<td>0,0405</td>
<td>0,811</td>
<td></td>
<td>0,092</td>
<td>1</td>
<td>0,092</td>
<td>1,004</td>
</tr>

<tr>
<td>3</td>
<td>0,2873</td>
<td>1</td>
<td>0,2873</td>
<td>1,641</td>
<td></td>
<td>0,0724</td>
<td>1</td>
<td>0,0724</td>
<td>1,162</td>
<td></td>
<td>0,217</td>
<td>1</td>
<td>0,217</td>
<td>1,716</td>
</tr>

<tr>
<td>3,5</td>
<td>0,2609</td>
<td>2</td>
<td>0,5218</td>
<td>1,194</td>
<td></td>
<td>0,069</td>
<td>2</td>
<td>0,138</td>
<td>1,290</td>
<td></td>
<td>0,1933</td>
<td>2</td>
<td>0,3866</td>
<td>1,155</td>
</tr>

<tr>
<td>4</td>
<td>0,1663</td>
<td>5</td>
<td>0,8315</td>
<td>0,932</td>
<td></td>
<td>0,067</td>
<td>5</td>
<td>0,335</td>
<td>1,774</td>
<td></td>
<td>0,1384</td>
<td>5</td>
<td>0,692</td>
<td>1,164</td>
</tr>

<tr>
<td>4,5</td>
<td>0,2182</td>
<td>5</td>
<td>1,091</td>
<td>0,543</td>
<td></td>
<td>0,142</td>
<td>5</td>
<td>0,71</td>
<td>1,502</td>
<td></td>
<td>0,1759</td>
<td>5</td>
<td>0,8795</td>
<td>0,480</td>
</tr>

<tr>
<td>5</td>
<td>0,1992</td>
<td>6</td>
<td>1,1952</td>
<td>0,182</td>
<td></td>
<td>0,1605</td>
<td>6</td>
<td>0,963</td>
<td>0,610</td>
<td></td>
<td>0,1843</td>
<td>6</td>
<td>1,1058</td>
<td>0,458</td>
</tr>
</tbody>
</table>

<p>-all strains in log phase at roughly 3-3,5h &ndash;<strong>Transformation should be done after 3-3,5h of incubation time after staring a culture!</strong></p>

<p><img src="/labjournal-entries/images/Curves.JPG" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-19" class="hidden">
      <label for="panel-19" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Preparation Sequences</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Preparation Sequences</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr, Ruben Caspary, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>Resuspension of the IDT sequences in 20 µL TE buffer.</p>

<p>Resulting concentration 50 ng/ µL.</p>

<p>Vortexing and incubation for 15 min at 50°C.</p>

<p>Vortexing and storage at -20°C.</p>

<p>Biotwist sequences and primer still not resuspended!
10 µL Filtertip-Spitzen</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-20" class="hidden">
      <label for="panel-20" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              2
              
          </p>
          <p class="inline-block">Amplify construct DNA</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Amplify construct DNA</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1</th>
<th>Attempt 2</th>
<th>Attempt 3</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µg (=0,5 µl)</td>
<td>2 µg (= 1 µl)</td>
<td>1 µg (= µl)</td>
</tr>

<tr>
<td>NEB buffer 3.1 (x10 )</td>
<td>5</td>
<td>5</td>
<td>-</td>
</tr>
</tbody>
</table>

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>dissolved primers for amplification in TE buffer to a concentration of 100mM</li>
<li>froze each primer diluted in TE with a concentration of 1mM (labeled according to construct)</li>
<li>dissolved all remaining DNA of the constructs in 20 µl TE buffer</li>
<li>prepared 2 1:100 pre-dilutions of the primer for the PCR, first in TE buffer, second in RNase-free water</li>
<li>performed PCR according to NEB protocol</li>
<li>stored PCR products in 4 °C fridge</li>
<li>prepared agarose gel for electrophresis (needs to be repeated)

<br />

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p><strong>Dissolving freeze-dried DNA</strong></p>

<ul>
<li>spin down aliquots</li>
<li>add required volume of TE buffer (20 µl)</li>
<li>vortex thoroughly</li>
<li>incubate on the heating block at 50°C for 20 min

<br /></li>
</ul>

<p><strong>PCR (according to the NEB protocol)</strong></p>

<table>
<thead>
<tr>
<th>template DNA</th>
<th>5 µl (250 ng)</th>
</tr>
</thead>

<tbody>
<tr>
<td>forward primer (10 µM)</td>
<td>1.25 µl (end concentration 0.5 µM)</td>
</tr>

<tr>
<td>reverse primer (10 µM)</td>
<td>1.25 µl (end concentration 0.5 µM)</td>
</tr>

<tr>
<td>Master Mix</td>
<td>12.5 µl</td>
</tr>

<tr>
<td>RNase-free water</td>
<td>5 µlpr</td>
</tr>

<tr>
<td></td>
<td>25 µl</td>
</tr>
</tbody>
</table>

<ul>
<li>programme is saved under Thomas Winkler &gt;Nina &gt;iGEM</li>
<li>can otherwise be found in the NEB protocol

<ul>
<li>30 cycles at 60°C

<br /></li>
</ul></li>
</ul>

<p><strong>Agarose gel</strong></p>

<ul>
<li>0.8 g Agarose</li>
<li>100 ml TAE buffer</li>
<li>10 µl SYBR Safe</li>

<li><p>medium chamber</p></li>

<li><p>slot chamber onto its sled with the open ends aligning with the edge of the sled</p></li>

<li><p>cautiously pour the liquid agarose into the chamber, avoid air bubbles</p></li>

<li><p>insert the comb</p></li>

<li><p><strong>DO NOT MOVE AFTER AGAROSE HAS BEEN ADDED!</strong> Otherwise liquid will slosh out and gel will be too thin</p></li>

<li><p>gels can be stored in a plastic dish with a little bit of TAE buffer at 4°C or wrapped in saran film</p></li>
</ul>

<p><strong>Calculations and Results</strong>
-   calculated volume for dissolving the primers:  <strong>V=m/(M*c)</strong>  with m and M found on the aliquots and c set as  <strong>100 mM</strong></p>

<table>
<thead>
<tr>
<th>Primer</th>
<th>m [mg]</th>
<th>M [g/mol]</th>
<th>V [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>Tac Promotor Primer fw 2.0</td>
<td>0.97</td>
<td>6.38</td>
<td>1520</td>
</tr>

<tr>
<td>K1 HisStop Primer re 2.0</td>
<td>0.61</td>
<td>6.467</td>
<td>943</td>
</tr>

<tr>
<td>K2A HisStop re 2.0</td>
<td>0.21</td>
<td>7.084</td>
<td>296</td>
</tr>

<tr>
<td>K3a HisStop Primer re 2.0</td>
<td>0.54</td>
<td>6.194</td>
<td>886</td>
</tr>

<tr>
<td>K4a StrepStop re 2.0</td>
<td>0.22</td>
<td>6.362</td>
<td>346</td>
</tr>

<tr>
<td>K4b StrepStop re 2.0</td>
<td>0.2</td>
<td>6.924</td>
<td>314</td>
</tr>
</tbody>
</table>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong>
-   run electrophoresis with PCR products</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-21" class="hidden">
      <label for="panel-21" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
          </p>
          <p class="inline-block">Create E.coli Starter Glycerol Stock</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Create E.coli Starter Glycerol Stock</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>see link</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Autoclaved 2x 500ml agar plate medium</li>
<li>Pour antibiotica-free plates</li>
<li>E.coli strains BL21, Star, Tuner, DH5alpha are cultivated on plates in 3 different dilutions: 1:10, 1:100</li>
<li>work very sterile
<a href="http://www.molbi.de/protocols/competent_cells_starter_cultures_v1_0.htm">http://www.molbi.de/protocols/competent_cells_starter_cultures_v1_0.htm</a>
<br /></li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Agar dissolves in autoclave</li>
<li><strong>work very sterile!!! No antibiotica</strong>

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>35g LB-Broth with agar

<br />

<br />

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Cultivate 2 colonies each in 50mL flasks. See protocol day 2 morning!</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-22" class="hidden">
      <label for="panel-22" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Primer Dilution</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Primer Dilution</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1</th>
<th>Attempt 2</th>
<th>Attempt 3</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µg (=0,5 µl)</td>
<td>2 µg (= 1 µl)</td>
<td>1 µg (= µl)</td>
</tr>

<tr>
<td>NEB buffer 3.1 (x10 )</td>
<td>5</td>
<td>5</td>
<td>-</td>
</tr>
</tbody>
</table>

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p><strong>Prepare Primer Solution:</strong></p>

<ul>
<li>centrifuge primers</li>
<li>add TE-buffer to dilute primers to a concentration of 100nM</li>
<li>Vortex</li>
<li>heat and shake at 50°C in thermomixer</li>
<li><strong>freeze and store at -20°C</strong>

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<ul>
<li><strong>Ran gelelectrophoresis with PCR products 02.07.2019 with 100x</strong></li>
</ul>

<table>
<thead>
<tr>
<th>DNA</th>
<th>2µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>Loading Dye(6x)</td>
<td>2µl</td>
</tr>

<tr>
<td>H20</td>
<td>16µl</td>
</tr>
</tbody>
</table>

<ul>
<li>Put 20µl in each pocket</li>
<li>put 6µl DNA Ladder in one pocket

<br /></li>
</ul>

<p><strong>Gibson - PCR:</strong></p>

<table>
<thead>
<tr>
<th>template DNA</th>
<th>5 µl (250 ng)</th>
</tr>
</thead>

<tbody>
<tr>
<td>forward primer (10 µM)</td>
<td>1.25 µl (end concentration 0.5 µM)</td>
</tr>

<tr>
<td>reverse primer (10 µM)</td>
<td>1.25 µl (end concentration 0.5 µM)</td>
</tr>

<tr>
<td>Master Mix</td>
<td>12.5 µl</td>
</tr>

<tr>
<td>Autoclaved H20</td>
<td>5 µlpr</td>
</tr>

<tr>
<td></td>
<td>25 µl</td>
</tr>
</tbody>
</table>

<ul>
<li>NEW PCR PROGRAM: igem gibson anhang</li>
<li>PCR is running over night. Finishes around 9pm and is then stored at 4°C in the PCR cycler

<br /></li>
</ul>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>No Protocol! People should make clear which protocol they want to use</li>
<li>Tested PCR Products on quantity with biophotometer: No clear results. Results are in our lab notebook

<br />

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>16 PCR Tubes and caps

<br />

<br /></li>
</ul>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Run Gel electrophoresis. See day before</li>
<li>Gibson Assembly of Fab Fragments</li>
<li>Gibson Assembly of different constructs into vector</li>
<li>Loading Dye 6x is stored in the fridge in Winkler&rsquo;s Lab</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-23" class="hidden">
      <label for="panel-23" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Primer Dilution</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Primer Dilution</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1</th>
<th>Attempt 2</th>
<th>Attempt 3</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µg (=0,5 µl)</td>
<td>2 µg (= 1 µl)</td>
<td>1 µg (= µl)</td>
</tr>

<tr>
<td>NEB buffer 3.1 (x10 )</td>
<td>5</td>
<td>5</td>
<td>-</td>
</tr>
</tbody>
</table>

<h2 id="protocol">Protocol</h2>

<p>**<br />
Notes**</p>

<p><strong>Prepare Primer Solution:</strong></p>

<ul>
<li>centrifuge primers</li>
<li>add TE-buffer to dilute primers to a concentration of 100nM</li>
<li>Vortex</li>
<li>heat and shake at 50°C in thermomixer</li>
<li><strong>freeze and store at -20°C</strong>

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<ul>
<li><strong>Ran gelelectrophoresis with PCR products 02.07.2019 with 100x</strong></li>
</ul>

<table>
<thead>
<tr>
<th>DNA</th>
<th>2µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>Loading Dye(6x)</td>
<td>2µl</td>
</tr>

<tr>
<td>H20</td>
<td>16µl</td>
</tr>
</tbody>
</table>

<ul>
<li>Put 20µl in each pocket</li>
<li>put 6µl DNA Ladder in one pocket

<br /></li>
</ul>

<p><strong>Gibson - PCR:</strong></p>

<table>
<thead>
<tr>
<th>template DNA</th>
<th>5 µl (250 ng)</th>
</tr>
</thead>

<tbody>
<tr>
<td>forward primer (10 µM)</td>
<td>1.25 µl (end concentration 0.5 µM)</td>
</tr>

<tr>
<td>reverse primer (10 µM)</td>
<td>1.25 µl (end concentration 0.5 µM)</td>
</tr>

<tr>
<td>Master Mix</td>
<td>12.5 µl</td>
</tr>

<tr>
<td>Autoclaved H20</td>
<td>5 µlpr</td>
</tr>

<tr>
<td></td>
<td>25 µl</td>
</tr>
</tbody>
</table>

<ul>
<li>NEW PCR PROGRAM: igem gibson anhang</li>
<li>PCR is running over night. Finishes around 9pm and is then stored at 4°C in the PCR cycler

<br /></li>
</ul>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>No Protocol! People should make clear which protocol they want to use</li>
<li>Tested PCR Products on quantity with biophotometer: No clear results. Results are in our lab notebook

<br />

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>16 PCR Tubes and caps

<br />

<br /></li>
</ul>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Run Gel electrophoresis. See day before</li>
<li>Gibson Assembly of Fab Fragments</li>
<li>Gibson Assembly of different constructs into vector</li>
<li>Loading Dye 6x is stored in the fridge in Winkler&rsquo;s Lab</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-24" class="hidden">
      <label for="panel-24" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Create E.coli Starter Glycerol Stock</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Create E.coli Starter Glycerol Stock</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>see link</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Autoclaved 2x 500ml agar plate medium</li>
<li>Pour antibiotica-free plates</li>
<li>E.coli strains BL21, Star, Tuner, DH5alpha are cultivated on plates in 3 different dilutions: 1:10, 1:100</li>
<li>work very sterile
<a href="http://www.molbi.de/protocols/competent_cells_starter_cultures_v1_0.htm">http://www.molbi.de/protocols/competent_cells_starter_cultures_v1_0.htm</a>
<br /></li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Agar dissolves in autoclave</li>
<li><strong>work very sterile!!! No antibiotica</strong>

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>35g LB-Broth with agar

<br />

<br />

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Cultivate 2 colonies each in 50mL flasks. See protocol day 2 morning!</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-25" class="hidden">
      <label for="panel-25" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
          </p>
          <p class="inline-block">Cultivation of E.coli in 15ml flasks with 10ml LB-medium</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Cultivation of E.coli in 15ml flasks with 10ml LB-medium</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>see links</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Too many colonies on plates</li>
<li>used 2 µl of each strain and put the whole tip into 10ml LB-medium</li>
<li>shake at 37°C and 200rpm. Incubate until the evening</li>
<li>autoclave 1L erlenmeyer flask until the evening

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p><a href="http://www.molbi.de/protocols/competent_cells_starter_cultures_v1_0.htm">http://www.molbi.de/protocols/competent_cells_starter_cultures_v1_0.htm</a>  Day 2: Morning</p>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Too many colonies. Use less amount for cultivation on agar plates

<br />

<br />

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>4 Falcons 15mL</li>
<li>1 10ml plastic tip

<br />

<br />

<br />

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Cultivate in 1L erlenmeyer flask with 250ml LB until the morning</li>
<li>OD measurement of cultivated E.coli</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-26" class="hidden">
      <label for="panel-26" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Gibson PCR (Overhang) - New 57°C</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson PCR (Overhang) - New 57°C</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>PCR (according to NEB protocol)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Defrost Primer on Ice

<br />

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p>Gibson - PCR:</p>

<table>
<thead>
<tr>
<th>Template DNA</th>
<th>5 µl (250 ng)</th>
</tr>
</thead>

<tbody>
<tr>
<td>Primer fw (10µM)</td>
<td>1.25 µl (end concentration 0.5 µM)</td>
</tr>

<tr>
<td>Primer rev (10µM)</td>
<td>1.25 µl (end concentration 0.5 µM)</td>
</tr>

<tr>
<td>Master Mix</td>
<td>12.5 µl</td>
</tr>
</tbody>
</table>

<ul>
<li>NEW PCR PROGRAM: igem gibson anhang &mdash;-now with 57°C instead of 65°C

<ul>
<li>Also 35 cycles instead of 30 cycles</li>
</ul></li>
<li>PCR is running over night

<br /></li>
</ul>

<p><strong>Calculations and Results</strong></p>

<p>-</p>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Gib pAC Promotor fw almost empty

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>8 PCR tubes

<br /></li>
</ul>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Run Gel electrophoresis. See day before</li>
<li>Gibson Assembly of Fab Fragments</li>
<li>Gibson Assembly of different constructs into vector</li>
<li><strong>Get new pAC fw primer!!!</strong></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-27" class="hidden">
      <label for="panel-27" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Gibson PCR (Overhang) - New 57°C</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson PCR (Overhang) - New 57°C</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>PCR (according to NEB protocol)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Defrost Primer on Ice

<br />

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p>Gibson - PCR:</p>

<table>
<thead>
<tr>
<th>Template DNA</th>
<th>5 µl (250 ng)</th>
</tr>
</thead>

<tbody>
<tr>
<td>Primer fw (10µM)</td>
<td>1.25 µl (end concentration 0.5 µM)</td>
</tr>

<tr>
<td>Primer rev (10µM)</td>
<td>1.25 µl (end concentration 0.5 µM)</td>
</tr>

<tr>
<td>Master Mix</td>
<td>12.5 µl</td>
</tr>
</tbody>
</table>

<ul>
<li>NEW PCR PROGRAM: igem gibson anhang &mdash;-now with 57°C instead of 65°C

<ul>
<li>Also 35 cycles instead of 30 cycles</li>
</ul></li>
<li>PCR is running over night

<br /></li>
</ul>

<p><strong>Calculations and Results</strong></p>

<p>-</p>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Gib pAC Promotor fw almost empty

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>8 PCR tubes

<br /></li>
</ul>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Run Gel electrophoresis. See day before</li>
<li>Gibson Assembly of Fab Fragments</li>
<li>Gibson Assembly of different constructs into vector</li>
<li><strong>Get new pAC fw primer!!!</strong></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-28" class="hidden">
      <label for="panel-28" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Gibson Assembly - cancelled!</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson Assembly - cancelled!</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>see link</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Antonio Maccataio, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<ul>
<li>Checked Gel electrophoresis from Gibson PCR

<ul>
<li>Didnt work/not enough DNA!!!!

<br />

<br /></li>
</ul></li>
</ul>

<p><strong>Protocol</strong></p>

<p><a href="https://international.neb.com/protocols/2012/09/25/gibson-assembly-master-mix-assembly">https://international.neb.com/protocols/2012/09/25/gibson-assembly-master-mix-assembly</a></p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p><strong>Materials used</strong></p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Repeat the Gibson Overhang PCR</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-29" class="hidden">
      <label for="panel-29" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(5)</strong>
              
          </p>
          <p class="inline-block">Gibson Assembly - cancelled!</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson Assembly - cancelled!</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>see link</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Antonio Maccataio, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<ul>
<li>Checked Gel electrophoresis from Gibson PCR

<ul>
<li>Didnt work/not enough DNA!!!!

<br />

<br /></li>
</ul></li>
</ul>

<p><strong>Protocol</strong></p>

<p><a href="https://international.neb.com/protocols/2012/09/25/gibson-assembly-master-mix-assembly">https://international.neb.com/protocols/2012/09/25/gibson-assembly-master-mix-assembly</a></p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p><strong>Materials used</strong></p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Repeat the Gibson Overhang PCR</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-30" class="hidden">
      <label for="panel-30" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(6)</strong>
              
          </p>
          <p class="inline-block">Gibson PCR Control Gel-Electrophoresis</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson PCR Control Gel-Electrophoresis</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>see link</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>0,8% Agarose gel</li>
<li>DNA-Mix of Gibson PCR

<ul>
<li>2 µl DNA</li>
<li>2 µl Loading Dye (6x)</li>
<li>16 µl dH2O</li>
</ul></li>
<li>Gelelectrophoresis @130V for 60 min</li>
<li>Used only 4µl of 1kb DNA Ladder

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p><a href="https://www.addgene.org/protocols/gel-electrophoresis/">https://www.addgene.org/protocols/gel-electrophoresis/</a></p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Gel seems to run a bit crooked</li>
<li>Amplification of unknown DNA</li>
<li>Only little desired DNA amplified</li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>Gelectrophoresis

<ul>
<li>0.8g Agarose</li>
<li>100ml TAE-Buffer (1x)</li>
</ul></li>
<li>DNA-Mix

<ul>
<li>2µl DNA</li>
<li>2µl Loading Dye (6x)</li>
<li>16µl dH2O</li>
</ul></li>
<li>4µl DNA Ladder (1kb)</li>
<li>8 PCR tubes

<br />

<br /></li>
</ul>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Gibson Assembly if correct bases are seen in the gelphoto</li>
</ul>

<p><img src="/labjournal-entries/images/Gelphoto_Gibson_PCR_04_07_19.jpg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-31" class="hidden">
      <label for="panel-31" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(7)</strong>
              
          </p>
          <p class="inline-block">Cultivation of E.coli in 15ml flasks with 10ml LB-medium</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Cultivation of E.coli in 15ml flasks with 10ml LB-medium</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>see links</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Too many colonies on plates</li>
<li>used 2 µl of each strain and put the whole tip into 10ml LB-medium</li>
<li>shake at 37°C and 200rpm. Incubate until the evening</li>
<li>autoclave 1L erlenmeyer flask until the evening

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p><a href="http://www.molbi.de/protocols/competent_cells_starter_cultures_v1_0.htm">http://www.molbi.de/protocols/competent_cells_starter_cultures_v1_0.htm</a>  Day 2: Morning</p>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Too many colonies. Use less amount for cultivation on agar plates

<br />

<br />

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>4 Falcons 15mL</li>
<li>1 10ml plastic tip

<br />

<br />

<br />

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Cultivate in 1L erlenmeyer flask with 250ml LB until the morning</li>
<li>OD measurement of cultivated E.coli</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-32" class="hidden">
      <label for="panel-32" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(8)</strong>
              
          </p>
          <p class="inline-block">Gibson PCR Control Gel-Electrophoresis</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson PCR Control Gel-Electrophoresis</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>see link</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>0,8% Agarose gel</li>
<li>DNA-Mix of Gibson PCR

<ul>
<li>2 µl DNA</li>
<li>2 µl Loading Dye (6x)</li>
<li>16 µl dH2O</li>
</ul></li>
<li>Gelelectrophoresis @130V for 60 min</li>
<li>Used only 4µl of 1kb DNA Ladder

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p><a href="https://www.addgene.org/protocols/gel-electrophoresis/">https://www.addgene.org/protocols/gel-electrophoresis/</a></p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Gel seems to run a bit crooked</li>
<li>Amplification of unknown DNA</li>
<li>Only little desired DNA amplified</li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>Gelectrophoresis

<ul>
<li>0.8g Agarose</li>
<li>100ml TAE-Buffer (1x)</li>
</ul></li>
<li>DNA-Mix

<ul>
<li>2µl DNA</li>
<li>2µl Loading Dye (6x)</li>
<li>16µl dH2O</li>
</ul></li>
<li>4µl DNA Ladder (1kb)</li>
<li>8 PCR tubes

<br />

<br /></li>
</ul>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Gibson Assembly if correct bases are seen in the gelphoto</li>
</ul>

<p><img src="/labjournal-entries/images/Gelphoto_Gibson_PCR_04_07_19.jpg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-33" class="hidden">
      <label for="panel-33" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              5
              
          </p>
          <p class="inline-block">Control electrophoresis Gibson Overhnag PCR from 07-04</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control electrophoresis Gibson Overhnag PCR from 07-04</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>checked Gibson Overhang</li>
<li>ran agarose gel at 130 V</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-34" class="hidden">
      <label for="panel-34" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              5
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Control electrophoresis Gibson Overhnag PCR from 07-04</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control electrophoresis Gibson Overhnag PCR from 07-04</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>checked Gibson Overhang</li>
<li>ran agarose gel at 130 V</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-35" class="hidden">
      <label for="panel-35" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              5
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Aliquot E. coli</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Aliquot E. coli</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>http://www.molbi.de/protocols/competent_cells_starter_cultures_v1_0.htm Day 3: Morning</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>PCR products from 2019-07-04 are stored at 4 °C</li>
<li>aliquots are stored at -80 °C, cells are not competent</li>
</ul>

<p><strong>Protocol:</strong></p>

<ul>
<li>transfer 200 mL E.coli suspension in four 50 mL flasks &ndash;Centrifuge at 4000 rpm for 10 mins with ultra centrifuge</li>
<li>dilute centrifuged E. coli in 2.5 ml glycerol and 2.5 mL LB-medium</li>
<li>use 4 mL of each flask to make 1 mL aliquots</li>
<li>making around 16x 1 mL aliquots of each strain: DH5alpha, Star, Tuner, BL21</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-36" class="hidden">
      <label for="panel-36" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              5
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Aliquot E. coli</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Aliquot E. coli</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>http://www.molbi.de/protocols/competent_cells_starter_cultures_v1_0.htm Day 3: Morning</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>PCR products from 2019-07-04 are stored at 4 °C</li>
<li>aliquots are stored at -80 °C, cells are not competent</li>
</ul>

<p><strong>Protocol:</strong>
* transfer 200 mL E.coli suspension in four 50 mL flasks &ndash;Centrifuge at 4000 rpm for 10 mins with ultra centrifuge
* dilute centrifuged E. coli in 2.5 ml glycerol and 2.5 mL LB-medium
* use 4 mL of each flask to make 1 mL aliquots
* making around 16x 1 mL aliquots of each strain: DH5alpha, Star, Tuner, BL21</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-37" class="hidden">
      <label for="panel-37" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              9
              
          </p>
          <p class="inline-block">Purify Konstruct DNA from Agarose Gel</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Purify Konstruct DNA from Agarose Gel</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<ul>
<li>Before the purification the according DNA bands were cut out of the gel</li>
<li>The gel pieces were transferred into Eppendorf Reaction Tubes, which were weighed beforehand;</li>
<li>The mass of the gel pieces was determined by repeated weighing, to determine the necessary amounts of reagents

<br />

<br />

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p>Protocol of the NucleoSpin Extract Kit</p>

<p><strong>Calculations and Results</strong></p>

<ul>
<li>Reaction Tube weight (empty): K2b 1,013g K3b 1,014g K4a 1,013g K4b 1,013g</li>

<li><p>Weight of Gelfragments: K2b 0,287g K3b 0,356g K4a 0,288g K4b 0,287g</p></li>

<li><p>For each 100mg gel 300ul of NT1 buffer should be used</p></li>

<li><p>Amount of NT1 buffer:  K2b 861ul K3b 1068ul K4a 863ul K4b 861ul</p></li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<p>The gel fragments were quite large; This might have a negative impact on the efficiency of the purification;</p>

<p>As the lab with the nanodrop was already closed the Concentration of the processed DNA could not be assesed at this day;</p>

<p><strong>Materials used</strong></p>

<ul>
<li>NucleoSpin Extract Kit

<br />

<br /></li>
</ul>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Measure DNA concentrations</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-38" class="hidden">
      <label for="panel-38" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              9
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Purify Konstruct DNA from Agarose Gel</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Purify Konstruct DNA from Agarose Gel</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<ul>
<li>Before the purification the according DNA bands were cut out of the gel</li>
<li>The gel pieces were transferred into Eppendorf Reaction Tubes, which were weighed beforehand;</li>
<li>The mass of the gel pieces was determined by repeated weighing, to determine the necessary amounts of reagents

<br />

<br />

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p>Protocol of the NucleoSpin Extract Kit</p>

<p><strong>Calculations and Results</strong></p>

<ul>
<li>Reaction Tube weight (empty): K2b 1,013g K3b 1,014g K4a 1,013g K4b 1,013g</li>

<li><p>Weight of Gelfragments: K2b 0,287g K3b 0,356g K4a 0,288g K4b 0,287g</p></li>

<li><p>For each 100mg gel 300ul of NT1 buffer should be used</p></li>

<li><p>Amount of NT1 buffer:  K2b 861ul K3b 1068ul K4a 863ul K4b 861ul</p></li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<p>The gel fragments were quite large; This might have a negative impact on the efficiency of the purification;</p>

<p>As the lab with the nanodrop was already closed the Concentration of the processed DNA could not be assesed at this day;</p>

<p><strong>Materials used</strong></p>

<ul>
<li>NucleoSpin Extract Kit

<br />

<br /></li>
</ul>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Measure DNA concentrations</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-39" class="hidden">
      <label for="panel-39" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              9
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Production of electrocompetent Bacteria (BL21, BL21 Star, Tuner)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Production of electrocompetent Bacteria (BL21, BL21 Star, Tuner)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>http://www.molbi.de/protocols/competent_cells_electro_v1_0.htm</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>used only 100 mL of LB medium for 3 flasks</li>
<li>Problems faced:

<ul>
<li>no ON culture, directly from -80 °C into 100 mL LB medium</li>
<li>lost Tuner completely &ndash;no competent Tuner bacteria on -80 °C</li>
<li>BL21 Star had a smeary pellet in the last washing step &ndash;could be problematic</li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-40" class="hidden">
      <label for="panel-40" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              9
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Production of electrocompetent Bacteria (BL21, BL21 Star, Tuner)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Production of electrocompetent Bacteria (BL21, BL21 Star, Tuner)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>http://www.molbi.de/protocols/competent_cells_electro_v1_0.htm</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>used only 100 mL of LB medium for 3 flasks</li>
<li>Problems faced:

<ul>
<li>no ON culture, directly from -80 °C into 100 mL LB medium</li>
<li>lost Tuner completely &ndash;no competent Tuner bacteria on -80 °C</li>
<li>BL21 Star had a smeary pellet in the last washing step &ndash;could be problematic</li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-41" class="hidden">
      <label for="panel-41" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              10
              
          </p>
          <p class="inline-block">PCR with K1, K2a, K3a 3% DMSO and gel-electrophoresis</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>PCR with K1, K2a, K3a 3% DMSO and gel-electrophoresis</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Antonio Maccataio, Lisa Vogg, Marie Wiedemann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>5 yL DNA</li>
<li>1,25 ym primer fw</li>
<li>1,25 yL primer re</li>
<li>5 yL DMSO 15% Stock -&gt;3% diluted</li>
<li>Mastermix 12,5 yL</li>
</ul>

<p>2x 1:100 Dilution</p>

<p>1mM TE (DNA storage at -20°)</p>

<p>Gibson</p>

<p>gel-electrophoresis</p>

<p><img src="/labjournal-entries/images/C374518F-2355-4212-A19A-9E432DE22E42.jpeg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-42" class="hidden">
      <label for="panel-42" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              10
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">PCR with K1, K2a, K3a 3% DMSO and gel-electrophoresis</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>PCR with K1, K2a, K3a 3% DMSO and gel-electrophoresis</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Antonio Maccataio, Lisa Vogg, Marie Wiedemann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>5 yL DNA</li>
<li>1,25 ym primer fw</li>
<li>1,25 yL primer re</li>
<li>5 yL DMSO 15% Stock -&gt;3% diluted</li>
<li>Mastermix 12,5 yL</li>
</ul>

<p>2x 1:100 Dilution</p>

<p>1mM TE (DNA storage at -20°)</p>

<p>Gibson</p>

<blockquote>
<p>see foto</p>
</blockquote>

<p>gel-electrophoresis</p>

<p>??</p>

<p><img src="/labjournal-entries/images/C374518F-2355-4212-A19A-9E432DE22E42.jpeg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-43" class="hidden">
      <label for="panel-43" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              10
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Repeat failed PCR for Gibson overhangs with added DMSO</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Repeat failed PCR for Gibson overhangs with added DMSO</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>NEB kit instructions, NEB troubleshooting</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lisa Vogg, Antonio Maccataio</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>aliquoted 15% DMSO</li>
<li>used original amplified DNA of K1, K2a, K3a</li>
<li>added DMSO to a final concentration of 3%</li>
<li>lowered running temperature of the PCR from 57 °C to 55 °C due to DMSO</li>
<li>prepared 1:100 dilution of the respective primers (&ndash;1mM) in TE buffer and stored them at -20 °C</li>
<li>prepared a further 1:100 dilution (&ndash;10 µM) in dH2O for the PCR</li>
</ul>

<table>
<thead>
<tr>
<th>DNA</th>
<th>5 µL</th>
</tr>
</thead>

<tbody>
<tr>
<td>Primer fw</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>Primer re</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>Master Mix</td>
<td>12.5 µL</td>
</tr>

<tr>
<td>DMSO (15%)</td>
<td>5 µL</td>
</tr>

<tr>
<td></td>
<td>25 µL</td>
</tr>
</tbody>
</table>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-44" class="hidden">
      <label for="panel-44" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              10
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Repeat failed PCR for Gibson overhangs with added DMSO</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Repeat failed PCR for Gibson overhangs with added DMSO</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>NEB kit instructions, NEB troubleshooting</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lisa Vogg, Antonio Maccataio</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>aliquoted 15% DMSO</li>
<li>used original amplified DNA of K1, K2a, K3a</li>
<li>added DMSO to a final concentration of 3%</li>
<li>lowered running temperature of the PCR from 57 °C to 55 °C due to DMSO</li>
<li>prepared 1:100 dilution of the respective primers (&ndash;1mM) in TE buffer and stored them at -20 °C</li>
<li>prepared a further 1:100 dilution (&ndash;10 µM) in dH2O for the PCR</li>
</ul>

<p>DNA         |5 µL<br />
&mdash;&mdash;&mdash;&mdash;&mdash;-|&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;|&mdash;
Primer fw   |1.25 µL
Primer re   |1.25 µL
Master Mix  |12.5 µL
DMSO (15%)  |5 µL
    |25 µL</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-45" class="hidden">
      <label for="panel-45" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              11
              
          </p>
          <p class="inline-block">Electroporation Control</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Electroporation Control</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>NEB electroporation protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>electroporation of pUC19 into BL21, BL21 Star</li>
<li>one H2O control in BL21 Star</li>
<li>generated own electroporation protocol in the GeneUser (&gt;Users&gt;igem&gt;igem1)</li>
<li>used 25 µL bacteria with 1 µL 1:5 diluted pUC19

<ul>
<li>should be 50 µL next time instead of 25 µL</li>
</ul></li>
</ul>

<p><strong>Protocol:</strong>
1. incubate labelled agar plates in the incubator at 37 °C
2. incubate medium at 37 °C
3. thaw bacteria from -80 °C on ice
4. when bacteria are thawed, add DNA
5. keep 1 mm cuvettes on ice
6. electroporation of the bacteria with 1 µL DNA &ndash;immediately add warm medium to cuvette and mix by pipetting up and down
7. put bacteria on shaker at 37 °C for 30-60 mi at 300 rpm
8. plate them on the 37 °C warm agar plates
9. incubate the bacteria on plates ON &ndash;not too long or they will grow too much</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-46" class="hidden">
      <label for="panel-46" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              11
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Electroporation Control</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Electroporation Control</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>NEB electroporation protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>electroporation of pUC19 into BL21, BL21 Star</li>
<li>one H2O control in BL21 Star</li>
<li>generated own electroporation protocol in the GeneUser (&gt;Users&gt;igem&gt;igem1)</li>
<li>used 25 µL bacteria with 1 µL 1:5 diluted pUC19

<ul>
<li>should be 50 µL next time instead of 25 µL</li>
</ul></li>
</ul>

<p><strong>Protocol:</strong>
1. incubate labelled agar plates in the incubator at 37 °C
2. incubate medium at 37 °C
3. thaw bacteria from -80 °C on ice
4. when bacteria are thawed, add DNA
5. keep 1 mm cuvettes on ice
6. electroporation of the bacteria with 1 µL DNA &ndash;immediately add warm medium to cuvette and mix by pipetting up and down
7. put bacteria on shaker at 37 °C for 30-60 mi at 300 rpm
8. plate them on the 37 °C warm agar plates
9. incubate the bacteria on plates ON &ndash;not too long or they will grow too much</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-47" class="hidden">
      <label for="panel-47" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              11
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">repeat failed PCR with correct amount of primer at 57 °C with and without DMSO</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>repeat failed PCR with correct amount of primer at 57 °C with and without DMSO</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>NEB PCR kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Lena Schorr, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>set up two PCR reactions:</li>
</ul>

<p>DNA         |5 µL<br />
&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;&mdash;&mdash;&mdash;|&mdash;|&mdash;
Primer fw (10 µM)   |1.25 µL
Primer re (10 µM)   |1.25 µL
Master Mix      |12.5 µL
dH2O or DMSO (15 %) |5 µL
        |25 µL</p>

<ul>
<li>run PCR at 57 °C

<ul>
<li>last PCR might have failed due to too low temperatures</li>
<li>last PCRs for overhang generation might have failed due to much too high primer concentrations, would explain very bright bands in the previous gel</li>
</ul></li>
<li>amplified constructs form the very first PCR were not sufficient fpr the reaction</li>
<li>set up another PCR to amplify the constructs K1, K2a and K3a in a 25 µL reaction at 60 °C</li>
<li>run 0.8 % agarose gel (small) for the amplified constructs

<ul>
<li>2 µL DNA, 18 µL total volume in each pocket</li>
<li>run at 130 V</li>
<li>3 µL 1 kb DNA ladder</li>
</ul></li>
<li>run 0.8 % gel (medium) for the overhang PCR

<ul>
<li>used 2 µL DNA and 6 µL of test vector from the day before</li>
</ul></li>
<li>autoclaved 1 L LB medium</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-48" class="hidden">
      <label for="panel-48" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              11
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">repeat failed PCR with correct amount of primer at 57 °C with and without DMSO</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>repeat failed PCR with correct amount of primer at 57 °C with and without DMSO</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>NEB PCR kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Lena Schorr, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>set up two PCR reactions:</li>
</ul>

<table>
<thead>
<tr>
<th>DNA</th>
<th>5 µL</th>
</tr>
</thead>

<tbody>
<tr>
<td>Primer fw (10 µM)</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>Primer re (10 µM)</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>Master Mix</td>
<td>12.5 µL</td>
</tr>

<tr>
<td>dH2O or DMSO (15 %)</td>
<td>5 µL</td>
</tr>

<tr>
<td></td>
<td>25 µL</td>
</tr>
</tbody>
</table>

<ul>
<li>run PCR at 57 °C

<ul>
<li>last PCR might have failed due to too low temperatures</li>
<li>last PCRs for overhang generation might have failed due to much too high primer concentrations, would explain very bright bands in the previous gel</li>
</ul></li>
<li>amplified constructs form the very first PCR were not sufficient fpr the reaction</li>
<li>set up another PCR to amplify the constructs K1, K2a and K3a in a 25 µL reaction at 60 °C</li>
<li>run 0.8 % agarose gel (small) for the amplified constructs

<ul>
<li>2 µL DNA, 18 µL total volume in each pocket</li>
<li>run at 130 V</li>
<li>3 µL 1 kb DNA ladder</li>
</ul></li>
<li>run 0.8 % gel (medium) for the overhang PCR

<ul>
<li>used 2 µL DNA and 6 µL of test vector from the day before</li>
</ul></li>
<li>autoclaved 1 L LB medium</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-49" class="hidden">
      <label for="panel-49" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              12
              
          </p>
          <p class="inline-block">Check electroporated E. coli on plates</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Check electroporated E. coli on plates</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel SChreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>no growth on plates &ndash;incubate longer</li>
<li>E. coli were possibily not transformed</li>
<li>use bigger amount of eletrocompetent bacteria</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-50" class="hidden">
      <label for="panel-50" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              12
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Check electroporated E. coli on plates</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Check electroporated E. coli on plates</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel SChreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>no growth on plates &ndash;incubate longer</li>
<li>E. coli were possibily not transformed</li>
<li>use bigger amount of eletrocompetent bacteria</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-51" class="hidden">
      <label for="panel-51" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              13
              
          </p>
          <p class="inline-block">make E. coli strains Tuner and DH5alpha electrocompetent</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>make E. coli strains Tuner and DH5alpha electrocompetent</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>http://www.molbi.de/protocols/competent_cells_electro_v1_0.htm</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>thawed stock bacteria on ice</li>
<li>inocculated 2 25 µL of each strain (Tuner and DH5alpha) in 100 mL LB medium and incubated for 1:45 h until OD600 was 0.4-0.6</li>
<li>not much 10 % glycerol available so the two cultures of each strain have been pooled and washed together</li>

<li><p>frozen in 50 µL aliquots</p></li>

<li><p>modification of used protocol: 8.7 % glycerol used instead of 10 %</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-52" class="hidden">
      <label for="panel-52" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              13
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">make E. coli strains Tuner and DH5alpha electrocompetent</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>make E. coli strains Tuner and DH5alpha electrocompetent</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>http://www.molbi.de/protocols/competent_cells_electro_v1_0.htm</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>thawed stock bacteria on ice</li>
<li>inocculated 2 25 µL of each strain (Tuner and DH5alpha) in 100 mL LB medium and incubated for 1:45 h until OD600 was 0.4-0.6</li>
<li>not much 10 % glycerol available so the two cultures of each strain have been pooled and washed together</li>

<li><p>frozen in 50 µL aliquots</p></li>

<li><p>modification of used protocol: 8.7 % glycerol used instead of 10 %</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-53" class="hidden">
      <label for="panel-53" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
          </p>
          <p class="inline-block">amplification of original constructs</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>amplification of original constructs</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>NEB kit instructions</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>used 1:10 dilution of 100 µM stock of the primers</li>
<li>amplified all constructs</li>
<li>PCR at 60 °C, 25 µL reaction with 5 µL DNA</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-54" class="hidden">
      <label for="panel-54" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">produce SOB medium</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>produce SOB medium</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Antonio Maccataio</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Protocol:</strong></p>

<p>Per liter: To 950 mL of deionized H2O, add:</p>

<table>
<thead>
<tr>
<th>Tryptone</th>
<th>20 g</th>
</tr>
</thead>

<tbody>
<tr>
<td>Yeast extract</td>
<td>5 g</td>
</tr>

<tr>
<td>NaCl</td>
<td>0.5 g</td>
</tr>
</tbody>
</table>

<p>SOC medium is identical to SOB medium, except that it contains 20 mM glucose. To prepare SOB medium, combine the above ingredients and shake until the solutes have dissolved. Add 10 mL of a 250 mM solution of KCl. (This solution is made by dissolving 1.86 g of KCl in 100 mL of deionized H2O.)
Adjust the pH of the medium to 7.0 with 5 N NaOH (∼0.2 mL). Adjust the volume of the solution to 1 L with deionized H2O. Sterilize by autoclaving for 20 min at 15 psi (1.05 kg/cm2) on liquid cycle.</p>

<p>Just before use, add 5 mL of a sterile solution of 2 M MgCl2 and 5ml of MGSO4. (This solution is made by dissolving 40,66 g of MgCl2 (Hydrate) and 49,3g MgSO4 (Hydrate) respectively in 90 mL of deionized H2O. Adjust the volume of the solution to 100 mL with deionized H2O and sterilize by autoclaving for 20 min at 15 psi [1.05 kg/cm2] on liquid cycle.)
After the SOB medium has been autoclaved, allow it to cool to 60°C or less.
Add 20 mL of a sterile 1 M solution of glucose. (This solution is made by dissolving 18 g of glucose in 90 mL of deionized H2O. After the sugar has dissolved, adjust the volume of the solution to 100 mL with deionized H2O and sterilize by passing it through a 0.22-µm filter.)</p>

<ul>
<li>only produced 100 mL</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-55" class="hidden">
      <label for="panel-55" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">amplification of original constructs</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>amplification of original constructs</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>NEB kit instructions</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>used 1:10 dilution of 100 µM stock of the primers</li>
<li>amplified all constructs</li>
<li>PCR at 60 °C, 25 µL reaction with 5 µL DNA</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-56" class="hidden">
      <label for="panel-56" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">produce SOB medium</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>produce SOB medium</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Antonio Maccataio</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Protocol:</strong></p>

<p>Per liter: To 950 mL of deionized H2O, add:
Tryptone        |20 g<br />
&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;&mdash;&mdash;&mdash;&mdash;|&ndash;|&ndash;
Yeast extract       |5 g
NaCl            |0.5 g</p>

<p>SOC medium is identical to SOB medium, except that it contains 20 mM glucose. To prepare SOB medium, combine the above ingredients and shake until the solutes have dissolved. Add 10 mL of a 250 mM solution of KCl. (This solution is made by dissolving 1.86 g of KCl in 100 mL of deionized H2O.)
Adjust the pH of the medium to 7.0 with 5 N NaOH (∼0.2 mL). Adjust the volume of the solution to 1 L with deionized H2O. Sterilize by autoclaving for 20 min at 15 psi (1.05 kg/cm2) on liquid cycle.</p>

<p>Just before use, add 5 mL of a sterile solution of 2 M MgCl2 and 5ml of MGSO4. (This solution is made by dissolving 40,66 g of MgCl2 (Hydrate) and 49,3g MgSO4 (Hydrate) respectively in 90 mL of deionized H2O. Adjust the volume of the solution to 100 mL with deionized H2O and sterilize by autoclaving for 20 min at 15 psi [1.05 kg/cm2] on liquid cycle.)
After the SOB medium has been autoclaved, allow it to cool to 60°C or less.
Add 20 mL of a sterile 1 M solution of glucose. (This solution is made by dissolving 18 g of glucose in 90 mL of deionized H2O. After the sugar has dissolved, adjust the volume of the solution to 100 mL with deionized H2O and sterilize by passing it through a 0.22-µm filter.)</p>

<ul>
<li>only produced 100 mL</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-57" class="hidden">
      <label for="panel-57" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              16
              
          </p>
          <p class="inline-block">electroporation control</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>electroporation control</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>https://international.neb.com/Protocols/0001/01/01/electroporation-protocol-c2986</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>control if BL21 blank/star/tuner and DH5alpha are electrocompetent</li>
<li>added 2 µL of pUC19 1:5 &ndash;electroporation with iGEM protocol</li>
<li>bacteria plated on 37 °C LB-Agar plates with ampicillin</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-58" class="hidden">
      <label for="panel-58" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              16
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">electroporation control</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>electroporation control</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>https://international.neb.com/Protocols/0001/01/01/electroporation-protocol-c2986</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>control if BL21 blank/star/tuner and DH5alpha are electrocompetent</li>
<li>added 2 µL of pUC19 1:5 &ndash;electroporation with iGEM protocol</li>
<li>bacteria plated on 37 °C LB-Agar plates with ampicillin</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-59" class="hidden">
      <label for="panel-59" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              17
              
          </p>
          <p class="inline-block">Heat shock of DH5alpha and BL21</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Heat shock of DH5alpha and BL21</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ol>
<li>Thaw chemical competent bacteria on ice (200 µL each)</li>
<li>Add 5 µL of pUC vector</li>
<li>30 min incubation on ice</li>
<li>1 min 42°C (450 rmp)</li>
<li>2 min on ice</li>
<li>Add 500 µL SOC medium</li>
<li>30 min 37°C (450 rpm)</li>
<li>Plate 200 µL bacs on Amp plates</li>
<li>Incubation over night at 37°C</li>
</ol>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-60" class="hidden">
      <label for="panel-60" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              17
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Preparation of chemically competent E. coli</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Preparation of chemically competent E. coli</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>https://www.helmholtz-muenchen.de/fileadmin/PEPF/Protocols/Chemically-competent-cells.pdf</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Daniel Schreyer, Ruben Caspary</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>prepared chemically competent E. coli: DH5alpha, BL21 Star, Tuner, BL21</li>
<li>used E. coli with the following OD:</li>
</ul>

<table>
<thead>
<tr>
<th>Strain</th>
<th>BL21</th>
<th>BL21 Star</th>
<th>Tuner</th>
<th>DH5alpha</th>
</tr>
</thead>

<tbody>
<tr>
<td>OD</td>
<td>0.55</td>
<td>0.499</td>
<td>0.43</td>
<td>0.42</td>
</tr>
</tbody>
</table>

<ul>
<li>20x 200 µL aliquots stored at -80 °C of each strain</li>
<li>CC buffer is stored at 4 °C and is sterilized through 0.22 µm filtration</li>
<li>used <sup>1</sup>&frasl;<sub>5</sub> of each medium &ndash;cultivated in 100 mL LB instead of 500 mL</li>
<li>used stock E. coli, not single colony &ndash;lower transformation rate possible</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-61" class="hidden">
      <label for="panel-61" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              17
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Preparation of chemically competent E. coli</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Preparation of chemically competent E. coli</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>https://www.helmholtz-muenchen.de/fileadmin/PEPF/Protocols/Chemically-competent-cells.pdf</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Daniel Schreyer, Ruben Caspary</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>prepared chemically competent E. coli: DH5alpha, BL21 Star, Tuner, BL21</li>
<li>used E. coli with the following OD:</li>
</ul>

<table>
<thead>
<tr>
<th>Strain</th>
<th>BL21</th>
<th>BL21 Star</th>
<th>Tuner</th>
<th>DH5alpha</th>
</tr>
</thead>

<tbody>
<tr>
<td>OD</td>
<td>0.55</td>
<td>0.499</td>
<td>0.43</td>
<td>0.42</td>
</tr>
</tbody>
</table>

<ul>
<li>20x 200 µL aliquots stored at -80 °C of each strain</li>
<li>CC buffer is stored at 4 °C and is sterilized through 0.22 µm filtration</li>
<li>used <sup>1</sup>&frasl;<sub>5</sub> of each medium &ndash;cultivated in 100 mL LB instead of 500 mL</li>
<li>used stock E. coli, not single colony &ndash;lower transformation rate possible</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-62" class="hidden">
      <label for="panel-62" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              17
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Heat shock of DH5alpha and BL21</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Heat shock of DH5alpha and BL21</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ol>
<li>Thaw chemical competent bacteria on ice (200 µL each)</li>
<li>Add 5 µL of pUC vector</li>
<li>30 min incubation on ice</li>
<li>1 min 42°C (450 rmp)</li>
<li>2 min on ice</li>
<li>Add 500 µL SOC medium</li>
<li>30 min 37°C (450 rpm)</li>
<li>Plate 200 µL bacs on Amp plates</li>
<li>Incubation over night at 37°C</li>
</ol>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-63" class="hidden">
      <label for="panel-63" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              19
              
          </p>
          <p class="inline-block">Check plates of transformation of 2019-07-18</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Check plates of transformation of 2019-07-18</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>checked LB agar plates with ampicillin oand our transformed bacteria of 2019-07-18</li>

<li><p>growth on both plates &ndash;heat shock success</p></li>

<li><p>pick on DH5alpha colony from the agar plate</p></li>

<li><p>cultivated in 250 mL LB-medium with 250 µL (50 mg/ml) Ampicillin</p></li>

<li><p>cultivate at 30 °C and 130 rpm for around 18 h</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-64" class="hidden">
      <label for="panel-64" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              19
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Check plates of transformation of 2019-07-18</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Check plates of transformation of 2019-07-18</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>checked LB agar plates with ampicillin oand our transformed bacteria of 2019-07-18</li>

<li><p>growth on both plates &ndash;heat shock success</p></li>

<li><p>pick on DH5alpha colony from the agar plate</p></li>

<li><p>cultivated in 250 mL LB-medium with 250 µL (50 mg/ml) Ampicillin</p></li>

<li><p>cultivate at 30 °C and 130 rpm for around 18 h</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-65" class="hidden">
      <label for="panel-65" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              19
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Miniprep of Gibson CHO Constructs (ON Cultures 18.08) / Preperation for MidiPrep of pUC19 /Control digestion of Gibson CHO Constructs</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Miniprep of Gibson CHO Constructs (ON Cultures 18.08) / Preperation for MidiPrep of pUC19 /Control digestion of Gibson CHO Constructs</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>**<br />
Notes**</p>

<ul>
<li>Did a MidiPrep of ON Cultures from 18.08

<ul>
<li>Concentration:</li>
</ul></li>
</ul>

<p>Agarose-Gel electrophoresis for validation</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>K1-1</th>
<th>K1-2</th>
<th>K1-3</th>
<th>K2a-1</th>
<th>K2a-2</th>
<th>K2a-3</th>
<th>K2b-1</th>
<th>K2b-2</th>
<th>K2b-3</th>
</tr>
</thead>

<tbody>
<tr>
<td>Conc [ng/µl]</td>
<td>33,9</td>
<td>31,9</td>
<td>27,4</td>
<td>38,7</td>
<td>31,7</td>
<td>27,4</td>
<td>28,5</td>
<td>44,4</td>
<td>26,1</td>
</tr>
</tbody>
</table>

<p>|Component|Amount|</p>

<p>|DNA|3 µl|
|6X Loading Dye|2 µl|
|dH2O|1 µl|</p>

<p>-Same Loading order as NanoDrop concentration measurment</p>

<ul>
<li><p>Did an ON Culture of a single conlony of pUC19 (18.08)</p>

<ul>
<li>Picked one colony and incubated it in 5 ml LB+Amp in the shaker @37°C</li>
<li>In the late afternoon &ndash;1:500 in 85ml LB+Amp &ndash;shake for 12-16h @37°C</li>
</ul></li>

<li><p>Control digestion of CHO Constructs</p>

<ul>
<li>Restriction enzymes: ApaI and HindIII-HF</li>
</ul></li>
</ul>

<table>
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>300 ng</td>
</tr>

<tr>
<td>ApaI</td>
<td>1 µl</td>
</tr>

<tr>
<td>HindIII-HF</td>
<td>1 µl</td>
</tr>
</tbody>
</table>

<table>
<thead>
<tr>
<th>CutSmart Buffer</th>
<th>5</th>
</tr>
</thead>

<tbody>
<tr>
<td>dH2O</td>
<td>ad 50 µl</td>
</tr>
</tbody>
</table>

<ul>
<li>-   Protocol
    -   Incubate ApaI at 25°C for 5-15 minutes
    -   Add 1.0 µl (or 10 units) of HindIII-HF and  Incubate at 37°C for 5-15 minutes
    -   Inactivate both at 80°C for 20 minutes</li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Low concentration of construc &ndash;more LB+amp Medium next time with longer incubation time in the shaker?

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>MidiPrep of ON Culture of pUC19</li>
<li>Empty constructs &ndash;repeat gibson</li>
</ul>

<p><img src="/labjournal-entries/images/Unbenannt.png" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/TS05376Aug192019.jpg" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/TS05375Aug192019.jpg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-66" class="hidden">
      <label for="panel-66" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              21
              
          </p>
          <p class="inline-block">Cultivation of transformed DH5alpha and BL21, PCR with new settings</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Cultivation of transformed DH5alpha and BL21, PCR with new settings</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Cultivation for miniprep of pUC19:</strong></p>

<ul>
<li>no bacteria grew in the cultivation of 2019-07-20</li>
<li>incubator did not heat up</li>
<li>new clones picked: 5x DH5alpha, 3x BL21</li>
<li>in 5 mL LB medium with Amp each</li>
<li>incubation at 37 °C</li>
</ul>

<p><strong>PCR overhang K1, K2a, K3a with DMSO:</strong></p>

<ul>
<li>primers used:

<ul>
<li>K1: fw Gib pAC Promotor fw 2.0, re Gib pACK1HisStop prim</li>
<li>K2a: fw Gib pAC Promotor fw 2.0, re Gib pAC K2a HisStop</li>
<li>K3a: fw Gib pAC Promotor fw 2.0, re Gib pAc K3a fusion re</li>
</ul></li>
<li>PCR program:

<ul>
<li>98 °C 3 min</li>
<li>98 °C 10 s</li>
<li>53.7 °C 30 s</li>
<li>72 °C 1 min</li>
<li>72 °C 10 min</li>
<li>35 cycles</li>
</ul></li>
<li>PCR products stored at 4 °C</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-67" class="hidden">
      <label for="panel-67" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              21
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Cultivation of transformed DH5alpha and BL21, PCR with new settings</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Cultivation of transformed DH5alpha and BL21, PCR with new settings</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Cultivation for miniprep of pUC19:</strong>
* no bacteria grew in the cultivation of 2019-07-20
* incubator did not heat up
* new clones picked: 5x DH5alpha, 3x BL21
* in 5 mL LB medium with Amp each
* incubation at 37 °C
<strong>PCR overhang K1, K2a, K3a with DMSO:</strong>
* primers used:
  * K1: fw Gib pAC Promotor fw 2.0, re Gib pACK1HisStop prim
  * K2a: fw Gib pAC Promotor fw 2.0, re Gib pAC K2a HisStop
  * K3a: fw Gib pAC Promotor fw 2.0, re Gib pAc K3a fusion re
* PCR program:
  * 98 °C 3 min
  * 98 °C 10 s
  * 53.7 °C 30 s
  * 72 °C 1 min
  * 72 °C 10 min
  * 35 cycles
*PCR products stored at 4 °C</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-68" class="hidden">
      <label for="panel-68" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              22
              
          </p>
          <p class="inline-block">Run test gel for new overhang PCR, mini prep of test transformation</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Run test gel for new overhang PCR, mini prep of test transformation</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Extract me! Mini kit instructions</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>0.8 % gel with 2 µL DNA, 3 µL loading dye, 13 µL dH2o</li>
<li>out of 8 cultivated colonies, only 2 showed bacterial growth</li>
<li>performed mini prep for those two cultures (BL21 colonies 2 and 3)</li>
<li>ran test gel for mini prep</li>
</ul>

<p><strong>Protocol:</strong>
1. Pellet cell from bacterial culture by centrifugation for 5 min at 3500g
2. discard supernatant and resuspend cell pellet thoroughly in 250 µl resuspension buffer. Transfer resuspended cell pellet to sterile 1.5 ml tube.
3. add 250 µl lysis buffer and invert tube 4-6 times. DO NOT VORTEX. If sample is not clear immediately, incubate for 1-2 min.
4. add 350 µl neutralization buffer and invert tube several times. DO NOT VORTEX.<br />
5. centrifuge for 5 min at 11000 g. Repeat if supernatant is not clear.
6. Carefully transfer supernatant into a purification microcolumn placed in a collection tube. Do not contaminate sample with pellet! Centrifuge for 60 s at 11000 g.
7. Optional: add 500 µl NW buffer and centrifuge 60 s at 11000g.
8. add 750 µl wash buffer and centrifuge for 60 s at 11000g. Discard filtrate and reuse collection tube.
9. centrifuge for 2 min at 11000g. Discard collection tube and carefully transfer column to a sterile 1.5 ml tube.
10. add 50-100 µl elution buffer directly onto the column. Incubate at room temperature for 60 s.
11. centrifuge for 60 s at 11000g. Remove microcolumn. Isolated DNA should be stored at 4 or -20°C.</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-69" class="hidden">
      <label for="panel-69" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              22
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Overnight culture of transformed E. coli for control</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Overnight culture of transformed E. coli for control</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>Picked 3 DH5alpha and 1 BL21 Heat shock puC19 transformed E.coli and incubated them over night in 2 ml LB+Amp at 37°C with 220 rpm</li>
<li>OD measurements in the morning to validate transformation success</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-70" class="hidden">
      <label for="panel-70" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              22
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Overnight culture of transformed E. coli for control</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Overnight culture of transformed E. coli for control</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>Picked 3 DH5alpha and 1 BL21 Heat shock puC19 transformed E.coli and incubated them over night in 2 ml LB+Amp at 37°C with 220 rpm</li>
<li>OD measurements in the morning to validate transformation success</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-71" class="hidden">
      <label for="panel-71" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              22
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Run test gel for new overhang PCR, mini prep of test transformation</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Run test gel for new overhang PCR, mini prep of test transformation</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Extract me! Mini kit instructions</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>0.8 % gel with 2 µL DNA, 3 µL loading dye, 13 µL dH2o</li>
<li>out of 8 cultivated colonies, only 2 showed bacterial growth</li>
<li>performed mini prep for those two cultures (BL21 colonies 2 and 3)</li>
<li>ran test gel for mini prep</li>
</ul>

<p><strong>Protocol:</strong></p>

<ol>
<li>Pellet cell from bacterial culture by centrifugation for 5 min at 3500g</li>
<li>discard supernatant and resuspend cell pellet thoroughly in 250 µl resuspension buffer. Transfer resuspended cell pellet to sterile 1.5 ml tube.</li>
<li>add 250 µl lysis buffer and invert tube 4-6 times. DO NOT VORTEX. If sample is not clear immediately, incubate for 1-2 min.</li>
<li>add 350 µl neutralization buffer and invert tube several times. DO NOT VORTEX.<br /></li>
<li>centrifuge for 5 min at 11000 g. Repeat if supernatant is not clear.</li>
<li>Carefully transfer supernatant into a purification microcolumn placed in a collection tube. Do not contaminate sample with pellet! Centrifuge for 60 s at 11000 g.</li>
<li>Optional: add 500 µl NW buffer and centrifuge 60 s at 11000g.</li>
<li>add 750 µl wash buffer and centrifuge for 60 s at 11000g. Discard filtrate and reuse collection tube.</li>
<li>centrifuge for 2 min at 11000g. Discard collection tube and carefully transfer column to a sterile 1.5 ml tube.</li>
<li>add 50-100 µl elution buffer directly onto the column. Incubate at room temperature for 60 s.</li>
<li>centrifuge for 60 s at 11000g. Remove microcolumn. Isolated DNA should be stored at 4 or -20°C.</li>
</ol>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-72" class="hidden">
      <label for="panel-72" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              23
              
          </p>
          <p class="inline-block">OD of over night culture DH5alpa with pUC19 and BL21 with pUC19</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>OD of over night culture DH5alpa with pUC19 and BL21 with pUC19</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie, Jannik</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>OD: DH5alpha with pUC19 = 2,79</li>
<li>OD: BL21 with pUC19= 4,4</li>
<li>Argarose gel for vector extraction for BL21</li>
<li>5 yL Eluate</li>
<li>3 yL dye</li>
<li>12 yL H2O</li>
<li>4 yL Ladder</li>
<li>Cut lower band out</li>
<li>Gel volume 230 mg</li>
<li>Used Qiagen QiAquick gel-extraction kit50 (see Protocol)</li>
<li>Used 80 yL EB-buffer to eluate</li>
<li>PCR amplification of K1 with

<ul>
<li>5 yL DNA</li>
<li>1,25 yL primer re &amp; fw</li>
<li>12 yL Master-mix</li>
<li>5 yL H2O

<br /></li>
</ul></li>
</ul>

<blockquote>
<p>BL21 pUC19 concentration: 6,53 ng/µL</p>
</blockquote>

<p><strong>Protocol</strong></p>

<p><img src="/labjournal-entries/images/45CC5796-285C-4552-9229-78437520F6DF.jpeg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-73" class="hidden">
      <label for="panel-73" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              23
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">OD of over night culture DH5alpa with pUC19 and BL21 with pUC19</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>OD of over night culture DH5alpa with pUC19 and BL21 with pUC19</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie, Jannik</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>OD: DH5alpha with pUC19 = 2,79</li>
<li>OD: BL21 with pUC19= 4,4</li>
<li>Argarose gel for vector extraction for BL21</li>
<li>5 yL Eluate</li>
<li>3 yL dye</li>
<li>12 yL H2O</li>
<li>4 yL Ladder</li>
<li>Cut lower band out</li>
<li>Gel volume 230 mg</li>
<li>Used Qiagen QiAquick gel-extraction kit50 (see Protocol)</li>
<li>Used 80 yL EB-buffer to eluate</li>
<li>PCR amplification of K1 with

<ul>
<li>5 yL DNA</li>
<li>1,25 yL primer re &amp; fw</li>
<li>12 yL Master-mix</li>
<li>5 yL H2O

<br /></li>
</ul></li>
</ul>

<blockquote>
<p>BL21 pUC19 concentration: 6,53 ng/yL</p>
</blockquote>

<p><strong>Protocol</strong></p>

<p><img src="/labjournal-entries/images/45CC5796-285C-4552-9229-78437520F6DF.jpeg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-74" class="hidden">
      <label for="panel-74" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              24
              
          </p>
          <p class="inline-block">Gibson assembly and transformation of K2a, K2b, K3a and K4a in pACYC in BL21</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson assembly and transformation of K2a, K2b, K3a and K4a in pACYC in BL21</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Gibson assembly instructions, heat shock protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Jannik Wagner, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>measured DNA concentrations of all constructs</li>

<li><p>calculated the amount of DNA and vector for the Gibson assembly</p></li>

<li><p>Gibson assembly of 2a, 2b, 3a+b, 4a+b</p></li>

<li><p>transformation of the constructs into BL21 via heat shock</p>

<ul>
<li>200 µL bacteria with 2 µL Gibson reagent</li>
</ul></li>

<li><p>preparation of chloramphenicol plates</p></li>

<li><p>plated transformed bacteria &ndash;incubator at 37 °C</p></li>
</ul>

<p><strong>Concentrations of DNA of the constructs:</strong></p>

<ul>
<li>K1: 808.9 ng/µL</li>
<li>K2a: 868.7 ng/µL</li>
<li>K2b: 847.2 ng/µL</li>
<li>K3a: 791 ng/µL</li>
<li>K3b: 819.1 ng/µL</li>
<li>K4a: 776.1 ng/µL</li>
<li>K4b: 805.9 ng/µL</li>
</ul>

<p><strong>Lengths of the DNA constructs:</strong></p>

<ul>
<li>K1: 1629 bp</li>
<li>K2a: 1278 bp</li>
<li>K2b: 1218 bp</li>
<li>K3a: 801 bp</li>
<li>K3b: 753 bp</li>
<li>K4a: 801 bp</li>
<li>K4b: 759 bp</li>
</ul>

<p><strong>Calculation of the amount of contruct and vector needed for the Assembly:</strong></p>

<ul>
<li>our constructs and vector in pmol/µl (with the formula: amount of DNA * 1000 / bp of the constructs + 650 Da)</li>
<li>K1: 1.04 pmol/µL</li>
<li>K2a: 1.097 pmol/µL</li>
<li>K2b: 1.46 pmol/µL</li>
<li>K3a: 1.36 pmol/µL</li>
<li>K3b: 1.49 pmol/µL</li>
<li>K4a: 1.34 pmol/µL</li>
<li>K4b: 1.46 pmol/µL
&ndash;1:3 dilution with H2O &ndash;1 µL was taken for the Gibson Assembly (=0.37 pmol/µl for construct 2a)
&ndash;&gt;Vector (pACYC): same amount: ~0.4 pmol/µL</li>
</ul>

<p><strong>Gibson Protocol:</strong></p>

<ul>
<li>0.02-0.5 pmol/µL DNA</li>
<li>10 µL Gibson assembly master mix</li>
<li>10 µL dH2O</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-75" class="hidden">
      <label for="panel-75" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              24
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Gibson assembly and transformation of K2a, K2b, K3a and K4a in pACYC in BL21</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson assembly and transformation of K2a, K2b, K3a and K4a in pACYC in BL21</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Gibson assembly instructions, heat shock protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Jannik Wagner, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>measured DNA concentrations of all constructs</li>

<li><p>calculated the amount of DNA and vector for the Gibson assembly</p></li>

<li><p>Gibson assembly of 2a, 2b, 3a+b, 4a+b</p></li>

<li><p>transformation of the constructs into BL21 via heat shock</p>

<ul>
<li>200 µL bacteria with 2 µL Gibson reagent</li>
</ul></li>

<li><p>preparation of chloramphenicol plates</p></li>

<li><p>plated transformed bacteria &ndash;incubator at 37 °C
<strong>Concentrations of DNA of the constructs:</strong></p></li>

<li><p>K1: 808.9 ng/µL</p></li>

<li><p>K2a: 868.7 ng/µL</p></li>

<li><p>K2b: 847.2 ng/µL</p></li>

<li><p>K3a: 791 ng/µL</p></li>

<li><p>K3b: 819.1 ng/µL</p></li>

<li><p>K4a: 776.1 ng/µL</p></li>

<li><p>K4b: 805.9 ng/µL
<strong>Lengths of the DNA constructs:</strong></p></li>

<li><p>K1: 1629 bp</p></li>

<li><p>K2a: 1278 bp</p></li>

<li><p>K2b: 1218 bp</p></li>

<li><p>K3a: 801 bp</p></li>

<li><p>K3b: 753 bp</p></li>

<li><p>K4a: 801 bp</p></li>

<li><p>K4b: 759 bp
<strong>Calculation of the amount of contruct and vector needed for the Assembly:</strong></p></li>

<li><p>our constructs and vector in pmol/µl (with the formula: amount of DNA * 1000 / bp of the constructs + 650 Da)</p></li>

<li><p>K1: 1.04 pmol/µL</p></li>

<li><p>K2a: 1.097 pmol/µL</p></li>

<li><p>K2b: 1.46 pmol/µL</p></li>

<li><p>K3a: 1.36 pmol/µL</p></li>

<li><p>K3b: 1.49 pmol/µL</p></li>

<li><p>K4a: 1.34 pmol/µL</p></li>

<li><p>K4b: 1.46 pmol/µL
&ndash;1:3 dilution with H2O &ndash;1 µL was taken for the Gibson Assembly (=0.37 pmol/µl for construct 2a)
&ndash;&gt;Vector (pACYC): same amount: ~0.4 pmol/µL
<strong>Gibson Protocol:</strong></p></li>

<li><p>0.02-0.5 pmol/µL DNA</p></li>

<li><p>10 µL Gibson assembly master mix</p></li>

<li><p>10 µL dH2O</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-76" class="hidden">
      <label for="panel-76" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              25
              
          </p>
          <p class="inline-block">Transformation of E.coli BL21 with Test Vector pUC</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Transformation of E.coli BL21 with Test Vector pUC</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Transformation mit Ca2&#43; - kompetente E.coli - Zellen</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Marie Wiedemann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1</th>
<th>Attempt 2</th>
<th>Attempt 3</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µg (=0,5 µl)</td>
<td>2 µg (= 1 µl)</td>
<td>1 µg (= µl)</td>
</tr>

<tr>
<td>NEB buffer 3.1 (x10 )</td>
<td>5</td>
<td>5</td>
<td>-</td>
</tr>
</tbody>
</table>

<h2 id="protocol">Protocol</h2>

<p><strong>Transformation of BL21 with Test vector pUC (Amp res.):</strong></p>

<ul>
<li><p>Protocol used is in folder page 1: Transformation mit Ca2+ - kompetente E.coli - Zellen</p>

<ul>
<li>Nanodrop: 6,53ng/µL</li>
<li>50ng DNA ( = 7,7µL) and 200 µL chemocompetent cells</li>
<li>heatshock 45s, 42 °C

<br /></li>
</ul></li>

<li><p>plate on Amp plates &ndash;test vector with Amp resitance</p></li>

<li><p>incubate at 37°C overnight</p></li>
</ul>

<p><strong>Result</strong>: succesfull transfortmation since the bacteria grew on the Amp plate</p>

<p>Agarose gel electropheres of PCR products from the morning,</p>

<p><strong>Amplification PCR of Construct 1</strong></p>

<ul>
<li>Protocol from the PCR kit</li>
<li>Q5 polymerase</li>
<li>4µL template</li>
<li>Procedure:

<ol>
<li>letting the cycler heat up and wait untilll actuall programm starts</li>
<li>then put in the reaction tube
<br /></li>
</ol></li>
</ul>

<p><strong>Result:</strong>  negative</p>

<ul>
<li>either bad primers</li>
<li>or because no DSMO was used</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-77" class="hidden">
      <label for="panel-77" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              25
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Transformation of E.coli BL21 with Test Vector pUC</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Transformation of E.coli BL21 with Test Vector pUC</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Transformation mit Ca2&#43; - kompetente E.coli - Zellen</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Marie Wiedemann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1</th>
<th>Attempt 2</th>
<th>Attempt 3</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µg (=0,5 µl)</td>
<td>2 µg (= 1 µl)</td>
<td>1 µg (= µl)</td>
</tr>

<tr>
<td>NEB buffer 3.1 (x10 )</td>
<td>5</td>
<td>5</td>
<td>-</td>
</tr>
</tbody>
</table>

<h2 id="protocol">Protocol</h2>

<p><strong>Transformation of BL21 with Test vector pUC (Amp res.):</strong></p>

<ul>
<li><p>Protocol used is in folder page 1: Transformation mit Ca2+ - kompetente E.coli - Zellen</p>

<ul>
<li>Nanodrop: 6,53ng/µL</li>
<li>50ng DNA ( = 7,7µL) and 200 µL chemocompetent cells</li>
<li>heatshock 45s, 42 °C

<br /></li>
</ul></li>

<li><p>plate on Amp plates &ndash;test vector with Amp resitance</p></li>

<li><p>incubate at 37°C overnight</p></li>
</ul>

<p><strong>Result</strong>: succesfull transfortmation since the bacteria grew on the Amp plate</p>

<p>Agarose gel electropheres of PCR products from the morning,</p>

<p><strong>Amplification PCR of Construct 1</strong></p>

<ul>
<li>Protocol from the PCR kit</li>
<li>Q5 polymerase</li>
<li>4µL template</li>
<li>Procedure:

<ol>
<li>letting the cycler heat up and wait untilll actuall programm starts</li>
<li>then put in the reaction tube</li>
<li>
<br /></li>
</ol></li>
</ul>

<p><strong>Result:</strong>  negatve</p>

<ul>
<li>either bad primers</li>
<li>or because no DSMO was used</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-78" class="hidden">
      <label for="panel-78" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              26
              
          </p>
          <p class="inline-block">Preparation of Media LB and SOC</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Preparation of Media LB and SOC</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Repeated PCR from this morning (26.07.19., entry 56) &ndash;Attempt to amplify K1 &ndash;not successful

<ul>
<li>2µL Template</li>
<li>25 µL x2 Q5 MasterMix</li>
<li>2,5 µL FW Primer</li>
<li>2,5 µL RV Primer</li>
<li>2µL H2O</li>
<li>16 µL 15% DSSO &ndash;~5% final conc.

<br /></li>
</ul></li>
</ul>

<p><strong>Prepartion of Media SOC and LB</strong></p>

<p>1L medium -25 g/L LB broth powder dissolved in dH2O</p>

<p>0,2 L SOB Medium for SOC Medium to make E.coli electrocompetent</p>

<ul>
<li>1,25 g Trypton</li>
<li>5 g yeast extract</li>
<li>0,117 g NaCl</li>
</ul>

<p>&ndash;autoclave 30 min 121 °C</p>

<p>Added the rest of the ingredients for SOC by Eva &amp; Lisa the next day after the Medium cooled off (entry 63)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-79" class="hidden">
      <label for="panel-79" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              26
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Preparation of Media LB and SOC</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Preparation of Media LB and SOC</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Repeated PCR from this morning (26.07.19., entry 56) &ndash;Attempt to amplify K1 &ndash;not successful

<ul>
<li>2µL Template</li>
<li>25 µL x2 Q5 MasterMix</li>
<li>2,5 µL FW Primer</li>
<li>2,5 µL RV Primer</li>
<li>2µL H2O</li>
<li>16 µL 15% DSSO &ndash;~5% final conc.

<br /></li>
</ul></li>
</ul>

<p><strong>Prepartion of Media SOC and LB</strong></p>

<p>1L medium -25 g/L LB broth powder dissolved in dH2O</p>

<p>0,2 L SOB Medium for SOC Medium to make E.coli electrocompetent</p>

<ul>
<li>1,25 g Trypton</li>
<li>5 g yeast extract</li>
<li>0,117 g NaCl</li>
</ul>

<p>&ndash;autoclave 30 min 121 °C</p>

<p>Added the rest of the ingredients for SOC by Eva &amp; Lisa the next day after the Medium cooled off (entry 63)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-80" class="hidden">
      <label for="panel-80" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              26
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Transformation of BL21 with Construct 2a, 2b, 3a &#43; 3b, 4a &#43; 4b in pACYC184 vector, Amplification of Construct 1 with original DNA</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Transformation of BL21 with Construct 2a, 2b, 3a &#43; 3b, 4a &#43; 4b in pACYC184 vector, Amplification of Construct 1 with original DNA</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Transformation mit Ca2&#43; - kompetente E.coli Zellen (Ordner Seite 1)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Ruben Caspary</td>
            </tr>
        </table>

        <div class="labjournal">
        

<p><strong>Notes</strong></p>

<ul>
<li>amplified 2µl original K1 DNA</li>
<li>H20 Control</li>
</ul>

<h2 id="protocol">Protocol</h2>

<table>
<thead>
<tr>
<th>DNA</th>
<th>2µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>2x MasterMix</td>
<td>10µl</td>
</tr>

<tr>
<td>Primer fw 10µM</td>
<td>1µl</td>
</tr>

<tr>
<td>Primer re 10µM</td>
<td>1µl</td>
</tr>

<tr>
<td>DMSO 3% in dH20</td>
<td>6µl</td>
</tr>
</tbody>
</table>

<p>Program used <strong>Construct Amplification (30 cycles):</strong></p>

<table>
<thead>
<tr>
<th>Denaturation</th>
<th>98°C</th>
<th>10s</th>
</tr>
</thead>

<tbody>
<tr>
<td>Annealing</td>
<td>51°C</td>
<td>30s</td>
</tr>

<tr>
<td>Extension</td>
<td>72°C</td>
<td>60s</td>
</tr>
</tbody>
</table>

<ul>
<li>Control with 0.8% Agarose-Gelelectrophoresis</li>
<li>17  <strong>µl K1 Amp remaining</strong></li>
</ul>

<p><strong>Transformation of BL21 with Construct 2a, 2b, 3a + 3b, 4a + 4b in pACYC184 vector:</strong></p>

<ul>
<li>Protocol used is in folder page 1: Transformation mit Ca2+ - kompetente E.coli - Zellen</li>
<li>Contructs used: 2a, 2b, 3a+b, 4a+b  <strong>2µl each</strong></li>

<li><p>positive Control with  <strong>NEB pUC Control vector: used 10µl -500pg + purified pUC Vector from colony 3 &ndash;4µl: around 25ng DNA</strong></p></li>

<li><p>plate on Amp plates: positive Controls</p></li>

<li><p>plate on Chloramphenicol plates from  <strong>29.06.2019 Construc 2a, 2b, 3a + 3b, 4a + 4b</strong></p></li>

<li><p><strong>Incubate in the afternoon or until tomorrow morning at 37°C</strong></p></li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li><strong>Used old plates 29.06.2019</strong></li>
<li>used SOC-Medium, which was opened on the 11.07.2019 &ndash;might be contaminated, even though medium was clear</li>
<li>used maybe too much DNA Ladder

<br />

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>3mL SOC-Medium</li>
<li>NEB pUC Vector: 10µl</li>
<li>0,4 Agarose</li>
<li>10µl Quick 1kb DNA Ladder

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Control of Gelelectrophoresis under UV-Light</li>
<li>autoclave SOC-Medium</li>
<li>Pour new plates: Chloramphenicol and Ampicillin plates</li>
<li>Check transformed bacteria on plates in the next morning 27.07.2019</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-81" class="hidden">
      <label for="panel-81" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              26
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Transformation of BL21 with Construct 2a, 2b, 3a &#43; 3b, 4a &#43; 4b in pACYC184 vector, Amplification of Construct 1 with original DNA</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Transformation of BL21 with Construct 2a, 2b, 3a &#43; 3b, 4a &#43; 4b in pACYC184 vector, Amplification of Construct 1 with original DNA</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Transformation mit Ca2&#43; - kompetente E.coli Zellen (Ordner Seite 1)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer, Ruben Caspary</td>
            </tr>
        </table>

        <div class="labjournal">
        

<p><strong>Notes</strong></p>

<ul>
<li>amplified 2µl original K1 DNA</li>
<li>H20 Control</li>
</ul>

<h2 id="protocol">Protocol</h2>

<table>
<thead>
<tr>
<th>DNA</th>
<th>2µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>2x MasterMix</td>
<td>10µl</td>
</tr>

<tr>
<td>Primer fw 10µM</td>
<td>1µl</td>
</tr>

<tr>
<td>Primer re 10µM</td>
<td>1µl</td>
</tr>

<tr>
<td>DMSO 3% in dH20</td>
<td>6µl</td>
</tr>
</tbody>
</table>

<p>Program used <strong>Construct Amplification (30 cycles):</strong></p>

<table>
<thead>
<tr>
<th>Denaturation</th>
<th>98°C</th>
<th>10s</th>
</tr>
</thead>

<tbody>
<tr>
<td>Annealing</td>
<td>51°C</td>
<td>30s</td>
</tr>

<tr>
<td>Extension</td>
<td>72°C</td>
<td>60s</td>
</tr>
</tbody>
</table>

<ul>
<li>Control with 0.8% Agarose-Gelelectrophoresis</li>
<li>17  <strong>µl K1 Amp remaining</strong></li>
</ul>

<p>**<br />
Transformation of BL21 with Construct 2a, 2b, 3a + 3b, 4a + 4b in pACYC184 vector:**</p>

<ul>
<li>Protocol used is in folder page 1: Transformation mit Ca2+ - kompetente E.coli - Zellen</li>
<li>Contructs used: 2a, 2b, 3a+b, 4a+b  <strong>2µl each</strong></li>

<li><p>positive Control with  <strong>NEB pUC Control vector: used 10µl -500pg + purified pUC Vector from colony 3 &ndash;4µl: around 25ng DNA</strong></p></li>

<li><p>plate on Amp plates: positive Controls</p></li>

<li><p>plate on Chloramphenicol plates from  <strong>29.06.2019 Construc 2a, 2b, 3a + 3b, 4a + 4b</strong></p></li>

<li><p><strong>Incubate in the afternoon or until tomorrow morning at 37°C</strong></p></li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li><strong>Used old plates 29.06.2019</strong></li>
<li>used SOC-Medium, which was opened on the 11.07.2019 &ndash;might be contaminated, even though medium was clear</li>
<li>used maybe too much DNA Ladder

<br />

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>3mL SOC-Medium</li>
<li>NEB pUC Vector: 10µl</li>
<li>0,4 Agarose</li>
<li>10µl Quick 1kb DNA Ladder

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Control of Gelelectrophoresis under UV-Light</li>
<li>autoclave SOC-Medium</li>
<li>Pour new plates: Chloramphenicol and Ampicillin plates</li>
<li>Check transformed bacteria on plates in the next morning 27.07.2019</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-82" class="hidden">
      <label for="panel-82" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              27
              
          </p>
          <p class="inline-block">Preparation of Media</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Preparation of Media</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lisa Vogg, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ol>
<li>1% Gel</li>
</ol>

<p>Sample from 26.07.2019 for the gel (2µL DNA, 3µL Stain, 13µL H2O)</p>

<p><strong>No bands visable!</strong></p>

<ol>
<li>Each 2 colonies of K2a, K2b, K4ab and one colonie of K3ab</li>
</ol>

<p>Incoluation overnight at 37°C in 5 mL LB with Camp</p>

<ol>
<li>Finishing the SOC medium from th 26.06.2019 by adding 1 M MgSo4 and 0,72 g Glucose</li>
</ol>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Miniprep of clones</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-83" class="hidden">
      <label for="panel-83" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              27
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Preparation of Media</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Preparation of Media</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lisa Vogg, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>TO BE EDITED WHEN I GET MY NOTES</strong></p>

<p><strong>Notes</strong></p>

<ol>
<li>1% Gel</li>
</ol>

<p>Sample from 26.07.2019 for the gel (2µL DNA, 3µL Stain, 13µL H2O)</p>

<p><strong>No bands visable!</strong></p>

<ol>
<li>Each 2 colonies of K2a, K2b, K4ab and one colonie of K3ab</li>
</ol>

<p>Incoluation overnight at 37°C in 5 mL LB with Camp</p>

<ol>
<li>Finishing the SOC medium from th 26.06.2019 by adding 1 M MgSo4 and 0,72 g Glucose

<br />

<br />

<br />

<br />

<br /></li>
</ol>

<p><strong>Protocol</strong></p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p><strong>Materials used</strong></p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Miniprep of clones</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-84" class="hidden">
      <label for="panel-84" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              28
              
          </p>
          <p class="inline-block">Store transformed E.coli, K1 Gel Extraction</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Store transformed E.coli, K1 Gel Extraction</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li><strong>No mini-prep columns left!! &ndash;Wait for new order.</strong></li>
<li>Transformed E.coli are stored at 4°C and are waiting for their mini-prep  <strong>(4</strong>.5 ml left)**</li>
<li><strong>Aliquoted 50% Glycerol in LB-medium in 15 ml Falcon</strong>

<br /></li>
</ul>

<p><strong>Storage of 500µl transformed E.coli at -80°C</strong></p>

<p><strong>Protocol:</strong></p>

<ol>
<li>Mix 500µl vortexed transformed E.coli with 500µl 100% Glycerol solution</li>
<li>Different colonies are labeled 1 and 2. Some Construct transformed E.colis have 2 aliquots, because of 2 colonies picked</li>
<li>Store at -80°C

<br /></li>
</ol>

<p><strong>K1 Gel Extraction:</strong></p>

<ul>
<li>Preparation for Blunt-End Cloning</li>
<li>Extract K1 amplificat (11.07.2019) out of a 0.8% agarose gel</li>
<li>Use 6µl of K1 and 15µl of H20 control &ndash;2µl of K1 are left for further PCRs</li>
<li>Run at 110V for 90 minutes</li>
<li>Plot: 4µl Ladder - empty - K1 - empty - H20

<br /></li>
</ul>

<p><strong>Plated 200µl K3a+b on Agar-Chloramphenicol plates</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Amplification PCR are not perfectly labeled. Unsure if it was K1 or Water control &ndash;Used both in our gel electrophoresis</li>
<li><strong>NO K1 Amplificat left in tube! Gel Extraction was not possible</strong>

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>4µl DNA Ladder</li>
<li>Gel extraction Kit (1 prep)</li>
<li>10ml Glycerol

<br />

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Blunt-End cloning of purified K1 in pACYC184</li>
<li>Mini-prep of all our transformed E.coli

<ul>
<li>PCR to validate Gibson Assembly and transformation success.</li>
</ul></li>
<li>Pick colony of K3a+b and Incubate in Lb - Chloramphenicol medium for mini-prep</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-85" class="hidden">
      <label for="panel-85" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              28
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Store transformed E.coli, K1 Gel Extraction</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Store transformed E.coli, K1 Gel Extraction</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li><strong>No mini-prep columns left!! &ndash;Wait for new order.</strong></li>
<li>Transformed E.coli are stored at 4°C and are waiting for their mini-prep  <strong>(4</strong>.5 ml left)**</li>
<li><strong>Aliquoted 50% Glycerol in LB-medium in 15 ml Falcon</strong>

<br /></li>
</ul>

<p><strong>Storage of 500µl transformed E.coli at -80°C</strong></p>

<p><strong>Protocol:</strong></p>

<ol>
<li>Mix 500µl vortexed transformed E.coli with 500µl 100% Glycerol solution</li>
<li>Different colonies are labeled 1 and 2. Some Construct transformed E.colis have 2 aliquots, because of 2 colonies picked</li>
<li>Store at -80°C

<br /></li>
</ol>

<p><strong>K1 Gel Extraction:</strong></p>

<ul>
<li>Preparation for Blunt-End Cloning</li>
<li>Extract K1 amplificat (11.07.2019) out of a 0.8% agarose gel</li>
<li>Use 6µl of K1 and 15µl of H20 control &ndash;2µl of K1 are left for further PCRs</li>
<li>Run at 110V for 90 minutes</li>
<li>Plot: 4µl Ladder - empty - K1 - empty - H20

<br /></li>
</ul>

<p><strong>Plated 200µl K3a+b on Agar-Chloramphenicol plates</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Amplification PCR are not perfectly labeled. Unsure if it was K1 or Water control &ndash;Used both in our gel electrophoresis</li>
<li><strong>NO K1 Amplificat left in tube! Gel Extraction was not possible</strong>

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>4µl DNA Ladder</li>
<li>Gel extraction Kit (1 prep)</li>
<li>10ml Glycerol

<br />

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Blunt-End cloning of purified K1 in pACYC184</li>
<li>Mini-prep of all our transformed E.coli

<ul>
<li>PCR to validate Gibson Assembly and transformation success.</li>
</ul></li>
<li>Pick colony of K3a+b and Incubate in Lb - Chloramphenicol medium for mini-prep</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-86" class="hidden">
      <label for="panel-86" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              30
              
          </p>
          <p class="inline-block">Cultivation of K3ab for Mini-Prep</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Cultivation of K3ab for Mini-Prep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li><strong>Picked 2 colonies of K3ab plate (overnight plate from the 29.07.19)</strong></li>
<li>Cultivate in Lb-Chloramphenicol medium

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Check growth on the 30.07.19</li>
<li>Perform mini-prep</li>
<li>Validate Insertion success of our Gibson Assembly</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-87" class="hidden">
      <label for="panel-87" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              30
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Cultivation of K3ab for Mini-Prep</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Cultivation of K3ab for Mini-Prep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li><strong>Picked 2 colonies of K3ab plate (overnight plate from the 29.07.19)</strong></li>
<li>Cultivate in Lb-Chloramphenicol medium

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Check growth on the 30.07.19</li>
<li>Perform mini-prep</li>
<li>Validate Insertion success of our Gibson Assembly</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-88" class="hidden">
      <label for="panel-88" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              31
              
          </p>
          <p class="inline-block">Mini-Prep of the reagenz K2a (2x), K2b (2x), K3ab (3x), K4ab (2x) = 9 samples</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Mini-Prep of the reagenz K2a (2x), K2b (2x), K3ab (3x), K4ab (2x) = 9 samples</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Protocol of the Kit &#34;ExtractMe&#34;</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>first we  down-centrifuged the over-night culture from K3ab</li>
<li>We then put 500µl of K3ab together with 500µl of Glycerol in the freezer at -80°C (box at the bottom of rang 12)</li>
<li>Then the Mini-Prep with the Extract gel was performed</li>
<li>The DNA was then stored at 4°C

<br />

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p>Protocol of the Extract me Kit</p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>We used 14ml overnight-culture, which was a little bit too much, because after the lysis there was a lot protein waste so we had to repeat this step several times</p>

<p><strong>Materials used</strong></p>

<ul>
<li>Glycerol</li>
<li>Extract Me Kit

<br />

<br /></li>
</ul>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Measure DNA concentrations</li>
<li>Performing a gel-electrophoresis with the isolated DNA (first cleave the DNA with restriction-enzymes)</li>
<li>Sent a part of the isolated DNA to sequencing-company</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-89" class="hidden">
      <label for="panel-89" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              31
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Concentration of Minipreps (K2a (2x), K2b (2x), K3ab (3x), K4ab (2x) = 9 samples)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Concentration of Minipreps (K2a (2x), K2b (2x), K3ab (3x), K4ab (2x) = 9 samples)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Eva’s Digest Protokoll</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie Wiedemann, Ruben Caspary</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Nanodrop of all Mini-Preps</li>
<li>Digest at 37° 350 rpm heat-block

<ul>
<li>12 yL H2O</li>
<li>5 yL Probe</li>
<li>2 yL CutSmart Buffer</li>
<li>0,5 yL Hind_III</li>
<li>0,5 yL BamHI-HF</li>
</ul></li>
<li>Concentrations similar to Eva’s vector preperation</li>
<li>Gel, 14 lanes for vector-insert digest of:

<ul>
<li>K2a (2x colonie 1 &amp; 2),</li>
<li>K2b (2x colonie 1 &amp; 2),</li>
<li>K3ab (3x colonie 1,2,3),</li>
<li>K4ab (2x colonie 1 &amp; 2 no Kol 3) = 9 samples

<br />

<br /></li>
</ul></li>
</ul>

<p><strong>Protocol</strong></p>

<p><strong>Calculations and Results</strong></p>

<table>
<thead>
<tr>
<th>Construct</th>
<th></th>
<th>Colonie 1</th>
<th>[c] ng/yL</th>
<th></th>
<th>Colonie 2</th>
<th></th>
<th>Colonie 3</th>
</tr>
</thead>

<tbody>
<tr>
<td>2a</td>
<td></td>
<td></td>
<td>40,22</td>
<td></td>
<td>46,00</td>
<td></td>
<td></td>
</tr>

<tr>
<td>2b</td>
<td></td>
<td></td>
<td>46,08</td>
<td></td>
<td>41,48</td>
<td></td>
<td></td>
</tr>

<tr>
<td>3ab</td>
<td></td>
<td></td>
<td>62,29</td>
<td></td>
<td>36,44</td>
<td></td>
<td>32,29</td>
</tr>

<tr>
<td>4ab</td>
<td></td>
<td></td>
<td>33,01</td>
<td></td>
<td></td>
<td></td>
<td>3,01</td>
</tr>
</tbody>
</table>

<p><strong>Problems faced / possible error causes</strong></p>

<p><strong>Materials used</strong></p>

<p>CutSmart Buffer</p>

<p>Hind_III</p>

<p>BamHI_HF</p>

<p>TAE-Buffer</p>

<p>Agarose</p>

<p><strong>Files used / generated</strong></p>

<p>eluate gel, send Insert</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-90" class="hidden">
      <label for="panel-90" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              31
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Mini-Prep of the reagenz K2a (2x), K2b (2x), K3ab (3x), K4ab (2x) = 9 samples</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Mini-Prep of the reagenz K2a (2x), K2b (2x), K3ab (3x), K4ab (2x) = 9 samples</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Protocol of the Kit &#34;ExtractMe&#34;</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>first we  down-centrifuged the over-night culture from K3ab</li>
<li>We then put 500µl of K3ab together with 500µl of Glycerol in the freezer at -80°C (box at the bottom of rang 12)</li>
<li>Then the Mini-Prep with the Extract gel was performed</li>
<li>The DNA was then stored at 4°C

<br />

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p>Protocol of the Extract me Kit</p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>We used 14ml overnight-culture, which was a little bit too much, because after the lysis there was a lot protein waste so we had to repeat this step several times</p>

<p><strong>Materials used</strong></p>

<ul>
<li>Glycerol</li>
<li>Extract Me Kit

<br />

<br /></li>
</ul>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Measure DNA concentrations</li>
<li>Performing a gel-electrophoresis with the isolated DNA (first cleave the DNA with restriction-enzymes)</li>
<li>Sent a part of the isolated DNA to sequencing-company</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-91" class="hidden">
      <label for="panel-91" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              31
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Concentration of Minipreps (K2a (2x), K2b (2x), K3ab (3x), K4ab (2x) = 9 samples)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Concentration of Minipreps (K2a (2x), K2b (2x), K3ab (3x), K4ab (2x) = 9 samples)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Eva’s Digest Protokoll</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie Wiedemann, Ruben Caspary</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Nanodrop of all Mini-Preps</li>
<li>Digest at 37° 350 rpm heat-block

<ul>
<li>12 yL H2O</li>
<li>5 yL Probe</li>
<li>2 yL CutSmart Buffer</li>
<li>0,5 yL Hind_III</li>
<li>0,5 yL BamHI-HF</li>
</ul></li>
<li>Concentrations similar to Eva’s vector preperation</li>
<li>Gel, 14 lanes for vector-insert digest of:

<ul>
<li>K2a (2x colonie 1 &amp; 2),</li>
<li>K2b (2x colonie 1 &amp; 2),</li>
<li>K3ab (3x colonie 1,2,3),</li>
<li>K4ab (2x colonie 1 &amp; 2 no Kol 3) = 9 samples

<br />

<br /></li>
</ul></li>
</ul>

<p><strong>Protocol</strong></p>

<p><strong>Calculations and Results</strong></p>

<table>
<thead>
<tr>
<th>Construct</th>
<th></th>
<th>Colonie 1</th>
<th>[c] ng/yL</th>
<th></th>
<th>Colonie 2</th>
<th></th>
<th>Colonie 3</th>
</tr>
</thead>

<tbody>
<tr>
<td>2a</td>
<td></td>
<td></td>
<td>40,22</td>
<td></td>
<td>46,00</td>
<td></td>
<td></td>
</tr>

<tr>
<td>2b</td>
<td></td>
<td></td>
<td>46,08</td>
<td></td>
<td>41,48</td>
<td></td>
<td></td>
</tr>

<tr>
<td>3ab</td>
<td></td>
<td></td>
<td>62,29</td>
<td></td>
<td>36,44</td>
<td></td>
<td>32,29</td>
</tr>

<tr>
<td>4ab</td>
<td></td>
<td></td>
<td>33,01</td>
<td></td>
<td></td>
<td></td>
<td>3,01</td>
</tr>
</tbody>
</table>

<p><strong>Problems faced / possible error causes</strong></p>

<p><strong>Materials used</strong></p>

<p>CutSmart Buffer</p>

<p>Hind_III</p>

<p>BamHI_HF</p>

<p>TAE-Buffer</p>

<p>Agarose</p>

<p><strong>Files used / generated</strong></p>

<p>eluate gel, send Insert</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

            
            </div>
            

            <h2 class="my-10 text-3xl">August</h2>
            

            <div class="accordion flex flex-col items-center justify-center">
            
        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-92" class="hidden">
      <label for="panel-92" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
          </p>
          <p class="inline-block">Transformation colony screening with PCR</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Transformation colony screening with PCR</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Standard PCR protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>used PCR primers for amplification to screen for our inserted sequence</li>
<li>forward primer the same for all the samples, used the respective reverse primers of each sample. For the ligated samples, the &ldquo;b&rdquo; primers were used.<br /></li>
</ul>

<table>
<thead>
<tr>
<th>Template DNA</th>
<th>5 µL</th>
</tr>
</thead>

<tbody>
<tr>
<td>Primer fw (10 µM)</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>Primer rev (10 µM)</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>Master Mix</td>
<td>12.5 µL</td>
</tr>

<tr>
<td>autoclaved H2O</td>
<td>5 µL</td>
</tr>

<tr>
<td></td>
<td>25 µL</td>
</tr>
</tbody>
</table>

<ul>
<li>checked success with a 1% agarose gel, run at 150 V for 1 h</li>
<li>PCR was negative

<ul>
<li>too little amplification primers</li>
<li>wrong primers used</li>
<li>or no insert in our vector</li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-93" class="hidden">
      <label for="panel-93" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Transformation colony screening with PCR</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Transformation colony screening with PCR</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Standard PCR protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>used PCR primers for amplification to screen for our inserted sequence</li>
<li>forward primer the same for all the samples, used the respective reverse primers of each sample. For the ligated samples, the &ldquo;b&rdquo; primers were used.<br /></li>
</ul>

<p>Template DNA        |5 µL<br />
&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;-|&mdash;-
Primer fw (10 µM)   |1.25 µL
Primer rev (10 µM)  |1.25 µL
Master Mix      |12.5 µL
autoclaved H2O      |5 µL<br />
        |25 µL</p>

<ul>
<li>checked success with a 1% agarose gel, run at 150 V for 1 h</li>
<li>PCR was negative

<ul>
<li>too little amplification primers</li>
<li>wrong primers used</li>
<li>or no insert in our vector</li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-94" class="hidden">
      <label for="panel-94" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Transformation Colony Screening with PCR</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Transformation Colony Screening with PCR</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Standard PCR Protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1</th>
<th>Attempt 2</th>
<th>Attempt 3</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µg (=0,5 µl)</td>
<td>2 µg (= 1 µl)</td>
<td>1 µg (= µl)</td>
</tr>

<tr>
<td>NEB buffer 3.1 (x10 )</td>
<td>5</td>
<td>5</td>
<td>-</td>
</tr>
</tbody>
</table>

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Use PCR Primers for Amplification to screen for our inserted sequence</li>
<li>Only if Primers can bind to our vector and PCR produces the right Insert size our sequence is in our vector and the colony can be used for dowstream experiments</li>
<li>Use undiluted amplification primers, because we don&rsquo;t trust the 100µM note.

<ul>
<li>We have enough Primers anyway</li>
</ul></li>
<li>used always the same forward Primer, but the reverse primer differed from construct to construct. Used always b re primers, if we assembled multiple constructs together</li>
<li><strong>Used 2 x 2a, 2 x 2b, 3 x 3ab, 2 x 4ab &ndash;labeled them respectively</strong>

<br /></li>
</ul>

<p><strong>Gel Electrophoresis:</strong></p>

<ul>
<li>1% Agarose gel. Run with 150 V for 1 hour

<br />

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p><strong>PCR:</strong></p>

<p>Primers:</p>

<p>fw: Tac Promoter fw 2.0 Primer</p>

<p>re: 2a: 2a re Primer, 2b: 2b re Primer, 3ab: 3b re Primer, 4ab: 4b re Primer</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Volume</th>
</tr>
</thead>

<tbody>
<tr>
<td>Template DNA</td>
<td>5 µl</td>
</tr>

<tr>
<td>Primer fw (10µM)</td>
<td>1.25 µl</td>
</tr>

<tr>
<td>Primer rev (10µM)</td>
<td>1.25 µl</td>
</tr>

<tr>
<td>Master Mix</td>
<td>12.5 µl</td>
</tr>

<tr>
<td>Autoclaved H2O</td>
<td>5 µl</td>
</tr>

<tr>
<td>Total Volume</td>
<td>25 µl</td>
</tr>
</tbody>
</table>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Too little amplification primers</li>
<li>Wrong amplification primers used</li>
<li>No Sequence in our vector

<br />

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>100 µl 2x MasterMix</li>
<li>12 PCR Tubes and lid

<br /></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-95" class="hidden">
      <label for="panel-95" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Transformation Colony Screening with PCR</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Transformation Colony Screening with PCR</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Standard PCR Protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1</th>
<th>Attempt 2</th>
<th>Attempt 3</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µg (=0,5 µl)</td>
<td>2 µg (= 1 µl)</td>
<td>1 µg (= µl)</td>
</tr>

<tr>
<td>NEB buffer 3.1 (x10 )</td>
<td>5</td>
<td>5</td>
<td>-</td>
</tr>
</tbody>
</table>

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Use PCR Primers for Amplification to screen for our inserted sequence</li>
<li>Only if Primers can bind to our vector and PCR produces the right Insert size our sequence is in our vector and the colony can be used for dowstream experiments</li>
<li>Use undiluted amplification primers, because we don&rsquo;t trust the 100µM note.

<ul>
<li>We have enough Primers anyway</li>
</ul></li>
<li>used always the same forward Primer, but the reverse primer differed from construct to construct. Used always b re primers, if we assembled multiple constructs together</li>
<li><strong>Used 2 x 2a, 2 x 2b, 3 x 3ab, 2 x 4ab &ndash;labeled them respectively</strong>

<br /></li>
</ul>

<p><strong>Gel Electrophoresis:</strong></p>

<ul>
<li>1% Agarose gel. Run with 150 V for 1 hour

<br />

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p><strong>PCR:</strong></p>

<p>Primers:</p>

<p>fw: Tac Promoter fw 2.0 Primer</p>

<p>re: 2a: 2a re Primer, 2b: 2b re Primer, 3ab: 3b re Primer, 4ab: 4b re Primer</p>

<table>
<thead>
<tr>
<th>Template DNA</th>
<th>5 µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>Primer fw (10µM)</td>
<td>1.25 µl</td>
</tr>

<tr>
<td>Primer rev (10µM)</td>
<td>1.25 µl</td>
</tr>

<tr>
<td>Master Mix</td>
<td>12.5 µl</td>
</tr>

<tr>
<td></td>
<td></td>
</tr>

<tr>
<td>Autoclaved H2O</td>
<td>5 µl</td>
</tr>

<tr>
<td>&mdash;Total Volume</td>
<td>25 µl</td>
</tr>
</tbody>
</table>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Too little amplification primers</li>
<li>Wrong amplification primers used</li>
<li>No Sequence in our vector

<br />

<br /></li>
</ul>

<p><strong>Materials used</strong></p>

<ul>
<li>100 µl 2x MasterMix</li>
<li>12 PCR Tubes and lid

<br />

<br />

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-96" class="hidden">
      <label for="panel-96" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(5)</strong>
              
          </p>
          <p class="inline-block">Control of the Gibson assembly</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control of the Gibson assembly</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>restriction digest with Ase1 and Pvu1: Ase1 cuts once in the vector and once in the promoter region of the insert, Pvu1 was used to cut the insert, however it only cuts in construct 2a

<ul>
<li>12 µL H2O</li>
<li>5 µL DNA</li>
<li>2 µL CutSmart Buffer 3.1</li>
<li>0.5 µL Ase1</li>
<li>0.5 µL Pvu1</li>
</ul></li>
<li>incubation for 1 h at 37 °C</li>
<li>gel electrophoresis in 1% agarose in TAE with SYBR Safe Gel Stain</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-97" class="hidden">
      <label for="panel-97" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(6)</strong>
              
          </p>
          <p class="inline-block">Control of the Gibson assembly</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control of the Gibson assembly</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>restriction digest with Ase1 and Pvu1: Ase1 cuts once in the vector and once in the promoter region of the insert, Pvu1 was used to cut the insert, however it only cuts in construct 2a

<ul>
<li>12 µL H2O</li>
<li>5 µL DNA</li>
<li>2 µL CutSmart Buffer 3.1</li>
<li>0.5 µL Ase1</li>
<li>0.5 µL Pvu1</li>
</ul></li>
<li>incubation for 1 h at 37 °C</li>
<li>gel electrophoresis in 1% agarose in TAE with SYBR Safe Gel Stain</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-98" class="hidden">
      <label for="panel-98" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(7)</strong>
              
          </p>
          <p class="inline-block">Control if the Assemblys worked</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control if the Assemblys worked</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li><p>Restriction digest with Ase1 and Pcu1.</p>

<ul>
<li>12 µL H2O</li>
<li>5 µL DNA</li>
<li>2 µL CutSmart Buffer 3.1</li>
<li>0,5 µL Ase1</li>
<li>0,5 µL Pvu1
<br /></li>
</ul>

<p>-incubation 1h 37 °C</p></li>
</ul>

<p>Gel Electrophoresis in 1% Agarose in TAE + SYBR green<br />
<strong>Calculations and Results</strong></p>

<p>Ase1 part of the vector and also part of each construct</p>

<p>-Ase1 cuts sticky ends -occurence of multimers</p>

<p>-Smallest piece is 329 kb, additinal bands due to multimers</p>

<p>-These bands can be found for every construct</p>

<p>Pvu1 cuts only 2a in two site</p>

<p>-many fragments from construct 2a</p>

<p><strong>Result</strong>: The Construct had been inserted into the vector successfully!</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Pvu1 was mistakingly used in an attempt to cut the insert. However, cutting site is only in 2a. Ase1 sufficient for the intepretation of the results.</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>confirm results with PCR of the insert</p>

<p><img src="/labjournal-entries/images/68ed1d2a-107d-4b25-a930-ffcaa0a7ade4.jpg" alt="&lt;Control if the Assemblys worked&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-99" class="hidden">
      <label for="panel-99" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(8)</strong>
              
          </p>
          <p class="inline-block">Control if the Assemblys worked</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control if the Assemblys worked</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li><p>Restriction digest with Ase1 and Pcu1.</p>

<ul>
<li>12 µL H2O</li>
<li>5 µL DNA</li>
<li>2 µL CutSmart Buffer 3.1</li>
<li>0,5 µL Ase1</li>
<li>0,5 µL Pvu1
<br /></li>
</ul>

<p>-incubation 1h 37 °C</p></li>
</ul>

<p>Gel Electrophoresis in 1% Agarose in TAE + SYBR green<br />
<strong>Calculations and Results</strong></p>

<p>Ase1 part of the vector and also part of each construct</p>

<p>-Ase1 cuts sticky ends -occurence of multimers</p>

<p>-Smallest piece is 329 kb, additinal bands due to multimers</p>

<p>-These bands can be found for every construct</p>

<p>Pvu1 cuts only 2a in two site</p>

<p>-many fragments from construct 2a</p>

<p><strong>Result</strong>: The Construct had been inserted into the vector successfully!</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Pvu1 was mistakingly used in an attempt to cut the insert. However, cutting site is only in 2a. Ase1 sufficient for the intepretation of the results.</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>confirm results with PCR of the insert</p>

<p><img src="/labjournal-entries/images/68ed1d2a-107d-4b25-a930-ffcaa0a7ade4.jpg" alt="&lt;Control if the Assemblys worked&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-100" class="hidden">
      <label for="panel-100" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              2
              
          </p>
          <p class="inline-block">Check RE digestion of the miniprep</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Check RE digestion of the miniprep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>ran 1% agarose gel: 2 µL plasmid DNA, 3 µL loading dye, 13 µL dH20 at 130 V for 1 h</li>
<li>dissolved newly arrived DNA for CHO expression and measurement part in 20 µL TE buffer, stored at -20 °C</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-101" class="hidden">
      <label for="panel-101" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              2
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Check RE digestion of the miniprep</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Check RE digestion of the miniprep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann, Lisa Vogg</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>ran 1% agarose gel: 2 µL plasmid DNA, 3 µL loading dye, 13 µL dH20 at 130 V for 1 h</li>
<li>dissolved newly arrived DNA for CHO expression and measurement part in 20 µL TE buffer, stored at -20 °C</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-102" class="hidden">
      <label for="panel-102" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              2
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Check RE digestion of the miniprep, dissolved new DNA samples</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Check RE digestion of the miniprep, dissolved new DNA samples</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lisa Vogg, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>ran gel (2 µl plasmid DNA, 3 µl loading dye, 13 µl dH20)</li>
<li>dissolved new DNA (CHO expression and measured part) in 20 µl TE buffer and stored at -20 °C</li>
<li>DNA should be thoroughly resuspended before usage just in the case it is not already</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-103" class="hidden">
      <label for="panel-103" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              2
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Check RE digestion of the miniprep, dissolved new DNA samples</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Check RE digestion of the miniprep, dissolved new DNA samples</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lisa Vogg, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>ran gel (2 µl plasmid DNA, 3 µl loading dye, 13 µl dH20)</li>
<li>dissolved new DNA (CHO expression and measured part) in 20 µl TE buffer and stored at -20 °C</li>
<li>DNA should be thoroughly resuspended before usage just in the case it is not already</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-104" class="hidden">
      <label for="panel-104" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              5
              
          </p>
          <p class="inline-block">Gibson overhang PCR, pACYC184 EcoRV digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson overhang PCR, pACYC184 EcoRV digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Digest:</strong>
* EcoRV digest with Eva&rsquo;s protocol
* digest mini-prep from 2019-08-02
* concentration of pACYC184: 154 ng/µL
* digest 1.5 µg plasmid: 9.74 µL at 37 °C for 60 min</p>

<p>CutSmart buffer     |5 µL<br />
&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;&mdash;|&mdash;&mdash;
DNA         |9.74 µL
EcoRV-HF enzyme     |1.5 µL
Water           |33.76 µL
 |50 µL</p>

<ul>
<li>inactivate for 20 min at 65 °C</li>
<li>run 0.8% agarose gel</li>
</ul>

<p><strong>PCR:</strong>
* PCR of constructs from 07-16 K1, K2a, K3a + DMSO control
* PCR of constructs from 07-04 K1, K2a, K2b, K3a, K3b, K4a, K4b + DMSO control
* used residual DNA of 2b, 3b, 4a, 4b from 07-16
* gel electrophoresis with 0.8% agarose gel</p>

<p>DMSO (15%)      |5 µL<br />
&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;&mdash;|&mdash;&mdash;
DNA         |5 µL
fw Primer       |1.25 µL
re Primer       |1.25 µL
Master Mix (2x)     |12.5 µL
|25 µL</p>

<p><strong>Second PCR with 3% DMSO:</strong>
* used overhang PCR products from 07-16 of K1, K2a, K3a</p>

<p><strong>Problems faced:</strong>
* used wrong plasmid concentration! after mini-prep concentration of 50 ng/µL
* used 100% DMSO instead of 15% as H2O replacement in the first PCR</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-105" class="hidden">
      <label for="panel-105" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              5
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Gibson overhang PCR, pACYC184 EcoRV digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson overhang PCR, pACYC184 EcoRV digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Digest:</strong></p>

<ul>
<li>EcoRV digest with Eva&rsquo;s protocol</li>
<li>digest mini-prep from 2019-08-02</li>
<li>concentration of pACYC184: 154 ng/µL</li>
<li>digest 1.5 µg plasmid: 9.74 µL at 37 °C for 60 min</li>
</ul>

<table>
<thead>
<tr>
<th>CutSmart buffer</th>
<th>5 µL</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>9.74 µL</td>
</tr>

<tr>
<td>EcoRV-HF enzyme</td>
<td>1.5 µL</td>
</tr>

<tr>
<td>Water</td>
<td>33.76 µL</td>
</tr>

<tr>
<td></td>
<td>50 µL</td>
</tr>
</tbody>
</table>

<ul>
<li>inactivate for 20 min at 65 °C</li>
<li>run 0.8% agarose gel</li>
</ul>

<p><strong>PCR:</strong></p>

<ul>
<li>PCR of constructs from 07-16 K1, K2a, K3a + DMSO control</li>
<li>PCR of constructs from 07-04 K1, K2a, K2b, K3a, K3b, K4a, K4b + DMSO control</li>
<li>used residual DNA of 2b, 3b, 4a, 4b from 07-16</li>
<li>gel electrophoresis with 0.8% agarose gel</li>
</ul>

<table>
<thead>
<tr>
<th>DMSO (15%)</th>
<th>5 µL</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>5 µL</td>
</tr>

<tr>
<td>fw Primer</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>re Primer</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>Master Mix (2x)</td>
<td>12.5 µL</td>
</tr>

<tr>
<td></td>
<td>25 µL</td>
</tr>
</tbody>
</table>

<p><strong>Second PCR with 3% DMSO:</strong></p>

<ul>
<li>used overhang PCR products from 07-16 of K1, K2a, K3a</li>
</ul>

<p><strong>Problems faced:</strong></p>

<ul>
<li>used wrong plasmid concentration! after mini-prep concentration of 50 ng/µL</li>
<li>used 100% DMSO instead of 15% as H2O replacement in the first PCR</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-106" class="hidden">
      <label for="panel-106" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              6
              
          </p>
          <p class="inline-block">Control of overhang PCR, purification of the cut pACYC184</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control of overhang PCR, purification of the cut pACYC184</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Monarch DNA Gel Extraction Kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>Control gel:

<ul>
<li>used 1% agarose gel for samples of the overhang PCR and the cut vector</li>
<li>run gel at 120 V for 1 h</li>
</ul></li>
<li>purification of the cut vector with <strong>Monarch DNA Gel Extraction Kit:</strong></li>
</ul>

<ol>
<li>Excise the DNA fragment from the agarose gel, taking care to trim excess agarose.Transfer to a 1.5 ml microfuge tube, and weigh the gel slice. Minimize exposure to UV light.</li>
<li>Add 4 volumes of Gel Dissolving Buffer to the gel slice (e.g., 400 μl buffer per 100 μl or 100 mg agarose).</li>
<li>Incubate the sample between 37-55°C (typically 50°C), vortexing periodically until the gel slice is completely dissolved (generally 5-10 minutes). For DNA fragments 8 kb, an additional 1.5 volumes of water should be added after the slice is dissolved to mitigate the tighter binding of larger pieces of DNA (e.g., 100 μl gel slice: 400 μl Gel Dissolving Buffer: 150 μl water).</li>
<li>Insert column into collection tube and load sample onto the column. Spin for 1 minute, then discard flow-through.</li>
<li>Re-insert column into collection tube. Add 200 μl DNA Wash Buffer and spin for 1 minute. Discarding flow-through is optional.</li>
<li>Repeat step 5.</li>
<li>Transfer column to a clean 1.5 ml microfuge tube. Use care to ensure that the tip of the column does not come into contact with the flow-through. If in doubt, re-spin for 1 minute.</li>
<li>Add ≥ 6 μl of DNA Elution Buffer to the center of the matrix. Wait for 1 minute, and spin for 1 minute to elute DNA. Typical elution volumes are 6-20 μl. Nuclease-free water (pH 7-8.5) can also be used to elute the DNA. Yield may slightly increase if a larger volume of DNA Elution Buffer is used, but the DNA will be less concentrated. For larger size DNA (≥ 10 kb), heating the elution buffer to 50°C prior to use can improve yield.</li>
</ol>

<ul>
<li>measurement of purified vector: 4.9 ng/µL</li>
<li>storage of purified cut vector at -20 °C</li>

<li><p>digestion of purified cut vector with EcoRV-HF enzyme</p></li>

<li><p>run gel with samples of the overhang PCR of K1, K2a, K3a from the previous day, the digested vector and uncut vector as control</p></li>

<li><p>extraction of the digested vector of the gel, store at 4 °C</p></li>
</ul>

<p><strong>Problems faced:</strong></p>

<ul>
<li>Control gel was negative, only K3a was visible in the overhang PCR</li>
<li>Gel for purification was empty (broken slide), had to be repeated</li>
<li>Second gel also negative except for K3a</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-107" class="hidden">
      <label for="panel-107" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              6
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Control of overhang PCR, purification of the cut pACYC184</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control of overhang PCR, purification of the cut pACYC184</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Monarch DNA Gel Extraction Kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>Control gel:

<ul>
<li>used 1% agarose gel for samples of the overhang PCR and the cut vector</li>
<li>run gel at 120 V for 1 h</li>
</ul></li>
<li>purification of the cut vector with <strong>Monarch DNA Gel Extraction Kit:</strong></li>
</ul>

<p>..1. Excise the DNA fragment from the agarose gel, taking care to trim excess agarose.Transfer to a 1.5 ml microfuge tube, and weigh the gel slice. Minimize exposure to UV light.
..2. Add 4 volumes of Gel Dissolving Buffer to the gel slice (e.g., 400 μl buffer per 100 μl or 100 mg agarose).
..3. Incubate the sample between 37-55°C (typically 50°C), vortexing periodically until the gel slice is completely dissolved (generally 5-10 minutes). For DNA fragments 8 kb, an additional 1.5 volumes of water should be added after the slice is dissolved to mitigate the tighter binding of larger pieces of DNA (e.g., 100 μl gel slice: 400 μl Gel Dissolving Buffer: 150 μl water).
..4. Insert column into collection tube and load sample onto the column. Spin for 1 minute, then discard flow-through.
..5. Re-insert column into collection tube. Add 200 μl DNA Wash Buffer and spin for 1 minute. Discarding flow-through is optional.
..6. Repeat step 5.
..7. Transfer column to a clean 1.5 ml microfuge tube. Use care to ensure that the tip of the column does not come into contact with the flow-through. If in doubt, re-spin for 1 minute.
..8. Add ≥ 6 μl of DNA Elution Buffer to the center of the matrix. Wait for 1 minute, and spin for 1 minute to elute DNA. Typical elution volumes are 6-20 μl. Nuclease-free water (pH 7-8.5) can also be used to elute the DNA. Yield may slightly increase if a larger volume of DNA Elution Buffer is used, but the DNA will be less concentrated. For larger size DNA (≥ 10 kb), heating the elution buffer to 50°C prior to use can improve yield.</p>

<ul>
<li>measurement of purified vector: 4.9 ng/µL</li>
<li>storage of purified cut vector at -20 °C</li>

<li><p>digestion of purified cut vector with EcoRV-HF enzyme</p></li>

<li><p>run gel with samples of the overhang PCR of K1, K2a, K3a from the previous day, the digested vector and uncut vector as control</p></li>

<li><p>extraction of the digested vector of the gel, store at 4 °C</p></li>
</ul>

<p><strong>Problems faced:</strong>
* Control gel was negative, only K3a was visible in the overhang PCR
* Gel for purification was empty (broken slide), had to be repeated
* Second gel also negative except for K3a</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-108" class="hidden">
      <label for="panel-108" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
          </p>
          <p class="inline-block">PCR of CFP part for measurement</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>PCR of CFP part for measurement</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>eluted the Measurement Primers to a concentration of 100 µM</li>
<li>used for amplification via PCR: annealing temperature 57 °C, elongation time 45 s</li>
</ul>

<table>
<thead>
<tr>
<th>Name</th>
<th>Volume</th>
</tr>
</thead>

<tbody>
<tr>
<td>Primer fw (10 µM)</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>Primer rev (10 µM)</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>template DNA</td>
<td>1 µL</td>
</tr>

<tr>
<td>Q5 2x Master Mix</td>
<td>12.5 µL</td>
</tr>

<tr>
<td>Nuclease free H2O</td>
<td>9 µL</td>
</tr>
</tbody>
</table>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-109" class="hidden">
      <label for="panel-109" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Create good Gibson overhangs and perform Gibson Assembly</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Create good Gibson overhangs and perform Gibson Assembly</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Q5 HiFi DNA-Pol NEB Protocol// Taq DNA-Pol NEB Protocol // NEBuilder HiFi DNA Assembly Kit Protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Antonio Maccataio, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>PCR:</strong>
* annealing temperature: 53.7 °C
* polymerase: Taq and Q5
* DNA: 2 µL
* nuclease-free water</p>

<p><strong>Gel:</strong>
* 1% agarose gel in TAE buffer
* SYBR Safe Gel Stain: 7:100,000
* run at 120 V</p>

<p><strong>Analysis:</strong>
* Taq PCR: total failure
* Q5: good amplification of 3a, 3b and 4b, rest: failure
* 3a and 3b measured by photometer
  * 500(?)ng 3a == 1,66µl (500ng/µl)
  * 500(?)ng 3b == 1,00µl (300ng/µl)
  * 370(?)ng pACYC == 2µl</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-110" class="hidden">
      <label for="panel-110" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">gel extraction of pSec Hygro Tag, CHO constructs K1, K2a, K2b</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>gel extraction of pSec Hygro Tag, CHO constructs K1, K2a, K2b</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Qiaquick Gel Extraction Kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>gel extraction of pSec Hygro Tag, and the CHO constructs K1, K2a, K2b</li>
<li>Qiaquick Gel Extraction Kit:</li>
</ul>

<ol>
<li>Add 5 volumes of Buffer PB to 1 volume of the PCR sample, and then mix. It is not necessary to remove mineral oil or kerosene. For example, add 500 μl of Buffer PB to 100 μl PCR sample (not including oil).</li>
<li>If pH Indicator I has been added to Buffer PB, check that the mixture’s color is yellow. If the color of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn yellow.</li>
<li>Place a QIAquick spin column in a provided 2 ml collection tube.</li>
<li>To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.</li>
<li>Discard flow-through. Place the QIAquick column back into the same tube. Collection tubes are reused to reduce plastic waste.</li>
<li>To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.</li>
<li>Discard flow-through and place the QIAquick column back into the same tube. Centrifuge the column for an additional 1 min. IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation.</li>
<li>Place QIAquick column in a clean 1.5 ml microcentrifuge tube.</li>
<li>To elute DNA, add 50 μl Buffer EB (10 mM Tris·Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min. Alternatively, for increased DNA concentration, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge. IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volumes are 48 μl from 50 μl elution buffer volume and 28 μl from 30 μl elution buffer. Elution efficiency is dependent on pH. Maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C because DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE buffer (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions.</li>
<li>If the purified DNA is to be analyzed on a gel, add 1 volume Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting it up and down before loading the gel. Loading Dye contains 3 marker dyes – bromophenol blue, xylene cyanol and orange G – that facilitate estimation of DNA-migration distance and optimization of the agarose gel run time. Refer to Table 2 (page 17) to identify the dyes according to migration distance and agarose gel percentage and type.</li>
</ol>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-111" class="hidden">
      <label for="panel-111" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">PCR of CHO constructs</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>PCR of CHO constructs</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>eluted the CHO Primers to a concentration of 100 µM</li>
<li>used for amplification via PCR: annealing temperature 57 °C, elongation time 45 s</li>
</ul>

<p>Primer fw (10 µM)   |1.25 µL
|&mdash; |&mdash; |&mdash; |<br />
Primer rev (10 µM)  |1.25 µL
template DNA        |1 µL
Q5 2x Master Mix    |12.5 µL
Nuclease free H2O   | 9 µL</p>

<ul>
<li>many unspecific bands, especially K1</li>
<li>new agarose gel with 10 µL PCR product and gel extraction</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-112" class="hidden">
      <label for="panel-112" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(5)</strong>
              
          </p>
          <p class="inline-block">Create good Gibson overhangs and perform Gibson Assembly</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Create good Gibson overhangs and perform Gibson Assembly</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Q5 HiFi DNA-Pol NEB Protocol// Taq DNA-Pol NEB Protocol // NEBuilder HiFi DNA Assembly Kit Protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Antonio Maccataio, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>PCR:</strong></p>

<ul>
<li>annealing temperature: 53.7 °C</li>
<li>polymerase: Taq and Q5</li>
<li>DNA: 2 µL</li>
<li>nuclease-free water</li>
</ul>

<p><strong>Gel:</strong></p>

<ul>
<li>1% agarose gel in TAE buffer</li>
<li>SYBR Safe Gel Stain: 7:100,000</li>
<li>run at 120 V</li>
</ul>

<p><strong>Analysis:</strong></p>

<ul>
<li>Taq PCR: total failure</li>
<li>Q5: good amplification of 3a, 3b and 4b, rest: failure</li>
<li>3a and 3b measured by photometer

<ul>
<li>500 ng 3a = 1,66µl (500ng/µl)</li>
<li>500 ng 3b = 1,00µl (300ng/µl)</li>
<li>370 ng pACYC = 2µl</li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-113" class="hidden">
      <label for="panel-113" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(6)</strong>
              
          </p>
          <p class="inline-block">PCR of CFP part for measurement</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>PCR of CFP part for measurement</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>eluted the Measurement Primers to a concentration of 100 µM</li>
<li>used for amplification via PCR: annealing temperature 57 °C, elongation time 45 s</li>
</ul>

<p>Primer fw (10 µM)   |1.25 µL<br />
&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;&mdash;&mdash;&mdash;&mdash;|&mdash;&mdash;&mdash;&mdash;&mdash;
Primer rev (10 µM)  |1.25 µL
template DNA        |1 µL
Q5 2x Master Mix    |12.5 µL
Nuclease free H2O   | 9 µL</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-114" class="hidden">
      <label for="panel-114" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(7)</strong>
              
          </p>
          <p class="inline-block">Digestion of pSec Hygro Tag</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of pSec Hygro Tag</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Digest protocol of NEB</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>digested pSec Hygro Tag with NheI</li>
<li>15 min at 37 °C, then 20 min at 80 °C</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-115" class="hidden">
      <label for="panel-115" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(8)</strong>
              
          </p>
          <p class="inline-block">Digestion of pSec Hygro Tag</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of pSec Hygro Tag</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Digest protocol of NEB</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>digested pSec Hygro Tag with NheI</li>
<li>15 min at 37 °C, then 20 min at 80 °C</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-116" class="hidden">
      <label for="panel-116" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(9)</strong>
              
          </p>
          <p class="inline-block">PCR of CHO constructs</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>PCR of CHO constructs</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>eluted the CHO Primers to a concentration of 100 µM</li>
<li>used for amplification via PCR: annealing temperature 57 °C, elongation time 45 s</li>
</ul>

<table>
<thead>
<tr>
<th>Name</th>
<th>Volume</th>
</tr>
</thead>

<tbody>
<tr>
<td>Primer fw (10 µM)</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>Primer rev (10 µM)</td>
<td>1.25 µL</td>
</tr>

<tr>
<td>template DNA</td>
<td>1 µL</td>
</tr>

<tr>
<td>Q5 2x Master Mix</td>
<td>12.5 µL</td>
</tr>

<tr>
<td>Nuclease free H2O</td>
<td>9 µL</td>
</tr>
</tbody>
</table>

<ul>
<li>many unspecific bands, especially K1</li>
<li>new agarose gel with 10 µL PCR product and gel extraction</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-117" class="hidden">
      <label for="panel-117" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(10)</strong>
              
          </p>
          <p class="inline-block">gel extraction of pSec Hygro Tag, CHO constructs K1, K2a, K2b</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>gel extraction of pSec Hygro Tag, CHO constructs K1, K2a, K2b</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Qiaquick Gel Extraction Kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>gel extraction of pSec Hygro Tag, and the CHO constructs K1, K2a, K2b</li>
<li>Qiaquick Gel Extraction Kit:
..1. Add 5 volumes of Buffer PB to 1 volume of the PCR sample, and then mix. It is not necessary to remove mineral oil or kerosene. For example, add 500 μl of Buffer PB to 100 μl PCR sample (not including oil).
..2. If pH Indicator I has been added to Buffer PB, check that the mixture’s color is yellow. If the color of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn yellow.
..3. Place a QIAquick spin column in a provided 2 ml collection tube.
..4. To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s.
..5. Discard flow-through. Place the QIAquick column back into the same tube. Collection tubes are reused to reduce plastic waste.
..6. To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s.
..7. Discard flow-through and place the QIAquick column back into the same tube. Centrifuge the column for an additional 1 min. IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation.
..8. Place QIAquick column in a clean 1.5 ml microcentrifuge tube.
..9. To elute DNA, add 50 μl Buffer EB (10 mM Tris·Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min. Alternatively, for increased DNA concentration, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge. IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volumes are 48 μl from 50 μl elution buffer volume and 28 μl from 30 μl elution buffer. Elution efficiency is dependent on pH. Maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C because DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE buffer (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions.
..10. If the purified DNA is to be analyzed on a gel, add 1 volume Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting it up and down before loading the gel. Loading Dye contains 3 marker dyes – bromophenol blue, xylene cyanol and orange G – that facilitate estimation of DNA-migration distance and optimization of the agarose gel run time. Refer to Table 2 (page 17) to identify the dyes according to migration distance and agarose gel percentage and type.</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-118" class="hidden">
      <label for="panel-118" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              10
              
          </p>
          <p class="inline-block">Heatshock of Gibson CHO-S (K1, K2a, K2b)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Heatshock of Gibson CHO-S (K1, K2a, K2b)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>NEB Heatshock</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>made 15 LB-Agar plates with Ampicilin</li>
<li>heatshock of the constructs of CHO constructs K1, K2a, K2b in BL21, BL21 Star, Tuner and DH5alpha, plus a positive and negative control in BL21 Star (=14 different transformations)

<ul>
<li>used 2 µL of the Gibson assembly</li>
<li>used SOC medium for the bacteria after the heatshock</li>
</ul></li>
<li>plated bacteria afterwards on the LB-Agar plates</li>
<li>incubation at 37 °C</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-119" class="hidden">
      <label for="panel-119" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              10
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Heatshock of Gibson CHO-S (K1, K2a, K2b)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Heatshock of Gibson CHO-S (K1, K2a, K2b)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>NEB Heatshock</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>made 15 LB-Agar plates with Ampicilin</li>
<li>heatshock of the constructs of CHO constructs K1, K2a, K2b in BL21, BL21 Star, Tuner and DH5alpha, plus a positive and negative control in BL21 Star (=14 different transformations)

<ul>
<li>used 2 µL of the Gibson assembly</li>
<li>used SOC medium for the bacteria after the heatshock</li>
</ul></li>
<li>plated bacteria afterwards on the LB-Agar plates</li>
<li>incubation at 37 °C</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-120" class="hidden">
      <label for="panel-120" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              12
              
          </p>
          <p class="inline-block">Production of single colonies (Star; K1, K2a,K2b of 11.08); Preparation of new amp-plates</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Production of single colonies (Star; K1, K2a,K2b of 11.08); Preparation of new amp-plates</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        
        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-121" class="hidden">
      <label for="panel-121" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              13
              
          </p>
          <p class="inline-block">Miniprep of ON Culture &amp; digestion; PCR clean-up</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Miniprep of ON Culture &amp; digestion; PCR clean-up</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li><p>Miniprep of the samples:</p>

<ul>
<li>K1-1 Tuner, K1-2 Tuner, K1-3 Tuner</li>
<li>K2a-1 Tuner, K2a-2 Tuner, K2a-3 Tuner</li>
<li>K2b-1 Tuner, K2b-2 Tuner, K2b-3 Tuner</li>
<li>K1 Star</li>
<li>negative control</li>
</ul></li>

<li><p>digestion of isolated DNA from Miniprep:</p></li>
</ul>

<table>
<thead>
<tr>
<th>Name</th>
<th>Volume</th>
</tr>
</thead>

<tbody>
<tr>
<td>EcoRV</td>
<td>1 µL</td>
</tr>

<tr>
<td>NheI</td>
<td>1 µL</td>
</tr>

<tr>
<td>CutSmart Buffer</td>
<td>5 µL</td>
</tr>

<tr>
<td>DNA</td>
<td>1 µL</td>
</tr>

<tr>
<td>dH2O</td>
<td>ad 50 µL</td>
</tr>
</tbody>
</table>

<ul>
<li><p>incubation for 15 min at 37 °C</p></li>

<li><p>PCR clean up of the samples:</p>

<ul>
<li>K2a, K2b, K3a from 2nd amplification from 15.7 !! no clear description on tubes of 2nd amplification and no picture of gel ==possible error source !!</li>
<li>K3b, K4a, K4b from 1st amplification</li>
<li>end volume 50 µL</li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-122" class="hidden">
      <label for="panel-122" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              13
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Miniprep of ON Culture &amp; digestion; PCR clean-up</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Miniprep of ON Culture &amp; digestion; PCR clean-up</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li><p>Miniprep of the samples:</p>

<ul>
<li>K1-1 Tuner, K1-2 Tuner, K1-3 Tuner</li>
<li>K2a-1 Tuner, K2a-2 Tuner, K2a-3 Tuner</li>
<li>K2b-1 Tuner, K2b-2 Tuner, K2b-3 Tuner</li>
<li>K1 Star</li>
<li>negative control</li>
</ul></li>

<li><p>digestion of isolated DNA from Miniprep:
EcoRV       |1 µL<br />
&mdash;&mdash;&mdash;&mdash;&mdash;-|&mdash;&mdash;&mdash;&mdash;&mdash; |&ndash;|
NheI        |1 µL
CutSmart Buffer |5 µL
DNA     |1 µL
dH2O        |ad 50 µL</p>

<ul>
<li>incubation for 15 min at 37 °C</li>
</ul></li>
</ul>

<p>*PCR clean up of the samples:
  * K2a, K2b, K3a from 2nd amplification from 15.7 !! no clear description on tubes of 2nd amplification and no picture of gel ==possible error source !!
  * K3b, K4a, K4b from 1st amplification
  * end volume 50 µL</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-123" class="hidden">
      <label for="panel-123" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              14
              
          </p>
          <p class="inline-block">PCR - Amplification, Overhang / Puc19 digest with Sma1 / Digested vector gel extraction / Mini-Prep CHO-S Parts</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>PCR - Amplification, Overhang / Puc19 digest with Sma1 / Digested vector gel extraction / Mini-Prep CHO-S Parts</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>PCR Amplification and overhang:</strong></p>

<ul>
<li>used every construct besides K1 from clean up</li>
</ul>

<p><strong>pUC19 digest:</strong></p>

<ul>
<li>digest for 1 h with Sma1 at 25 °C in CutSmart Buffer</li>
<li>5 µL pUC19 (Bl21 K3 from gel 07-23)</li>
<li>2 µL restriction enzyme</li>
<li>pUC19 gel extraction: 0.8% agarose gel electrophoresis with undigested pUC19 as control</li>
<li>no pUC19 was digested! BL21 K3 is empty!</li>
</ul>

<p><strong>Miniprep from the ON cultures 2019-08-13:</strong></p>

<ul>
<li>extract-me miniprep kit used</li>
<li>nanodrop measurements are on the respective tubes</li>
<li>control digestion with Pst1

<ul>
<li>used 500 ng of each miniprep</li>
<li>used 1 µL Pst1-HF and 5 µL CutSmart Buffer</li>
<li>stopped reaction with 10 µL 6x loading dye and used 20 µL for gel electrophoresis</li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-124" class="hidden">
      <label for="panel-124" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              14
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">PCR - Amplification, Overhang / Puc19 digest with Sma1 / Digested vector gel extraction / Mini-Prep CHO-S Parts</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>PCR - Amplification, Overhang / Puc19 digest with Sma1 / Digested vector gel extraction / Mini-Prep CHO-S Parts</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Daniel Schreyer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>PCR Amplification and overhang:</strong>
* used every construct besides K1 from clean up</p>

<p><strong>pUC19 digest:</strong>
* digest for 1 h with Sma1 at 25 °C in CutSmart Buffer
* 5 µL pUC19 (Bl21 K3 from gel 07-23)
* 2 µL restriction enzyme
* pUC19 gel extraction: 0.8% agarose gel electrophoresis with undigested pUC19 as control
* no pUC19 was digested! BL21 K3 is empty!</p>

<p><strong>Miniprep from the ON cultures 2019-08-13:</strong>
* extract-me miniprep kit used
* nanodrop measurements are on the respective tubes
* control digestion with Pst1
  * used 500 ng of each miniprep
  * used 1 µL Pst1-HF and 5 µL CutSmart Buffer
  * stopped reaction with 10 µL 6x loading dye and used 20 µL for gel electrophoresis</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-125" class="hidden">
      <label for="panel-125" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
          </p>
          <p class="inline-block">Gel extraction of Gibson Overhang PCR (K2a -K4b) (from 14.08)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gel extraction of Gibson Overhang PCR (K2a -K4b) (from 14.08)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>no bands in K2b and K3a</li>
<li>many different bands in K2a</li>
<li>K3b - K4b look good
&ndash;gel extraction only of K2a, K3b, K4a, K4b</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-126" class="hidden">
      <label for="panel-126" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Gel extraction of Gibson Overhang PCR (K2a -K4b) (from 14.08)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gel extraction of Gibson Overhang PCR (K2a -K4b) (from 14.08)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>no bands in K2b and K3a</li>
<li>many different bands in K2a</li>
<li>K3b - K4b look good
&ndash;gel extraction only of K2a, K3b, K4a, K4b</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-127" class="hidden">
      <label for="panel-127" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Control Digestion of pSec/Tag2/Hygro</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control Digestion of pSec/Tag2/Hygro</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Jannik Pfister, Felix Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>Did single enzyme digestion of pSec</li>
<li>EcoRI &ndash;2 sites</li>
<li>EcoRV &ndash;1 site</li>
<li>NheI &ndash;1 site
*See picture attachment for estimation and results</li>
</ul>

<p>|DNA |pSec + EcorI |Laddder|pSec + EcorV|pSec + NheI|
|&mdash;|&mdash;|&mdash;|
|</p>

<p><img src="/labjournal-entries/images/WhatsApp_Image_2019-08-15_at_14.53.19.jpeg" alt="&lt;Hygro digestion with EcoRV,EcoRI,NheI&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-128" class="hidden">
      <label for="panel-128" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">pSec&#43;CHO (K1, K2a K2b) / Gel extraction</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>pSec&#43;CHO (K1, K2a K2b) / Gel extraction</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>used 2x 20 µL of the miniprep of the vector+construct</li>
<li>did not start the gel extraction</li>
<li>ran agarose gel</li>
<li>gel extraction only of K1E1-1, K1E1-2, K2aE1-3</li>
<li>gel weight for the extraction:

<ul>
<li>K1E1-1: 0.17 g</li>
<li>K1E1-2: 0.16 g</li>
<li>K2aE1-3: 0.12 g
<br /></li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-129" class="hidden">
      <label for="panel-129" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(5)</strong>
              
          </p>
          <p class="inline-block">pSec&#43;CHO (K1, K2a K2b) / Gel extraction</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>pSec&#43;CHO (K1, K2a K2b) / Gel extraction</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>used 2x 20 µL of the miniprep of the vector+construct</li>
<li>did not start the gel extraction</li>
<li>ran agarose gel</li>
<li>gel extraction only of K1E1-1, K1E1-2, K2aE1-3</li>
<li>gel weight for the extraction:

<ul>
<li>K1E1-1: 0.17 g</li>
<li>K1E1-2: 0.16 g</li>
<li>K2aE1-3: 0.12 g</li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-130" class="hidden">
      <label for="panel-130" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(6)</strong>
              
          </p>
          <p class="inline-block">pUC19 / MiniPrep / Nanodrop / Gel-Extraction / Dephosphorylation Test / Heat-Shock</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>pUC19 / MiniPrep / Nanodrop / Gel-Extraction / Dephosphorylation Test / Heat-Shock</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>Miniprep: used 30 µL MiliQ for eluation</li>
<li>Nanodrop:

<ul>
<li>colony 1: 286 ng/µL</li>
<li>colony 2: 283 ng/µL</li>
<li>colony 3: 306 ng/µL</li>
</ul></li>
<li>gel weight for the extraction:

<ul>
<li>pUC19 1: 0.31 g</li>
<li>pUC19 2: 0.23 g</li>
<li>pUC19 3: 0.25 g
<br /></li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-131" class="hidden">
      <label for="panel-131" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(7)</strong>
              
          </p>
          <p class="inline-block">pUC19 / MiniPrep / Nanodrop / Gel-Extraction / Dephosphorylation Test / Heat-Shock</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>pUC19 / MiniPrep / Nanodrop / Gel-Extraction / Dephosphorylation Test / Heat-Shock</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <ul>
<li>Miniprep: used 30 µL MiliQ for eluation</li>
<li>Nanodrop:

<ul>
<li>colony 1: 286 ng/µL</li>
<li>colony 2: 283 ng/µL</li>
<li>colony 3: 306 ng/µL</li>
</ul></li>
<li>gel weight for the extraction:

<ul>
<li>pUC19 1: 0.31 g</li>
<li>pUC19 2: 0.23 g</li>
<li>pUC19 3: 0.25 g</li>
</ul></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-132" class="hidden">
      <label for="panel-132" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              17
              
          </p>
          <p class="inline-block">Produce new Amp&#43; Agar Plates/ Measure Miniprep of pUC19/ Digest,De-phosphorylate pUC19 with SmaI/ Seed new single colonies from Gibson CHO-S K1, K2a, K2b</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Produce new Amp&#43; Agar Plates/ Measure Miniprep of pUC19/ Digest,De-phosphorylate pUC19 with SmaI/ Seed new single colonies from Gibson CHO-S K1, K2a, K2b</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie Wiedemann, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<p><strong>Prepare new Amp plates:</strong></p>

<p>Prepare 500ml LB-Broth with Agar (17,5 g Agar) for 19 new colony plates;</p>

<p>Autoclave the Mixture to make it sterile;</p>

<p>After a cooling phase Amp got added (500 µl &ndash;concentration: 50 µg/ml);</p>

<p>Pour the plates under sterile conditions and put them in the refrigerator @4 °C;</p>

<p>For the digestion of pUC19 with SmaI the miniprep samples were measured in the nanodrop;</p>

<table>
<thead>
<tr>
<th>Sample Name</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th>ng/µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>pUC19 c1</td>
<td>1,143</td>
<td>0,606</td>
<td>1,89</td>
<td>57,15</td>
</tr>

<tr>
<td>pUC19 c2</td>
<td>1,426</td>
<td>0,873</td>
<td>1,63</td>
<td>71,30</td>
</tr>

<tr>
<td>pUC19 c3</td>
<td>1,073</td>
<td>0,533</td>
<td>2,01</td>
<td>53,65</td>
</tr>
</tbody>
</table>

<p>As the measurement diverged greatly from the measurement the day before that was done right after the miniprep, but with out further purification two approaches were prepared. One with the old nanodrop results from the day before and one with the new results. It is thought that the approach from the day before won&rsquo;t have the right amount of Vector inside but it is prepared either way to be on the safe side.</p>

<p>Approach one:</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>3,3</td>
</tr>

<tr>
<td>CutSmart x10 NEB</td>
<td>5</td>
</tr>

<tr>
<td>Arctic Buffer x10 NEB</td>
<td>5</td>
</tr>

<tr>
<td>Arctic Phosphatase NEB</td>
<td>1</td>
</tr>

<tr>
<td>SmaI NEB</td>
<td>0,5</td>
</tr>

<tr>
<td>H2O to 50µl</td>
<td>35,2</td>
</tr>
</tbody>
</table>

<p>Approach two:</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>14</td>
</tr>

<tr>
<td>CutSmart x10 NEB</td>
<td>5</td>
</tr>

<tr>
<td>Arctic Buffer x10 NEB</td>
<td>5</td>
</tr>

<tr>
<td>Arctic Phosphatase NEB</td>
<td>1</td>
</tr>

<tr>
<td>SmaI NEB</td>
<td>0,5</td>
</tr>

<tr>
<td>H2O to 50µl</td>
<td>24,5</td>
</tr>
</tbody>
</table>

<p>Incubate at 37°C for 1 h;</p>

<p>Inactivate at 80 °C for 20 min (80 °C necessary to inactivate the phosphatase, SmaI itself 65 °C for 20 min would be sufficient);</p>

<p><strong>Preparation of 0,8 % Agarose Gel:</strong></p>

<p>Weigh in 0,8g Agarose and solve in 100ml TAE-buffer &ndash;solve Agarose by microwaving;</p>

<p>After cooling adding of SybrGreen 7 µl;</p>

<p><strong>Preparation of Single Colonies from CHO-S K1, K2a, K2b</strong></p>

<p>For each of the three constructs three colonies were picked and carefully plated under sterile conditions onto the Agar (Amp+) Plates that were earlier prepared;</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>The Gel did not show a clear enough band to extract the digested vector. This is due incubation temperature (37 °C);<br />
SmaI does only require 25 °C to cut properly and shows only 50 % performance at 37 °C;</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li><strong>Measurement part!: Repeat digestion (SmaI) and dephosphorylation of puc19 at 25 °C (Stored at 4 °C);</strong></li>
<li><strong>Control: Gel electrophoresis after digest;</strong></li>
<li><strong>Perform Gibson Assembly puc19 + CFP;</strong></li>
<li><strong>Check Plates seeded today (Incubator at 37 °C);</strong></li>
<li><strong>New Gibson Assembly with K1, K2a, K2b (important)</strong></li>
</ul>

<p><img src="/labjournal-entries/images/pUC19_SmaI_-_Benchling.png" alt="&lt;pUC19&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-133" class="hidden">
      <label for="panel-133" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              17
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Produce new Amp&#43; Agar Plates/ Measure Miniprep of pUC19/ Digest,De-phosphorylate pUC19 with SmaI/ Seed new single colonies from Gibson CHO-S K1, K2a, K2b</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Produce new Amp&#43; Agar Plates/ Measure Miniprep of pUC19/ Digest,De-phosphorylate pUC19 with SmaI/ Seed new single colonies from Gibson CHO-S K1, K2a, K2b</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie Wiedemann, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<p><strong>Prepare new Amp plates:</strong></p>

<p>Prepare 500ml LB-Broth with Agar (17,5 g Agar) for 19 new colony plates;</p>

<p>Autoclave the Mixture to make it sterile;</p>

<p>After a cooling phase Amp got added (500 µl &ndash;concentration: 50 µg/ml);</p>

<p>Pour the plates under sterile conditions and put them in the refrigerator @4 °C;</p>

<p>For the digestion of pUC19 with SmaI the miniprep samples were measured in the nanodrop;</p>

<table>
<thead>
<tr>
<th>Sample Name</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th>ng/µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>pUC19 c1</td>
<td>1,143</td>
<td>0,606</td>
<td>1,89</td>
<td>57,15</td>
</tr>

<tr>
<td>pUC19 c2</td>
<td>1,426</td>
<td>0,873</td>
<td>1,63</td>
<td>71,30</td>
</tr>

<tr>
<td>pUC19 c3</td>
<td>1,073</td>
<td>0,533</td>
<td>2,01</td>
<td>53,65</td>
</tr>
</tbody>
</table>

<p>As the measurement diverged greatly from the measurement the day before that was done right after the miniprep, but with out further purification two approaches were prepared. One with the old nanodrop results from the day before and one with the new results. It is thought that the approach from the day before won&rsquo;t have the right amount of Vector inside but it is prepared either way to be on the safe side.</p>

<p>Approach one:</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>3,3</td>
</tr>

<tr>
<td>CutSmart x10 NEB</td>
<td>5</td>
</tr>

<tr>
<td>Arctic Buffer x10 NEB</td>
<td>5</td>
</tr>

<tr>
<td>Arctic Phosphatase NEB</td>
<td>1</td>
</tr>

<tr>
<td>SmaI NEB</td>
<td>0,5</td>
</tr>

<tr>
<td>H2O to 50µl</td>
<td>35,2</td>
</tr>
</tbody>
</table>

<p>Approach two:</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>14</td>
</tr>

<tr>
<td>CutSmart x10 NEB</td>
<td>5</td>
</tr>

<tr>
<td>Arctic Buffer x10 NEB</td>
<td>5</td>
</tr>

<tr>
<td>Arctic Phosphatase NEB</td>
<td>1</td>
</tr>

<tr>
<td>SmaI NEB</td>
<td>0,5</td>
</tr>

<tr>
<td>H2O to 50µl</td>
<td>24,5</td>
</tr>
</tbody>
</table>

<p>Incubate at 37°C for 1 h;</p>

<p>Inactivate at 80 °C for 20 min (80 °C necessary to inactivate the phosphatase, SmaI itself 65 °C for 20 min would be sufficient);</p>

<p><strong>Preparation of 0,8 % Agarose Gel:</strong></p>

<p>Weigh in 0,8g Agarose and solve in 100ml TAE-buffer &ndash;solve Agarose by microwaving;</p>

<p>After cooling adding of SybrGreen 7 µl;</p>

<p>1kb DNA Ladder (5µl)|empty|Approach 1 (30µl):5µl Staining
|&mdash; |&mdash; |&mdash; |&mdash; |&mdash; |
25 µl Sample|empty|Approach 2 (30 µl):5µl Staining 25 µl Sample|</p>

<p><strong>Preparation of Single Colonies from CHO-S K1, K2a, K2b</strong></p>

<p>For each of the three constructs three colonies were picked and carefully plated under sterile conditions onto the Agar (Amp+) Plates that were earlier prepared;</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>The Gel did not show a clear enough band to extract the digested vector. This is due incubation temperature (37 °C);<br />
SmaI does only require 25 °C to cut properly and shows only 50 % performance at 37 °C;</p>

<p><strong>P</strong>ending tasks, next steps, comments for next people in the Lab**</p>

<ul>
<li><strong>Measurement part!: Repeat digestion (SmaI) and dephosphorylation of puc19 at 25 °C (Stored at 4 °C);</strong></li>
<li><strong>Control: Gel electrophoresis after digest;</strong></li>
<li><strong>Perform Gibson Assembly puc19 + CFP;</strong></li>
<li><strong>Check Plates seeded today (Incubator at 37 °C);</strong></li>
<li><strong>New Gibson Assembly with K1, K2a, K2b (important</strong>)**</li>
</ul>

<p><img src="/labjournal-entries/images/pUC19_SmaI_-_Benchling.png" alt="&lt;pUC19&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-134" class="hidden">
      <label for="panel-134" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              18
              
          </p>
          <p class="inline-block">New Attempts for pUC19 Digest/ Heatshock pSEC and ON culture/ ON cultures from K1,K2a,K2b</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>New Attempts for pUC19 Digest/ Heatshock pSEC and ON culture/ ON cultures from K1,K2a,K2b</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie Wiedemann, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em>**<br />
Notes**</em></p>

<p><strong>Check the Plates from 17.8.19 CHO-S Constructs K1, K2a, K2b:</strong></p>

<p>On all of the plates the seeding worked fine and clones can be picked later on for ON cultures;
<strong>New Attempt to digest pUC19:</strong></p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1 [µl]</th>
<th>Attempt 2 [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>17,5 (pUC19 c1)</td>
<td>18,6 (pUC19 c3)</td>
</tr>

<tr>
<td>CutSmart x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td>AntarcticBuffer x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td>Antarctic Phosphatase</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td>SmaI</td>
<td>0,5</td>
<td>1</td>
</tr>

<tr>
<td>H2O</td>
<td>21</td>
<td>19,4</td>
</tr>
</tbody>
</table>

<p>The first attempt was @25 °C for 1 hour;</p>

<p>Inactivation @ 80 °C for 20 min;</p>

<p>Second attempt incubation @37 °C for 1 hour;</p>

<p>Inactivation @ 80 °C for 20 min;</p>

<p>Both of the samples were to find better conditions as the digest from 17.8.19 were not optimal;</p>

<p>25 °C is the optimal temperature for SmaI, whereas 37 °C is the optimum for the phosphatase;</p>

<p>With 1 hour incubation time also the 25 °C should be suitable for the phosphatase and likewise the 37 °C are acceptable for SmaI, given that the efficacy of the digest is lowered by approx 50% (information on the NEB site);</p>

<p>None of the Digests showed optimal results;</p>

<p>Both were looked at in an 0,8% agarose gel, each for their own.</p>

<p>The order looked like the following:</p>

<p>| 1kb DNA Ladder (5µl)| empty | Half Sample (30µl):5 µl Staining |
|&mdash; |&mdash; |&mdash; |&mdash; |&mdash; |
| 25 µl Sample|empty|Half Sample (30 µl):5 µl Staining25 µl Sample|</p>

<p>Attempt 1 of todays pUC19 digest did show a similar result to yesterdays digestion</p>

<p>Expected was one band of 2,7 kb length (see picture from virtual digest 17.8.19);</p>

<p>The suspected bands were cut out and extracted from the gel (QiaQuick Gel Extraction Kit);</p>

<p>Sample 1: 0,36 g; Sample2: 0,3 g;</p>

<p>&ndash;1060 µl QG; &ndash;900 µl QG</p>

<p>Protocoll of the Kit was used, the DNA was solved in 50 µl H2O;</p>

<p>The resulting DNA was measured in the Nanodrop;</p>

<table>
<thead>
<tr>
<th>Sample Name</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th>ng/µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>Attempt 1 first half</td>
<td>0,026</td>
<td>0,030</td>
<td>0,87</td>
<td>1,29</td>
</tr>

<tr>
<td>Attempt 1 second half</td>
<td>0,033</td>
<td>0,041</td>
<td>0,81</td>
<td>1,66</td>
</tr>
</tbody>
</table>

<p>With these low DNA concentrations Gibson Assembly is not feasible without increasing the reaction volume substantially;<br />
Therefor one should ask one of the TAs for advise to increase the DNA concentrationin a reliable way;</p>

<p><strong>ON culture of Single Colonies from CHO-S K1, K2a, K2b</strong></p>

<p>For each of the three constructs three colonies were picked and carefully seeded in 5 ml LB-Medium (Amp+) under sterile conditions;</p>

<p>Placed in the incubator @ 37 °C 130 rpm (time approx. 18:30);</p>

<p><strong>Heatshock pSec:</strong></p>

<p>Stored pSec has a concentration of 2 µg/µl &ndash;a 1:10 dilution was performed to pipette 100 ng of vector DNA for the heatshock procedure;</p>

<p>The 100 ng of pSec vector DNA were used for 200 µl DH5a;</p>

<p>Evas&rsquo; protocoll was used;</p>

<p>A plate of LB-Agar (Amp+) was seeded and put in the incubator @ 37 °C (time approx. 18:30);</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Gel bands did indicate a non optimal digest &ndash;Not suspected bands and maybe also religation??;</p>

<p>The other digestion Attempt worked as bad as the first or the one from 17.8.19;</p>

<p>Not sure which of the bands is the actual cut vector as one bamd is visible at approx. 3kb, one band looks more like 2 kb and maybe there is a very thin band at 2,7 kb which would be the expected position;</p>

<p>Ask TA with the gel photo for advise and maybe also advise for a proper digestion &ndash;Might be that it was not to few but rather to much enzyme??</p>

<p>Also DNA purification from the Gel yielded only an extremly low amount of DNA &ndash;Gibson not performable;</p>

<p><strong>P</strong>ending tasks, next steps, comments for next people in the Lab**</p>

<p>Take the overnight cultures of Gibson CHO K1, K2a, K2b out of the incubator</p>

<p>Take the plate from pSec out of the incubator</p>

<p>Mini-Prep of pSec &amp; freeze</p>

<p>Speak to the TA or PhD: Discuss the gel bands of the digests &amp; how to increase the concentration of pUC19 digest 2 to continue with gibson assembly</p>

<p>Gibson assembly of pUC19 and measurement part (has conc of 1000 ng in 20 µl according to TAs advice</p>

<p>Heatshock of pUC19 and measurement part in competent E.coli</p>

<p>Generate more pUC19 vector</p>

<p><img src="/labjournal-entries/images/Verdau_2_18.8_pUC19_SmaI_Dephos.jpeg" alt="&lt;pUC19 Digest&gt;" />
<img src="/labjournal-entries/images/Verdau_3_18.8_pUC19_SmaI_Dephos.jpeg" alt="&lt;pUC19 Digest&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-135" class="hidden">
      <label for="panel-135" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              19
              
          </p>
          <p class="inline-block">Miniprep of Gibson CHO Constructs (ON Cultures 18.08) / Preperation for MidiPrep of pUC19 /Control digestion of Gibson CHO Constructs</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Miniprep of Gibson CHO Constructs (ON Cultures 18.08) / Preperation for MidiPrep of pUC19 /Control digestion of Gibson CHO Constructs</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Did a MidiPrep of ON Cultures from 18.08

<ul>
<li>Concentration:</li>
</ul></li>
</ul>

<p>Agarose-Gel electrophoresis for validation</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>K1-1</th>
<th>K1-2</th>
<th>K1-3</th>
<th>K2a-1</th>
<th>K2a-2</th>
<th>K2a-3</th>
<th>K2b-1</th>
<th>K2b-2</th>
<th>K2b-3</th>
</tr>
</thead>

<tbody>
<tr>
<td>Conc [ng/µl]</td>
<td>33,9</td>
<td>31,9</td>
<td>27,4</td>
<td>38,7</td>
<td>31,7</td>
<td>27,4</td>
<td>28,5</td>
<td>44,4</td>
<td>26,1</td>
</tr>
</tbody>
</table>

<table>
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>3 µl</td>
</tr>

<tr>
<td>6X Loading Dye</td>
<td>2 µl</td>
</tr>

<tr>
<td>dH2O</td>
<td>1 µl</td>
</tr>
</tbody>
</table>

<p>-Same Loading order as NanoDrop concentration measurment</p>

<ul>
<li><p>Did an ON Culture of a single conlony of pUC19 (18.08)</p>

<ul>
<li>Picked one colony and incubated it in 5 ml LB+Amp in the shaker @37°C</li>
<li>In the late afternoon &ndash;1:500 in 85ml LB+Amp &ndash;shake for 12-16h @37°C</li>
</ul></li>

<li><p>Control digestion of CHO Constructs</p>

<ul>
<li>Restriction enzymes: ApaI and HindIII-HF</li>
</ul></li>
</ul>

<table>
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>300 ng</td>
</tr>

<tr>
<td>ApaI</td>
<td>1 µl</td>
</tr>

<tr>
<td>HindIII-HF</td>
<td>1 µl</td>
</tr>
</tbody>
</table>

<table>
<thead>
<tr>
<th>CutSmart Buffer</th>
<th>5</th>
</tr>
</thead>

<tbody>
<tr>
<td>dH2O</td>
<td>ad 50 µl</td>
</tr>
</tbody>
</table>

<ul>
<li>-   Protocol
    -   Incubate ApaI at 25°C for 5-15 minutes
    -   Add 1.0 µl (or 10 units) of HindIII-HF and  Incubate at 37°C for 5-15 minutes
    -   Inactivate both at 80°C for 20 minutes</li>
</ul>

<p><strong>Problems faced / possible error causes</strong></p>

<ul>
<li>Low concentration of construc &ndash;more LB+amp Medium next time with longer incubation time in the shaker?

<br /></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>MidiPrep of ON Culture of pUC19</li>
<li>Empty constructs</li>
<li>repeat gibson</li>
</ul>

<p><img src="/labjournal-entries/images/Unbenannt-with-annotations.png" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/TS05376Aug192019.jpg" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/TS05375Aug192019.jpg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-136" class="hidden">
      <label for="panel-136" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              19
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">New Attempts for pUC19 Digest/ Heatshock pSEC and ON culture/ ON cultures from K1,K2a,K2b</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>New Attempts for pUC19 Digest/ Heatshock pSEC and ON culture/ ON cultures from K1,K2a,K2b</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie Wiedemann, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p><strong>Check the Plates from 17.8.19 CHO-S Constructs K1, K2a, K2b:</strong></p>

<p>On all of the plates the seeding worked fine and clones can be picked later on for ON cultures;</p>

<p><strong>New Attempt to digest pUC19:</strong></p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1 [µl]</th>
<th>Attempt 2 [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>17,5 (pUC19 c1)</td>
<td>18,6 (pUC19 c3)</td>
</tr>

<tr>
<td>CutSmart x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td>AntarcticBuffer x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td>Antarctic Phosphatase</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td>SmaI</td>
<td>0,5</td>
<td>1</td>
</tr>

<tr>
<td>H2O</td>
<td>21</td>
<td>19,4</td>
</tr>
</tbody>
</table>

<p>The first attempt was @25 °C for 1 hour;</p>

<p>Inactivation @ 80 °C for 20 min;</p>

<p>Second attempt incubation @37 °C for 1 hour;</p>

<p>Inactivation @ 80 °C for 20 min;</p>

<p>Both of the samples were to find better conditions as the digest from 17.8.19 were not optimal;</p>

<p>25 °C is the optimal temperature for SmaI, whereas 37 °C is the optimum for the phosphatase;</p>

<p>With 1 hour incubation time also the 25 °C should be suitable for the phosphatase and likewise the 37 °C are acceptable for SmaI, given that the efficacy of the digest is lowered by approx 50% (information on the NEB site);</p>

<p>None of the Digests showed optimal results;</p>

<p>Both were looked at in an 0,8% agarose gel, each for their own.</p>

<p>The order looked like the following:</p>

<p>|1kb DNA Ladder (5µl)|Half Sample (30µl):5 µl Staining|25 µl Sample|Half Sample (30 µl):5 µl Staining25 µl Sample|</p>

<p>Attempt 1 of todays pUC19 digest did show a similar result to yesterdays digestion</p>

<p>Expected was one band of 2,7 kb length (see picture from virtual digest 17.8.19);</p>

<p>The suspected bands were cut out and extracted from the gel (QiaQuick Gel Extraction Kit);</p>

<p>Sample 1: 0,36 g; Sample2: 0,3 g;</p>

<p>&ndash;1060 µl QG; &ndash;900 µl QG</p>

<p>Protocoll of the Kit was used, the DNA was solved in 50 µl H2O;</p>

<p>The resulting DNA was measured in the Nanodrop;</p>

<table>
<thead>
<tr>
<th>Sample Name</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th>ng/µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>Attempt 1 first half</td>
<td>0,026</td>
<td>0,030</td>
<td>0,87</td>
<td>1,29</td>
</tr>

<tr>
<td>Attempt 1 second half</td>
<td>0,033</td>
<td>0,041</td>
<td>0,81</td>
<td>1,66</td>
</tr>
</tbody>
</table>

<p>With these low DNA concentrations Gibson Assembly is not feasible without increasing the reaction volume substantially;<br />
Therefor one should ask one of the TAs for advise to increase the DNA concentrationin a reliable way;</p>

<p><strong>ON culture of Single Colonies from CHO-S K1, K2a, K2b</strong></p>

<p>For each of the three constructs three colonies were picked and carefully seeded in 5 ml LB-Medium (Amp+) under sterile conditions;</p>

<p>Placed in the incubator @ 37 °C 130 rpm (time approx. 18:30);</p>

<p><strong>Heatshock pSec:</strong></p>

<p>Stored pSec has a concentration of 2 µg/µl &ndash;a 1:10 dilution was performed to pipette 100 ng of vector DNA for the heatshock procedure;</p>

<p>The 100 ng of pSec vector DNA were used for 200 µl DH5a;</p>

<p>Evas&rsquo; protocoll was used;</p>

<p>A plate of LB-Agar (Amp+) was seeded and put in the incubator @ 37 °C (time approx. 18:30);</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Gel bands did indicate a non optimal digest &ndash;Not suspected bands and maybe also religation??;</p>

<p>The other digestion Attempt worked as bad as the first or the one from 17.8.19;</p>

<p>Not sure which of the bands is the actual cut vector as one bamd is visible at approx. 3kb, one band looks more like 2 kb and maybe there is a very thin band at 2,7 kb which would be the expected position;</p>

<p>Ask TA with the gel photo for advise and maybe also advise for a proper digestion &ndash;Might be that it was not to few but rather to much enzyme??</p>

<p>Also DNA purification from the Gel yielded only an extremly low amount of DNA &ndash;Gibson not performable;</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Take the overnight cultures of Gibson CHO K1, K2a, K2b out of the incubator</p>

<p>Take the plate from pSec out of the incubator</p>

<p>Mini-Prep of pSec &amp; freeze</p>

<p>Speak to the TA or PhD: Discuss the gel bands of the digests &amp; how to increase the concentration of pUC19 digest 2 to continue with gibson assembly</p>

<p>Gibson assembly of pUC19 and measurement part (has conc of 1000 ng in 20 µl according to TAs advice</p>

<p>Heatshock of pUC19 and measurement part in competent E.coli</p>

<p>Generate more pUC19 vector</p>

<p><img src="/labjournal-entries/images/Verdau_2_18.8_pUC19_SmaI_Dephos.jpeg" alt="&lt;pUC19 Digest&gt;" />
<img src="/labjournal-entries/images/Verdau_3_18.8_pUC19_SmaI_Dephos.jpeg" alt="&lt;pUC19 Digest&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-137" class="hidden">
      <label for="panel-137" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              20
              
          </p>
          <p class="inline-block">Midi-prep of overnight-culture from DH5a with puc19; Digestion of puc19 with SmaI at 37°C</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Midi-prep of overnight-culture from DH5a with puc19; Digestion of puc19 with SmaI at 37°C</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Midi prep protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li><p><strong>performance of a midi prep of the overnight culture of DH5a with puC19</strong></p>

<ul>
<li>2 x 25ml from overnight culture</li>
<li>See procotol below</li>
<li>467,5 ng/µl (total: 400µl)</li>
<li>Storage in -20°C in 15ml falcon

<br /></li>
</ul></li>

<li><p><strong>Digestion of the puC19 vector with SmaI under new conditions</strong></p>

<ul>
<li>all materials in the fridge of TW (-20°C)</li>
<li>digestion at 37°C for 1 h</li>
<li>Inactivation at 80°C for 20 minutes</li>
<li>Storage at -20°C (annotated with attempt 1; attempt 2)<br /></li>
</ul></li>
</ul>

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1 [µl]</th>
<th>Attempt 2 [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>10µl (=6µg)</td>
<td>20µl (=12µg)</td>
</tr>

<tr>
<td>CutSmart x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td>AntarcticBuffer x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td>Antarctic Phosphatase</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td>SmaI</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>H2O</td>
<td>28,5</td>
<td>18,5</td>
</tr>
</tbody>
</table>

<p><strong>Protocol Midi:</strong></p>

<ol>
<li><p>Harvest overnight bacterial culture by centrifuging at 6000 x g for 15 min at 4°C. (at the fuge in the Behrensen Lab!!!)</p></li>

<li><p>Resuspend the bacterial pellet in 4 ml Buffer P1.</p></li>

<li><p>Add 4 ml Buffer P2, mix thoroughly by vigorously inverting 4–6 times and incubate at room temperature (15–25°C) for 5 min. If using LyseBlue reagent, the solution will turn blue.</p></li>

<li><p>Add 4 ml Buffer P3, mix thoroughly by vigorously inverting 4–6 times. Incubate on ice for 15 min. If using LyseBlue reagent, mix the solution until it is colorless.</p></li>

<li><p>Centrifuge at ≥20,000 x g for 30 min at 4°C. Re-centrifuge the supernatant at ≥20,000 x g for 15 min at 4°C.</p></li>

<li><p>Equilibrate a QIAGEN-tip 100 by applying 4 ml Buffer QBT, and allow column to empty by gravity flow.</p></li>

<li><p>Apply the supernatant from step 5 to the QIAGEN-tip and allow it to enter the resin by gravity flow.</p></li>

<li><p>Wash the QIAGEN-tip with 2 x 10 ml Buffer QC. Allow Buffer QC to move through the QIAGEN-tip by gravity flow.</p></li>

<li><p>Elute DNA with 5 ml Buffer QF into a clean 15 ml vessel. For constructs larger than 45 kb, prewarming the elution buffer to 65°C may help to increase the yield.</p></li>

<li><p>Precipitate DNA by adding 3.5 ml roomtemperature isopropanol to the eluted DNA and mix. Centrifuge at ≥15,000 x g for 30 min at 4°C. Carefully decant the supernatant.</p></li>

<li><p>Wash the DNA pellet with 2 ml room-temperature 70% ethanol and centrifuge at ≥15,000 x g for 10 min. Carefully decant supernatant.</p></li>

<li><p>Air-dry pellet for 5–10 min and redissolve DNA in a suitable volume of appropriate buffer (e.g., TE buffer, pH 8.0, or 10 mM Tris·Cl, pH 8.5).</p></li>
</ol>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>0,8% agarose gel of the digested puc vector (attempt 1 and 2)</li>
<li>Performing gibson assembly with construct 1, and 2a+b in psec for CHOs</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-138" class="hidden">
      <label for="panel-138" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              20
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Midi-prep of overnight-culture from DH5a with puc19; Digestion of puc19 with SmaI at 37°C</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Midi-prep of overnight-culture from DH5a with puc19; Digestion of puc19 with SmaI at 37°C</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Midi prep protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li><p><strong>performance of a midi prep of the overnight culture of DH5a with puC19</strong></p>

<ul>
<li>2 x 25ml from overnight culture</li>
<li>See procotol below</li>
<li>467,5 ng/µl (total: 400µl)</li>
<li>Storage in -20°C in 15ml falcon

<br /></li>
</ul></li>

<li><p><strong>Digestion of the puC19 vector with SmaI under new conditions</strong></p>

<ul>
<li>all materials in the fridge of TW (-20°C)</li>
<li>digestion at 37°C for 1 h</li>
<li>Inactivation at 80°C for 20 minutes</li>
<li>Storage at -20°C (annotated with attempt 1; attempt 2)<br /></li>
</ul></li>
</ul>

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1 [µl]</th>
<th>Attempt 2 [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>10µl (=6µg)</td>
<td>20µl (=12µg)</td>
</tr>

<tr>
<td>CutSmart x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td>AntarcticBuffer x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td>Antarctic Phosphatase</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td>SmaI</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>H2O</td>
<td>28,5</td>
<td>18,5</td>
</tr>
</tbody>
</table>

<p><strong>Protocol Midi:</strong></p>

<ol>
<li><p>Harvest overnight bacterial culture by centrifuging at 6000 x g for 15 min at 4°C. (at the fuge in the Behrensen Lab!!!)</p></li>

<li><p>Resuspend the bacterial pellet in 4 ml Buffer P1.</p></li>

<li><p>Add 4 ml Buffer P2, mix thoroughly by vigorously inverting 4–6 times and incubate at room temperature (15–25°C) for 5 min. If using LyseBlue reagent, the solution will turn blue.</p></li>

<li><p>Add 4 ml Buffer P3, mix thoroughly by vigorously inverting 4–6 times. Incubate on ice for 15 min. If using LyseBlue reagent, mix the solution until it is colorless.</p></li>

<li><p>Centrifuge at ≥20,000 x g for 30 min at 4°C. Re-centrifuge the supernatant at ≥20,000 x g for 15 min at 4°C. 6. Equilibrate a QIAGEN-tip 100 by applying 4 ml Buffer QBT, and allow column to empty by gravity flow.</p></li>

<li><p>Apply the supernatant from step 5 to the QIAGEN-tip and allow it to enter the resin by gravity flow.</p></li>

<li><p>Wash the QIAGEN-tip with 2 x 10 ml Buffer QC. Allow Buffer QC to move through the QIAGEN-tip by gravity flow.</p></li>

<li><p>Elute DNA with 5 ml Buffer QF into a clean 15 ml vessel. For constructs larger than 45 kb, prewarming the elution buffer to 65°C may help to increase the yield.</p></li>

<li><p>Precipitate DNA by adding 3.5 ml roomtemperature isopropanol to the eluted DNA and mix. Centrifuge at ≥15,000 x g for 30 min at 4°C. Carefully decant the supernatant.</p></li>

<li><p>Wash the DNA pellet with 2 ml room-temperature 70% ethanol and centrifuge at ≥15,000 x g for 10 min. Carefully decant supernatant.</p></li>
</ol>

<p>12.Air-dry pellet for 5–10 min and redissolve DNA in a suitable volume of appropriate buffer (e.g., TE buffer, pH 8.0, or 10 mM Tris·Cl, pH 8.5).</p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p><strong>Materials used</strong></p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>0,8% agarose gel of the digested puc vector (attempt 1 and 2)</li>
<li>Performing gibson assembly with construct 1, and 2a+b in psec for CHOs</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-139" class="hidden">
      <label for="panel-139" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              21
              
          </p>
          <p class="inline-block">Digestion of pSEC with EcoR5 and NHeI for Gibson Assembly, agarose gel (0,8%) for control of digestion of pSec and pUC19, Extraction of digested pSEC vector from gel</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of pSEC with EcoR5 and NHeI for Gibson Assembly, agarose gel (0,8%) for control of digestion of pSec and pUC19, Extraction of digested pSEC vector from gel</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Extraction protocol from Qiagen</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Digestion of pSEC with EcoR5 and NHE1</li>
</ul>

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1 [µl]</th>
<th>Attempt 2 [µl]</th>
<th>Attempt 3 [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µg (=0,5µl)</td>
<td>2 µg (=1µl)</td>
<td>1 µg (=0,5µl)</td>
</tr>

<tr>
<td>NEB buffer 3.1 (x10)</td>
<td>5</td>
<td>5</td>
<td>-</td>
</tr>

<tr>
<td>Cut-Smart (10x)</td>
<td>-</td>
<td>-</td>
<td>3</td>
</tr>

<tr>
<td>EcoR5</td>
<td>1µl</td>
<td>1µl</td>
<td>1µl</td>
</tr>

<tr>
<td>NHE1</td>
<td>1µl</td>
<td>1µl</td>
<td>1µl</td>
</tr>

<tr>
<td>H2O</td>
<td>42,5µl</td>
<td>42µl</td>
<td>42µl</td>
</tr>
</tbody>
</table>

<ul>
<li>attempt 1 and 2:

<ul>
<li>15 minutes on 37°C and stop reaction by adding 10 µl 6 x loading dye</li>
</ul></li>
<li>attempt 3:

<ul>
<li>60 minutes on 37°C and inactivation at 80°C

<br /></li>
</ul></li>
</ul>

<p>Preparation of a 0,8% agarose gel</p>

<ul>
<li>scheme see below</li>
<li>25 µl of the samples and 5µl ladder were used</li>
<li>gel run for 45 minutes at 120 V</li>
</ul>

<p>Extraction of line 7 (digested pSEC vector with EcoRV and NheI)</p>

<ul>
<li>Extraction kit was used</li>
<li>sample stored at -20°C</li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li><p>new digestion of puC19 and maybe pSEC</p></li>

<li><p>Gibson Assembly of CHO constructs in pSEC and iGEM Part in Puc18</p></li>

<li><p>Heat Shock</p></li>
</ul>

<p><img src="/labjournal-entries/images/Puc_and_pSEC_digestion_prediction.PNG" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/digestion_puc19_and_pSEC.png" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-140" class="hidden">
      <label for="panel-140" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              21
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Digestion of pSEC with EcoR5 and NHeI for Gibson Assembly, agarose gel (0,8%) for control of digestion of pSec and pUC19, Extraction of digested pSEC vector from gel</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of pSEC with EcoR5 and NHeI for Gibson Assembly, agarose gel (0,8%) for control of digestion of pSec and pUC19, Extraction of digested pSEC vector from gel</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Extraction protocol from Qiagen</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Digestion of pSEC with EcoR5 and NHE1</li>
</ul>

<table>
<thead>
<tr>
<th>Name</th>
<th>Attempt 1 [µl]</th>
<th>Attempt 2 [µl]</th>
<th>Attempt 3 [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µg (=0,5µl)</td>
<td>2 µg (=1µl)</td>
<td>1 µg (=0,5µl)</td>
</tr>

<tr>
<td>NEB buffer 3.1 (x10)</td>
<td>5</td>
<td>5</td>
<td>-</td>
</tr>

<tr>
<td>Cut-Smart (10x)</td>
<td>-</td>
<td>-</td>
<td>3</td>
</tr>

<tr>
<td>EcoR5</td>
<td>1µl</td>
<td>1µl</td>
<td>1µl</td>
</tr>

<tr>
<td>NHE1</td>
<td>1µl</td>
<td>1µl</td>
<td>1µl</td>
</tr>

<tr>
<td>H2O</td>
<td>42,5µl</td>
<td>42µl</td>
<td>42µl</td>
</tr>
</tbody>
</table>

<ul>
<li>attempt 1 and 2:

<ul>
<li>15 minutes on 37°C and stop reaction by adding 10 µl 6 x loading dye</li>
</ul></li>
<li>attempt 3:

<ul>
<li>60 minutes on 37°C and inactivation at 80°C

<br /></li>
</ul></li>
</ul>

<p>Preparation of a 0,8% agarose gel</p>

<ul>
<li>scheme see below</li>
<li>25 µl of the samples and 5µl ladder were used</li>
<li>gel run for 45 minutes at 120 V</li>
</ul>

<p>Extraction of line 7 (digested pSEC vector with EcoRV and NheI)</p>

<ul>
<li>Extraction kit was used</li>
<li>sample stored at -20°C

<br />

<br />

<br />

<br />

<br />

<br /></li>
</ul>

<p><strong>Protocol</strong></p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>&ndash;&gt;Tip: if you want Andrea to order anything, just pin a postlet at her place</p>

<p><strong>Materials used</strong></p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>-new digestion of puC19 and maybe pSEC</p>

<p>-Gibson Assembly of CHO constructs in pSEC and iGEM Part in Puc18</p>

<p>-Heat Shock</p>

<p><img src="/labjournal-entries/images/Puc_and_pSEC_digestion_prediction.PNG" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/digestion_puc19_and_pSEC.png" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-141" class="hidden">
      <label for="panel-141" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              22
              
          </p>
          <p class="inline-block">Digestion of pSEC with EcoR5 and NHeI for Gibson Assembly, agarose gel (0,8%) for control of digestion of pSec and pUC19, Extraction of digested pSEC vector from gel</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of pSEC with EcoR5 and NHeI for Gibson Assembly, agarose gel (0,8%) for control of digestion of pSec and pUC19, Extraction of digested pSEC vector from gel</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Extraction protocol from Qiagen</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<p>Digestion of pSEC with EcoR5 and NHE1</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µl (=2 µg)</td>
</tr>

<tr>
<td>Cut-Smart (10x)</td>
<td>5</td>
</tr>

<tr>
<td>EcoR5</td>
<td>0,5</td>
</tr>

<tr>
<td>NHE1</td>
<td>0,5</td>
</tr>

<tr>
<td>H2O</td>
<td>43</td>
</tr>
</tbody>
</table>

<ul>
<li>60 minutes @ 37°C</li>
<li>20 minutes inactivation @ 80°C</li>
</ul>

<p>Digestion of pUC19 with SmaI + Antarctic Phosphatase</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>10,6 µl (of Midiprep)</td>
</tr>

<tr>
<td>Cut-Smart (10x)</td>
<td>5</td>
</tr>

<tr>
<td>Antarctic Buffer x10</td>
<td>5</td>
</tr>

<tr>
<td>Antarctic Phosphatase</td>
<td>1</td>
</tr>

<tr>
<td>EcoR5</td>
<td>0,5</td>
</tr>

<tr>
<td>H2O</td>
<td>27,9</td>
</tr>
</tbody>
</table>

<ul>
<li>60 minutes @ 37°C</li>
<li>20 minutes inactivation @ 80°C

<br /></li>
</ul>

<p>Preparation of a 0,8% agarose gel</p>

<ul>
<li>scheme see below</li>
<li>25 µl of the samples + 5 µl of 6x loading dye</li>
<li>5µl ladder</li>

<li><p>1h @130V</p></li>

<li><p>Extraction of line 5, 6 (digested pSEC vector with EcoRV and NheI)</p></li>

<li><p>QiaGen Gelextraction kit was used</p></li>

<li><p>sample stored at -20°C</p></li>
</ul>

<p><strong>Protocol</strong></p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p><strong>Materials used</strong></p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>-new digestion of pUC19 better would be to ask wether they have pUC19 stored somewhere as ours is out of bounds</p>

<p>-Gibson Assembly of CHO constructs in pSEC and potentially pUC19</p>

<p>-Heat Shock</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-142" class="hidden">
      <label for="panel-142" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              22
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Digestion of pSEC with EcoR5 and NHeI for Gibson Assembly, agarose gel (0,8%) for control of digestion of pSec and pUC19, Extraction of digested pSEC vector from gel</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of pSEC with EcoR5 and NHeI for Gibson Assembly, agarose gel (0,8%) for control of digestion of pSec and pUC19, Extraction of digested pSEC vector from gel</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Extraction protocol from Qiagen</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p>Digestion of pSEC with EcoR5 and NHE1</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>1 µl (=2 µg)</td>
</tr>

<tr>
<td>Cut-Smart (10x)</td>
<td>5</td>
</tr>

<tr>
<td>EcoR5</td>
<td>0,5</td>
</tr>

<tr>
<td>NHE1</td>
<td>0,5</td>
</tr>

<tr>
<td>H2O</td>
<td>43</td>
</tr>
</tbody>
</table>

<ul>
<li>60 minutes @ 37°C</li>
<li>20 minutes inactivation @ 80°C</li>
</ul>

<p>Digestion of pUC19 with SmaI + Antarctic Phosphatase</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>10,6 µl (of Midiprep)</td>
</tr>

<tr>
<td>Cut-Smart (10x)</td>
<td>5</td>
</tr>

<tr>
<td>Antarctic Buffer x10</td>
<td>5</td>
</tr>

<tr>
<td>Antarctic Phosphatase</td>
<td>1</td>
</tr>

<tr>
<td>EcoR5</td>
<td>0,5</td>
</tr>

<tr>
<td>H2O</td>
<td>27,9</td>
</tr>
</tbody>
</table>

<ul>
<li>60 minutes @ 37°C</li>
<li>20 minutes inactivation @ 80°C

<br /></li>
</ul>

<p>Preparation of a 0,8% agarose gel</p>

<ul>
<li>scheme see below</li>
<li>25 µl of the samples + 5 µl of 6x loading dye</li>
<li>5µl ladder</li>

<li><p>1h @130V</p></li>

<li><p>Extraction of line 5, 6 (digested pSEC vector with EcoRV and NheI)</p></li>

<li><p>QiaGen Gelextraction kit was used</p></li>

<li><p>sample stored at -20°C</p></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li><p>new digestion of pUC19 better would be to ask wether they have pUC19 stored somewhere as ours is out of bounds</p></li>

<li><p>Gibson Assembly of CHO constructs in pSEC and potentially pUC19</p></li>

<li><p>Heat Shock</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-143" class="hidden">
      <label for="panel-143" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              23
              
          </p>
          <p class="inline-block">Gibson Assembly of CHO K1, K2a, K2b and measurement part in pSec; Heatshock in DH5a bacs; Gel electrophoresis of Gibson Assembly Samples</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson Assembly of CHO K1, K2a, K2b and measurement part in pSec; Heatshock in DH5a bacs; Gel electrophoresis of Gibson Assembly Samples</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Gibson Assembly; Heatshock; Gel electrophoresis</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie Wiedemann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p><strong>Gibson Assembly</strong></p>

<p>According to 2.8.19: CHO constructs and measurement part were taken up in 20 µl TE buffer. &ndash;concentration: 50ng/µl</p>

<p>concentration pSec: 26,8 ng/µl</p>

<p>using 50 ng pSec 1,86 µl</p>

<p>using insert three-fold: 150 ng ~3 µl ; used inserts: CHO K1, CHO K2a, CHO K2b, measurement part</p>

<p>also performed a positive control with the provided kit.</p>

<table>
<thead>
<tr>
<th>Amount in µl</th>
<th>Name</th>
</tr>
</thead>

<tbody>
<tr>
<td>1,86</td>
<td>Vector pSec</td>
</tr>

<tr>
<td>3</td>
<td>Insert</td>
</tr>

<tr>
<td>10</td>
<td>Gibson Assembly Master Mix</td>
</tr>

<tr>
<td>5,14</td>
<td>dH2O</td>
</tr>
</tbody>
</table>

<p>Incubated at 50 °C in thermocycler for 30mins.</p>

<p><strong>Heatshock</strong></p>

<p>performed a heatshock with the Gibson Assembly with DH5a bacteria, following Evas protocoll. Used 18 µl of the assembly samples.</p>

<p>Plated the bacteria on Amp plates and put in incubator to be continued with tomorrow.</p>

<p><strong>Gel electrophoresis</strong></p>

<p>prepared a 0.8% agarose gel with 0.8 g agarose in 100 ml TE buffer. microwaved and added SybrGreen after short cooling period.</p>

<p>Mixed 2 µl of DNA sample with 2 µl of DNA Purple Dye and 16 µl of water.</p>

<p>Added 7 µl DNA Ladder</p>

<p>Order: wayward DNA Ladder - empty - DNA Ladder - CHO K1 - CHO K2a - CHO K2b - Measurement Part - Positive control</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Nothing just DNA ladder visible on gel. Might have used not enough DNA sample?</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>take plates out of incubator, pick colonies</li>
<li>find a new pUC19 vector to try</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-144" class="hidden">
      <label for="panel-144" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              23
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Gibson Assembly of CHO K1, K2a, K2b and measurement part in pSec; Heatshock in DH5a bacs; Gel electrophoresis of Gibson Assembly Samples</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gibson Assembly of CHO K1, K2a, K2b and measurement part in pSec; Heatshock in DH5a bacs; Gel electrophoresis of Gibson Assembly Samples</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Gibson Assembly; Heatshock; Gel electrophoresis</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie Wiedemann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>**<br />
Notes**</p>

<p>**Gibson Assembly<br />
**</p>

<p>According to 2.8.19: CHO constructs and measurement part were taken up in 20 µl TE buffer. &ndash;concentration: 50ng/µl</p>

<p>concentration pSec: 26,8 ng/µl</p>

<p>using 50 ng pSec &ndash;1,86 µl</p>

<p>using insert three-fold: 150 ng &ndash;3 µl ; used inserts: CHO K1, CHO K2a, CHO K2b, measurement part</p>

<p>also performed a positive control with the provided kit.</p>

<table>
<thead>
<tr>
<th>Amount in µl</th>
<th>Name</th>
</tr>
</thead>

<tbody>
<tr>
<td>1,86</td>
<td>Vector pSec</td>
</tr>

<tr>
<td>3</td>
<td>Insert</td>
</tr>

<tr>
<td>10</td>
<td>Gibson Assembly Master Mix</td>
</tr>

<tr>
<td>5,14</td>
<td>dH2O</td>
</tr>
</tbody>
</table>

<p>Incubated at 50 °C in thermocycler for 30mins.</p>

<p><strong>Heatshock</strong></p>

<p>performed a heatshock with the Gibson Assembly with DH5a bacteria, following Evas protocoll. Used 18 µl of the assembly samples.</p>

<p>Plated the bacteria on Amp plates and put in incubator to be continued with tomorrow.</p>

<p><strong>Gel electrophoresis</strong></p>

<p>prepared a 0.8% agarose gel with 0,8 g agarose in 100 ml TE buffer. microwaved and added SybrGreen after short cooling period.</p>

<p>Mixed 2 µl of DNA sample with 2 µl of DNA Purple Dye and 16 µl of water.</p>

<p>Added 7 µl DNA Ladder</p>

<p>Order: wayward DNA Ladder - empty - DNA Ladder - CHO K1 - CHO K2a - CHO K2b - Measurement Part - Positive control</p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Nothing just DNA ladder visible on gel. Might have used not enough DNA sample?</p>

<p><strong>Materials used</strong></p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>take plates out of incubator, pick colonies</li>
<li>find a new pUC19 vector to try</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-145" class="hidden">
      <label for="panel-145" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              27
              
          </p>
          <p class="inline-block">Isolation of DNA (CHO constructs) from overnight culture</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Isolation of DNA (CHO constructs) from overnight culture</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Extract Me - for mini prep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>BACS stock of

<ul>
<li>CHO K1, clone 2</li>
<li>CHO K1, clone 1</li>
<li>CHO K2a, clone 4</li>
<li>DH5a positive control

<ul>
<li>storage at -80°C (in box of electrocompetent tuner and DH5a)</li>
</ul></li>
</ul></li>
<li>Mini-prep of overnight culture

<ul>
<li>eluation with 50µl MiliQ water</li>
</ul></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Measerment of the DNA concentrations</li>
<li>gel of Mini-Prep (did Gibson work?)</li>
<li>Digestion of puC19 with ecoRV</li>
<li>Gibson Assembly</li>
<li>sent CHOs to Kiel<br /></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-146" class="hidden">
      <label for="panel-146" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              27
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Isolation of DNA (CHO constructs) from overnight culture</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Isolation of DNA (CHO constructs) from overnight culture</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Extract Me - for mini prep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>BACS stock of

<ul>
<li>CHO K1, clone 2</li>
<li>CHO K1, clone 1</li>
<li>CHO K2a, clone 4</li>
<li>DH5a positive control

<ul>
<li>storage at -80°C (in box of electrocompetent tuner and DH5a)</li>
</ul></li>
</ul></li>
<li>Mini-prep of overnight culture

<ul>
<li>eluation with 50µl MiliQ water

<br />

<br /></li>
</ul></li>
</ul>

<p><strong>Protocol</strong></p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p><strong>Materials used</strong></p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Measerment of the DNA concentrations</li>
<li>gel of Mini-Prep (did Gibson work?)</li>
<li>Digestion of puC19 with ecoRV</li>
<li>Gibson Assembly</li>
<li>sent CHOs to Kiel<br /></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-147" class="hidden">
      <label for="panel-147" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              28
              
          </p>
          <p class="inline-block">Mini Prep from ON cultures (K1,K2a, K2b), Gel electrophoresis, PCR</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Mini Prep from ON cultures (K1,K2a, K2b), Gel electrophoresis, PCR</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Extract Me - Plasmid Preparation Kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>Mini-prep of overnight cultures (K1, K2a, K2b) of each 5 clones

<ul>
<li>eluation with 50µl MiliQ water</li>
</ul></li>
<li>Ran a 0,8% Gel</li>
<li>Measured Plasmid DNA at Nanodrop<br /></li>
</ul>

<table>
<thead>
<tr>
<th>Sample Name</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th>ng/µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>K1 c1</td>
<td>0,525</td>
<td>0,293</td>
<td>1,79</td>
<td>26,25</td>
</tr>

<tr>
<td>K1 c2</td>
<td>1,408</td>
<td>0,748</td>
<td>1,88</td>
<td>70,42</td>
</tr>

<tr>
<td>K1 c3</td>
<td>1,088</td>
<td>0,619</td>
<td>1,76</td>
<td>54,39</td>
</tr>

<tr>
<td>K1 c4</td>
<td>1,275</td>
<td>0,694</td>
<td>1,84</td>
<td>63,77</td>
</tr>

<tr>
<td>K1 c5</td>
<td>1,404</td>
<td>0,760</td>
<td>1,85</td>
<td>70,21</td>
</tr>

<tr>
<td>K2a c1</td>
<td>1,809</td>
<td>0,983</td>
<td>1,84</td>
<td>90,45</td>
</tr>

<tr>
<td>K2a c2</td>
<td>1,503</td>
<td>0,830</td>
<td>1,81</td>
<td>75,14</td>
</tr>

<tr>
<td>K2a c3</td>
<td>1,214</td>
<td>0,665</td>
<td>1,83</td>
<td>60,72</td>
</tr>

<tr>
<td>K2a c4</td>
<td>1,241</td>
<td>0,649</td>
<td>1,91</td>
<td>62,07</td>
</tr>

<tr>
<td>K2a c5</td>
<td>1,227</td>
<td>0,658</td>
<td>1,86</td>
<td>61,37</td>
</tr>

<tr>
<td>K2b c1</td>
<td>0,409</td>
<td>0,239</td>
<td>1,71</td>
<td>20,45</td>
</tr>

<tr>
<td>K2b c2</td>
<td>1,024</td>
<td>0,562</td>
<td>1,82</td>
<td>51,18</td>
</tr>

<tr>
<td>K2b c3</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>K2b c4</td>
<td>1,222</td>
<td>0,649</td>
<td>1,88</td>
<td>61,12</td>
</tr>

<tr>
<td>K2b c5</td>
<td>0,960</td>
<td>0,476</td>
<td>2,02</td>
<td>47,99</td>
</tr>
</tbody>
</table>

<p>Performed a PCR with all of the Minis to confirm bands that are potentially depicting vector + insert</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>K1 c1</th>
<th>K1 c2</th>
<th>K1 c3</th>
<th>K1 c4</th>
<th>K1 c5</th>
<th>K2a c1</th>
<th>K2a c2</th>
<th>K2a c3</th>
<th>K2a c4</th>
<th>K2a c5</th>
<th>K2b c1</th>
<th>K2b c2</th>
<th>K2b c3</th>
<th>K2b c4</th>
<th>K2b c5</th>
</tr>
</thead>

<tbody>
<tr>
<td>CHO-S fw</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
</tr>

<tr>
<td>CHO-S Primer re</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
</tr>

<tr>
<td>DNA</td>
<td>3,8</td>
<td>1,4</td>
<td>1,8</td>
<td>1,6</td>
<td>1,4</td>
<td>1,1</td>
<td>1,3</td>
<td>1,6</td>
<td>1,6</td>
<td>1,6</td>
<td>4,9</td>
<td>2</td>
<td></td>
<td>1,6</td>
<td>2,1</td>
</tr>

<tr>
<td>Water</td>
<td>6,2</td>
<td>8,6</td>
<td>8,2</td>
<td>8,4</td>
<td>8,6</td>
<td>8,9</td>
<td>8,7</td>
<td>8,4</td>
<td>8,4</td>
<td>8,4</td>
<td>5,1</td>
<td>8</td>
<td></td>
<td>8,4</td>
<td>7,9</td>
</tr>

<tr>
<td>Q5 Hifi MM 2x</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
</tr>

<tr>
<td>Total</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
</tr>
</tbody>
</table>

<p>Primer for this PCR are stored at -20°C the original primer tubes have a concentration of 100µM. Besides them there are also Eppis with the ready to pcr concentration of 10µM;</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Not a real problem but teh bands of corerct lenght were quite weak in comparison to what is probably just oure vector at 5,7kb;<br />
To tackle this issue PCR was performed. If the PCR is successfull, then Samples with vector + Insert will show a band of the corresponding inserts length in a gel;</p>

<p><strong>Materials used</strong></p>

<p>Q5 Hifi Polymerase <strong>empty!</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Perform Gel with PCR Products;</li>
<li>Send positive ones to Kiel (call Matthias Peipp beforehand)</li>
<li>Furhter try to make the Measurement Part work &ndash;Puc digest or also possible to open any other vector blunt and clone the part behind a promotor</li>
<li>If successfull heatshock</li>
<li>Plate out</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-148" class="hidden">
      <label for="panel-148" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              28
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Mini Prep from ON cultures (K1,K2a, K2b), Gel electrophoresis, PCR</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Mini Prep from ON cultures (K1,K2a, K2b), Gel electrophoresis, PCR</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Extract Me - Plasmid Preparation Kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<ul>
<li>Mini-prep of overnight cultures (K1, K2a, K2b) of each 5 clones

<ul>
<li>eluation with 50µl MiliQ water</li>
</ul></li>
<li>Ran a 0,8% Gel</li>
<li>Measured Plasmid DNA at Nanodrop<br /></li>
</ul>

<table>
<thead>
<tr>
<th>Sample Name</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th>ng/µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>K1 c1</td>
<td>0,525</td>
<td>0,293</td>
<td>1,79</td>
<td>26,25</td>
</tr>

<tr>
<td>K1 c2</td>
<td>1,408</td>
<td>0,748</td>
<td>1,88</td>
<td>70,42</td>
</tr>

<tr>
<td>K1 c3</td>
<td>1,088</td>
<td>0,619</td>
<td>1,76</td>
<td>54,39</td>
</tr>

<tr>
<td>K1 c4</td>
<td>1,275</td>
<td>0,694</td>
<td>1,84</td>
<td>63,77</td>
</tr>

<tr>
<td>K1 c5</td>
<td>1,404</td>
<td>0,760</td>
<td>1,85</td>
<td>70,21</td>
</tr>

<tr>
<td>K2a c1</td>
<td>1,809</td>
<td>0,983</td>
<td>1,84</td>
<td>90,45</td>
</tr>

<tr>
<td>K2a c2</td>
<td>1,503</td>
<td>0,830</td>
<td>1,81</td>
<td>75,14</td>
</tr>

<tr>
<td>K2a c3</td>
<td>1,214</td>
<td>0,665</td>
<td>1,83</td>
<td>60,72</td>
</tr>

<tr>
<td>K2a c4</td>
<td>1,241</td>
<td>0,649</td>
<td>1,91</td>
<td>62,07</td>
</tr>

<tr>
<td>K2a c5</td>
<td>1,227</td>
<td>0,658</td>
<td>1,86</td>
<td>61,37</td>
</tr>

<tr>
<td>K2b c1</td>
<td>0,409</td>
<td>0,239</td>
<td>1,71</td>
<td>20,45</td>
</tr>

<tr>
<td>K2b c2</td>
<td>1,024</td>
<td>0,562</td>
<td>1,82</td>
<td>51,18</td>
</tr>

<tr>
<td>K2b c3</td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>K2b c4</td>
<td>1,222</td>
<td>0,649</td>
<td>1,88</td>
<td>61,12</td>
</tr>

<tr>
<td>K2b c5</td>
<td>0,960</td>
<td>0,476</td>
<td>2,02</td>
<td>47,99</td>
</tr>
</tbody>
</table>

<p>Performed a PCR with all of the Minis to confirm bands that are potentially depicting vector + insert</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>K1 c1</th>
<th>K1 c2</th>
<th>K1 c3</th>
<th>K1 c4</th>
<th>K1 c5</th>
<th>K2a c1</th>
<th>K2a c2</th>
<th>K2a c3</th>
<th>K2a c4</th>
<th>K2a c5</th>
<th>K2b c1</th>
<th>K2b c2</th>
<th>K2b c3</th>
<th>K2b c4</th>
<th>K2b c5</th>
</tr>
</thead>

<tbody>
<tr>
<td>CHO-S fw</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
</tr>

<tr>
<td>CHO-S Primer re</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
</tr>

<tr>
<td>DNA</td>
<td>3,8</td>
<td>1,4</td>
<td>1,8</td>
<td>1,6</td>
<td>1,4</td>
<td>1,1</td>
<td>1,3</td>
<td>1,6</td>
<td>1,6</td>
<td>1,6</td>
<td>4,9</td>
<td>2</td>
<td></td>
<td>1,6</td>
<td>2,1</td>
</tr>

<tr>
<td>Water</td>
<td>6,2</td>
<td>8,6</td>
<td>8,2</td>
<td>8,4</td>
<td>8,6</td>
<td>8,9</td>
<td>8,7</td>
<td>8,4</td>
<td>8,4</td>
<td>8,4</td>
<td>5,1</td>
<td>8</td>
<td></td>
<td>8,4</td>
<td>7,9</td>
</tr>

<tr>
<td>Q5 Hifi MM 2x</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
</tr>

<tr>
<td>Total</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
<td>50</td>
</tr>
</tbody>
</table>

<p>Primer for this PCR are stored at -20°C the original primer tubes have a concentration of 100µM. Besides them there are also Eppis with the ready to pcr concentration of 10µM;</p>

<p><strong>Protocol</strong></p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Not a real problem but teh bands of corerct lenght were quite weak in comparison to what is probably just oure vector at 5,7kb;<br />
To tackle this issue PCR was performed. If the PCR is successfull, then Samples with vector + Insert will show a band of the corresponding inserts length in a gel;</p>

<p><strong>Materials used</strong></p>

<p>Q5 Hifi Polymerase <strong>empty!</strong>!</p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Perform Gel with PCR Products;</li>
<li>Send positive ones to Kiel (call Matthias Peipp beforehand)</li>
<li>Furhter try to make the Measurement Part work &ndash;Puc digest or also possible to open any other vector blunt and clone the part behind a promotor</li>
<li>If successfull heatshock &ndash;Plate out</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-149" class="hidden">
      <label for="panel-149" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              29
              
          </p>
          <p class="inline-block">Gel of PCR, Digest of pUC19 (EcoRI, HindIII), ON cultures, Gel electrophoresis of unclear samples</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gel of PCR, Digest of pUC19 (EcoRI, HindIII), ON cultures, Gel electrophoresis of unclear samples</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p>The PCR samples from 28.8 were applied to two 0,8% Agarose Gel and ran at 130V for 1 hour.</p>

<p>Gel 1:</p>

<p>Ladder K1 c1 K1 c2 K1 c3 K1c4 K1 c5 K2a c1 K2a c2 K2a c3 K2a c4 K2a c5</p>

<p>Gel 2:</p>

<p>Ladder K2b c1 K2b c2 K2b c3 K2b c4 K2b c5 Control</p>

<p>The PCR was performed to see whether the desired Insert was implemented into the vector, as the Gel from 28.8 allowed no precise determination.</p>

<p>The PCR Water-control showed a clear band at approximately 1,2 kb. This band was also seen in every other samples, hinting towards contamination of the water or PCR mix.</p>

<p>Other than that, the PCR samples did not show bands that would allow the assumption of successful Gibson assembly. Probably all the vectors are empty.</p>

<p>Another Digestion of pUC19 was performed to exclude potential problems with the blunt cutting enzymes due to re-ligation.</p>

<p>EcoRI and HindIII were used and were expected to yield a 2,6 kb and a 0,1 kb fragment.<br />
Of EcoRI and HindIII a 1:10 Dilution was performed for easier pipetting.</p>

<p>Digestion Mix:</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA (pUC Midi 467,5 ng/µl)</td>
<td>4,27</td>
</tr>

<tr>
<td>CutSmart x10</td>
<td>5</td>
</tr>

<tr>
<td>EcoRI</td>
<td>1</td>
</tr>

<tr>
<td>HindIII</td>
<td>2</td>
</tr>

<tr>
<td>Water</td>
<td>37,73</td>
</tr>

<tr>
<td>Total Amount</td>
<td>50</td>
</tr>
</tbody>
</table>

<p>Digestion @ 37°C for 1h;</p>

<p>Inactivation @ 80°C for 20min;</p>

<p>Samples applied to a 0,8% Agarose Gel: Ladder pUC19</p>

<p>The digest gave the same result as every other digest with blunt end enzymes, indicating that the pUC19 sample we use is in fact contaminated, or no pUC19 at all!</p>

<p>To clarify which of the stored samples in the freezer @ -20°C and in the refrigerator @ 4°C are still of use Gel electrophoresis was performed.</p>

<table>
<thead>
<tr>
<th>Gel 1 part 1</th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA Ladder</td>
<td>Gelex K2a 16.8</td>
<td>Gelex K3b 16.8</td>
<td>Gelex K4a 16.8</td>
<td>Gelex K4b 16.8</td>
<td>Gelex K2b 09.7</td>
<td>Gelex K3b 09.7</td>
<td>Gelex K4b 09.7</td>
<td>Gelex K4b 09.7</td>
<td>PC</td>
<td>Vector 4b/a</td>
<td>3ab pACYC184 p3</td>
<td>Psec digested Gelex 21.08</td>
<td>Attempt 3 pSec</td>
</tr>

<tr>
<td>Gel 1 part 2</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>DNA Ladder</td>
<td>K2ab 2 30.07</td>
<td>K2a 1 30.07</td>
<td>K2a 2 30.07</td>
<td>K2b 2 30.07</td>
<td>K2b 1 30.07</td>
<td>Gelex K1 CHO-S 8.8</td>
<td>Gelex K2a CHO-S 8.8</td>
<td>Gelex K2b CHO-S 8.8</td>
<td>Purified CFP 9.8</td>
<td>/</td>
<td>3ab 1</td>
<td>3ab 2</td>
<td>3ab 3</td>
</tr>

<tr>
<td>Gel 2</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>DNA Ladder</td>
<td>25.8 K1 c1</td>
<td>25.8 K1 c2</td>
<td>25.8 K1 c3</td>
<td>25.8 K1 c4</td>
<td>25.8 K2a c1</td>
<td>25.8 K2a c3</td>
<td>25.8 K2a c4</td>
<td>25.8 K2a c6</td>
<td>DNA CHO K2b c1 25.8</td>
<td>DNA CHO K2b c2 25.8</td>
<td>DNA CHO K2b c3 25.8</td>
<td>DNA CHO K2b c4 25.8</td>
<td></td>
</tr>

<tr>
<td>Gel 3</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>Ladder</td>
<td>2a</td>
<td>2a 1</td>
<td>2a 2</td>
<td>2b</td>
<td>2b 1</td>
<td>4ab</td>
<td>3ab</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Decision which of the tested samples remains usefull and should therefore be kept in storage;</li>
<li>Find alternative to pUC19 for the measurement part

<ul>
<li>One can  <strong>not</strong>  perform blunt end cloning with the ordered gBlocks</li>
<li>However it should be possible with the PCr products of the gBlocks as the have 5&rsquo; phosphate groups</li>
</ul></li>
</ul>

<p><img src="/labjournal-entries/images/Gel_1_doppel.png" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/Gel_2_einzel_heller.png" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/Gel_2_einzel.png" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/Gel_3_einzel_klein.png" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-150" class="hidden">
      <label for="panel-150" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              29
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Gel of PCR, Digest of pUC19 (EcoRI, HindIII), ON cultures, Gel electrophoresis of unclear samples</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Gel of PCR, Digest of pUC19 (EcoRI, HindIII), ON cultures, Gel electrophoresis of unclear samples</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<p>The PCR samples from 28.8 were applied to two 0,8% Agarose Gel and ran at 130V for 1 hour.</p>

<p>Gel 1:</p>

<p>Ladder K1 c1 K1 c2 K1 c3 K1c4 K1 c5 K2a c1 K2a c2 K2a c3 K2a c4 K2a c5</p>

<p>Gel 2:</p>

<p>Ladder K2b c1 K2b c2 K2b c3 K2b c4 K2b c5 Control</p>

<p>The PCR was performed to see whether the desired Insert was implemented into the vector, as the Gel from 28.8 allowed no precise determination.</p>

<p>The PCR Water-control showed a clear band at approximately 1,2 kb. This band was also seen in every other samples, hinting towards contamination of the water or PCR mix.</p>

<p>Other than that, the PCR samples did not show bands that would allow the assumption of successful Gibson assembly. Probably all the vectors are empty.</p>

<p>Another Digestion of pUC19 was performed to exclude potential problems with the blunt cutting enzymes due to re-ligation.</p>

<p>EcoRI and HindIII were used and were expected to yield a 2,6 kb and a 0,1 kb fragment.<br />
Of EcoRI and HindIII a 1:10 Dilution was performed for easier pipetting.</p>

<p>Digestion Mix:</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA (pUC Midi 467,5 ng/µl)</td>
<td>4,27</td>
</tr>

<tr>
<td>CutSmart x10</td>
<td>5</td>
</tr>

<tr>
<td>EcoRI</td>
<td>1</td>
</tr>

<tr>
<td>HindIII</td>
<td>2</td>
</tr>

<tr>
<td>Water</td>
<td>37,73</td>
</tr>

<tr>
<td>Total Amount</td>
<td>50</td>
</tr>
</tbody>
</table>

<p>Digestion @ 37°C for 1h;</p>

<p>Inactivation @ 80°C for 20min;</p>

<p>Samples applied to a 0,8% Agarose Gel: Ladder pUC19</p>

<p>The digest gave the same result as every other digest with blunt end enzymes, indicating that the pUC19 sample we use is in fact contaminated, or no pUC19 at all!</p>

<p>To clarify which of the stored samples in the freezer @ -20°C and in the refrigerator @ 4°C are still of use Gel electrophoresis was performed.</p>

<table>
<thead>
<tr>
<th>Gel 1 part 1</th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA Ladder</td>
<td>Gelex K2a 16.8</td>
<td>Gelex K3b 16.8</td>
<td>Gelex K4a 16.8</td>
<td>Gelex K4b 16.8</td>
<td>Gelex K2b 09.7</td>
<td>Gelex K3b 09.7</td>
<td>Gelex K4b 09.7</td>
<td>Gelex K4b 09.7</td>
<td>PC</td>
<td>Vector 4b/a</td>
<td>3ab pACYC184 p3</td>
<td>Psec digested Gelex 21.08</td>
<td>Attempt 3 pSec</td>
</tr>

<tr>
<td>Gel 1 part 2</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>DNA Ladder</td>
<td>K2ab 2 30.07</td>
<td>K2a 1 30.07</td>
<td>K2a 2 30.07</td>
<td>K2b 2 30.07</td>
<td>K2b 1 30.07</td>
<td>Gelex K1 CHO-S 8.8</td>
<td>Gelex K2a CHO-S 8.8</td>
<td>Gelex K2b CHO-S 8.8</td>
<td>Purified CFP 9.8</td>
<td>/</td>
<td>3ab 1</td>
<td>3ab 2</td>
<td>3ab 3</td>
</tr>

<tr>
<td>Gel 2</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>DNA Ladder</td>
<td>25.8 K1 c1</td>
<td>25.8 K1 c2</td>
<td>25.8 K1 c3</td>
<td>25.8 K1 c4</td>
<td>25.8 K2a c1</td>
<td>25.8 K2a c3</td>
<td>25.8 K2a c4</td>
<td>25.8 K2a c6</td>
<td>DNA CHO K2b c1 25.8</td>
<td>DNA CHO K2b c2 25.8</td>
<td>DNA CHO K2b c3 25.8</td>
<td>DNA CHO K2b c4 25.8</td>
<td></td>
</tr>

<tr>
<td>Gel 3</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>Ladder</td>
<td>2a</td>
<td>2a 1</td>
<td>2a 2</td>
<td>2b</td>
<td>2b 1</td>
<td>4ab</td>
<td>3ab</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

<p><strong>Protocol</strong></p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p><strong>Materials used</strong></p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li>Decision which of the tested samples remains usefull and should therefore be kept in storage;</li>
<li>Find alternative to pUC19 for the measurement part

<ul>
<li>One can  <strong>not</strong>  perform blunt end cloning with the ordered gBlocks</li>
<li>However it should be possible with the PCr products of the gBlocks as the have 5&rsquo; phosphate groups</li>
</ul></li>
</ul>

<p><img src="/labjournal-entries/images/Gel_1_doppel.png" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/Gel_2_einzel_heller.png" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/Gel_2_einzel.png" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/Gel_3_einzel_klein.png" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-151" class="hidden">
      <label for="panel-151" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              30
              
          </p>
          <p class="inline-block">MiniPrep of Gibson Assembly Colonies from 23.08 / MiniPrep of sequenced CHO K1 E1-2 and CHO K2a E1-3 (16.08)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>MiniPrep of Gibson Assembly Colonies from 23.08 / MiniPrep of sequenced CHO K1 E1-2 and CHO K2a E1-3 (16.08)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>MiniPrep of Gibson Assembly Colonies K1, K2a, K2b (5 each)

<ul>
<li>Eluated in 50 µl dH2O</li>
</ul></li>
<li>MiniPrep of CHO K1 E1-2 and K2a E1-3

<ul>
<li>Lost K1 CHO K1 E1-2</li>
<li>Eluated in 50 µl dH2O</li>
</ul></li>

<li><p>Control per Electrophoresis</p>

<ul>
<li>Loading

<ul>
<li>DNA Ladder/K1-1/K1-2/K1-3/K1-4/K1-5/empty/K2a-1/K2a-2/K2a-3/K2a-4/K2a-5/</li>
<li>DNA Ladder/K2b-1/K2b-2/K2b-3/K2b-4/K2b-5/empty/CHO K2a E1-3

<br />

<br /></li>
</ul></li>
</ul></li>

<li><p>Control digestion of both MP</p></li>
</ul>

<p><img src="/labjournal-entries/images/TS05396Aug302019.jpg" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/Unbenannt.PNG" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-152" class="hidden">
      <label for="panel-152" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              30
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">MiniPrep of Gibson Assembly Colonies from 23.08 / MiniPrep of sequenced CHO K1 E1-2 and CHO K2a E1-3 (16.08)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>MiniPrep of Gibson Assembly Colonies from 23.08 / MiniPrep of sequenced CHO K1 E1-2 and CHO K2a E1-3 (16.08)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>MiniPrep of Gibson Assembly Colonies K1, K2a, K2b (5 each)

<ul>
<li>Eluated in 50 µl dH2O</li>
</ul></li>
<li>MiniPrep of CHO K1 E1-2 and K2a E1-3

<ul>
<li>Lost K1 CHO K1 E1-2</li>
<li>Eluated in 50 µl dH2O</li>
</ul></li>

<li><p>Control per Electrophoresis</p>

<ul>
<li>Loading

<ul>
<li>DNA Ladder/K1-1/K1-2/K1-3/K1-4/K1-5/empty/K2a-1/K2a-2/K2a-3/K2a-4/K2a-5/</li>
<li>DNA Ladder/K2b-1/K2b-2/K2b-3/K2b-4/K2b-5/empty/CHO K2a E1-3

<br />

<br /></li>
</ul></li>
</ul></li>

<li><p>Control digestion of both MP</p></li>
</ul>

<p><img src="/labjournal-entries/images/Unbenannt.PNG" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/TS05396Aug302019.jpg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-153" class="hidden">
      <label for="panel-153" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              30
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">MiniPrep of Gibson Assembly Colonies from 23.08 / MiniPrep of sequenced CHO K1 E1-2 and CHO K2a E1-3 (16.08)</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>MiniPrep of Gibson Assembly Colonies from 23.08 / MiniPrep of sequenced CHO K1 E1-2 and CHO K2a E1-3 (16.08)</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Jannik Wagner</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>MiniPrep of Gibson Assembly Colonies K1, K2a, K2b (5 each)

<ul>
<li>Eluated in 50 µl dH2O</li>
</ul></li>
<li>MiniPrep of CHO K1 E1-2 and K2a E1-3

<ul>
<li>Lost K1 CHO K1 E1-2</li>
<li>Eluated in 50 µl dH2O</li>
</ul></li>

<li><p>Control per Electrophoresis</p>

<ul>
<li>Loading

<ul>
<li>DNA Ladder/K1-1/K1-2/K1-3/K1-4/K1-5/empty/K2a-1/K2a-2/K2a-3/K2a-4/K2a-5/</li>
<li>DNA Ladder/K2b-1/K2b-2/K2b-3/K2b-4/K2b-5/empty/CHO K2a E1-3

<br />

<br /></li>
</ul></li>
</ul></li>

<li><p>Control digestion of both MP</p></li>
</ul>

<p><img src="/labjournal-entries/images/Unbenannt.PNG" alt="&lt;Beschreibung&gt;" />
<img src="/labjournal-entries/images/TS05396Aug302019.jpg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-154" class="hidden">
      <label for="panel-154" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              31
              
          </p>
          <p class="inline-block">PCR of measurment part, gel-electrophoresis of measurment part, gel extraction of measurment part</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>PCR of measurment part, gel-electrophoresis of measurment part, gel extraction of measurment part</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Q5 high fidelity PCR Kit, qiagen gel extraction kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>PCR of the iGEM measurment part CFP

<ul>
<li>50µl reaction</li>
</ul></li>
</ul>

<table>
<thead>
<tr>
<th></th>
<th>1x</th>
<th>2,5x</th>
</tr>
</thead>

<tbody>
<tr>
<td>Q5 high fidelity master mix</td>
<td>25µl</td>
<td>62,5µl</td>
</tr>

<tr>
<td>10µM forward primer</td>
<td>1,25 µl</td>
<td>3,125µl</td>
</tr>

<tr>
<td>10µM reverse primer</td>
<td>1,25µl</td>
<td>3,125µl</td>
</tr>

<tr>
<td>template DNA (CFP)</td>
<td>0,5µl (=25ng)</td>
<td>-</td>
</tr>

<tr>
<td>nuclease free water</td>
<td>22µl</td>
<td>55µl</td>
</tr>
</tbody>
</table>

<ul>
<li>with melting temperature of 56°C</li>
<li>program from the protocol</li>
<li>PCR products are stored at -20°C

<ul>
<li>M = measurement part</li>
<li>C = control with water</li>
</ul></li>
<li>gel electrophoresis

<ul>
<li>1% gel</li>
<li>run for 1h, 120V</li>
</ul></li>
</ul>

<p><strong>Scheme:</strong></p>

<p>ladder (7µl) - measurment part (25µl &ndash;&gt;10 µl PCR product + 10µl MilliQ + 5µl loading Dye) - water control (&ldquo;&rdquo;)</p>

<p>&ndash;&gt;see below</p>

<ul>
<li>gel extraction via qiagen gel extraction kit

<ul>
<li>band of CFP at 723 bp (see picture below)</li>
<li>elution with 50µl MiliQ water</li>
<li>storage at -20°C
<br /></li>
</ul></li>
</ul>

<p><strong>Protocol</strong></p>

<p>Q5 high fidelity PCR Kit, qiagen gel extraction kit</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Measurement of DNA concentration of extracted PCR product (CFP &ndash;stored at -20°C)</p>

<p><img src="/labjournal-entries/images/gel_picture_PCR_CFP_new.jpg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-155" class="hidden">
      <label for="panel-155" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              31
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">PCR of measurment part, gel-electrophoresis of measurment part, gel extraction of measurment part</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>PCR of measurment part, gel-electrophoresis of measurment part, gel extraction of measurment part</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Q5 high fidelity PCR Kit, qiagen gel extraction kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ul>
<li>PCR of the iGEM measurment part CFP

<ul>
<li>50µl reaction</li>
</ul></li>
</ul>

<table>
<thead>
<tr>
<th></th>
<th>1x</th>
<th>2,5x</th>
</tr>
</thead>

<tbody>
<tr>
<td>Q5 high fidelity master mix</td>
<td>25µl</td>
<td>62,5µl</td>
</tr>

<tr>
<td>10µM forward primer</td>
<td>1,25 µl</td>
<td>3,125µl</td>
</tr>

<tr>
<td>10µM reverse primer</td>
<td>1,25µl</td>
<td>3,125µl</td>
</tr>

<tr>
<td>template DNA (CFP)</td>
<td>0,5µl (=25ng)</td>
<td>-</td>
</tr>

<tr>
<td>nuclease free water</td>
<td>22µl</td>
<td>55µl</td>
</tr>
</tbody>
</table>

<ul>
<li>with melting temperature of 56°C</li>
<li>program from the protocol</li>
<li>PCR products are stored at -20°C

<ul>
<li>M = measurement part</li>
<li>C = control with water</li>
</ul></li>
<li>gel electrophoresis

<ul>
<li>1% gel</li>
<li>run for 1h, 120V</li>
</ul></li>
</ul>

<p><strong>Scheme:</strong></p>

<p>ladder (7µl) - measurment part (25µl &ndash;&gt;10 µl PCR product + 10µl MilliQ + 5µl loading Dye) - water control (&ldquo;&rdquo;)</p>

<p>&ndash;&gt;see below</p>

<ul>
<li>gel extraction via qiagen gel extraction kit

<ul>
<li>band of CFP at 723 bp (see picture below)</li>
<li>elution with 50µl MiliQ water</li>
<li>storage at -20°C

<br />

<br />

<br /></li>
</ul></li>
</ul>

<p><strong>Protocol</strong></p>

<p>Q5 high fidelity PCR Kit, qiagen gel extraction kit</p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Problems faced / possible error causes</strong></p>

<p><strong>Materials used</strong></p>

<p><strong>Files used / generated</strong></p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Measurement of DNA concentration of extracted PCR product (CFP &ndash;stored at -20°C)</p>

<p><img src="/labjournal-entries/images/gel_picture_PCR_CFP_new.jpg" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

            
            </div>
            

            <h2 class="my-10 text-3xl">September</h2>
            

            <div class="accordion flex flex-col items-center justify-center">
            
        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-156" class="hidden">
      <label for="panel-156" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
          </p>
          <p class="inline-block">Digestion of pUC19 and CFP with Xbal and EcoRI</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of pUC19 and CFP with Xbal and EcoRI</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Heat Shock Protocol, NEB Ligation Protocol (M0202), Q5 Hifi PCR Protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Nanodrop measurement of gelextraction from 31.07. and and 01.08.</p>

<p>CFP PCR gel extr. 31.07.: 13,6</p>

<p>CFP PCR gel extr. 01.08. 1: 21,82</p>

<p>CFP PCR gel extr. 01.08. 2: 16,96</p>

<p>Digest of gelextr. (31.08.) and pUC19 with EcoRI and XbaI:</p>

<p>43µL DNA + 5 µL cutsmart + 1 µL of each enzyme (HF)</p>

<p>15 min 37°C</p>

<p>20 min 65°C heat inactivation</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Ligation of puc19 and CFP</p>

<p>New PCR of CFP</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-157" class="hidden">
      <label for="panel-157" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Ligation of pUC19 with Measurement Part, Heatshock into DH5a, PCR of Measurement Part</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Ligation of pUC19 with Measurement Part, Heatshock into DH5a, PCR of Measurement Part</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Heat Shock Protocol, NEB Ligation Protocol (M0202), Q5 Hifi PCR Protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Ligation of Measurement Part with cut pUC19 from earlier of the day;</p>

<p>Relative amounts of vector 1 : Insert 4;</p>

<p>10 ng of Vector and approx. 10 ng of Insert as the Insert is only 700 bp long;</p>

<p>Ligation with T4 DNA Ligase Protocol (M020) from NEB;</p>

<p>Reaction set up on ice;</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
<th></th>
</tr>
</thead>

<tbody>
<tr>
<td>T4 DNA Ligase Buffer x10</td>
<td>3</td>
<td></td>
</tr>

<tr>
<td>Vector DNA</td>
<td>20</td>
<td></td>
</tr>

<tr>
<td>Insert DNA</td>
<td>0,5</td>
<td></td>
</tr>

<tr>
<td>Water</td>
<td>5,5,</td>
<td></td>
</tr>

<tr>
<td>T4 Ligase</td>
<td>1</td>
<td></td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>30 µl</strong></td>
<td></td>
</tr>
</tbody>
</table>

<p>Incubate @RT for 10 min;</p>

<p>Heat inactivate @65 for 10 min;</p>

<p>With the sample <strong>Heat shock</strong> was performed after Evas Protocol;</p>

<p>The transformed cells were seeded on an Amp+ plate and placed into the incubator @ 37 °C;</p>

<p>For the case of a failed ligation a PCR with the measurement part was set up;</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>0,5</td>
</tr>

<tr>
<td>Q5 Hifi Polymerase MasterMix x2</td>
<td>12,5</td>
</tr>

<tr>
<td>Measurement Primer fw</td>
<td>1,25</td>
</tr>

<tr>
<td>Measurement Primer re</td>
<td>1,25</td>
</tr>

<tr>
<td>Nuclease free water</td>
<td>9,5</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>30 µl</strong></td>
</tr>
</tbody>
</table>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Q5 Hifi Polymerase is stored in the topmost drawer in the -20 °C fridge in the winkler lab and no longer in the Plastik Box in drawer 2;</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li><p>Check Plate in the Incubator and pick colonies if transformation was successful;</p></li>

<li><p>IF not new digestion with the ordered pUC19 (XbaI/EcoRI)</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-158" class="hidden">
      <label for="panel-158" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Ligation of pUC19 with Measurement Part, Heatshock into DH5a, PCR of Measurement Part</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Ligation of pUC19 with Measurement Part, Heatshock into DH5a, PCR of Measurement Part</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Heat Shock Protocol, NEB Ligation Protocol (M0202), Q5 Hifi PCR Protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>Ligation of Measurement Part with cut pUC19 from earlier of the day;</p>

<p>Relative amounts of vector 1 : Insert 4;</p>

<p>10 ng of Vector and approx. 10 ng of Insert as the Insert is only 700 bp long;</p>

<p>Ligation with T4 DNA Ligase Protocol (M020) from NEB;</p>

<p>Reaction set up on ice;</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
<th></th>
</tr>
</thead>

<tbody>
<tr>
<td>T4 DNA Ligase Buffer x10</td>
<td>3</td>
<td></td>
</tr>

<tr>
<td>Vector DNA</td>
<td>20</td>
<td></td>
</tr>

<tr>
<td>Insert DNA</td>
<td>0,5</td>
<td></td>
</tr>

<tr>
<td>Water</td>
<td>5,5,</td>
<td></td>
</tr>

<tr>
<td>T4 Ligase</td>
<td>1</td>
<td></td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>30 µl</strong></td>
<td></td>
</tr>
</tbody>
</table>

<p>Incubate @RT for 10 min;</p>

<p>Heat inactivate @65 for 10 min;</p>

<p>With the sample <strong>Heat shock</strong> was performed after Evas Protocol;</p>

<p>The transformed cells were seeded on an Amp+ plate and placed into the incubator @ 37 °C;</p>

<p>For the case of a failed ligation a PCR with the measurement part was set up;</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>0,5</td>
</tr>

<tr>
<td>Q5 Hifi Polymerase MasterMix x2</td>
<td>12,5</td>
</tr>

<tr>
<td>Measurement Primer fw</td>
<td>1,25</td>
</tr>

<tr>
<td>Measurement Primer re</td>
<td>1,25</td>
</tr>

<tr>
<td>Nuclease free water</td>
<td>9,5</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>30 µl</strong></td>
</tr>
</tbody>
</table>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Q5 Hifi Polymerase is stored in the topmost drawer in the -20 °C fridge in the winkler lab and no longer in the Plastik Box in drawer 2;</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>- Check Plate in the Incubator and pick colonies if transformation was successful;</p>

<p>- IF not new digestion with the ordered pUC19 (XbaI/EcoRI)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-159" class="hidden">
      <label for="panel-159" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Digestion of pUC19 and CFP with Xbal and EcoRI</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of pUC19 and CFP with Xbal and EcoRI</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Lsg Sequenzing Anleitung</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Nanodrop measurement of gelextraction from 31.07. and and 01.08.</p>

<p>CFP PCR gel extr. 31.07.: 13,6</p>

<p>CFP PCR gel extr. 01.08. 1: 21,82</p>

<p>CFP PCR gel extr. 01.08. 2: 16,96</p>

<p>Digest of gelextr. (31.08.) and pUC19 with EcoRI and XbaI:</p>

<p>43µL DNA + 5 µL cutsmart + 1 µL of each enzyme (HF)</p>

<p>15 min 37°C</p>

<p>20 min 65°C heat inactivation</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Ligation of puc19 and CFP</p>

<p>New PCR of CFP</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-160" class="hidden">
      <label for="panel-160" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(5)</strong>
              
          </p>
          <p class="inline-block">Digestion of pUC19 and CFP with Xbal and EcoRI</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of pUC19 and CFP with Xbal and EcoRI</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Nanodrop measurement of gelextraction from 31.07. and and 01.08.</p>

<p>CFP PCR gel extr. 31.07.: 13,6</p>

<p>CFP PCR gel extr. 01.08. 1: 21,82</p>

<p>CFP PCR gel extr. 01.08. 2: 16,96</p>

<p>Digest of gelextr. (31.08.) and pUC19 with EcoRI and XbaI:</p>

<p>43µL DNA + 5 µL cutsmart + 1 µL of each enzyme (HF)</p>

<p>15 min 37°C</p>

<p>20 min 65°C heat inactivation</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Ligation of puc19 and CFP</p>

<p>New PCR of CFP</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-161" class="hidden">
      <label for="panel-161" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(6)</strong>
              
          </p>
          <p class="inline-block">Digestion of pUC19 and CFP with Xbal and EcoRI</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of pUC19 and CFP with Xbal and EcoRI</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Felix Pfister, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Nanodrop measurement of gelextraction from 31.07. and and 01.08.</p>

<p>CFP PCR gel extr. 31.07.: 13,6</p>

<p>CFP PCR gel extr. 01.08. 1: 21,82</p>

<p>CFP PCR gel extr. 01.08. 2: 16,96</p>

<p>Digest of gelextr. (31.08.) and pUC19 with EcoRI and XbaI:</p>

<p>43µL DNA + 5 µL cutsmart + 1 µL of each enzyme (HF)</p>

<p>15 min 37°C</p>

<p>20 min 65°C heat inactivation</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Ligation of puc19 and CFP</p>

<p>New PCR of CFP</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-162" class="hidden">
      <label for="panel-162" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(7)</strong>
              
          </p>
          <p class="inline-block">CHO parts of 30.07. sequencing &#43; ampicilin aliquots</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CHO parts of 30.07. sequencing &#43; ampicilin aliquots</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Lsg Sequenzing Anleitung</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Total volume should be 15 µL with a concentration of 100 µg/µL</p>

<p>Accodring to the nanodrop results of 30.08. following DNA volumes were used:</p>

<p>K 1-1: 5,9 µL (C159961)</p>

<p>K 1-2: 6,2 µL (C159960)</p>

<p>K 1-3: 2,96 µL (C159959)</p>

<p>K 1-4: 2,8 µL (C159958)</p>

<p>K 1-5: 4,2 µL (C159957)</p>

<p>K 2a-1: 2,44 µL (C159956)</p>

<p>K 2a-2: 5,3 µL (C159955)</p>

<p>K 2a-3: 3,2 µL (C159954)</p>

<p>K 2a-4: 2,9 µL (C159953)</p>

<p>K 2a-5: 2,66 µL (C159952)</p>

<p>K 2b-1: 6,82 µL (C159951)</p>

<p>K 2b-2: 3,5 µL (C159950)</p>

<p>K 2b-3: 4,5 µL (C159949)</p>

<p>K 2b-4: 4,52 µL (C159948)</p>

<p>K 2b-5: 4,1 µL (C159947)</p>

<p>ad 15 µL with milliQ water</p>

<p>Amp aliqouts:</p>

<p>50 mg/ mL</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-163" class="hidden">
      <label for="panel-163" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(8)</strong>
              
          </p>
          <p class="inline-block">CHO parts of 30.07. sequencing &#43; ampicilin aliquots</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CHO parts of 30.07. sequencing &#43; ampicilin aliquots</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Lsg Sequenzing Anleitung</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Total volume should be 15 µL with a concentration of 100 µg/µL</p>

<p>Accodring to the nanodrop results of 30.08. following DNA volumes were used:</p>

<p>K 1-1: 5,9 µL (C159961)</p>

<p>K 1-2: 6,2 µL (C159960)</p>

<p>K 1-3: 2,96 µL (C159959)</p>

<p>K 1-4: 2,8 µL (C159958)</p>

<p>K 1-5: 4,2 µL (C159957)</p>

<p>K 2a-1: 2,44 µL (C159956)</p>

<p>K 2a-2: 5,3 µL (C159955)</p>

<p>K 2a-3: 3,2 µL (C159954)</p>

<p>K 2a-4: 2,9 µL (C159953)</p>

<p>K 2a-5: 2,66 µL (C159952)</p>

<p>K 2b-1: 6,82 µL (C159951)</p>

<p>K 2b-2: 3,5 µL (C159950)</p>

<p>K 2b-3: 4,5 µL (C159949)</p>

<p>K 2b-4: 4,52 µL (C159948)</p>

<p>K 2b-5: 4,1 µL (C159947)</p>

<p>ad 15 µL with milliQ water</p>

<p>Amp aliqouts:</p>

<p>50 mg/ mL</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-164" class="hidden">
      <label for="panel-164" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(9)</strong>
              
          </p>
          <p class="inline-block">CHO parts of 30.07. sequencing &#43; ampicilin aliquots</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CHO parts of 30.07. sequencing &#43; ampicilin aliquots</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Lsg Sequenzing Anleitung</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Lena Schorr, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Total volume should be 15 µL with a concentration of 100 µg/µL</p>

<p>Accodring to the nanodrop results of 30.08. following DNA volumes were used:</p>

<p>K 1-1: 5,9 µL (C159961)</p>

<p>K 1-2: 6,2 µL (C159960)</p>

<p>K 1-3: 2,96 µL (C159959)</p>

<p>K 1-4: 2,8 µL (C159958)</p>

<p>K 1-5: 4,2 µL (C159957)</p>

<p>K 2a-1: 2,44 µL (C159956)</p>

<p>K 2a-2: 5,3 µL (C159955)</p>

<p>K 2a-3: 3,2 µL (C159954)</p>

<p>K 2a-4: 2,9 µL (C159953)</p>

<p>K 2a-5: 2,66 µL (C159952)</p>

<p>K 2b-1: 6,82 µL (C159951)</p>

<p>K 2b-2: 3,5 µL (C159950)</p>

<p>K 2b-3: 4,5 µL (C159949)</p>

<p>K 2b-4: 4,52 µL (C159948)</p>

<p>K 2b-5: 4,1 µL (C159947)</p>

<p>ad 15 µL with milliQ water</p>

<p>Amp aliqouts:</p>

<p>50 mg/ mL</p>

<table>
<thead>
<tr>
<th><strong>Aim/Task:</strong></th>
<th>Ligation of pUC19 with Measurement Part, Heatshock into DH5a, PCR of Measurement Part</th>
</tr>
</thead>

<tbody>
<tr>
<td><strong>Protocoll used:</strong></td>
<td>Heat Shock Protocol, NEB Ligation Protocol (M0202), Q5 Hifi PCR Protocol</td>
</tr>

<tr>
<td><strong>Date, Site:</strong></td>
<td>03.09.19 , nfz</td>
</tr>

<tr>
<td><strong>Name:</strong></td>
<td>Felix, Eva</td>
</tr>
</tbody>
</table>

<p><strong><em>Notes</em></strong></p>

<p>Ligation of Measurement Part with cut pUC19 from earlier of the day;</p>

<p>Relative amounts of vector 1 : Insert 4;</p>

<p>10 ng of Vector and approx. 10 ng of Insert as the Insert is only 700 bp long;</p>

<p>Ligation with T4 DNA Ligase Protocol (M020) from NEB;</p>

<p>Reaction set up on ice;</p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>Amount [µl]</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>T4 DNA Ligase Buffer x10</td>
<td>3</td>
</tr>

<tr>
<td>Vector DNA</td>
<td>20</td>
</tr>

<tr>
<td>Insert DNA</td>
<td>0,5</td>
</tr>

<tr>
<td>Water</td>
<td>5,5,</td>
</tr>

<tr>
<td>T4 Ligase</td>
<td>1</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>30 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @RT for 10 min;</p>

<p>Heat inactivate @65 for 10 min;</p>

<p>With the sample <strong>Heat shock</strong> was performed after Evas Protocol;</p>

<p>The transformed cells were seeded on an Amp+ plate and placed into the incubator @ 37 °C;</p>

<p>For the case of a failed ligation a PCR with the measurement part was set up;</p>

<p>||
| &mdash; || &mdash; |
| <strong>Name</strong>                        | <strong>Amount [µl]</strong> |
| &mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;- | &mdash;&mdash;&mdash;&mdash;&mdash; |
| DNA                             | 0,5             |
| Q5 Hifi Polymerase MasterMix x2 | 12,5            |
| Measurement Primer fw           | 1,25            |
| Measurement Primer re           | 1,25            |
| Nuclease free water             | 9,5             |
| <strong>Total amount</strong>                | <strong>30 µl</strong>       |</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Q5 Hifi Polymerase is stored in the topmost drawer in the -20 °C fridge in the winkler lab and no longer in the Plastik Box in drawer 2;</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>- Check Plate in the Incubator and pick colonies if transformation was successful;</p>

<p>- IF not new digestion with the ordered pUC19 (XbaI/EcoRI)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-165" class="hidden">
      <label for="panel-165" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
          </p>
          <p class="inline-block">Digest of new pUC19 (XbaI/EcoRI), PCR clean up of CFP, Gel extraction of CFP, Digest purified CFP, Ligation of pUC19 with CFP, Heatshock</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digest of new pUC19 (XbaI/EcoRI), PCR clean up of CFP, Gel extraction of CFP, Digest purified CFP, Ligation of pUC19 with CFP, Heatshock</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p><strong>Digestion of pUC19 with EcoRI and XbaI:</strong></p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>pUC19 DNa</td>
<td>2              (= 2µg &ndash;DNA conc. 100ng/µl)</td>
</tr>

<tr>
<td>EcoRI-HF</td>
<td>1</td>
</tr>

<tr>
<td>XbaI</td>
<td>1</td>
</tr>

<tr>
<td>Cut Smart Buffer x10</td>
<td>2</td>
</tr>

<tr>
<td>Water</td>
<td>14</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @37 °C for 20 min;</p>

<p>Inactivate @ 65 °C for 20 min;</p>

<hr />

<p><strong>PCR Purification:</strong></p>

<p>Purified PCR sample from 3.9.19 with the PCR Purification Kit and followed the Kits protocol;</p>

<p>Eluted DNA in 30 µl Buffer EB;</p>

<hr />

<p><strong>Gel Extraction:</strong></p>

<p>Extracted DNA from the Gel prepared earlier this DNA to extract the Measurement Part;</p>

<p>Used the QiaQuick Gel Extraction Kit and followed its protocol;</p>

<p>Weight of Gel fragment ~0,02g &ndash;60 µl Buffer QG for Gel solving;</p>

<p>Eluted the DNA in 35 µl Buffer EB;</p>

<hr />

<p><strong>DNA Measuring at Nanodrop:</strong></p>

<table>
<thead>
<tr>
<th>Name</th>
<th>ng/µl</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th><sup>260</sup>&frasl;<sub>230</sub></th>
</tr>
</thead>

<tbody>
<tr>
<td>PCR Clean Up CFP</td>
<td>45,6</td>
<td>0,912</td>
<td>0,500</td>
<td>1,82</td>
<td>1,69</td>
</tr>

<tr>
<td>Gel Ex CFP</td>
<td>8,5</td>
<td>0,169</td>
<td>0,085</td>
<td>2,00</td>
<td>0,02</td>
</tr>
</tbody>
</table>

<hr />

<p><strong>Digestion of the purified samples (Gel Ex CFP, PCR Clean Up CFP):</strong></p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl] PCR Purified</th>
<th>Amount [µl] Gel Ex CFP</th>
</tr>
</thead>

<tbody>
<tr>
<td>CFP DNA</td>
<td>30 (= 1,37 µg)</td>
<td>35 (= 927,5 ng)</td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>EcoRI-HF</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>XbaI</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>Cut Smart Buffer x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>Water</td>
<td>15</td>
<td>8</td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>50 µl</strong></td>
<td><strong>50 µl</strong></td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

<p>Incubate @37 °C for 20 min;</p>

<p>Inactivate @ 65 °C for 20 min;</p>

<p>Ligation of Measurement Part with the digested pUC in different ratios</p>

<p>Ligation with T4 DNA Ligase Protocol (M020) from NEB;</p>

<p>Reaction set up on ice;</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]           CFP PCR 1:5</th>
<th>Amount [µl]            CFP PCR 1:7</th>
<th>Amount [µl]        CFP PCR 1:10</th>
<th>Amount [µl]          Gel Ex 1:5</th>
</tr>
</thead>

<tbody>
<tr>
<td>T4 DNA Ligase Buffer x10</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
</tr>

<tr>
<td>Vector DNA</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>Insert DNA</td>
<td>1,4</td>
<td>2</td>
<td>2,84</td>
<td>11,76   ( = 100 ng)</td>
</tr>

<tr>
<td>Water</td>
<td>15,1</td>
<td>14,5</td>
<td>13,7</td>
<td>5,24</td>
</tr>

<tr>
<td>T4 Ligase</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @RT for 10 min;</p>

<p>Heat inactivate @65 for 10 min;</p>

<p>With the sample <strong>Heat shock</strong> was performed after Evas Protocol;</p>

<p>The transformed cells and one sample being ony digested vector were seeded on Amp+ plates and placed into the incubator @ 37 °C overnight;</p>

<hr />

<p>Placed the colonies picked earlier by Eva into the Shaker overnight @ 37 °C and 230 rpm;</p>

<p><strong>Files used / generated</strong></p>

<p>In parallel designed Primer for the restriction digestion of the Constructs for the <em>E. coli</em> expression;</p>

<p>Designed them with the intent to open the pACYC184 vector with ClaI and SalI and accordingly also digest the constructs with the same enzymes to perform classical cloning;</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li><p>Mini Prep from the Overnight cultures in the Shaker (one could also look at the cells under the fluorescence microscope as successfull ligation + transformation should produce CFP);</p></li>

<li><p>Gel electrophoresis thereof</p></li>

<li><p>Heatshock into other Strains BL21, Star, Tuner;</p></li>

<li><p>Check Plates in the Incubator, prepare ON cultures;</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-166" class="hidden">
      <label for="panel-166" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Digest of new pUC19 (XbaI/EcoRI), PCR clean up of CFP, Gel extraction of CFP, Digest purified CFP, Ligation of pUC19 with CFP, Heatshock</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digest of new pUC19 (XbaI/EcoRI), PCR clean up of CFP, Gel extraction of CFP, Digest purified CFP, Ligation of pUC19 with CFP, Heatshock</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p><strong>Digestion of pUC19 with EcoRI and XbaI:</strong></p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]</th>
</tr>
</thead>

<tbody>
<tr>
<td>pUC19 DNa</td>
<td>2              (= 2µg &ndash;DNA conc. 100ng/µl)</td>
</tr>

<tr>
<td>EcoRI-HF</td>
<td>1</td>
</tr>

<tr>
<td>XbaI</td>
<td>1</td>
</tr>

<tr>
<td>Cut Smart Buffer x10</td>
<td>2</td>
</tr>

<tr>
<td>Water</td>
<td>14</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @37 °C for 20 min;</p>

<p>Inactivate @ 65 °C for 20 min;</p>

<hr />

<p><strong>PCR Purification:</strong></p>

<p>Purified PCR sample from 3.9.19 with the PCR Purification Kit and followed the Kits protocol;</p>

<p>Eluted DNA in 30 µl Buffer EB;</p>

<hr />

<p><strong>Gel Extraction:</strong></p>

<p>Extracted DNA from the Gel prepared earlier this DNA to extract the Measurement Part;</p>

<p>Used the QiaQuick Gel Extraction Kit and followed its protocol;</p>

<p>Weight of Gel fragment ~0,02g &ndash;60 µl Buffer QG for Gel solving;</p>

<p>Eluted the DNA in 35 µl Buffer EB;</p>

<hr />

<p><strong>DNA Measuring at Nanodrop:</strong></p>

<table>
<thead>
<tr>
<th>Name</th>
<th>ng/µl</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th><sup>260</sup>&frasl;<sub>230</sub></th>
</tr>
</thead>

<tbody>
<tr>
<td>PCR Clean Up CFP</td>
<td>45,6</td>
<td>0,912</td>
<td>0,500</td>
<td>1,82</td>
<td>1,69</td>
</tr>

<tr>
<td>Gel Ex CFP</td>
<td>8,5</td>
<td>0,169</td>
<td>0,085</td>
<td>2,00</td>
<td>0,02</td>
</tr>
</tbody>
</table>

<hr />

<p><strong>Digestion of the purified samples (Gel Ex CFP, PCR Clean Up CFP):</strong></p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl] PCR Purified</th>
<th>Amount [µl] Gel Ex CFP</th>
</tr>
</thead>

<tbody>
<tr>
<td>CFP DNA</td>
<td>30 (= 1,37 µg)</td>
<td>35 (= 927,5 ng)</td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>EcoRI-HF</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>XbaI</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>Cut Smart Buffer x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>Water</td>
<td>15</td>
<td>8</td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>50 µl</strong></td>
<td><strong>50 µl</strong></td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

<p>Incubate @37 °C for 20 min;</p>

<p>Inactivate @ 65 °C for 20 min;</p>

<p>Ligation of Measurement Part with the digested pUC in different ratios</p>

<p>Ligation with T4 DNA Ligase Protocol (M020) from NEB;</p>

<p>Reaction set up on ice;</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>Amount [µl]           CFP PCR 1:5</th>
<th>Amount [µl]            CFP PCR 1:7</th>
<th>Amount [µl]        CFP PCR 1:10</th>
<th>Amount [µl]          Gel Ex 1:5</th>
</tr>
</thead>

<tbody>
<tr>
<td>T4 DNA Ligase Buffer x10</td>
<td>2</td>
<td>2</td>
<td></td>
<td></td>
</tr>

<tr>
<td>2</td>
<td>2</td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>Vector DNA</td>
<td>0,5</td>
<td>0,5</td>
<td></td>
<td></td>
</tr>

<tr>
<td>0,5</td>
<td>0,5</td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>Insert DNA</td>
<td>1,4</td>
<td>2</td>
<td></td>
<td></td>
</tr>

<tr>
<td>2,84</td>
<td>11,76   ( = 100 ng)</td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>Water</td>
<td>15,1</td>
<td>14,5</td>
<td></td>
<td></td>
</tr>

<tr>
<td>13,7</td>
<td>5,24</td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td>T4 Ligase</td>
<td>1</td>
<td>1</td>
<td></td>
<td></td>
</tr>

<tr>
<td>1</td>
<td>1</td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @RT for 10 min;</p>

<p>Heat inactivate @65 for 10 min;</p>

<p>With the sample <strong>Heat shock</strong> was performed after Evas Protocol;</p>

<p>The transformed cells and one sample being ony digested vector were seeded on Amp+ plates and placed into the incubator @ 37 °C overnight;</p>

<hr />

<p>Placed the colonies picked earlier by Eva into the Shaker overnight @ 37 °C and 230 rpm;</p>

<p><strong>Files used / generated</strong></p>

<p>In parallel designed Primer for the restriction digestion of the Constructs for the <em>E. coli</em> expression;</p>

<p>Designed them with the intent to open the pACYC184 vector with ClaI and SalI and accordingly also digest the constructs with the same enzymes to perform classical cloning;</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>- Mini Prep from the Overnight cultures in the Shaker (one could also look at the cells under the fluorescence microscope as successfull ligation + transformation should produce CFP);</p>

<p>- Gel electrophoresis thereof</p>

<p>- Heatshock into other Strains BL21, Star, Tuner;</p>

<p>- Check Plates in the Incubator, prepare ON cultures;</p>

<p><img src="/labjournal-entries/images/blank.gif" alt="&lt;Beschreibung&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-167" class="hidden">
      <label for="panel-167" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">CFP ligation with restrictionenzyme digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CFP ligation with restrictionenzyme digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p><strong>Plates of Heatshock (03.09.2019)</strong></p>

<p>4 small clonies detactable</p>

<p>Plate back to incubator</p>

<p>Later clone picking</p>

<p><strong>1% Agarose gel</strong></p>

<p>M (5µL) | 1 CFP 01. | 2 CFP 01. | puc19 Verdau 3.9. | - | - | M(5µL) |  PCR L | PCR R</p>

<p>For: 1 CFP 01., 2 CFP 01., puc19 Verdau 3.9. following composition was used:</p>

<p>5µL DNA, 3,3 µL Loading Dye, 11,7 µL MilliQ water</p>

<p>For: PCR R and PCR L following composition was used:</p>

<p>12µL DNA, 3,3 µL Loading Dye, 4,7 µL MilliQ water</p>

<p>Gel running running at 110V</p>

<p>No band for the vector -probably not enough DNA loaded on the gel</p>

<p>Cutting out the CFP from the PCR (PCR L was the probe and R the control)</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Plate was the wrong way round in the incubator</p>

<p>No negative control for the heatshock -clones might be just undigested vector</p>

<p><strong>insert picture from 4.9.19 here; name on labfolder:TS05405Sept42019.jpg</strong></p>

<table>
<thead>
<tr>
<th><strong>Aim/Task:</strong></th>
<th>Digest of new pUC19 (XbaI/EcoRI), PCR clean up of CFP, Gel extraction of CFP, Digest purified CFP, Ligation of pUC19 with CFP, Heatshock</th>
</tr>
</thead>

<tbody>
<tr>
<td><strong>Protocoll used:</strong></td>
<td></td>
</tr>

<tr>
<td><strong>Date:</strong></td>
<td>2019-09-04</td>
</tr>

<tr>
<td><strong>Name:</strong></td>
<td>Leon</td>
</tr>
</tbody>
</table>

<p><strong><em>Notes</em></strong></p>

<p><strong>Digestion of pUC19 with EcoRI and XbaI:</strong></p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>Amount [µl]</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>pUC19 DNa</td>
<td>2              (= 2µg &ndash;DNA conc. 100ng/µl)</td>
</tr>

<tr>
<td>EcoRI-HF</td>
<td>1</td>
</tr>

<tr>
<td>XbaI</td>
<td>1</td>
</tr>

<tr>
<td>Cut Smart Buffer x10</td>
<td>2</td>
</tr>

<tr>
<td>Water</td>
<td>14</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @37 °C for 20 min;</p>

<p>Inactivate @ 65 °C for 20 min;</p>

<hr />

<p><strong>PCR Purification:</strong></p>

<p>Purified PCR sample from 3.9.19 with the PCR Purification Kit and followed the Kits protocol;</p>

<p>Eluted DNA in 30 µl Buffer EB;</p>

<hr />

<p><strong>Gel Extraction:</strong></p>

<p>Extracted DNA from the Gel prepared earlier this DNA to extract the Measurement Part;</p>

<p>Used the QiaQuick Gel Extraction Kit and followed its protocol;</p>

<p>Weight of Gel fragment ~0,02g &ndash;60 µl Buffer QG for Gel solving;</p>

<p>Eluted the DNA in 35 µl Buffer EB;</p>

<hr />

<p><strong>DNA Measuring at Nanodrop:</strong></p>

<table>
<thead>
<tr>
<th>Name</th>
<th>ng/µl</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th><sup>260</sup>&frasl;<sub>230</sub></th>
</tr>
</thead>

<tbody>
<tr>
<td>PCR Clean Up CFP</td>
<td>45,6</td>
<td>0,912</td>
<td>0,500</td>
<td>1,82</td>
<td>1,69</td>
</tr>

<tr>
<td>Gel Ex CFP</td>
<td>8,5</td>
<td>0,169</td>
<td>0,085</td>
<td>2,00</td>
<td>0,02</td>
</tr>
</tbody>
</table>

<hr />

<p><strong>Digestion of the purified samples (Gel Ex CFP, PCR Clean Up CFP):</strong></p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>Amount [µl] PCR Purified</strong></th>
<th><strong>Amount [µl] Gel Ex CFP</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>CFP DNA</td>
<td>30 (= 1,37 µg)</td>
<td>35 (= 927,5 ng)</td>
</tr>

<tr>
<td>EcoRI-HF</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td>XbaI</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td>Cut Smart Buffer x10</td>
<td>5</td>
<td>5</td>
</tr>

<tr>
<td>Water</td>
<td>15</td>
<td>8</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>50 µl</strong></td>
<td><strong>50 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @37 °C for 20 min;</p>

<p>Inactivate @ 65 °C for 20 min;</p>

<p>Ligation of Measurement Part with the digested pUC in different ratios</p>

<p>Ligation with T4 DNA Ligase Protocol (M020) from NEB;</p>

<p>Reaction set up on ice;</p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>Amount [µl]           CFP PCR 1:5</strong></th>
<th><strong>Amount [µl]            CFP PCR 1:7</strong></th>
<th><strong>Amount [µl]</strong>        <strong>CFP PCR 1:10</strong></th>
<th><strong>Amount [µl]          Gel Ex 1:5</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>T4 DNA Ligase Buffer x10</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
</tr>

<tr>
<td>Vector DNA</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>Insert DNA</td>
<td>1,4</td>
<td>2</td>
<td>2,84</td>
<td>11,76   ( = 100 ng)</td>
</tr>

<tr>
<td>Water</td>
<td>15,1</td>
<td>14,5</td>
<td>13,7</td>
<td>5,24</td>
</tr>

<tr>
<td>T4 Ligase</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @RT for 10 min;</p>

<p>Heat inactivate @65 for 10 min;</p>

<p>With the sample <strong>Heat shock</strong> was performed after Evas Protocol;</p>

<p>The transformed cells and one sample being ony digested vector were seeded on Amp+ plates and placed into the incubator @ 37 °C overnight;</p>

<hr />

<p>Placed the colonies picked earlier by Eva into the Shaker overnight @ 37 °C and 230 rpm;</p>

<p><strong>Files used / generated</strong></p>

<p>In parallel designed Primer for the restriction digestion of the Constructs for the <em>E. coli</em> expression;</p>

<p>Designed them with the intent to open the pACYC184 vector with ClaI and SalI and accordingly also digest the constructs with the same enzymes to perform classical cloning;</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>- Mini Prep from the Overnight cultures in the Shaker (one could also look at the cells under the fluorescence microscope as successfull ligation + transformation should produce CFP);</p>

<p>- Gel electrophoresis thereof</p>

<p>- Heatshock into other Strains BL21, Star, Tuner;</p>

<p>- Check Plates in the Incubator, prepare ON cultures;</p>

<p><img src="/labjournal-entries/images/TS05405Sept42019.jpg" alt="&lt;CFP_gel&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-168" class="hidden">
      <label for="panel-168" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">CFP ligation with restrictionenzyme digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CFP ligation with restrictionenzyme digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p><strong>Plates of Heatshock (03.09.2019)</strong></p>

<p>4 small clonies detactable</p>

<p>Plate back to incubator</p>

<p>Later clone picking</p>

<p><strong>1% Agarose gel</strong></p>

<p>M (5µL) | 1 CFP 01. | 2 CFP 01. | puc19 Verdau 3.9. | - | - | M(5µL) |  PCR L | PCR R</p>

<p>For: 1 CFP 01., 2 CFP 01., puc19 Verdau 3.9. following composition was used:</p>

<p>5µL DNA, 3,3 µL Loading Dye, 11,7 µL MilliQ water</p>

<p>For: PCR R and PCR L following composition was used:</p>

<p>12µL DNA, 3,3 µL Loading Dye, 4,7 µL MilliQ water</p>

<p>Gel running running at 110V</p>

<p>No band for the vector -probably not enough DNA loaded on the gel</p>

<p>Cutting out the CFP from the PCR (PCR L was the probe and R the control)</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Plate was the wrong way round in the incubator</p>

<p>No negative control for the heatshock -clones might be just undigested vector</p>

<p><img src="/labjournal-entries/images/TS05405Sept42019.jpg" alt="&lt;Agarose_gel_cfp&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-169" class="hidden">
      <label for="panel-169" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              4
              
              <strong class="inline-block opacity-75 text-sm">(5)</strong>
              
          </p>
          <p class="inline-block">CFP ligation with restrictionenzyme digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CFP ligation with restrictionenzyme digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p><strong>Plates of Heatshock (03.09.2019)</strong></p>

<p>4 small clonies detactable</p>

<p>Plate back to incubator</p>

<p>Later clone picking</p>

<p><strong>1% Agarose gel</strong></p>

<p>M (5µL) | 1 CFP 01. | 2 CFP 01. | puc19 Verdau 3.9. | - | - | M(5µL) |  PCR L | PCR R</p>

<p>For: 1 CFP 01., 2 CFP 01., puc19 Verdau 3.9. following composition was used:</p>

<p>5µL DNA, 3,3 µL Loading Dye, 11,7 µL MilliQ water</p>

<p>For: PCR R and PCR L following composition was used:</p>

<p>12µL DNA, 3,3 µL Loading Dye, 4,7 µL MilliQ water</p>

<p>Gel running running at 110V</p>

<p>No band for the vector -probably not enough DNA loaded on the gel</p>

<p>Cutting out the CFP from the PCR (PCR L was the probe and R the control)</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Plate was the wrong way round in the incubator</p>

<p>No negative control for the heatshock -clones might be just undigested vector</p>

<p><img src="/labjournal-entries/images/TS05405Sept42019.jpg" alt="&lt;Agarose_gel_cfp&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-170" class="hidden">
      <label for="panel-170" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              5
              
          </p>
          <p class="inline-block">CFP ligation with restrictionenzyme digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CFP ligation with restrictionenzyme digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>ON Plates in the incubator were checked. The Control plate with only cut vector had 4 very small colonies on it. Whereas all Plates with Vector + Insert (pUC19 + CFP) had much higher numbers. Comparing the plates, the Gel Ex. Plate 1:5 had the least amount and within the PCR Plates a nice gradient was visible. The higher the Vector: Insert ratio was, the larger was the number of colonies.</p>

<p>With the fluorescence microscope of the AG Behrens the plates were screened for positive clones.</p>

<p>Number of CFp pos. clones was rather low &lt;10 %, positive clones were marked and later on picked to set up ON cultures;</p>

<p>Incubated ON cultures @ 37 °C, 230 rpm;</p>

<p>The ON cultures in the shaker were split. A 500 µl Aliquot was stored at -20 °C after spinning down the cells.
The other 4,5 ml were spinned down in the Megafuge (Mouse Lab), Miniprep not performed due to the low efficycy of the better ligation approach, so it is to be assumed that these clones are all negative;</p>

<p>Sequencing data was analyzed &ndash;all the sent in clones seem to be positive</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-171" class="hidden">
      <label for="panel-171" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              5
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">CFP ligation with restrictionenzyme digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CFP ligation with restrictionenzyme digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>ON Plates in the incubator were checked. The Control plate with only cut vector had 4 very small colonies on it. Whereas all Plates with Vector + Insert (pUC19 + CFP) had much higher numbers. Comparing the plates, the Gel Ex. Plate 1:5 had the least amount and within the PCR Plates a nice gradient was visible. The higher the Vector: Insert ratio was, the larger was the number of colonies.</p>

<p>With the fluorescence microscope of the AG Behrens the plates were screened for positive clones.</p>

<p>Number of CFp pos. clones was rather low &lt;10 %, positive clones were marked and later on picked to set up ON cultures;</p>

<p>Incubated ON cultures @ 37 °C, 230 rpm;</p>

<p>The ON cultures in the shaker were split. A 500 µl Aliquot was stored at -20 °C after spinning down the cells.
The other 4,5 ml were spinned down in the Megafuge (Mouse Lab), Miniprep not performed due to the low efficycy of the better ligation approach, so it is to be assumed that these clones are all negative;</p>

<p>Sequencing data was analyzed &ndash;all the sent in clones seem to be positive;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-172" class="hidden">
      <label for="panel-172" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              6
              
          </p>
          <p class="inline-block">estriction OH PCR with the E. coli Parts, Mini Prep of the CFP clones, Control Digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>estriction OH PCR with the E. coli Parts, Mini Prep of the CFP clones, Control Digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>To generate the necessary restriction sites for the classical cloning PCr was performed with the constructs for the E. coli expression.</p>

<table>
<thead>
<tr>
<th>Restriction OH PCR</th>
<th>K1</th>
<th>K2a</th>
<th>K2b</th>
<th>K3a</th>
<th>K3b</th>
<th>K4a</th>
<th>K4b</th>
<th>Water Control</th>
</tr>
</thead>

<tbody>
<tr>
<td>Q5 MM x2</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
</tr>

<tr>
<td>fw Primer</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
</tr>

<tr>
<td>re Primer</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
</tr>

<tr>
<td>DNA</td>
<td>0,5</td>
<td>0,5</td>
<td>2,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0</td>
</tr>

<tr>
<td>Water</td>
<td>9,5</td>
<td>9,5</td>
<td>7,5</td>
<td>9,5</td>
<td>9,5</td>
<td>9,5</td>
<td>9,5</td>
<td>10</td>
</tr>

<tr>
<td><strong>Total</strong></td>
<td><strong>25</strong></td>
<td>25</td>
<td>25</td>
<td>25</td>
<td>25</td>
<td>25</td>
<td>25</td>
<td>25</td>
</tr>
</tbody>
</table>

<p>The normal PCR Amplification Program was used in the PCR cycler, Samples were stored @ 4 °C;</p>

<p>Miniprep of the ON cultures that were now for 4 more hours in hte Shaker was performed;</p>

<p>Eluted the samples in 50 µl of EB Buffer;</p>

<p>DNA concentration of the samples was measured with the Nanodrop;</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>ng/µl</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th><sup>260</sup>&frasl;<sub>230</sub></th>
</tr>
</thead>

<tbody>
<tr>
<td>C1</td>
<td>77,25</td>
<td>1,545</td>
<td>0,706</td>
<td>2,19</td>
<td>2,28</td>
</tr>

<tr>
<td>C2</td>
<td>166,47</td>
<td>3,329</td>
<td>1,552</td>
<td>2,14</td>
<td>2.25</td>
</tr>

<tr>
<td>C3</td>
<td>50,54</td>
<td>1,011</td>
<td>0,519</td>
<td>1,95</td>
<td>1,82</td>
</tr>

<tr>
<td>C4</td>
<td>34,62</td>
<td>0,692</td>
<td>0,356</td>
<td>1,95</td>
<td>1,91</td>
</tr>

<tr>
<td>C5</td>
<td>80,80</td>
<td>1,616</td>
<td>0,782</td>
<td>2,07</td>
<td>1,99</td>
</tr>

<tr>
<td>C6</td>
<td>24,83</td>
<td>0,497</td>
<td>0,227</td>
<td>2,19</td>
<td>2,04</td>
</tr>

<tr>
<td>C7</td>
<td>33,04</td>
<td>0,661</td>
<td>0,308</td>
<td>2,14</td>
<td>2,08</td>
</tr>

<tr>
<td>C8</td>
<td>22,01</td>
<td>0,440</td>
<td>0,217</td>
<td>2,02</td>
<td>1,88</td>
</tr>
</tbody>
</table>

<p><strong>Test Digest</strong></p>

<p>Miniprep samples were digested with EcoRI HF and SalI</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>C1</th>
<th>C2</th>
<th>C3</th>
<th>C4</th>
<th>C5</th>
<th>C6</th>
<th>C7</th>
<th>C8</th>
</tr>
</thead>

<tbody>
<tr>
<td>SalI</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>EcoRI HF</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>Cut Smart x 10</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
</tr>

<tr>
<td>DNA</td>
<td>6,5</td>
<td>3</td>
<td>10</td>
<td>14,5</td>
<td>6,2</td>
<td>20</td>
<td>15,2</td>
<td>0</td>
</tr>

<tr>
<td>Water</td>
<td>15</td>
<td>18,5</td>
<td>11,5</td>
<td>7</td>
<td>15,3</td>
<td>1,5</td>
<td>6,3</td>
<td>21,5</td>
</tr>

<tr>
<td><strong>Total</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
</tr>
</tbody>
</table>

<p>Incubation @ 37 °C for 15 min;</p>

<p>Inactivation @ 65 °C for 20 min;</p>

<p><strong><em>Problems faced / possible error causes</em></strong></p>

<p>During the Miniprep a washing step was forgotten so the eluted DNA presumable also contained genomic DNA;
To assay this the samples were once more transferred onto purifiation säulchen to be washed and finally aeluted again;</p>

<p><strong><em>Pending tasks, next steps, comments for next people in the Lab</em></strong></p>

<ul>
<li><p>Run a Gel of the digested Miniprep samples</p></li>

<li><p>Run a Gel of the PCR samples</p></li>

<li><p>Prepare new competent Cell ( DH5a, Tuner, BL21, Star)</p></li>

<li><p>Heat Shock with the PCR samples (Tuner, Bl21, Star) and positive Miniprep Samples (Tuner, BL21, Star)</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-173" class="hidden">
      <label for="panel-173" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              6
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">estriction OH PCR with the E. coli Parts, Mini Prep of the CFP clones, Control Digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>estriction OH PCR with the E. coli Parts, Mini Prep of the CFP clones, Control Digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <pre><code>                                             |
</code></pre>

<p><strong><em>Notes</em></strong></p>

<p>To generate the necessary restriction sites for the classical cloning PCr was performed with the constructs for the E. coli expression.</p>

<table>
<thead>
<tr>
<th>Restriction OH PCR</th>
<th>K1</th>
<th>K2a</th>
<th>K2b</th>
<th>K3a</th>
<th>K3b</th>
<th>K4a</th>
<th>K4b</th>
<th>Water Control</th>
</tr>
</thead>

<tbody>
<tr>
<td>Q5 MM x2</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
<td>12,5</td>
</tr>

<tr>
<td>fw Primer</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
</tr>

<tr>
<td>re Primer</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
</tr>

<tr>
<td>DNA</td>
<td>0,5</td>
<td>0,5</td>
<td>2,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0</td>
</tr>

<tr>
<td>Water</td>
<td>9,5</td>
<td>9,5</td>
<td>7,5</td>
<td>9,5</td>
<td>9,5</td>
<td>9,5</td>
<td>9,5</td>
<td>10</td>
</tr>

<tr>
<td><strong>Total</strong></td>
<td><strong>25</strong></td>
<td>25</td>
<td>25</td>
<td>25</td>
<td>25</td>
<td>25</td>
<td>25</td>
<td>25</td>
</tr>
</tbody>
</table>

<p>The normal PCR Amplification Program was used in the PCR cycler, Samples were stored @ 4 °C;</p>

<hr />

<p>Miniprep of the ON cultures that were now for 4 more hours in hte Shaker was performed;</p>

<p>Eluted the samples in 50 µl of EB Buffer;</p>

<p>DNA concentration of the samples was measured with the Nanodrop;</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>ng/µl</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th><sup>260</sup>&frasl;<sub>230</sub></th>
</tr>
</thead>

<tbody>
<tr>
<td>C1</td>
<td>77,25</td>
<td>1,545</td>
<td>0,706</td>
<td>2,19</td>
<td>2,28</td>
</tr>

<tr>
<td>C2</td>
<td>166,47</td>
<td>3,329</td>
<td>1,552</td>
<td>2,14</td>
<td>2.25</td>
</tr>

<tr>
<td>C3</td>
<td>50,54</td>
<td>1,011</td>
<td>0,519</td>
<td>1,95</td>
<td>1,82</td>
</tr>

<tr>
<td>C4</td>
<td>34,62</td>
<td>0,692</td>
<td>0,356</td>
<td>1,95</td>
<td>1,91</td>
</tr>

<tr>
<td>C5</td>
<td>80,80</td>
<td>1,616</td>
<td>0,782</td>
<td>2,07</td>
<td>1,99</td>
</tr>

<tr>
<td>C6</td>
<td>24,83</td>
<td>0,497</td>
<td>0,227</td>
<td>2,19</td>
<td>2,04</td>
</tr>

<tr>
<td>C7</td>
<td>33,04</td>
<td>0,661</td>
<td>0,308</td>
<td>2,14</td>
<td>2,08</td>
</tr>

<tr>
<td>C8</td>
<td>22,01</td>
<td>0,440</td>
<td>0,217</td>
<td>2,02</td>
<td>1,88</td>
</tr>
</tbody>
</table>

<hr />

<p><strong>Test Digest</strong></p>

<p>Miniprep samples were digested with EcoRI HF and SalI</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>C1</th>
<th>C2</th>
<th>C3</th>
<th>C4</th>
<th>C5</th>
<th>C6</th>
<th>C7</th>
<th>C8</th>
</tr>
</thead>

<tbody>
<tr>
<td>SalI</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>EcoRI HF</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>Cut Smart x 10</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
<td>2,5</td>
</tr>

<tr>
<td>DNA</td>
<td>6,5</td>
<td>3</td>
<td>10</td>
<td>14,5</td>
<td>6,2</td>
<td>20</td>
<td>15,2</td>
<td>0</td>
</tr>

<tr>
<td>Water</td>
<td>15</td>
<td>18,5</td>
<td>11,5</td>
<td>7</td>
<td>15,3</td>
<td>1,5</td>
<td>6,3</td>
<td>21,5</td>
</tr>

<tr>
<td><strong>Total</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
<td><strong>25</strong></td>
</tr>
</tbody>
</table>

<p>Incubation @ 37 °C for 15 min;</p>

<p>Inactivation @ 65 °C for 20 min;</p>

<p><strong><em>Problems faced / possible error causes</em></strong></p>

<p>During the Miniprep a washing step was forgotten so the eluted DNA presumable also contained genomic DNA;
To assay this the samples were once more transferred onto purifiation säulchen to be washed and finally aeluted again;</p>

<p><strong><em>Pending tasks, next steps, comments for next people in the Lab</em></strong></p>

<p>- Run a Gel of the digested Miniprep samples</p>

<p>- Run a Gel of the PCR samples</p>

<p>- Prepare new competent Cell ( DH5a, Tuner, BL21, Star)</p>

<p>- Heat Shock with the PCR samples (Tuner, Bl21, Star) and positive Miniprep Samples (Tuner, BL21, Star)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-174" class="hidden">
      <label for="panel-174" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              6
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Control Gel of PCR and Mini, PCR Purification/Gel Extraction, New Amp&#43; Plates, Heatshock of CFP c2 into Star/BL21/Tuner</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control Gel of PCR and Mini, PCR Purification/Gel Extraction, New Amp&#43; Plates, Heatshock of CFP c2 into Star/BL21/Tuner</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann, Lena Schorr, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p><strong>Control Gel electrophoresis:</strong></p>

<p>A 1% Agarose Gel was prepared and the samples from the digested Mini Prep (6.7.) and the E. coli Restriction PCR (6.7.) were applied;</p>

<p>Gel 1:</p>

<table>
<thead>
<tr>
<th>Ladder</th>
<th>C1</th>
<th>C2</th>
<th>C3</th>
<th>C4</th>
<th>C5</th>
<th>C6</th>
<th>C7</th>
<th>C8</th>
</tr>
</thead>

<tbody>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

<p>Gel 2:</p>

<table>
<thead>
<tr>
<th>Ladder</th>
<th>K1</th>
<th>K2a</th>
<th>K2b</th>
<th>K3a</th>
<th>H20</th>
<th>H20</th>
<th>K4a</th>
<th>K4b</th>
<th>K3b</th>
</tr>
</thead>

<tbody>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

<p>Of the PCR 3 µl were used to create a 20 µl sample with loading dye and water;</p>

<p>Of the digested Mini Prep 25 µl were used together with 4,2 µl;</p>

<p>To the gel 15 µl of each sample were applied;</p>

<p><strong>PCR Purification of K2b, K3a, K3b, K4a, K4b:</strong></p>

<p>Performed PCR Purification according to the QiaQuick Purification Protocol and eluted the samples in 30 µl EB-Buffer;</p>

<p><strong>Gel extraction K1, K2a:</strong></p>

<p>For the Gel extraction another Gel of K1 and K2a with a higher sample concentration was prepared;</p>

<p>For K1 the band at approx 1,7 kb was extracted;</p>

<p>Construct 2a with the Sites for Restriction cloning is approx. 1400 bp long, to be safe though also the bad that was seen at the height of roughly 1200 bp was extracted and purified;</p>

<p>The samples were eluted in 50 µl EB Buffer;</p>

<p><strong>Cast new Amp+ Plates:</strong></p>

<p>New AMP Plates were necessary so 35g of LB-Agar Powder were mixed with 1l of MilliQ Water and mixed thoroughly.
Mixture was autoclaved and after sufficient cooling Amp (50 mg/ml) was added to reach a concentration of 50 µg/ml;</p>

<p>With the Broth new Plates were casted;</p>

<hr />

<p><strong>Heatshock of CFP c2 into Star/Bl21/Tuner:</strong></p>

<p>Heatshock with CFP+Puc19 clone 2 into Bl21, Star and Tuner with 2 µl DNA;</p>

<p>Eva Heatshock Protocol was used plus an extra Step with 200 µl SOC Medium after the 2 min incubation on ice;</p>

<p>Plated 200 µl of each sample onto Amp+ plates and stored the remaining 200 µl @ 4 °C;</p>

<p><strong>Digestion of pACYC184 and purified samples with ClaI, SalI-HF:</strong></p>

<p>pACYC184:</p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>Amount [µl]      pACYC184</strong></th>
<th><strong>Amount [µl]       PCR purified K2b, K3b ,K4a, K4b</strong></th>
<th><strong>Amount [µl]           Gel Ex. K1, K2a</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>5,32</td>
<td>15</td>
<td>40</td>
</tr>

<tr>
<td>ClaI</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td>SalI-HF</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td>Cut Smart Buffer x10</td>
<td>5</td>
<td>2,5</td>
<td>5</td>
</tr>

<tr>
<td>Water</td>
<td>44,86</td>
<td>4,5</td>
<td>4</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>50 µl</strong></td>
<td><strong>25 µl</strong></td>
<td><strong>50 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @37 °C for 15 min;</p>

<p>Inactivate @ 65 °C for 20 min;</p>

<p>To avoid uncut vector, that might reduce the efficacy of a following ligation Gel Extraction of the 3,6 kb long fragment was performed;</p>

<p>The sample was eluted in 30 µl Buffer EB;</p>

<p><strong>Ligation of digested and gel extracted pACYC184 with the digested the digested E. coli constructs (K1-K4b) in a 1:7 ratio (vector:insert):</strong></p>

<p>Before the Ligation the amount DNA amount was measured in the Nanodrop:</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>ng/µl</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th><sup>260</sup>&frasl;<sub>230</sub></th>
</tr>
</thead>

<tbody>
<tr>
<td>pACYC184</td>
<td>14,94</td>
<td>0,290</td>
<td>0,160</td>
<td>1,81</td>
<td>0,02</td>
</tr>

<tr>
<td>K1 dig</td>
<td>75,94</td>
<td>1,519</td>
<td>0,867</td>
<td>1,75</td>
<td>0,45</td>
</tr>

<tr>
<td>K2a 1 dig (oben)</td>
<td>127,04</td>
<td>2,541</td>
<td>1,624</td>
<td>1,56</td>
<td>0,44</td>
</tr>

<tr>
<td>K2a 2 dig (unten)</td>
<td>123,97</td>
<td>2,479</td>
<td>1,584</td>
<td>1,56</td>
<td>0,43</td>
</tr>

<tr>
<td>K2b dig</td>
<td>71,40</td>
<td>1,428</td>
<td>0,839</td>
<td>1,70</td>
<td>1,01</td>
</tr>

<tr>
<td>K3a dig</td>
<td>71,79</td>
<td>1,463</td>
<td>0,830</td>
<td>1,73</td>
<td>0,87</td>
</tr>

<tr>
<td>K3b dig</td>
<td>63,78</td>
<td>1,276</td>
<td>0,776</td>
<td>1,64</td>
<td>0,81</td>
</tr>

<tr>
<td>K4a dig</td>
<td>56.60</td>
<td>1,132</td>
<td>0,665</td>
<td>1,70</td>
<td>0,86</td>
</tr>

<tr>
<td>K4b dig</td>
<td>76,38</td>
<td>1,528</td>
<td>0,993</td>
<td>1,54</td>
<td>0,80</td>
</tr>
</tbody>
</table>

<p>With the results from the Nanidrop Measurement Ligation with T4 DNA Ligase Protocol (M020) from NEB was performed;</p>

<p>Reaction set up on ice;</p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>Amount [µl]           K1</strong></th>
<th><strong>Amount [µl]            K2a 1</strong></th>
<th><strong>Amount [µl]</strong>        <strong>K2a 2</strong></th>
<th><strong>Amount [µl]          K2b</strong></th>
<th><strong>Amount [µl]             K3a</strong></th>
<th><strong>Amount [µl]            K3b</strong></th>
<th><strong>Amount [µl]          K4a</strong></th>
<th><strong>Amount [µl]            K4b</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>T4 DNA Ligase Buffer x10</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
</tr>

<tr>
<td>Vector DNA</td>
<td>3,5</td>
<td>3,5</td>
<td>3,5</td>
<td>3,5</td>
<td>3,5</td>
<td>2,7</td>
<td>2,7</td>
<td>2,7</td>
</tr>

<tr>
<td>Insert DNA</td>
<td>4,6</td>
<td>2,75</td>
<td>2,8</td>
<td>3,5</td>
<td>3,5</td>
<td>5,5</td>
<td>6,2</td>
<td>4,6</td>
</tr>

<tr>
<td>Water</td>
<td>8,9</td>
<td>10,75</td>
<td>10,7</td>
<td>10</td>
<td>10</td>
<td>8</td>
<td>7,3</td>
<td>8,9</td>
</tr>

<tr>
<td>T4 Ligase</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Short Spin the Samples and incubate @RT for 10 min;</p>

<p>With the samples <strong>Heat shock</strong> was performed after Evas Protocol + SOC Step;</p>

<p>The 200µl of the ransformed cells were seeded on Amp+ plates and placed into the incubator @ 37 °C overnight;</p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Control Gel electrophoresis:</strong></p>

<p>The control gel electrophoresis showed that the digestion in most of the samples were successfull. In some (4,6,7,8) uncut DNA was visible;</p>

<p>Sample 5 and 8 do also not show the expected band at approx 0,7 kb;</p>

<p>All PCR samples showed bands at their expected heights;</p>

<p>Just K1 and K2a had a strong unspecific bands. This is why the correct band was extracted out of the Gel instead of PCR Purification like with the other samples;</p>

<p><strong><em>Problems faced / possible error causes</em></strong></p>

<p>Forgot to inactivate the Ligase @ 65 °C for 10 min;</p>

<p><strong>pirctures have to be inserted here:</strong></p>

<p>7.9_Restriction_PCR,_Miniprep_Gel.jpeg</p>

<p>7.9_Restriction_K1,_K2a_for_Gel_Ex.jpeg</p>

<p><img src="/labjournal-entries/images/7.9_Restriction_K1,_K2a_for_Gel_Ex.jpeg-with-annotations.png" alt="&lt;Restriction_gel&gt;" />
<img src="/labjournal-entries/images/7.9_Restriction_PCR,_Miniprep_Gel.jpeg-with-annotations.png" alt="&lt;Restriction_PCR&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-175" class="hidden">
      <label for="panel-175" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              6
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Control Gel of PCR and Mini, PCR Purification/Gel Extraction, New Amp&#43; Plates, Heatshock of CFP c2 into Star/BL21/Tuner</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control Gel of PCR and Mini, PCR Purification/Gel Extraction, New Amp&#43; Plates, Heatshock of CFP c2 into Star/BL21/Tuner</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann, Lena Schorr, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <pre><code>                                      |
</code></pre>

<p><strong><em>Notes</em></strong></p>

<p><strong>Control Gel electrophoresis:</strong></p>

<p>A 1% Agarose Gel was prepared and the samples from the digested Mini Prep (6.7.) and the E. coli Restriction PCR (6.7.) were applied;</p>

<p>Gel 1:</p>

<table>
<thead>
<tr>
<th>Ladder</th>
<th>C1</th>
<th>C2</th>
<th>C3</th>
<th>C4</th>
<th>C5</th>
<th>C6</th>
<th>C7</th>
<th>C8</th>
</tr>
</thead>

<tbody>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

<p>Gel 2:</p>

<table>
<thead>
<tr>
<th>Ladder</th>
<th>K1</th>
<th>K2a</th>
<th>K2b</th>
<th>K3a</th>
<th>H20</th>
<th>H20</th>
<th>K4a</th>
<th>K4b</th>
<th>K3b</th>
</tr>
</thead>

<tbody>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

<p>Of the PCR 3 µl were used to create a 20 µl sample with loading dye and water;</p>

<p>Of the digested Mini Prep 25 µl were used together with 4,2 µl;</p>

<p>To the gel 15 µl of each sample were applied;</p>

<hr />

<p><strong>PCR Purification of K2b, K3a, K3b, K4a, K4b:</strong></p>

<p>Performed PCR Purification according to the QiaQuick Purification Protocol and eluted the samples in 30 µl EB-Buffer;</p>

<hr />

<p><strong>Gel extraction K1, K2a:</strong></p>

<p>For the Gel extraction another Gel of K1 and K2a with a higher sample concentration was prepared;</p>

<p>For K1 the band at approx 1,7 kb was extracted;</p>

<p>Construct 2a with the Sites for Restriction cloning is approx. 1400 bp long, to be safe though also the bad that was seen at the height of roughly 1200 bp was extracted and purified;</p>

<p>The samples were eluted in 50 µl EB Buffer;</p>

<hr />

<p><strong>Cast new Amp+ Plates:</strong></p>

<p>New AMP Plates were necessary so 35g of LB-Agar Powder were mixed with 1l of MilliQ Water and mixed thoroughly.
Mixture was autoclaved and after sufficient cooling Amp (50 mg/ml) was added to reach a concentration of 50 µg/ml;</p>

<p>With the Broth new Plates were casted;</p>

<hr />

<p><strong>Heatshock of CFP c2 into Star/Bl21/Tuner:</strong></p>

<p>Heatshock with CFP+Puc19 clone 2 into Bl21, Star and Tuner with 2 µl DNA;</p>

<p>Eva Heatshock Protocol was used plus an extra Step with 200 µl SOC Medium after the 2 min incubation on ice;</p>

<p>Plated 200 µl of each sample onto Amp+ plates and stored the remaining 200 µl @ 4 °C;</p>

<hr />

<p><strong>Digestion of pACYC184 and purified samples with ClaI, SalI-HF:</strong></p>

<p>pACYC184:</p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>Amount [µl]      pACYC184</strong></th>
<th><strong>Amount [µl]       PCR purified K2b, K3b ,K4a, K4b</strong></th>
<th><strong>Amount [µl]           Gel Ex. K1, K2a</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>5,32</td>
<td>15</td>
<td>40</td>
</tr>

<tr>
<td>ClaI</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td>SalI-HF</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td>Cut Smart Buffer x10</td>
<td>5</td>
<td>2,5</td>
<td>5</td>
</tr>

<tr>
<td>Water</td>
<td>44,86</td>
<td>4,5</td>
<td>4</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>50 µl</strong></td>
<td><strong>25 µl</strong></td>
<td><strong>50 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @37 °C for 15 min;</p>

<p>Inactivate @ 65 °C for 20 min;</p>

<p>To avoid uncut vector, that might reduce the efficacy of a following ligation Gel Extraction of the 3,6 kb long fragment was performed;</p>

<p>The sample was eluted in 30 µl Buffer EB;</p>

<hr />

<p><strong>Ligation of digested and gel extracted pACYC184 with the digested the digested E. coli constructs (K1-K4b) in a 1:7 ratio (vector:insert):</strong></p>

<p>Before the Ligation the amount DNA amount was measured in the Nanodrop:</p>

<table>
<thead>
<tr>
<th>Name</th>
<th>ng/µl</th>
<th>260</th>
<th>280</th>
<th><sup>260</sup>&frasl;<sub>280</sub></th>
<th><sup>260</sup>&frasl;<sub>230</sub></th>
</tr>
</thead>

<tbody>
<tr>
<td>pACYC184</td>
<td>14,94</td>
<td>0,290</td>
<td>0,160</td>
<td>1,81</td>
<td>0,02</td>
</tr>

<tr>
<td>K1 dig</td>
<td>75,94</td>
<td>1,519</td>
<td>0,867</td>
<td>1,75</td>
<td>0,45</td>
</tr>

<tr>
<td>K2a 1 dig (oben)</td>
<td>127,04</td>
<td>2,541</td>
<td>1,624</td>
<td>1,56</td>
<td>0,44</td>
</tr>

<tr>
<td>K2a 2 dig (unten)</td>
<td>123,97</td>
<td>2,479</td>
<td>1,584</td>
<td>1,56</td>
<td>0,43</td>
</tr>

<tr>
<td>K2b dig</td>
<td>71,40</td>
<td>1,428</td>
<td>0,839</td>
<td>1,70</td>
<td>1,01</td>
</tr>

<tr>
<td>K3a dig</td>
<td>71,79</td>
<td>1,463</td>
<td>0,830</td>
<td>1,73</td>
<td>0,87</td>
</tr>

<tr>
<td>K3b dig</td>
<td>63,78</td>
<td>1,276</td>
<td>0,776</td>
<td>1,64</td>
<td>0,81</td>
</tr>

<tr>
<td>K4a dig</td>
<td>56.60</td>
<td>1,132</td>
<td>0,665</td>
<td>1,70</td>
<td>0,86</td>
</tr>

<tr>
<td>K4b dig</td>
<td>76,38</td>
<td>1,528</td>
<td>0,993</td>
<td>1,54</td>
<td>0,80</td>
</tr>
</tbody>
</table>

<p>With the results from the Nanidrop Measurement Ligation with T4 DNA Ligase Protocol (M020) from NEB was performed;</p>

<p>Reaction set up on ice;</p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>Amount [µl]           K1</strong></th>
<th><strong>Amount [µl]            K2a 1</strong></th>
<th><strong>Amount [µl]</strong>        <strong>K2a 2</strong></th>
<th><strong>Amount [µl]          K2b</strong></th>
<th><strong>Amount [µl]             K3a</strong></th>
<th><strong>Amount [µl]            K3b</strong></th>
<th><strong>Amount [µl]          K4a</strong></th>
<th><strong>Amount [µl]            K4b</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>T4 DNA Ligase Buffer x10</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
</tr>

<tr>
<td>Vector DNA</td>
<td>3,5</td>
<td>3,5</td>
<td>3,5</td>
<td>3,5</td>
<td>3,5</td>
<td>2,7</td>
<td>2,7</td>
<td>2,7</td>
</tr>

<tr>
<td>Insert DNA</td>
<td>4,6</td>
<td>2,75</td>
<td>2,8</td>
<td>3,5</td>
<td>3,5</td>
<td>5,5</td>
<td>6,2</td>
<td>4,6</td>
</tr>

<tr>
<td>Water</td>
<td>8,9</td>
<td>10,75</td>
<td>10,7</td>
<td>10</td>
<td>10</td>
<td>8</td>
<td>7,3</td>
<td>8,9</td>
</tr>

<tr>
<td>T4 Ligase</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Short Spin the Samples and incubate @RT for 10 min;</p>

<p>With the samples <strong>Heat shock</strong> was performed after Evas Protocol + SOC Step;</p>

<p>The 200µl of the ransformed cells were seeded on Amp+ plates and placed into the incubator @ 37 °C overnight;</p>

<p><strong>Calculations and Results</strong></p>

<p><strong>Control Gel electrophoresis:</strong></p>

<p>The control gel electrophoresis showed that the digestion in most of the samples were successfull. In some (4,6,7,8) uncut DNA was visible;</p>

<p>Sample 5 and 8 do also not show the expected band at approx 0,7 kb;</p>

<p>All PCR samples showed bands at their expected heights;</p>

<p>Just K1 and K2a had a strong unspecific bands. This is why the correct band was extracted out of the Gel instead of PCR Purification like with the other samples;</p>

<p><strong><em>Problems faced / possible error causes</em></strong></p>

<p>Forgot to inactivate the Ligase @ 65 °C for 10 min;</p>

<p><strong>pirctures have to be inserted here:</strong></p>

<p>7.9_Restriction_PCR,_Miniprep_Gel.jpeg</p>

<p>7.9_Restriction_K1,_K2a_for_Gel_Ex.jpeg</p>

<p><img src="/labjournal-entries/images/7.9_Restriction_K1,_K2a_for_Gel_Ex.jpeg-with-annotations.png" alt="&lt;Restriction_gel&gt;" />
<img src="/labjournal-entries/images/7.9_Restriction_PCR,_Miniprep_Gel.jpeg-with-annotations.png" alt="&lt;Restriction_PCR&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-176" class="hidden">
      <label for="panel-176" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              6
              
              <strong class="inline-block opacity-75 text-sm">(5)</strong>
              
          </p>
          <p class="inline-block">CFP ligation with restrictionenzyme digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CFP ligation with restrictionenzyme digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>Checked the ON cultures in the Shaker &ndash;A 500 ml Aliquot was frozen away after centrifuging and getting rid of the supernatant;</p>

<p>The other 4,5 ml were spinned down in hte Megafuge for 5 min, 4000 rpm &ndash;All the pellets were rather small so the Samples were put into the Shaker under the same conditions for a couple more hours;</p>

<p>The ordered Primer for Restriction Overhang PCR were diluted in TE-Buffer, so that the concentration is 100 µM;</p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>DNA [nmol]</strong></th>
<th><strong>TE-Buffer [µl]</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>K1 restriction fw</td>
<td>31,9</td>
<td>319</td>
</tr>

<tr>
<td>K1 restriction re</td>
<td>32,4</td>
<td>324</td>
</tr>

<tr>
<td>Restriction fw</td>
<td>99,9</td>
<td>999</td>
</tr>

<tr>
<td>Restriction re</td>
<td>106,8</td>
<td>1068</td>
</tr>
</tbody>
</table>

<p>Incubation @ 50 °C to elute the Primers;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-177" class="hidden">
      <label for="panel-177" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              6
              
              <strong class="inline-block opacity-75 text-sm">(6)</strong>
              
          </p>
          <p class="inline-block">CFP ligation with restrictionenzyme digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CFP ligation with restrictionenzyme digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>Checked the ON cultures in the Shaker &ndash;A 500 ml Aliquot was frozen away after centrifuging and getting rid of the supernatant;</p>

<p>The other 4,5 ml were spinned down in hte Megafuge for 5 min, 4000 rpm &ndash;Alle the pellets were rather small so the Samples were put into the Shaker under the same conditions for a couple more hours;</p>

<p>The ordered Primer for Restriction Overhang PCR were diluted in TE-Buffer, so that the concentration is 100 µM;</p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>DNA [nmol]</strong></th>
<th><strong>TE-Buffer [µl]</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>K1 restriction fw</td>
<td>31,9</td>
<td>319</td>
</tr>

<tr>
<td>K1 restriction re</td>
<td>32,4</td>
<td>324</td>
</tr>

<tr>
<td>Restriction fw</td>
<td>99,9</td>
<td>999</td>
</tr>

<tr>
<td>Restriction re</td>
<td>106,8</td>
<td>1068</td>
</tr>
</tbody>
</table>

<p>Incubation @ 50 °C to elute the Primers;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-178" class="hidden">
      <label for="panel-178" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
          </p>
          <p class="inline-block">CFP ligation with restrictionenzyme digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CFP ligation with restrictionenzyme digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Checked the Plates in the Incubator;</p>

<p>The Cells with the E.  coli Restriction constructs were seeded on the wrong plates AMp+ Plates instead of CAM plates;</p>

<p>The Cells with the Measurement Part (CFP) have grown nicely; Most colonies grew with the Tuner strain;</p>

<p>2l New LB Medium with 20g LB powder per Liter were prepared and autoclaved;</p>

<p>New Heatshock with the Rest of the Ligation samples (15 µl) into Tuner;</p>

<p>200 µl of the Heatshock samples were plated on CAM+ Plates and Placed in the incubator @ 37 °C; (Rest stored @ 4 °C)</p>

<p>Made 3 ON cultures with LB Amp+ (300 ml) one for each of the Measurement Part Plates (BL21, Tuner, Star);</p>

<p>Put them in the Shaker @ 37 °C, 230 rpm</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li><p>Prepare new competent cells (BL21, Star, Tuner, DH5a);</p></li>

<li><p>Check Plates in the Incubator and prepare cultures &ndash;Mini Prep;</p></li>

<li><p>Check Cultures in the Shaker and prepare for CFP Measuring;</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-179" class="hidden">
      <label for="panel-179" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">CFP ligation with restrictionenzyme digest</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>CFP ligation with restrictionenzyme digest</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>Checked the Plates in the Incubator;</p>

<p>The Cells with the E.  coli Restriction constructs were seeded on the wrong plates AMp+ Plates instead of CAM plates;</p>

<p>The Cells with the Measurement Part (CFP) have grown nicely; Most colonies grew with the Tuner strain;</p>

<p>2l New LB Medium with 20g LB powder per Liter were prepared and autoclaved;</p>

<p>New Heatshock with the Rest of the Ligation samples (15 µl) into Tuner;</p>

<p>200 µl of the Heatshock samples were plated on CAM+ Plates and Placed in the incubator @ 37 °C; (Rest stored @ 4 °C)</p>

<p>Made 3 ON cultures with LB Amp+ (300 ml) one for each of the Measurement Part Plates (BL21, Tuner, Star);</p>

<p>Put them in the Shaker @ 37 °C, 230 rpm</p>

<p><strong><em>Pending tasks, next steps, comments for next people in the Lab</em></strong></p>

<p>- Prepare new competent cells (BL21, Star, Tuner, DH5a);</p>

<p>- Check Plates in the Incubator and prepare cultures &ndash;Mini Prep;</p>

<p>- Check Cultures in the Shaker and prepare for CFP Measuring;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-180" class="hidden">
      <label for="panel-180" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              9
              
          </p>
          <p class="inline-block">colony PCR</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>colony PCR</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Preparation of labled Eppis with 1 mL LB with Chloramphenicol.</p>

<p>Preparation of Mastermixes for the colony PCR with: Q5 MM, Primer, Water like following:</p>

<table>
<thead>
<tr>
<th></th>
<th>K1</th>
<th>All other constructs</th>
</tr>
</thead>

<tbody>
<tr>
<td>Q5 MM</td>
<td>75 µL</td>
<td>475 µL</td>
</tr>

<tr>
<td>fw primer</td>
<td>7,5 µL</td>
<td>47,5 µL</td>
</tr>

<tr>
<td>re primer</td>
<td>7,5 µL</td>
<td>47,5 µL</td>
</tr>

<tr>
<td>H2O</td>
<td>60 µL</td>
<td>380 µL</td>
</tr>
</tbody>
</table>

<p>25 µL per colony master mix in the according PCR tube.</p>

<p>Picking of clones with a pipet. Putting the pipette in the PCR tube and then into the LB.</p>

<p>PCR program: Konstrukt amplification</p>

<p>Running gel on a 1% agarose gel.</p>

<p>5 µL Loading dye was added to the 25µL PCR and then 15µL loaded on the gel.</p>

<p>Running at 130 V.</p>

<p>Loading scheme of the gel:</p>

<p>Gel 1:</p>

<p>Marker | K1.1 | K1.2 | K1.3 | K1.4 | K1.5 | K2a.1 | K2a.2 | K2a.3 | K2a.4 | K2a.5 | K2a.1obere |K2a.2obere| K2a.3obere</p>

<p>Gel 2:</p>

<p>Marker| K2a.4obere | K2a.5obere | K2b.1 | K2b.2 | K2b.3 | K2b.4 | K3a.1 | K3a.2 | K3a.3 | K3a.4 | K3a.5 | K3b.1 | K3b.2</p>

<p>Gel 3:</p>

<p>Marker| K3b.3 | K3b.4 | K3b.5 | K4a.1| K4a.2| K4a.3| K4a.4|K4b.1|K4b.2|K4b.3|K4b.3|K4b.4|h20 PCR|milliq</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-181" class="hidden">
      <label for="panel-181" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              9
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">colony PCR</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>colony PCR</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Preparation of labled Eppis with 1 mL LB with Chloramphenicol.</p>

<p>Preparation of Mastermixes for the colony PCR with: Q5 MM, Primer, Water like following:</p>

<table>
<thead>
<tr>
<th></th>
<th>K1</th>
<th>All other constructs</th>
</tr>
</thead>

<tbody>
<tr>
<td>Q5 MM</td>
<td>75 µL</td>
<td>475 µL</td>
</tr>

<tr>
<td>fw primer</td>
<td>7,5 µL</td>
<td>47,5 µL</td>
</tr>

<tr>
<td>re primer</td>
<td>7,5 µL</td>
<td>47,5 µL</td>
</tr>

<tr>
<td>H2O</td>
<td>60 µL</td>
<td>380 µL</td>
</tr>
</tbody>
</table>

<p>25 µL per colony master mix in the according PCR tube.</p>

<p>Picking of clones with a pipet. Putting the pipette in the PCR tube and then into the LB.</p>

<p>PCR program: Konstrukt amplification</p>

<p>Running gel on a 1% agarose gel.</p>

<p>5 µL Loading dye was added to the 25µL PCR and then 15µL loaded on the gel.</p>

<p>Running at 130 V.</p>

<p>Loading scheme of the gel:</p>

<p>Gel 1:</p>

<p>Marker | K1.1 | K1.2 | K1.3 | K1.4 | K1.5 | K2a.1 | K2a.2 | K2a.3 | K2a.4 | K2a.5 | K2a.1obere |K2a.2obere| K2a.3obere</p>

<p>Gel 2:</p>

<p>Marker| K2a.4obere | K2a.5obere | K2b.1 | K2b.2 | K2b.3 | K2b.4 | K3a.1 | K3a.2 | K3a.3 | K3a.4 | K3a.5 | K3b.1 | K3b.2</p>

<p>Gel 3:</p>

<p>Marker| K3b.3 | K3b.4 | K3b.5 | K4a.1| K4a.2| K4a.3| K4a.4|K4b.1|K4b.2|K4b.3|K4b.3|K4b.4|h20 PCR|milliq</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-182" class="hidden">
      <label for="panel-182" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              10
              
          </p>
          <p class="inline-block">colony PCR</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>colony PCR</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p><strong>Colony PCR</strong></p>

<p>Preparation of labled Eppis with 1 mL LB with Chloramphenicol.</p>

<p>Preparation of Mastermixes for the colony PCR with: Q5 MM, Primers, Water as followed (3 colonies of each construct):</p>

<table>
<thead>
<tr>
<th></th>
<th>K1 (3 reactions)</th>
<th>All other constructs (21 reactions)</th>
</tr>
</thead>

<tbody>
<tr>
<td>Q5 MM</td>
<td>37,5 µL</td>
<td>262,5 µL</td>
</tr>

<tr>
<td>fw primer</td>
<td>3,75 µL</td>
<td>26,25 µL</td>
</tr>

<tr>
<td>re primer</td>
<td>3,75 µL</td>
<td>26,25 µL</td>
</tr>

<tr>
<td>H2O</td>
<td>30 µL</td>
<td>210 µL</td>
</tr>
</tbody>
</table>

<p>Primers:</p>

<ul>
<li>​    K1: &ldquo;K1 restriction fw&rdquo; and &ldquo;K1 restriction re&rdquo;</li>
<li>​    all other constructs: &ldquo;restriction fw&rdquo; and &ldquo;restriction re&rdquo;</li>
</ul>

<p>25 µL per colony master mix in the according PCR tube.</p>

<p>Picking of clones with a pipet tip</p>

<p>PCR program: Konstrukt amplification</p>

<p>​    modifcation: 1st step at 90°C 10 min</p>

<p>Gel electrophores (1% agarose, 130 V)</p>

<p>2 µL Loading dye was added to the 10 µL PCR and then loaded on the gel.</p>

<p>Loading scheme of the gel (2 x 14 pockets, 1mm wide):</p>

<p>First Row:</p>

<p>1 kb Ladder | K1.1 | K1.2 | K1.3 |K2a1.1 | K2a1.2 | K2a1.3 | K2b.1| K2b.2 | K2b.3 | K3a.1 | K3a.2 |   K3a.3</p>

<p>Second Row:</p>

<p>1 kb Ladder   K3b.1 | K3b.2 | K3b.3 | K4a.1 | K4a.2 | K4a.3 | K4b.1 | K4b.2 | K4b.3 | K2a2.1 | K2a2.2 | K2a.3</p>

<p><strong>Calculations and Results</strong></p>

<p>Weak bands</p>

<p>​    might be repeated with with more cycles!</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Didn&rsquo;t prepare 1 replicate in the MM, therefore the last tube had a little less volume than the others.</p>

<p>Switched poles during electrophoresis, therefore lost the first row.</p>

<p><strong>Mini Prep</strong></p>

<p>Miniprep of ON cultures of the colonies that were incubated after colony PCR (09.09.19)</p>

<p>Assessment of success using gel electrophoresis (1% gel, 130 V)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-183" class="hidden">
      <label for="panel-183" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              10
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">colony PCR</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>colony PCR</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Felix Pfister</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p><strong>Colony PCR</strong></p>

<p>Preparation of labled Eppis with 1 mL LB with Chloramphenicol.</p>

<p>Preparation of Mastermixes for the colony PCR with: Q5 MM, Primers, Water as followed (3 colonies of each construct):</p>

<table>
<thead>
<tr>
<th></th>
<th>K1 (3 reactions)</th>
<th>All other constructs (21 reactions)</th>
</tr>
</thead>

<tbody>
<tr>
<td>Q5 MM</td>
<td>37,5 µL</td>
<td>262,5 µL</td>
</tr>

<tr>
<td>fw primer</td>
<td>3,75 µL</td>
<td>26,25 µL</td>
</tr>

<tr>
<td>re primer</td>
<td>3,75 µL</td>
<td>26,25 µL</td>
</tr>

<tr>
<td>H2O</td>
<td>30 µL</td>
<td>210 µL</td>
</tr>
</tbody>
</table>

<p>Primers:</p>

<ul>
<li>​    K1: &ldquo;K1 restriction fw&rdquo; and &ldquo;K1 restriction re&rdquo;</li>
<li>​    all other constructs: &ldquo;restriction fw&rdquo; and &ldquo;restriction re&rdquo;</li>
</ul>

<p>25 µL per colony master mix in the according PCR tube.</p>

<p>Picking of clones with a pipet tip</p>

<p>PCR program: Konstrukt amplification</p>

<p>​    modifcation: 1st step at 90°C 10 min</p>

<p>Gel electrophores (1% agarose, 130 V)</p>

<p>2 µL Loading dye was added to the 10 µL PCR and then loaded on the gel.</p>

<p>Loading scheme of the gel (2 x 14 pockets, 1mm wide):</p>

<p>First Row:</p>

<p>1 kb Ladder | K1.1 | K1.2 | K1.3 |K2a1.1 | K2a1.2 | K2a1.3 | K2b.1| K2b.2 | K2b.3 | K3a.1 | K3a.2 |   K3a.3</p>

<p>Second Row:</p>

<p>1 kb Ladder   K3b.1 | K3b.2 | K3b.3 | K4a.1 | K4a.2 | K4a.3 | K4b.1 | K4b.2 | K4b.3 | K2a2.1 | K2a2.2 | K2a.3</p>

<p><strong>Calculations and Results</strong></p>

<p>Weak bands</p>

<p>​    might be repeated with with more cycles!</p>

<p><strong>Problems faced / possible error causes</strong></p>

<p>Didn&rsquo;t prepare 1 replicate in the MM, therefore the last tube had a little less volume than the others.</p>

<p>Switched poles during electrophoresis, therefore lost the first row.</p>

<p><strong>Mini Prep</strong></p>

<p>Miniprep of ON cultures of the colonies that were incubated after colony PCR (09.09.19)</p>

<p>Assessment of success using gel electrophoresis (1% gel, 130 V)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-184" class="hidden">
      <label for="panel-184" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              11
              
          </p>
          <p class="inline-block">Testing of miniprep from 10.09.</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Testing of miniprep from 10.09.</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>https://www.helmholtz-muenchen.de/fileadmin/PEPF/Protocols/Chemically-competent-cells.pdf</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie Wiedemann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<ul>
<li>Used prepared bacteria from -80 °C (DH5a, Star, BL21, Tuner)</li>
<li>defrosted them at RT, put in 125 ml LB Medium</li>
<li>incubated at 30 °C for 1h</li>
<li>checked oD until between 0.4 and 0.6</li>
<li>oD: DH5a: 0,564; Star: 0,654; BL21: 0,401; Tuner: 0,82</li>
<li>diluted Tuner to oD of 0,628</li>
<li>Following steps consitently on ice</li>
<li>split in 50ml falcons</li>
<li>incubate ion ice 10 min</li>
<li>spin in ultracentrifuge 10 min 4000rpm 4 °C</li>
<li>resuspend pellet in 25 ml CC-Puffer (already prepared)</li>
<li>incubate 10 min on ice</li>
<li>centrifuge 10 min 4000 rpm 4 °C</li>
<li>resuspend pellet in 4,65 ml CC-Puffer + 0,35 ml 15% DMSO per falcon</li>
<li>incubate on ice 10 min</li>
<li>aliquoted in 1,5 ml eppis 200 µl per eppi</li>
<li>freezed in -80 °C</li>
</ul>

<p><strong>Materials used</strong></p>

<p>CC-Puffer</p>

<p>DMSO</p>

<p>LB-Medium</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-185" class="hidden">
      <label for="panel-185" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              11
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Testing of miniprep from 10.09.</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Testing of miniprep from 10.09.</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Marie Wiedemann</td>
            </tr>
        </table>

        <div class="labjournal">
        <table>
<thead>
<tr>
<th><strong>Aim/Task:</strong></th>
<th>Produce chemically competent E.coli</th>
</tr>
</thead>

<tbody>
<tr>
<td><strong>Protocoll used:</strong></td>
<td></td>
</tr>

<tr>
<td><strong>Date, Site:</strong></td>
<td>11.09.19 , nfz</td>
</tr>

<tr>
<td><strong>Name:</strong></td>
<td>Marie</td>
</tr>
</tbody>
</table>

<p><a href="https://www.helmholtz-muenchen.de/fileadmin/PEPF/Protocols/Chemically-competent-cells.pdf">https://www.helmholtz-muenchen.de/fileadmin/PEPF/Protocols/Chemically-competent-cells.pdf</a></p>

<p><strong><em>Notes</em></strong></p>

<ul>
<li>Used prepared bacteria from -80 °C (DH5a, Star, BL21, Tuner)</li>
<li>defrosted them at RT, put in 125 ml LB Medium</li>
<li>incubated at 30 °C for 1h</li>
<li>checked oD until between 0.4 and 0.6</li>
<li>oD: DH5a: 0,564; Star: 0,654; BL21: 0,401; Tuner: 0,82</li>
<li>diluted Tuner to oD of 0,628</li>
<li>Following steps consitently on ice</li>
<li>split in 50ml falcons</li>
<li>incubate ion ice 10 min</li>
<li>spin in ultracentrifuge 10 min 4000rpm 4 °C</li>
<li>resuspend pellet in 25 ml CC-Puffer (already prepared)</li>
<li>incubate 10 min on ice</li>
<li>centrifuge 10 min 4000 rpm 4 °C</li>
<li>resuspend pellet in 4,65 ml CC-Puffer + 0,35 ml 15% DMSO per falcon</li>
<li>incubate on ice 10 min</li>
<li>aliquoted in 1,5 ml eppis 200 µl per eppi</li>
<li>freezed in -80 °C</li>
</ul>

<p><strong>Materials used</strong></p>

<p>CC-Puffer</p>

<p>DMSO</p>

<p>LB-Medium</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-186" class="hidden">
      <label for="panel-186" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              11
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Testing of miniprep from 10.09.</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Testing of miniprep from 10.09.</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Measurement of the miniprep samples from 10.09. at the nanodrop.</p>

<p>Concentration in ng/µL:</p>

<p>K1.1: 64,2; K1.2 168,7; K2a.1.1 61,1; K2a.1.2 179,4; K2a2.1 132,6; K2a2.2 88,9; K2b.1 201,4; K2b.2 152,0; K3a.1 191; K3a.2 173,1; K3b.1 189,5; K3b.2 175; K4a.1 91,9; K4a.2 72,5</p>

<p>K4b.1 73,5; K4b.2 166,2</p>

<p>Testdigestion with 500 ng DNA, 0,5 µL Cla-I and 0,5 µL SalI, 5 µL Cut smart ad 25 µL for 15 min at 37°C</p>

<p>Adding of 5 µL Loading Dye to the digest. Running a 0,8% gel with 5µL Marker and 15µL of the digestion samples at 130V.</p>

<p>Pipetting scheme was this:</p>

<p>Gel1:</p>

<p>M | K1.1 | K1.2 | K2a.1.1 | K2a.1.2 | K2a2.1 | K2a2.2 | K2b.1| K2b.2 | K3a.1| K3a.2 | K3b.1 | K3b.2 | K4a.1</p>

<p>Gel2:</p>

<p>M | K4a.2 | K4b.1 | K4b.2</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-187" class="hidden">
      <label for="panel-187" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              11
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Testing of miniprep from 10.09.</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Testing of miniprep from 10.09.</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Measurement of the miniprep samples from 10.09. at the nanodrop.</p>

<p>Concentration in ng/µL:</p>

<p>K1.1: 64,2; K1.2 168,7; K2a.1.1 61,1; K2a.1.2 179,4; K2a2.1 132,6; K2a2.2 88,9; K2b.1 201,4; K2b.2 152,0; K3a.1 191; K3a.2 173,1; K3b.1 189,5; K3b.2 175; K4a.1 91,9; K4a.2 72,5</p>

<p>K4b.1 73,5; K4b.2 166,2</p>

<p>Testdigestion with 500 ng DNA, 0,5 µL Cla-I and 0,5 µL SalI, 5 µL Cut smart ad 25 µL for 15 min at 37°C</p>

<p>Adding of 5 µL Loading Dye to the digest. Running a 0,8% gel with 5µL Marker and 15µL of the digestion samples at 130V.</p>

<p>Pipetting scheme was this:</p>

<p>Gel1:</p>

<p>M | K1.1 | K1.2 | K2a.1.1 | K2a.1.2 | K2a2.1 | K2a2.2 | K2b.1| K2b.2 | K3a.1| K3a.2 | K3b.1 | K3b.2 | K4a.1</p>

<p>Gel2:</p>

<p>M | K4a.2 | K4b.1 | K4b.2</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-188" class="hidden">
      <label for="panel-188" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              12
              
          </p>
          <p class="inline-block">Digestion of E.coli constructs, ligation of vector and constructs, Heat-Shock of CHO-constructs in DH5a, overhang PCR with bacterial constructs</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of E.coli constructs, ligation of vector and constructs, Heat-Shock of CHO-constructs in DH5a, overhang PCR with bacterial constructs</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Lena Schorr, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<ul>
<li>Digestion of paCYC and K1, K2a, K2b, K3a, K3b, K4a, K4b with ClaI and SmaI &ndash;wrong enzymes!!! (instead of SmaI, SalI should have been unsed)</li>
<li>Heat-Shock of constructs (K1-1, K1-2, K2a-1, K2a-2, K2b-1, K2b-2) in DH5a

<ul>
<li>with 0,5µl of Mini samples from 30.8.19</li>
<li>plating the constructs on Amp plates</li>
<li>storage in the incubator</li>
</ul></li>
<li>Overhang PCR with K1, K2a, K2b, K3a, K3b, K4a, K4b (from initial IDT constructs) and K2a, K2b, K3a, K4b from PCR clean up from 13.08.</li>
</ul>

<table>
<thead>
<tr>
<th></th>
<th>1x</th>
<th>16x</th>
</tr>
</thead>

<tbody>
<tr>
<td>Q5 high fidelity</td>
<td>25µl</td>
<td>400µl</td>
</tr>

<tr>
<td>10µM forward primer</td>
<td>2,5µl</td>
<td>40µl</td>
</tr>

<tr>
<td>10µM reverse primer</td>
<td>2,5µl</td>
<td>40µl</td>
</tr>

<tr>
<td>DNA template</td>
<td>1µl</td>
<td>-</td>
</tr>

<tr>
<td>nuclease free water</td>
<td>19µl</td>
<td>304µl</td>
</tr>
</tbody>
</table>

<ul>
<li><p>gel electrophoresis (0,8% and 1%)</p>

<ul>
<li>0,8%: ladder - K1 - K2a - K2b - K3a - K3b - K4a - K4b - H2O (from the initiall constructs)</li>
<li>1%: ladder - K2a - K2b - K3a - K3b - K4a - K4b - H20</li>
</ul></li>

<li><p>Preparation of overnight culture of heat shocked bacteria with LB and Amp</p></li>
</ul>

<p><img src="/labjournal-entries/images/Overhang_PCR_restriction.jpg" alt="&lt;Overhang_PCR_restriction&gt;" />
<img src="/labjournal-entries/images/overhang_PCR_restriction_2.jpg" alt="&lt;Overhang_PCR_restriction2&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-189" class="hidden">
      <label for="panel-189" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              12
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Digestion of E.coli constructs, ligation of vector and constructs, Heat-Shock of CHO-constructs in DH5a, overhang PCR with bacterial constructs</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Digestion of E.coli constructs, ligation of vector and constructs, Heat-Shock of CHO-constructs in DH5a, overhang PCR with bacterial constructs</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Lena Schorr, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<ul>
<li>Digestion of paCYC and K1, K2a, K2b, K3a, K3b, K4a, K4b with ClaI and SmaI &ndash;wrong enzymes!!! (instead of SmaI, SalI should have been unsed)</li>
<li>Heat-Shock of constructs (K1-1, K1-2, K2a-1, K2a-2, K2b-1, K2b-2) in DH5a

<ul>
<li>with 0,5µl of Mini samples from 30.8.19</li>
<li>plating the constructs on Amp plates</li>
<li>storage in the incubator</li>
</ul></li>
<li>Overhang PCR with K1, K2a, K2b, K3a, K3b, K4a, K4b (from initial IDT constructs) and K2a, K2b, K3a, K4b from PCR clean up from 13.08.</li>
</ul>

<table>
<thead>
<tr>
<th></th>
<th>1x</th>
<th>16x</th>
</tr>
</thead>

<tbody>
<tr>
<td>Q5 high fidelity</td>
<td>25µl</td>
<td>400µl</td>
</tr>

<tr>
<td>10µM forward primer</td>
<td>2,5µl</td>
<td>40µl</td>
</tr>

<tr>
<td>10µM reverse primer</td>
<td>2,5µl</td>
<td>40µl</td>
</tr>

<tr>
<td>DNA template</td>
<td>1µl</td>
<td>-</td>
</tr>

<tr>
<td>nuclease free water</td>
<td>19µl</td>
<td>304µl</td>
</tr>
</tbody>
</table>

<ul>
<li><p>gel electrophoresis (0,8% and 1%)</p>

<ul>
<li>0,8%: ladder - K1 - K2a - K2b - K3a - K3b - K4a - K4b - H2O (from the initiall constructs)</li>
<li>1%: ladder - K2a - K2b - K3a - K3b - K4a - K4b - H20</li>
</ul></li>

<li><p>Preparation of overnight culture of heat shocked bacteria with LB and Amp</p></li>
</ul>

<p><img src="/labjournal-entries/images/Overhang_PCR_restriction.jpg" alt="&lt;Overhang_PCR_restriction&gt;" />
<img src="/labjournal-entries/images/overhang_PCR_restriction_2.jpg" alt="&lt;Overhang_PCR_restriction2&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-190" class="hidden">
      <label for="panel-190" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              14
              
          </p>
          <p class="inline-block">Midi Prep of ON cultures (K1-1, K1-2, K2a-1, K2a-2, K2b-1, K2b-2); Preparation of new LB Medium, PCR purification of PCR samples from the 12.9.; inoculation of clones from the heatshock from the 13.9. in 5ml LB-Amp</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Midi Prep of ON cultures (K1-1, K1-2, K2a-1, K2a-2, K2b-1, K2b-2); Preparation of new LB Medium, PCR purification of PCR samples from the 12.9.; inoculation of clones from the heatshock from the 13.9. in 5ml LB-Amp</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Midi prep protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<ul>
<li><p>PCR purification o</p>

<ul>
<li><p>PCR samples from the 12.9</p></li>

<li><p>Elution in 150 µl MiliQ water</p></li>

<li><p>concentration:</p></li>
</ul></li>

<li><p>Preparation of 3 1l bottles LB medium</p></li>

<li><p>Midi Prep of the overnight cultures of</p></li>
</ul>

<p>K1-1, K1-2, K2a-1, K2a-2, K2b-1, K2b-2 (CHO constructs) in DH5a</p>

<ul>
<li>protocol ion labfolder at the 20.8.</li>

<li><p>eluation in 100µl MilliQ water</p></li>

<li><p>Inoculation of clones from the heatshock (13.9.) in 5ml LB-Amb</p>

<ul>
<li>over night in the shaker at 37°</li>
</ul></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>&ndash;&gt;Mini Prep of the overnight cultures</p>

<p>&ndash;&gt;gel-electrophoresis of the midi prep + from the current mini-prep</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-191" class="hidden">
      <label for="panel-191" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              14
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Midi Prep of ON cultures (K1-1, K1-2, K2a-1, K2a-2, K2b-1, K2b-2); Preparation of new LB Medium, PCR purification of PCR samples from the 12.9.; inoculation of clones from the heatshock from the 13.9. in 5ml LB-Amp</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Midi Prep of ON cultures (K1-1, K1-2, K2a-1, K2a-2, K2b-1, K2b-2); Preparation of new LB Medium, PCR purification of PCR samples from the 12.9.; inoculation of clones from the heatshock from the 13.9. in 5ml LB-Amp</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Midi prep protocol</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Lena Schorr</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<ul>
<li>PCR purification o</li>
</ul>

<p>f PCR samples from the 12.9</p>

<ul>
<li>Elution in 150 µl MiliQ water</li>

<li><p>concentration:</p></li>

<li><p>Preparation of 3 1l bottles LB medium</p></li>

<li><p>Midi Prep of the overnight cultures of</p></li>
</ul>

<p>K1-1, K1-2, K2a-1, K2a-2, K2b-1, K2b-2 (CHO constructs) in DH5a</p>

<ul>
<li>protocol ion labfolder at the 20.8.</li>

<li><p>eluation in 100µl MilliQ water</p></li>

<li><p>Inoculation of clones from the heatshock (13.9.) in 5ml LB-Amb</p>

<ul>
<li>over night in the shaker at 37°</li>
</ul></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>&ndash;&gt;Mini Prep of the overnight cultures</p>

<p>&ndash;&gt;gel-electrophoresis of the midi prep + from the current mini-prep</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-192" class="hidden">
      <label for="panel-192" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
          </p>
          <p class="inline-block">iniprep of ON cultures from  14.09.19 &#43; Gel electrophoresis &#43; Gel electrophoresis from Midiprep from 14.09.19</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>iniprep of ON cultures from  14.09.19 &#43; Gel electrophoresis &#43; Gel electrophoresis from Midiprep from 14.09.19</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>ExtractMe Plasmid Mini Kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<ul>
<li><p>Miniprep of over night cultures from 14.09.19 using &ldquo;ExtractMe Plasmid Mini Kit&rdquo;</p>

<ul>
<li>Elution in 60 µL (50µL left after taking 10 µL for electrophoresis)</li>
</ul></li>

<li><p>Preparation of Gel electrophoresis</p>

<ul>
<li>Dilution of Midiprep samples from 14.09.19

<br /></li>
</ul>

<p>1:100</p>

<p>to avoid extrem bright fluorescence that might disturb the picture of the upcoming gel electrophoresis (samples have a DNA conc. of up to 5700 ng/µL)</p>

<ul>
<li><p>10 µL sample + 990 µL H2O</p></li>

<li><p>Sample &ldquo;Midi K2-a-2&rdquo; diluted 1:10 since its DNA conentration was only 337  ng/µL)</p></li>

<li><p>10 µL sample + 90 µL H2O</p></li>
</ul></li>

<li><p>Gel electrophoresis in 0,8% agarose gel</p>

<ul>
<li>Loading scheme: 1kb ladder | Mini K1-1 | Mini K1-2 | Mini K2a-1 | Mini K2a-2 | Mini K2b-1 | Mini K2b-2 | Midi K1-1 | Midi K1-2 | Midi K2a-1 | Midi K2a-2 | Midi K2b-1 | Midi K2b-2</li>
</ul></li>
</ul>

<p><img src="/labjournal-entries/images/gel_photo_14.09..JPG" alt="&lt;Miniprep_Gel_electrophoresis&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-193" class="hidden">
      <label for="panel-193" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">iniprep of ON cultures from  14.09.19 &#43; Gel electrophoresis &#43; Gel electrophoresis from Midiprep from 14.09.19</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>iniprep of ON cultures from  14.09.19 &#43; Gel electrophoresis &#43; Gel electrophoresis from Midiprep from 14.09.19</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>ExtractMe Plasmid Mini Kit</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<ul>
<li><p>Miniprep of over night cultures from 14.09.19 using &ldquo;ExtractMe Plasmid Mini Kit&rdquo;</p>

<ul>
<li>Elution in 60 µL (50µL left after taking 10 µL for electrophoresis)</li>
</ul></li>

<li><p>Preparation of Gel electrophoresis</p>

<ul>
<li>Dilution of Midiprep samples from 14.09.19

<br /></li>
</ul>

<p>1:100</p>

<p>to avoid extrem bright fluorescence that might disturb the picture of the upcoming gel electrophoresis (samples have a DNA conc. of up to 5700 ng/µL)</p>

<ul>
<li><p>10 µL sample + 990 µL H2O</p></li>

<li><p>Sample &ldquo;Midi K2-a-2&rdquo; diluted 1:10 since its DNA conentration was only 337  ng/µL)</p></li>

<li><p>10 µL sample + 90 µL H2O</p></li>
</ul></li>

<li><p>Gel electrophoresis in 0,8% agarose gel</p>

<ul>
<li>Loading scheme: 1kb ladder | Mini K1-1 | Mini K1-2 | Mini K2a-1 | Mini K2a-2 | Mini K2b-1 | Mini K2b-2 | Midi K1-1 | Midi K1-2 | Midi K2a-1 | Midi K2a-2 | Midi K2b-1 | Midi K2b-2</li>
</ul></li>
</ul>

<p><img src="/labjournal-entries/images/gel_photo_14.09..JPG" alt="&lt;Miniprep_Gel_electrophoresis&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-194" class="hidden">
      <label for="panel-194" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              18
              
          </p>
          <p class="inline-block">Control Digest of MidiPrep samples (14.9) and Mini Prep samples (15.9); Prepare samples for Kiel</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control Digest of MidiPrep samples (14.9) and Mini Prep samples (15.9); Prepare samples for Kiel</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>Digest of MidiPrep samples (14.9) and Mini Prep samples (15.9) with EcoRI-HF and NHeI-HF to check properly check the samples;</p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>Mini Prep samples [µl]</strong></th>
<th><strong>Midi Prep samples [µl]</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>2</td>
<td>0,5</td>
</tr>

<tr>
<td>EcoRI-HF</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>NheI-HF</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>Cut Smart Buffer x10</td>
<td>2</td>
<td>2</td>
</tr>

<tr>
<td>Water</td>
<td>15</td>
<td>1,5</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @37 °C for 20 min;</p>

<p>Inactivate @ 65 °C for 20 min;</p>

<p>Prepare 0,8 % Agarose Gel to examine digested samples;</p>

<p><strong>Loading order:</strong></p>

<p>| Ladder | Midi K1-1 | Midi K1-2 | Midi K2a- 1 | Midi K2a-2 | Midi K2b-1 | Midi K2b-2 | Mini K1-1 | Mini K1-2 | Mini K2a-1 | Mini K2a-2 | Mini K2b-1 | Mini K2b-2 | Ladder |</p>

<p>Prepared 100 µg of the Midi Prep samples K1-2, K2a-1, K2b-1 for Kiel and sent them there via Post;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-195" class="hidden">
      <label for="panel-195" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              18
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Control Digest of MidiPrep samples (14.9) and Mini Prep samples (15.9); Prepare samples for Kiel</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Control Digest of MidiPrep samples (14.9) and Mini Prep samples (15.9); Prepare samples for Kiel</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>Digest of MidiPrep samples (14.9) and Mini Prep samples (15.9) with EcoRI-HF and NHeI-HF to check properly check the samples;</p>

<table>
<thead>
<tr>
<th><strong>Name</strong></th>
<th><strong>Mini Prep samples [µl]</strong></th>
<th><strong>Midi Prep samples [µl]</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>DNA</td>
<td>2</td>
<td>0,5</td>
</tr>

<tr>
<td>EcoRI-HF</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>NheI-HF</td>
<td>0,5</td>
<td>0,5</td>
</tr>

<tr>
<td>Cut Smart Buffer x10</td>
<td>2</td>
<td>2</td>
</tr>

<tr>
<td>Water</td>
<td>15</td>
<td>1,5</td>
</tr>

<tr>
<td><strong>Total amount</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubate @37 °C for 20 min;</p>

<p>Inactivate @ 65 °C for 20 min;</p>

<p>Prepare 0,8 % Agarose Gel to examine digested samples;</p>

<p><strong>Loading order:</strong></p>

<table>
<thead>
<tr>
<th>Ladder</th>
<th>Midi K1-1</th>
<th>Midi K1-2</th>
<th>Midi K2a- 1</th>
<th>Midi K2a-2</th>
<th>Midi K2b-1</th>
<th>Midi K2b-2</th>
<th>Mini K1-1</th>
<th>Mini K1-2</th>
<th>Mini K2a-1</th>
<th>Mini K2a-2</th>
<th>Mini K2b-1</th>
<th>Mini K2b-2</th>
<th>Ladder</th>
</tr>
</thead>

<tbody>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

<p>Prepared 100 µg of the Midi Prep samples K1-2, K2a-1, K2b-1 for Kiel and sent them there via Post;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-196" class="hidden">
      <label for="panel-196" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              22
              
          </p>
          <p class="inline-block">Splitting of the 293T cells</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Splitting of the 293T cells</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes:</strong></p>

<p>-293T cells were not properly attached
-Probably most of them died</p>

<p><strong>Work done:</strong></p>

<p>-Mediumchange of the cells</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Check if cells are growing</p>

<p>Getting new cells and culture them proberly :)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-197" class="hidden">
      <label for="panel-197" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              22
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Splitting of the 293T cells</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Splitting of the 293T cells</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p>Notes:
-293T cells were not properly attached
-Probably most of them died</p>

<p>Work done:</p>

<p>-Mediumchange of the cells</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Check if cells are growing</p>

<p>Getting new cells and culture them proberly :)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-198" class="hidden">
      <label for="panel-198" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              23
              
          </p>
          <p class="inline-block">Splitting 239T cells</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Splitting 239T cells</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Check HEKs in the Incubator &ndash;All of them died</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Get new HEKs</p>

<p>Proceed with Measurement Part</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-199" class="hidden">
      <label for="panel-199" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              23
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Splitting 239T cells</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Splitting 239T cells</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>Check HEKs in the Incubator &ndash;All of them died</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Get new HEKs</p>

<p>Proceed with Measurement Part</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-200" class="hidden">
      <label for="panel-200" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              24
              
          </p>
          <p class="inline-block">Splitting 239T cells</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Splitting 239T cells</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Repeat the IPTG</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Check if cells are growing</p>

<p>Getting new cells and culture them properly</p>

<p><strong>CFP part:</strong></p>

<p>Cheking plates in the incubator</p>

<p>Plenty colonies in Tuner but not BL21 and STAR (Those were the &ldquo;new&rdquo; competent cells)</p>

<p>=BL21 and STAR not properly competent!!</p>

<p>New heatshock with old stock of competent bacs (STAR and BL21) acording to Evas protocol</p>

<p>HEK:</p>

<p>Check the flasks in the incubator: Live and happy ~ 50% confluent</p>

<p>Seewing of 4 x 10^6 cells in a plate (10 mL) like following.</p>

<ol>
<li><p>Discard medium</p></li>

<li><p>Washing with 3 mL Trypsin</p></li>

<li><p>Incubation with 3 mL trypsin</p></li>

<li><p>When cells were detached mixing with 7 mL Medium</p></li>

<li><p>Counting in neubauer chamber (total cell number 10,2 x 10^6 cells)</p></li>

<li><p>8 mL cell mix in the plate + 2 mL medium</p></li>
</ol>

<p>Rest of the cells back in the flask, ad 20 mL with medium</p>

<p>E. coli:</p>

<p>Digest of pACYC184: 5µL cutsmart, 11µL DNA, 1µL Cla-I, 1µL Sal-I</p>

<p>Water was not added! (WRONG)</p>

<p>Digest of the constructs which were already digested on the 07.09. but not for 1 h just 15 min.</p>

<p>1 h digest with Cla-I and Sal-I (each 0,5µL)</p>

<p>Ligation</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-201" class="hidden">
      <label for="panel-201" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              24
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Splitting 239T cells</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Splitting 239T cells</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Martin Borgmann, Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<ul>
<li>Repeat the IPTG</li>
<li></li>
</ul>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<p>Check if cells are growing</p>

<p>Getting new cells and culture them properly</p>

<p><strong>CFP part:</strong></p>

<p>Cheking plates in the incubator</p>

<p>Plenty colonies in Tuner but not BL21 and STAR (Those were the &ldquo;new&rdquo; competent cells)</p>

<p>=BL21 and STAR not properly competent!!</p>

<p>New heatshock with old stock of competent bacs (STAR and BL21) acording to Evas protocol</p>

<p>HEK:</p>

<p>Check the flasks in the incubator: Live and happy ~ 50% confluent</p>

<p>Seewing of 4 x 10^6 cells in a plate (10 mL) like following.</p>

<p>\1. Discard medium</p>

<p>\2. Washing with 3 mL Trypsin</p>

<p>\3. Incubation with 3 mL trypsin</p>

<p>\4. When cells were detached mixing with 7 mL Medium</p>

<p>\5. Counting in neubauer chamber (total cell number 10,2 x 10^6 cells)</p>

<p>\6. 8 mL cell mix in the plate + 2 mL medium</p>

<p>Rest of the cells back in the flask, ad 20 mL with medium</p>

<p>Coli:</p>

<p>Digest of pACYC184: 5µL cutsmart, 11µL DNA, 1µL Cla-I, 1µL Sal-I</p>

<p>Water was not added! (WRONG)</p>

<p>Digest of the constructs which were already digested on the 07.09. but not for 1 h just 15 min.</p>

<p>1 h digest with Cla-I and Sal-I (each 0,5µL)</p>

<p>Ligation</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-202" class="hidden">
      <label for="panel-202" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              25
              
          </p>
          <p class="inline-block">HEK transfer, vector Digestion, Ligation, Heatshock</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>HEK transfer, vector Digestion, Ligation, Heatshock</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>CFP part:</p>

<p>Checking plates in the incubator</p>

<p>Plenty colonies in Tuner but not BL21 and STAR (Those were the &ldquo;new&rdquo; competent cells)</p>

<p>=BL21 and STAR not properly competent!!</p>

<p>New heatshock with old stock of competent bacs (STAR and BL21) acording to Evas protocol</p>

<p>HEK:</p>

<p>Check the flasks in the incubator: Live and happy ~ 50% confluent</p>

<p>Seeding of 4 x 10^6 cells in a cell culture plate (10 ml) like following.</p>

<p>\1. Discard medium</p>

<p>\2. Washing with 3 ml trypsin</p>

<p>\3. Incubation with 3 ml trypsin</p>

<p>\4. When cells are detached mixing with 7 ml Medium</p>

<p>\5. Counting in neubauer chamber (total cell number 10,2 x 10^6 cells)</p>

<p>\6. 8 ml cell mix in the plate + 2 ml medium</p>

<p>Rest of the cells back in the flask, ad 20 ml with medium</p>

<p>Coli:</p>

<p>Digest of pACYC184: 5µL cutsmart, 11µL DNA, 1µL Cla-I, 1µL Sal-I</p>

<p>Water was not added! (WRONG);</p>

<p>Incubation @37 °C for 1h and following inactivation @65 °C for 20 min;</p>

<p>Proceeded nonetheless and added water aftewards to have a convenient volume of 50 µl</p>

<p>Digest of the constructs (K1 - K4b) which were already digested on the 07.09.19. ClaI is not a time saver qualified enzyme and thus the digestion time of 15 minutes might have not been sufficient.
1 h digest with Cla-I and Sal-I (each 0,5µL) in a volume of 20 µl @37 °C;</p>

<p>Inactivation @65 °C for 20 min;</p>

<p>With the sample concentration determined on 7.9 the new sample concentration was calculated, as the digest altered the volume;</p>

<p>Sample Concentration:</p>

<table>
<thead>
<tr>
<th><strong>K1</strong></th>
<th><strong>K2a</strong></th>
<th><strong>K2b</strong></th>
<th><strong>K3a</strong></th>
<th><strong>K3b</strong></th>
<th><strong>K4a</strong></th>
<th><strong>K4b</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>67.5 ng</td>
<td>110 ng</td>
<td>58,4 ng</td>
<td>71,8 ng</td>
<td>63,8 ng</td>
<td>56,6 ng</td>
<td>62,9 ng</td>
</tr>
</tbody>
</table>

<p>Only 6 Chloramphenicol plates were left in the fridge, so ligation was only performed with K1, K2a and K2b;</p>

<p>Amounts necessary for the ligation were calculated for all constructs;</p>

<table>
<thead>
<tr>
<th><strong>Construct</strong></th>
<th><strong>1:10 ng</strong></th>
<th><strong>1:10 µl</strong></th>
<th><strong>1:15 ng</strong></th>
<th><strong>1:15 µl</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td><strong>K1</strong></td>
<td>207,4</td>
<td>3</td>
<td>311,1</td>
<td>4,5</td>
</tr>

<tr>
<td><strong>K2a</strong></td>
<td>151,<strong>4</strong></td>
<td>1,4</td>
<td>227,1</td>
<td>2,1</td>
</tr>

<tr>
<td><strong>K2b</strong></td>
<td>144,2</td>
<td>2,46</td>
<td>216,3</td>
<td>3,7</td>
</tr>

<tr>
<td><strong>K3a</strong></td>
<td>105,6</td>
<td>1,8</td>
<td>158,4</td>
<td>2,7</td>
</tr>

<tr>
<td><strong>K3b</strong></td>
<td>100</td>
<td>1,9</td>
<td>150</td>
<td>2,85</td>
</tr>

<tr>
<td><strong>K4a</strong></td>
<td>105,6</td>
<td>2,3</td>
<td>158,4</td>
<td>3,45</td>
</tr>

<tr>
<td><strong>K4b</strong></td>
<td>100,6</td>
<td>1,6</td>
<td>150,9</td>
<td>2,4</td>
</tr>
</tbody>
</table>

<p>Ligation</p>

<table>
<thead>
<tr>
<th></th>
<th><strong>K1</strong></th>
<th></th>
<th><strong>K2a</strong></th>
<th></th>
<th><strong>K2b</strong></th>
<th></th>
</tr>
</thead>

<tbody>
<tr>
<td></td>
<td><strong>1:10</strong></td>
<td><strong>1:15</strong></td>
<td><strong>1:10</strong></td>
<td><strong>1:15</strong></td>
<td><strong>1:10</strong></td>
<td><strong>1:15</strong></td>
</tr>

<tr>
<td><strong>DNA</strong></td>
<td>3</td>
<td>4,5</td>
<td>1,4</td>
<td>2,1</td>
<td>2,46</td>
<td>3,7</td>
</tr>

<tr>
<td><strong>Vector</strong></td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
</tr>

<tr>
<td><strong>Ligase Buffer x10</strong></td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
</tr>

<tr>
<td><strong>Ligase</strong></td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td><strong>H20</strong></td>
<td>12,75</td>
<td>11,25</td>
<td>14,35</td>
<td>13,65</td>
<td>13,3</td>
<td>12,05</td>
</tr>

<tr>
<td><strong>Total</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubation @RT for 20 min;</p>

<p>Inactivation @65 °C for 20 min;</p>

<p>Perform Heatshock after Evas Protocol;</p>

<p>Heatshock samples were not plated on plates, as the remaining Chloramphenicol<strong>+</strong> were found to be contaminated;
Samples were therefor stored @4 °C;</p>

<p><strong><em>Pending tasks, next steps, comments for next people in the Lab</em></strong></p>

<p>- Produce new Chloramph Plates</p>

<p>- Test Transfection of HEK293T cells with K1</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-203" class="hidden">
      <label for="panel-203" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              25
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">HEK transfer, vector Digestion, Ligation, Heatshock</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>HEK transfer, vector Digestion, Ligation, Heatshock</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>CFP part:</p>

<p>Checking plates in the incubator</p>

<p>Plenty colonies in Tuner but not BL21 and STAR (Those were the &ldquo;new&rdquo; competent cells)</p>

<p>=BL21 and STAR not properly competent!!</p>

<p>New heatshock with old stock of competent bacs (STAR and BL21) acording to Evas protocol</p>

<p>HEK:</p>

<p>Check the flasks in the incubator: Live and happy ~ 50% confluent</p>

<p>Seeding of 4 x 10^6 cells in a cell culture plate (10 ml) like following.</p>

<ol>
<li><p>Discard medium</p></li>

<li><p>Washing with 3 ml trypsin</p></li>

<li><p>Incubation with 3 ml trypsin</p></li>

<li><p>When cells are detached mixing with 7 ml Medium</p></li>

<li><p>Counting in neubauer chamber (total cell number 10,2 x 10^6 cells)</p></li>

<li><p>8 ml cell mix in the plate + 2 ml medium</p></li>
</ol>

<p>Rest of the cells back in the flask, ad 20 ml with medium</p>

<p>E. coli:</p>

<p>Digest of pACYC184: 5µL cutsmart, 11µL DNA, 1µL Cla-I, 1µL Sal-I</p>

<p>Water was not added! (WRONG);</p>

<p>Incubation @37 °C for 1h and following inactivation @65 °C for 20 min;</p>

<p>Proceeded nonetheless and added water aftewards to have a convenient volume of 50 µl</p>

<p>Digest of the constructs (K1 - K4b) which were already digested on the 07.09.19. ClaI is not a time saver qualified enzyme and thus the digestion time of 15 minutes might have not been sufficient.
1 h digest with Cla-I and Sal-I (each 0,5µL) in a volume of 20 µl @37 °C;</p>

<p>Inactivation @65 °C for 20 min;</p>

<p>With the sample concentration determined on 7.9 the new sample concentration was calculated, as the digest altered the volume;</p>

<p>Sample Concentration:</p>

<table>
<thead>
<tr>
<th><strong>K1</strong></th>
<th><strong>K2a</strong></th>
<th><strong>K2b</strong></th>
<th><strong>K3a</strong></th>
<th><strong>K3b</strong></th>
<th><strong>K4a</strong></th>
<th><strong>K4b</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td>67.5 ng</td>
<td>110 ng</td>
<td>58,4 ng</td>
<td>71,8 ng</td>
<td>63,8 ng</td>
<td>56,6 ng</td>
<td>62,9 ng</td>
</tr>
</tbody>
</table>

<p>Only 6 Chloramphenicol plates were left in the fridge, so ligation was only performed with K1, K2a and K2b;</p>

<p>Amounts necessary for the ligation were calculated for all constructs;</p>

<table>
<thead>
<tr>
<th><strong>Construct</strong></th>
<th><strong>1:10 ng</strong></th>
<th><strong>1:10 µl</strong></th>
<th><strong>1:15 ng</strong></th>
<th><strong>1:15 µl</strong></th>
</tr>
</thead>

<tbody>
<tr>
<td><strong>K1</strong></td>
<td>207,4</td>
<td>3</td>
<td>311,1</td>
<td>4,5</td>
</tr>

<tr>
<td><strong>K2a</strong></td>
<td>151,4</td>
<td>1,4</td>
<td>227,1</td>
<td>2,1</td>
</tr>

<tr>
<td><strong>K2b</strong></td>
<td>144,2</td>
<td>2,46</td>
<td>216,3</td>
<td>3,7</td>
</tr>

<tr>
<td><strong>K3a</strong></td>
<td>105,6</td>
<td>1,8</td>
<td>158,4</td>
<td>2,7</td>
</tr>

<tr>
<td><strong>K3b</strong></td>
<td>100</td>
<td>1,9</td>
<td>150</td>
<td>2,85</td>
</tr>

<tr>
<td><strong>K4a</strong></td>
<td>105,6</td>
<td>2,3</td>
<td>158,4</td>
<td>3,45</td>
</tr>

<tr>
<td><strong>K4b</strong></td>
<td>100,6</td>
<td>1,6</td>
<td>150,9</td>
<td>2,4</td>
</tr>
</tbody>
</table>

<p>Ligation</p>

<table>
<thead>
<tr>
<th></th>
<th><strong>K1</strong></th>
<th></th>
<th><strong>K2a</strong></th>
<th></th>
<th><strong>K2b</strong></th>
<th></th>
</tr>
</thead>

<tbody>
<tr>
<td></td>
<td><strong>1:10</strong></td>
<td><strong>1:15</strong></td>
<td><strong>1:10</strong></td>
<td><strong>1:15</strong></td>
<td><strong>1:10</strong></td>
<td><strong>1:15</strong></td>
</tr>

<tr>
<td><strong>DNA</strong></td>
<td>3</td>
<td>4,5</td>
<td>1,4</td>
<td>2,1</td>
<td>2,46</td>
<td>3,7</td>
</tr>

<tr>
<td><strong>Vector</strong></td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
<td>1,25</td>
</tr>

<tr>
<td><strong>Ligase Buffer x10</strong></td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
<td>2</td>
</tr>

<tr>
<td><strong>Ligase</strong></td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>

<tr>
<td><strong>H20</strong></td>
<td>12,75</td>
<td>11,25</td>
<td>14,35</td>
<td>13,65</td>
<td>13,3</td>
<td>12,05</td>
</tr>

<tr>
<td><strong>Total</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
<td><strong>20 µl</strong></td>
</tr>
</tbody>
</table>

<p>Incubation @RT for 20 min;</p>

<p>Inactivation @65 °C for 20 min;</p>

<p>Perform Heatshock after Evas Protocol;</p>

<p>Heatshock samples were not plated on plates, as the remaining Chloramphenicol<strong>+</strong> were found to be contaminated;
Samples were therefor stored @4 °C;</p>

<p><strong>Pending tasks, next steps, comments for next people in the Lab</strong></p>

<ul>
<li><p>Produce new Chloramph Plates</p></li>

<li><p>Test Transfection of HEK293T cells with K1</p></li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-204" class="hidden">
      <label for="panel-204" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              26
              
          </p>
          <p class="inline-block">calciumchoride transfection of heks with K1</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>calciumchoride transfection of heks with K1</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>Take Plates with negative control for measurement Part out of the incubator, colonies grew on alle of them!</p>

<p>Picked clones and prepared ON cultures with 8 ml LB-Amp, put them in the fridge until later;</p>

<p>Medium Change 1h  prior to Transfection of the HEKs, DMEM without Hepes (cells were already in Medium without Hepes);</p>

<p>Confluence ~ 70%;</p>

<p>Prepare new LB Agar for new Chloramphenicol Plates;
35 g LB Agar were weighed in and mixed thorougly with 1 l Water;</p>

<p>Autoclaved Medium;</p>

<p>Performed Calciumchlorid Transfection:</p>

<p>Perpared two  mixes:</p>

<p>A) 1 ml of 2x HBS in a 15 ml Falcon;</p>

<p>B)  DNA Mix:</p>

<table>
<thead>
<tr>
<th>Water</th>
<th>860 µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>Plasmid</td>
<td>20 µl (1 µg/µl)</td>
</tr>

<tr>
<td>2,5 M CaCl2</td>
<td>100 µl</td>
</tr>

<tr>
<td>2,5 mM Chloroquine</td>
<td>20 µl</td>
</tr>
</tbody>
</table>

<p>Carefully apply the DNA mix to the 2x HBS over the time of 1 min, while constantly blowing air into the HBS with a Pipette boy;</p>

<p>Vortex the resulting Mix thorougly and apply it dropwise to the Cells;</p>

<p>After the prepared Medium has cooled sufficiently added Chloamphenicol (35 mg/ml) to reach a concentration of 25 µg/ml &ndash;735 µl;</p>

<p>Poured new Chloramph Plates;
Seeded the Heatshock Samples from 25.9 that were stored in the fridge onto Chloramphenicol Plates;</p>

<p>Placed them in the Incubator;</p>

<p>Placed the earlier prepared ON cultures in the shaker;</p>

<p>Performed Medium Change 6 h post transfection with 10 ml Medium;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-205" class="hidden">
      <label for="panel-205" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              26
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">calciumchoride transfection of heks with K1</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>calciumchoride transfection of heks with K1</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p>Take Plates with negative control for measurement Part out of the incubator, colonies grew on alle of them!</p>

<p>Picked clones and prepared ON cultures with 8 ml LB-Amp, put them in the fridge until later;</p>

<p>Medium Change 1h  prior to Transfection of the HEKs, DMEM without Hepes (cells were already in Medium without Hepes);</p>

<p>Confluence ~ 70%;</p>

<p>Prepare new LB Agar for new Chloramphenicol Plates;
35 g LB Agar were weighed in and mixed thorougly with 1 l Water;</p>

<p>Autoclaved Medium;</p>

<p>Performed Calciumchlorid Transfection:</p>

<p>Perpared two  mixes:</p>

<p>A) 1 ml of 2x HBS in a 15 ml Falcon;</p>

<p>B)  DNA Mix:</p>

<table>
<thead>
<tr>
<th>Water</th>
<th>860 µl</th>
</tr>
</thead>

<tbody>
<tr>
<td>Plasmid</td>
<td>20 µl (1 µg/µl)</td>
</tr>

<tr>
<td>2,5 M CaCl2</td>
<td>100 µl</td>
</tr>

<tr>
<td>2,5 mM Chloroquine</td>
<td>20 µl</td>
</tr>
</tbody>
</table>

<p>Carefully apply the DNA mix to the 2x HBS over the time of 1 min, while constantly blowing air into the HBS with a Pipette boy;</p>

<p>Vortex the resulting Mix thorougly and apply it dropwise to the Cells;</p>

<p>After the prepared Medium has cooled sufficiently added Chloamphenicol (35 mg/ml) to reach a concentration of 25 µg/ml &ndash;735 µl;</p>

<p>Poured new Chloramph Plates;
Seeded the Heatshock Samples from 25.9 that were stored in the fridge onto Chloramphenicol Plates;</p>

<p>Placed them in the Incubator;</p>

<p>Placed the earlier prepared ON cultures in the shaker;</p>

<p>Performed Medium Change 6 h post transfection with 10 ml Medium;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-206" class="hidden">
      <label for="panel-206" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              27
              
          </p>
          <p class="inline-block">measuring measuring part</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>measuring measuring part</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p><strong>Measurement part:</strong></p>

<p>Take ON cultures out of shaker</p>

<p>Measure the smaples with spectrometer</p>

<p>Bl21 puc19 neg: 1,098</p>

<p>STAR puC19 neg: 1,17</p>

<p>TUNER pUC19: CFP neg: 1,41</p>

<p>Prepared samples and induced them with IPTG</p>

<table>
<thead>
<tr>
<th>Culture [mL]</th>
<th>LB Amp [mL]</th>
<th>IPTG 1mM with LB amp [mL]</th>
<th></th>
</tr>
</thead>

<tbody>
<tr>
<td>BL21 pos</td>
<td>5</td>
<td>0</td>
<td>5</td>
</tr>

<tr>
<td>STAR pos</td>
<td>5</td>
<td>0</td>
<td>5</td>
</tr>

<tr>
<td>TUNER pos</td>
<td>5</td>
<td>0</td>
<td>5</td>
</tr>

<tr>
<td>BL21 negative</td>
<td>2,5</td>
<td>2,5</td>
<td>5</td>
</tr>

<tr>
<td>STAR negative</td>
<td>2,5</td>
<td>2,5</td>
<td>5</td>
</tr>

<tr>
<td>TUNER negative</td>
<td>2</td>
<td>3</td>
<td>5</td>
</tr>
</tbody>
</table>

<p>Incubation for 3 hours at 37°C</p>

<p>Sonification:</p>

<p>Measuring cell desnity at spectrometer</p>

<p>acording to this 10^7 cells/ mL were added for a 10 mL endvolume solution</p>

<p>centrifugation for 10 min 4000 g</p>

<p>Resuspend in 10 mL sonification buffer</p>

<p>3 10s pulses at 30W</p>

<p>Remove cell debris at 10.000g for 10 min</p>

<p>Measuring in ELISA reader:</p>

<p>Not possible ELISA has just OD detection no flourescence</p>

<p><strong>E. Coli constructs:</strong></p>

<p>Plates from heatshock out of the incubator</p>

<p>they were all overgrown</p>

<p>K1 293T:</p>

<p>Medium chance</p>

<p>1:5 splitting of the 293T cell flask</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-207" class="hidden">
      <label for="panel-207" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              27
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">measuring measuring part</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>measuring measuring part</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong><em>Notes</em></strong></p>

<p><strong>Measurement part:</strong></p>

<p>Take ON cultures out of shaker</p>

<p>Measure the smaples with spectrometer</p>

<p>Bl21 puc19 neg: 1,098</p>

<p>STAR puC19 neg: 1,17</p>

<p>TUNER pUC19: CFP neg: 1,41</p>

<p>Prepared samples and induced them with IPTG</p>

<pre><code>            | Culture [mL] | LB Amp [mL] | IPTG 1mM with LB amp [mL] |
</code></pre>

<p>| &mdash;&mdash;&mdash;&mdash;&ndash; | &mdash;&mdash;&mdash;&mdash; | &mdash;&mdash;&mdash;&ndash; | &mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;- |
| BL21 pos       | 5            | 0           | 5                         |
| STAR pos       | 5            | 0           | 5                         |
| TUNER pos      | 5            | 0           | 5                         |
| BL21 negative  | 2,5          | 2,5         | 5                         |
| STAR negative  | 2,5          | 2,5         | 5                         |
| TUNER negative | 2            | 3           | 5                         |</p>

<p>Incubation for 3 hours at 37°C</p>

<p>Sonification:</p>

<p>Measuring cell desnity at spectrometer</p>

<p>acording to this 10^7 cells/ mL were added for a 10 mL endvolume solution</p>

<p>centrifugation for 10 min 4000 g</p>

<p>Resuspend in 10 mL sonification buffer</p>

<p>3 10s pulses at 30W</p>

<p>Remove cell debris at 10.000g for 10 min</p>

<p>Measuring in ELISA reader:</p>

<p>Not possible ELISA has just OD detection no flourescence</p>

<p><strong>E. Coli constructs:</strong></p>

<p>Plates from heatshock out of the incubator</p>

<p>they were all overgrown</p>

<p>K1 293T:</p>

<p>Medium chance</p>

<p>1:5 splitting of the 293T cell flask</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-208" class="hidden">
      <label for="panel-208" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              29
              
          </p>
          <p class="inline-block">ITPG induction of ON culture</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>ITPG induction of ON culture</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>OD measurement of the ON cultures</p>

<table>
<thead>
<tr>
<th>Culture</th>
<th>OD</th>
<th>amount ON culture for OD of 0,5 in 5 mL [mL]</th>
</tr>
</thead>

<tbody>
<tr>
<td>Tuner CFP pos</td>
<td>2,2075</td>
<td>1,13</td>
</tr>

<tr>
<td>Star CFP pos</td>
<td>2,59</td>
<td>0,96</td>
</tr>

<tr>
<td>BL21 CFP pos</td>
<td>1,757</td>
<td>1,48</td>
</tr>

<tr>
<td>Tuner CFP neg</td>
<td>1,97</td>
<td>1,25</td>
</tr>

<tr>
<td>Star CFP neg</td>
<td>1,769</td>
<td>1,43</td>
</tr>

<tr>
<td>BL21 CFP neg</td>
<td>1,672</td>
<td>1,5</td>
</tr>
</tbody>
</table>

<p>5mL induction culture for each ON culture with</p>

<ol>
<li><p>0,5 mM IPTG (2,5mL of 1mM IPTG solution in LB Amp)</p></li>

<li><p>0,3 mM IPTG (1,5mL of 1mM IPTG solution in LB Amp)</p></li>
</ol>

<p>ad 5 mL with LB</p>

<p>ON cultures filled up to ca 5 mL with LB amp</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-209" class="hidden">
      <label for="panel-209" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              29
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">ITPG induction of ON culture</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>ITPG induction of ON culture</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td></td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        <p><strong>Notes</strong></p>

<p>OD measurement of the ON cultures</p>

<p>|||
|&mdash; |&mdash; |&mdash; |</p>

<table>
<thead>
<tr>
<th>Culture</th>
<th>OD</th>
<th>amount ON culture for OD of 0,5 in 5 mL [mL]</th>
</tr>
</thead>

<tbody>
<tr>
<td>Tuner CFP pos</td>
<td>2,2075</td>
<td>1,13</td>
</tr>

<tr>
<td>Star CFP pos</td>
<td>2,59</td>
<td>0,96</td>
</tr>

<tr>
<td>BL21 CFP pos</td>
<td>1,757</td>
<td>1,48</td>
</tr>

<tr>
<td>Tuner CFP neg</td>
<td>1,97</td>
<td>1,25</td>
</tr>

<tr>
<td>Star CFP neg</td>
<td>1,769</td>
<td>1,43</td>
</tr>

<tr>
<td>BL21 CFP neg</td>
<td>1,672</td>
<td>1,5</td>
</tr>
</tbody>
</table>

<p>5mL induction culture for each ON culture with</p>

<p>\1. 0,5 mM IPTG (2,5mL of 1mM IPTG solution in LB Amp)</p>

<p>\2. 0,3 mM IPTG (1,5mL of 1mM IPTG solution in LB Amp)</p>

<p>ad 5 mL with LB</p>

<p>ON cultures filled up to ca 5 mL with LB amp</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-210" class="hidden">
      <label for="panel-210" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              30
              
          </p>
          <p class="inline-block">transfection</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>transfection</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<p>Medium changen of the HEK plates (K2a, K2b) 1 h pre-transfection;</p>

<p>Harvesting supernatant from last weeks K1;</p>

<p>Splitting of the 5 flasks with HEK cells (ratio 1:5);</p>

<p>DNA for transfection stems from the Midi-Prep of 14th of September;</p>

<p>20 µg of DNA were brought into a 1 µg/µl concentration for the transfection;</p>

<p>Transfection with K2a, K2b according to Annes&rsquo; protocol, time of transfection approx. 11:00;</p>

<p>Changed Medium 8 h post-transfection with 10 ml DMEM;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-211" class="hidden">
      <label for="panel-211" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              30
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">transfection</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>transfection</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p>Medium changen of the HEK plates (K2a, K2b) 1 h pre-transfection;</p>

<p>Harvesting supernatant from last weeks K1;</p>

<p>Splitting of the 5 flasks with HEK cells (ratio 1:5);</p>

<p>DNA for transfection stems from the Midi-Prep of 14th of September;</p>

<p>20 µg of DNA were brought into a 1 µg/µl concentration for the transfection;</p>

<p>Transfection with K2a, K2b according to Annes&rsquo; protocol, time of transfection approx. 11:00;</p>

<p>Changed Medium 8 h post-transfection with 10 ml DMEM;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-212" class="hidden">
      <label for="panel-212" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              30
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">transfection</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>transfection</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<p>Medium changen of the HEK plates (K2a, K2b) 1 h pre-transfection;</p>

<p>Harvesting supernatant from last weeks K1;</p>

<p>Splitting of the 5 flasks with HEK cells (ratio 1:5);</p>

<p>DNA for transfection stems from the Midi-Prep of 14th of September;</p>

<p>20 µg of DNA were brought into a 1 µg/µl concentration for the transfection;</p>

<p>Transfection with K2a, K2b according to Annes&rsquo; protocol, time of transfection approx. 11:00;</p>

<p>Changed Medium 8 h post-transfection with 10 ml DMEM;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

            
            </div>
            

            <h2 class="my-10 text-3xl">October</h2>
            

            <div class="accordion flex flex-col items-center justify-center">
            
        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-213" class="hidden">
      <label for="panel-213" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
          </p>
          <p class="inline-block">Western blot</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Western blot</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<ol>
<li><p>Preperation of a 10% Bis-tris Gel.</p></li>

<li><p>Mixing of 10 µL harvest of K1 day1, K1 day2, K2a, and K2b with 2x loading buffer</p></li>

<li><p>95°C for 5 min</p></li>

<li><p>Loading of the gel with 5µL marker |K1 day1|K1 day4|K2a|K2b (samples 15µL)</p></li>

<li><p>Running gel for 1 h at 130V in 1x MOPS buffer</p></li>

<li><p>Blotting to the membrane at 65mA for 1h30</p></li>

<li><p>Blocking membrane in 10% milkpowder for 1h30</p></li>

<li><p>Washing with PBST</p></li>

<li><p>Adding primary antibody (mouse anti-Histag in 1:2000 dilution) 10mL in 10% milkpowder</p></li>

<li><p>Shaking overnight at 4°C</p></li>

<li><p>3 times washing for 10 min with PBST</p></li>

<li><p>Adding secondary antibody (HRP anti mouse in 1:2000 dilution) 10 ml in 10% milkpowder</p></li>

<li><p>Shaking at RT for 1h</p></li>

<li><p>3 times washing for 10 min with PBST</p></li>

<li><p>Adding 2mL of the reaction mix directly before taking the picture of the blot</p></li>
</ol>

<p><img src="/labjournal-entries/images/Westernblot_02_10_2019_Marker.tif" alt="&lt;Westernblot_02_10_2019_Marker&gt;" />
<img src="/labjournal-entries/images/Westernblot_02_10_2019.tif" alt="&lt;Westernblot_02_10_2019&gt;" />
<img src="/labjournal-entries/images/WB_02.10.png" alt="&lt;Westernblot_02_10_2019&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-214" class="hidden">
      <label for="panel-214" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Harvesting K2a and K2b supernatant</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Harvesting K2a and K2b supernatant</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p>Harvesting K2a and K2b supernatant, reapply 4 ml of Medium to the plates;</p>

<p>Prepared new plates for transfection;</p>

<p>Took 3 flasks with HEk Cells and collected the cells in a 50 ml Falcon after making them loose with trypsin and inactivating it again with Medum;</p>

<p>Cells were counted using a Neubauer Chamber in a 1:1 Dilution with Trypan Blue;<br />
Counted: 115/131/119 &ndash;Mean is 121,6 Cells per quadrant &ndash;Therefore 2,432*106  cells/ml;</p>

<p>Seed 4*106 cells per Plate for the transfection tomorrow (1,64 ml of the cell suspension);</p>

<p>The remaining 5,4 ml (~13*106  cells) were applied to a new cell culture flask;<br />
Everything was put in the incubator @37 °C, 5% Co2;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-215" class="hidden">
      <label for="panel-215" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Harvesting K2a and K2b supernatant</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Harvesting K2a and K2b supernatant</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<p>Harvesting K2a and K2b supernatant, reapply 4 ml of Medium to the plates;</p>

<p>Prepared new plates for transfection;</p>

<p>Took 3 flasks with HEk Cells and collected the cells in a 50 ml Falcon after making them loose with trypsin and inactivating it again with Medum;</p>

<p>Cells were counted using a Neubauer Chamber in a 1:1 Dilution with Trypan Blue;<br />
Counted: 115/131/119 &ndash;Mean is 121,6 Cells per quadrant &ndash;Therefore 2,432*106  cells/ml;</p>

<p>Seed 4*106 cells per Plate for the transfection tomorrow (1,64 ml of the cell suspension);</p>

<p>The remaining 5,4 ml (~13*106  cells) were applied to a new cell culture flask;<br />
Everything was put in the incubator @37 °C, 5% Co2;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-216" class="hidden">
      <label for="panel-216" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              1
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Western blot</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Western blot</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<ol>
<li><p>Preperation of a 10% Bis-tris Gel.</p></li>

<li><p>Mixing of 10 µL harvest of K1 day1, K1 day2, K2a, and K2b with 2x loading buffer</p></li>

<li><p>95°C for 5 min</p></li>

<li><p>Loading of the gel with 5µL marker |K1 day1|K1 day4|K2a|K2b (samples 15µL)</p></li>

<li><p>Running gel for 1 h at 130V in 1x MOPS buffer</p></li>

<li><p>Blotting to the membrane at 65mA for 1h30</p></li>

<li><p>Blocking membrane in 10% milkpowder for 1h30</p></li>

<li><p>Washing with PBST</p></li>

<li><p>Adding primary antibody (mouse anti-Histag in 1:2000 dilution) 10mL in 10% milkpowder</p></li>

<li><p>Shaking overnight at 4°C</p></li>

<li><p>3 times washing for 10 min with PBST</p></li>

<li><p>Adding secondary antibody (HRP anti mouse in 1:2000 dilution) 10 ml in 10% milkpowder</p></li>

<li><p>Shaking at RT for 1h</p></li>

<li><p>3 times washing for 10 min with PBST</p></li>

<li><p>Adding 2mL of the reaction mix directly before taking the picture of the blot</p></li>
</ol>

<p><img src="/labjournal-entries/images/WB_02.10.png" alt="&lt;Westernblot_02_10_2019&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-217" class="hidden">
      <label for="panel-217" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              2
              
          </p>
          <p class="inline-block">Harvesting supernatant</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Harvesting supernatant</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p>Change Medium 1h pre-transfection on the 6 prepared plates;</p>

<p>Harvesting supernatant from yesterdays transformation (K2a, K2b);</p>

<p>DNA for transfection stems from the Midi-Prep of 14th of September (K1-2, K2a-1, K2b-1);</p>

<p>20 µg of each DNA were brought into a 1 µg/µl concentration for the transfection;</p>

<p>With each Construct two plates were transfected (K1, K2a, K2b) according to Annes&rsquo; protocol, time of transfection approx. 11:30;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-218" class="hidden">
      <label for="panel-218" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              2
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Harvesting supernatant</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Harvesting supernatant</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em>**<br />
Notes**</em></p>

<p>Change Medium 1h pre-transfection on the 6 prepared plates;</p>

<p>Harvesting supernatant from yesterdays transformation (K2a, K2b);</p>

<p>DNA for transfection stems from the Midi-Prep of 14th of September (K1-2, K2a-1, K2b-1);</p>

<p>20 µg of each DNA were brought into a 1 µg/µl concentration for the transfection;</p>

<p>With each Construct two plates were transfected (K1, K2a, K2b) according to Annes&rsquo; protocol, time of transfection approx. 11:30;</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-219" class="hidden">
      <label for="panel-219" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
          </p>
          <p class="inline-block">Test Miniprep of used strains for Measurement Part</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Test Miniprep of used strains for Measurement Part</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Miniprep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>Notes</p>

<p>Check XGal/IPTG/Amp+ Plates &ndash;overgrown, all colonies blue (on negative and positive plates!)
Induced the ON cultures with 0 mM, 0,33 mM and 0,5 mM IPTG;
Placed them in the incubator
Meanwhile Miniprep with 1 ml of each of the uninduced CFP pos cultures was performed.
Applied 10 µl sample + 3 µl 6xDye + 5 µl H2O to a 0.8% Agarose Gel</p>

<p>Gel:     Ladder/Bl21/Star/Tuner</p>

<p>&ndash;Tuner does not contain the desired insert. This notion is corraborated by the pellets colour.</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-220" class="hidden">
      <label for="panel-220" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              3
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Test Miniprep of used strains for Measurement Part</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Test Miniprep of used strains for Measurement Part</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Miniprep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>Notes</p>

<p>Check XGal/IPTG/Amp+ Plates &ndash;overgrown, all colonies blue (on negative and positive plates!)
Induced the ON cultures with 0 mM, 0,33 mM and 0,5 mM IPTG;
Placed them in the incubator
Meanwhile Miniprep with 1 ml of each of the uninduced CFP pos cultures was performed.
Applied 10 µl sample + 3 µl 6xDye + 5 µl H2O to a 0.8% Agarose Gel</p>

<p>Gel:     Ladder/Bl21/Star/Tuner</p>

<p>&ndash;Tuner does not contain the desired insert. This notion is corraborated by the pellets colour.</p>

<p><img src="/labjournal-entries/images/WhatsApp_Image_2019-10-20_at_23.46.26.jpeg" alt="&lt;Agarose Gel of Measurement Part&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-221" class="hidden">
      <label for="panel-221" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              5
              
          </p>
          <p class="inline-block">Harvesting</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Harvesting</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<p>Harvesting of all transfected cells:</p>

<p>collection of the medium and centrifugation at max for 10 minutes.</p>

<p>The supernatant still needs to be filtrated.</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-222" class="hidden">
      <label for="panel-222" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              5
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Harvesting</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Harvesting</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p>Harvesting of all transfected cells:</p>

<p>collection of the medium and centrifugation at max for 10 minutes.</p>

<p>The supernatant still needs to be filtrated.</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-223" class="hidden">
      <label for="panel-223" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
          </p>
          <p class="inline-block">Westernblot</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Westernblot</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<ol>
<li><p>Removal of of the storage buffer by centrifugation with 100 g for 1 min</p></li>

<li><p>Adding 600 µL binding buffer and centrifugation with 100 g for 1 min</p></li>

<li><p>Adding the whole sample in multiple steps to the colume with each time centrifugation for 30s at 100g (Keeping the flow through)</p></li>

<li><p>Washing once with 600 µL of binding buffer</p></li>

<li><p>First Elution step: Adding 200µL Elution buffer to the colum</p></li>

<li><p>Second Elution step: Adding 200 µL Elution buffer to the colum</p></li>
</ol>

<p>Running 15% SDS Gel at 300V for 30 min</p>

<p>10 µL of the samples were applied and 7 µL of the ladder</p>

<p>Pipetting scheme:</p>

<p>Ladder | Wash K1 | Wash K2a | Wash K2b | Elution1 K1 | Elution 1 K2a | Elution 1 K2b | Elution 2 K1 | Elution 2 K2a | Elution 2 K2b</p>

<p>Stain the finished Gel with Coomasie and unstain it sufficiently to see clear bands;</p>

<p>Measure the samples in the Nanodrop A280:</p>

<p>Determine the Extinction Coefficient for each of the constructs with ProtParam (OnlineTool);</p>

<table>
<thead>
<tr>
<th></th>
<th></th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>Mean</th>
</tr>
</thead>

<tbody>
<tr>
<td>Elution 1</td>
<td>K1</td>
<td>25,54</td>
<td>25,08</td>
<td>27,04</td>
<td>25,9</td>
</tr>

<tr>
<td></td>
<td>K2a</td>
<td>75,6</td>
<td>72,57</td>
<td>72,02</td>
<td>73,22</td>
</tr>

<tr>
<td></td>
<td>K2b</td>
<td>47,28</td>
<td>48,16</td>
<td>48,99</td>
<td>48,14</td>
</tr>

<tr>
<td>Elution 2</td>
<td>K1</td>
<td>13,0</td>
<td>12,02</td>
<td>12,41</td>
<td>12,47</td>
</tr>

<tr>
<td></td>
<td>K2a</td>
<td>11,74</td>
<td>11,58</td>
<td>12,53</td>
<td>11,95</td>
</tr>

<tr>
<td></td>
<td>K2b</td>
<td>25,09</td>
<td>24,75</td>
<td>24,78</td>
<td>24,87</td>
</tr>
</tbody>
</table>

<p>For a 1:1 ratio of K2a and b following volumes were added together:</p>

<p>E1: K2b 100µL with K2a 65,9µL</p>

<p>E2: K2a 100µL with K2b 48µL</p>

<p>Samples were sended to Kiel for killing assay.</p>

<p>10µL of each sample was kept for a westernblot to verify that the tag and catcherreaction happend.</p>

<p><img src="/labjournal-entries/images/Comassiegel.tif" alt="&lt;Comassiegel&gt;" /></p>

<p><strong>Protocol</strong></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-224" class="hidden">
      <label for="panel-224" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              8
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Westernblot</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Westernblot</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer, Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<ol>
<li><p>Removal of of the storage buffer by centrifugation with 100 g for 1 min</p></li>

<li><p>Adding 600 µL binding buffer and centrifugation with 100 g for 1 min</p></li>

<li><p>Adding the whole sample in multiple steps to the colume with each time centrifugation for 30s at 100g (Keeping the flow through)</p></li>

<li><p>Washing once with 600 µL of binding buffer</p></li>

<li><p>First Elution step: Adding 200µL Elution buffer to the colum</p></li>

<li><p>Second Elution step: Adding 200 µL Elution buffer to the colum</p></li>
</ol>

<p>Running 15% SDS Gel at 300V for 30 min</p>

<p>10 µL of the samples were applied and 7 µL of the ladder</p>

<p>Pipetting scheme:</p>

<p>Ladder | Wash K1 | Wash K2a | Wash K2b | Elution1 K1 | Elution 1 K2a | Elution 1 K2b | Elution 2 K1 | Elution 2 K2a | Elution 2 K2b</p>

<p>Stain the finished Gel with Coomasie and unstain it sufficiently to see clear bands;</p>

<p>Measure the samples in the Nanodrop A280:</p>

<p>Determine the Extinction Coefficient for each of the constructs with ProtParam (OnlineTool);</p>

<table>
<thead>
<tr>
<th></th>
<th></th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>Mean</th>
</tr>
</thead>

<tbody>
<tr>
<td>Elution 1</td>
<td>K1</td>
<td>25,54</td>
<td>25,08</td>
<td>27,04</td>
<td>25,9</td>
</tr>

<tr>
<td></td>
<td>K2a</td>
<td>75,6</td>
<td>72,57</td>
<td>72,02</td>
<td>73,22</td>
</tr>

<tr>
<td></td>
<td>K2b</td>
<td>47,28</td>
<td>48,16</td>
<td>48,99</td>
<td>48,14</td>
</tr>

<tr>
<td>Elution 2</td>
<td>K1</td>
<td>13,0</td>
<td>12,02</td>
<td>12,41</td>
<td>12,47</td>
</tr>

<tr>
<td></td>
<td>K2a</td>
<td>11,74</td>
<td>11,58</td>
<td>12,53</td>
<td>11,95</td>
</tr>

<tr>
<td></td>
<td>K2b</td>
<td>25,09</td>
<td>24,75</td>
<td>24,78</td>
<td>24,87</td>
</tr>
</tbody>
</table>

<p>For a 1:1 ratio of K2a and b following volumes were added together:</p>

<p>E1: K2b 100µL with K2a 65,9µL</p>

<p>E2: K2a 100µL with K2b 48µL</p>

<p>Samples were sended to Kiel for killing assay.</p>

<p>10µL of each sample was kept for a westernblot to verify that the tag and catcherreaction happend.</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-225" class="hidden">
      <label for="panel-225" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              9
              
          </p>
          <p class="inline-block">Validate data from 3.10.19</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Validate data from 3.10.19</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p>Notes</p>

<p>Performed restriction digest with Tuner and BL21 (control) to check the result from 3rd October.
Took Miniprep samples from 3.10.19 and digested the samples with SalI and EcoRI;
Incubate @ 37 °C for 45 min;
Inactivation fro 20 min @ 65 °C;</p>

<p>Applied digested samples to 0.8% Agarose Gel:
Ladder/Bl21/Tuner</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-226" class="hidden">
      <label for="panel-226" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              9
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Validate data from 3.10.19</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Validate data from 3.10.19</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<p>Notes</p>

<p>Performed restriction digest with Tuner and BL21 (control) to check the result from 3rd October.
Took Miniprep samples from 3.10.19 and digested the samples with SalI and EcoRI;
Incubate @ 37 °C for 45 min;
Inactivation fro 20 min @ 65 °C;</p>

<p>Applied digested samples to 0.8% Agarose Gel:
Ladder/Bl21/Tuner</p>

<p><img src="/labjournal-entries/images/d57c97c0-85f0-4e0c-8171-5f10e47b804f.jfif" alt="&lt;Westernblot_1&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-227" class="hidden">
      <label for="panel-227" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              10
              
          </p>
          <p class="inline-block">Westernblot of the purified proteins</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Westernblot of the purified proteins</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>10% BisTris Gel</p>

<p>10µL purified samples were mixed with 2x loading dye.</p>

<p>95°C for 5 min</p>

<p>Running gel for 1 h 130V:</p>

<p>Pipetting scheme of the gel: 5µL each</p>

<p>Ladder | E1 K1 | E1 K2a | K1 K2b | K1 K2ab | E2 K1 | E2 K2a | E2 K2b | E2 K2ab</p>

<p>Blotting for 1:30 h at 65mA</p>

<p>Blocking the membrane with 10% milkpowdersolution at 4°C for 5h</p>

<p>1x washing with PBST</p>

<p>Adding his-Ab (1:5000 in milkpowdersolution) overnight at 4°C</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-228" class="hidden">
      <label for="panel-228" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              10
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Westernblot of the purified proteins</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Westernblot of the purified proteins</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>10% BisTris Gel</p>

<p>10µL purified samples were mixed with 2x loading dye.</p>

<p>95°C for 5 min</p>

<p>Running gel for 1 h 130V:</p>

<p>Pipetting scheme of the gel: 5µL each</p>

<p>Ladder | E1 K1 | E1 K2a | K1 K2b | K1 K2ab | E2 K1 | E2 K2a | E2 K2b | E2 K2ab</p>

<p>Blotting for 1:30 h at 65mA</p>

<p>Blocking the membrane with 10% milkpowdersolution at 4°C for 5h</p>

<p>1x washing with PBST</p>

<p>Adding his-Ab (1:5000 in milkpowdersolution) overnight at 4°C</p>

<p><img src="/labjournal-entries/images/wb2019-10-20" alt="&lt;Westernblot&gt;" /></p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-229" class="hidden">
      <label for="panel-229" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
          </p>
          <p class="inline-block">New Transfection</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>New Transfection</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p>Splitting HEK293T cells in 3 wells of a 6 well plate</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-230" class="hidden">
      <label for="panel-230" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              15
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">New Transfection</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>New Transfection</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em>
Notes</p>

<p>Splitting HEK293T cells in 3 wells of a 6 well plate</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-231" class="hidden">
      <label for="panel-231" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              16
              
          </p>
          <p class="inline-block">New Transfection, Purification, Western blot</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>New Transfection, Purification, Western blot</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<ul>
<li>Medium change 1 h before transfection</li>
<li>For transfection 2 µg DNA were used of K1-1, K2a-1 and K2b-1: Diluted 500 µL medium with nothing added</li>
<li>Mixed with 500µL Medium with nothing added with 4µL genjet</li>
<li>Incubation for 10 min at RT</li>
</ul>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-232" class="hidden">
      <label for="panel-232" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              16
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">New Transfection, Purification, Western blot</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>New Transfection, Purification, Western blot</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>Medium change 1 h before transfection
For transfection 2 µg DNA were used of K1-1, K2a-1 and K2b-1: Diluted 500 µL medium with nothing added
Mixed with 500µL Medium with nothing added with 4µL genjet
Incubation for 10 min at RT</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-233" class="hidden">
      <label for="panel-233" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              18
              
          </p>
          <p class="inline-block">New Transfection, Purification, Western blot</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>New Transfection, Purification, Western blot</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>Harvesting of the cells
Centrifuagtion at 2500 rpm for 5 min
600µL of supernatant were used for the purification via Histrap</p>

<p>Histrap Purification:
centrifuge at 4°C
open columns a quater and placing it in a 2 mL eppi
Centrifuation at 500 rpm for 1 min
Adding 600 µL Binding buffer
Centrifugation at 500 rpm for 1 min
Adding 600µL sample
Centrifugation at 500 rpm for 1 min
Washing with 600µL binding buffer
Centrifugation at 500 rpm for 1 min
3x Elution step: Adding 100µL Elution buffer and centrifugation at 500 rpm for 1 min</p>

<p>Tag-Catcher reaction:
Adding same colume K2a and K2b and incubate them 20 min at RT
For this samples: Supernatant (SN) harvest, Elution (E) 1-3</p>

<p>Gel running :</p>

<p>at 130V for 1 h with following pipetting scheme</p>

<p>Gel1
M, SN K1, SN K2a, SN K2b, SN K2ab, E1 K1, E1 K2a, E1 K2b, E1 K2ab, E2 K1, E2 K2a, E2 K2b, E2 K2ab</p>

<p>Gel 2
M, SN K1, SN K2a, SN K2b, SN K2ab, E3 K1, E3 K2a, E3 K2b, E3 K2ab</p>

<p>Marker 5µL, Samples 20µL (10µL sample+10µL loading dye)
After loading dye adding samples were cooked for 2 min at 95°C except the SN samples of the gel 2</p>

<p>Blotting:
Both gels were blotted semidry for 1:23 h at 130mA</p>

<p>Blocking:
Membranes were blocked for 1h 45 with 10% milkpowdersolution</p>

<p>Primary Ab:
2 washing steps with PBST
anti-Histag ab was diluted 1:2000 in 10mL 10% milkpowdersolution and added to the membrane
Incubation overnight at 4°C</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-234" class="hidden">
      <label for="panel-234" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              18
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">New Transfection, Purification, Western blot</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>New Transfection, Purification, Western blot</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>Harvesting of the cells
Centrifuagtion at 2500 rpm for 5 min
600µL of supernatant were used for the purification via Histrap</p>

<p>Histrap Purification:
centrifuge at 4°C
open columns a quater and placing it in a 2 mL eppi
Centrifuation at 500 rpm for 1 min
Adding 600 µL Binding buffer
Centrifugation at 500 rpm for 1 min
Adding 600µL sample
Centrifugation at 500 rpm for 1 min
Washing with 600µL binding buffer
Centrifugation at 500 rpm for 1 min
3x Elution step: Adding 100µL Elution buffer and centrifugation at 500 rpm for 1 min</p>

<p>Tag-Catcher reaction:
Adding same colume K2a and K2b and incubate them 20 min at RT
For this samples: Supernatant (SN) harvest, Elution (E) 1-3</p>

<p>Gel running :</p>

<p>at 130V for 1 h with following pipetting scheme</p>

<p>Gel1
M, SN K1, SN K2a, SN K2b, SN K2ab, E1 K1, E1 K2a, E1 K2b, E1 K2ab, E2 K1, E2 K2a, E2 K2b, E2 K2ab</p>

<p>Gel 2
M, SN K1, SN K2a, SN K2b, SN K2ab, E3 K1, E3 K2a, E3 K2b, E3 K2ab</p>

<p>Marker 5µL, Samples 20µL (10µL sample+10µL loading dye)
After loading dye adding samples were cooked for 2 min at 95°C except the SN samples of the gel 2</p>

<p>Blotting:
Both gels were blotted semidry for 1:23 h at 130mA</p>

<p>Blocking:
Membranes were blocked for 1h 45 with 10% milkpowdersolution</p>

<p>Primary Ab:
2 washing steps with PBST
anti-Histag ab was diluted 1:2000 in 10mL 10% milkpowdersolution and added to the membrane
Incubation overnight at 4°C</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-235" class="hidden">
      <label for="panel-235" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              19
              
          </p>
          <p class="inline-block">Test Miniprep of used strains for Measurement Part</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Test Miniprep of used strains for Measurement Part</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Miniprep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p>3x washing with PBST for each time 10 min at RT
Adding primary antibody which was diluted 1:2000 in 10% milkpowdersolution
1h RT incubation
3x washing with PBST for wach time 10 min at RT
Adding for each blot directly for taking the picture the solution for HRP (2mL)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-236" class="hidden">
      <label for="panel-236" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              19
              
              <strong class="inline-block opacity-75 text-sm">(2)</strong>
              
          </p>
          <p class="inline-block">Test Miniprep of used strains for Measurement Part</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Test Miniprep of used strains for Measurement Part</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>Miniprep</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Leon Altmann</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p>Notes</p>

<p>3x washing with PBST for each time 10 min at RT
Adding primary antibody which was diluted 1:2000 in 10% milkpowdersolution
1h RT incubation
3x washing with PBST for wach time 10 min at RT
Adding for each blot directly for taking the picture the solution for HRP (2mL)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-237" class="hidden">
      <label for="panel-237" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              19
              
              <strong class="inline-block opacity-75 text-sm">(3)</strong>
              
          </p>
          <p class="inline-block">Westernblot</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Westernblot</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><strong>Notes</strong></p>

<p>3x washing with PBST for each time 10 min at RT
Adding primary antibody which was diluted 1:2000 in 10% milkpowdersolution
1h RT incubation
3x washing with PBST for wach time 10 min at RT
Adding for each blot directly for taking the picture the solution for HRP (2mL)</p>

        </div>

      </div>
    </div>

    


    

        
        

        
            
        

        

    <div class="w-full">
      <input type="checkbox" name="panel" id="panel-238" class="hidden">
      <label for="panel-238" class="accordion__label">
          <p class="inline-block font-bold text-xl mr-5 opacity-50">
              19
              
              <strong class="inline-block opacity-75 text-sm">(4)</strong>
              
          </p>
          <p class="inline-block">Westernblot</p>
      </label>
      <div class="accordion__content overflow-hidden bg-grey-lighter shadow-2xl rounded-lg my-2">
        <p class="accordion__body" id="panel1">
            
        <table class="mb-10">
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Aim / Task</td>
                <td>Westernblot</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Protocol</td>
                <td>-</td>
            </tr>
            <tr class="p-2">
                <td class="font-bold text-xs text-gray-700 pr-5">Participants</td>
                <td>Eva Neugebauer</td>
            </tr>
        </table>

        <div class="labjournal">
        

<h2 id="protocol">Protocol</h2>

<p><em><strong>Notes</strong></em></p>

<p>Notes</p>

<p>3x washing with PBST for each time 10 min at RT
Adding primary antibody which was diluted 1:2000 in 10% milkpowdersolution
1h RT incubation
3x washing with PBST for wach time 10 min at RT
Adding for each blot directly for taking the picture the solution for HRP (2mL)</p>

<p><img src="/labjournal-entries/images/Gel_1.png" alt="&lt;Westernblot_1&gt;" />
<img src="/labjournal-entries/images/Gel_2.png" alt="&lt;Westernblot_2&gt;" /></p>

        </div>

      </div>
    </div>

    


    

</div>

</div>



<footer class="bg-grey-darkest w-full">

    <div class="container mx-auto py-10">

        <div class="table w-full">

            <div class="block sm:table-cell sm:w-1/6">
                <p class="uppercase text-grey-700 text-sm sm:mb-6">&nbsp;</p>
                <ul class="list-reset text-xs mb-6">
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/" class="no-underline text-grey hover:text-grey-dark">Home</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/safety/" class="no-underline text-grey hover:text-grey-dark">Safety</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/human-practices/human-practices/" class="no-underline text-grey hover:text-grey-dark">Human Practices</a></li>
                </ul>
            </div>

            <div class="block sm:table-cell sm:w-1/6">
                <p class="uppercase text-yellow-700 font-bold text-sm sm:mb-6">Team</p>
                <ul class="list-reset text-xs mb-6">
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/team-members/" class="no-underline text-grey hover:text-grey-dark">Team Members</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/collaborations/" class="no-underline text-grey hover:text-grey-dark">Collaborations</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/attributions/" class="no-underline text-grey hover:text-grey-dark">Attributions</a></li>
                </ul>
            </div>

            <div class="block sm:table-cell sm:w-1/6">
                <p class="uppercase text-green-700 font-bold text-sm sm:mb-6">Project</p>
                <ul class="list-reset text-xs mb-6">
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/project/description/" class="no-underline text-grey hover:text-grey-dark">Description</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/project/experiments/" class="no-underline text-grey hover:text-grey-dark">Experiments</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/labjournal-entries/" class="no-underline text-grey hover:text-grey-dark">Notebook</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/project/contribution/" class="no-underline text-grey hover:text-grey-dark">Contribution</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/project/results/" class="no-underline text-grey hover:text-grey-dark">Results</a></li>
                </ul>
            </div>

            <div class="block sm:table-cell sm:w-1/6">
                <p class="uppercase text-blue-700 font-bold text-sm sm:mb-6">Parts</p>
                <ul class="list-reset text-xs mb-6">
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/parts/overview/" class="no-underline text-grey hover:text-grey-dark">Parts Overview</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/parts/basic/" class="no-underline text-grey hover:text-grey-dark">Basic Parts</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/parts/composite/" class="no-underline text-grey hover:text-grey-dark">Composite Parts</a></li>
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/parts/collection/" class="no-underline text-grey hover:text-grey-dark">Parts Collection</a></li>
                </ul>
            </div>

            

            <div class="block sm:table-cell sm:w-1/6">
                <p class="uppercase text-red-700 font-bold text-sm sm:mb-6">Awards</p>
                <ul class="list-reset text-xs mb-6">
                    
                    
                    
                    <li class="mt-2 inline-block mr-2 sm:block sm:mr-0"><a href="http://example.org/awards/model/" class="no-underline text-grey hover:text-grey-dark">Model</a></li>
                    
                    
                </ul>
            </div>

        </div>

        
        <div class="flex items-center justify-end mr-32 align-middle">
            <div class="text-xs text-grey-dark mr-5">Join us on social media</div>
            <ul class="list-reset">
            
            <li class="inline-block text-2xl m-2"><a class="no-underline hover:text-gray-600 text-grey" href="https://facebook.com/igemErlangen" target="_blank"><i class="fab fa-facebook-square"></i></a></li>
            
            
            <li class="inline-block text-2xl m-2"><a class="no-underline hover:text-gray-600 text-grey" target="_blank" href="https://www.twitter.com/iGEM_Erlangen"><i class="fab fa-twitter"></i></a></li>
            
            
            <li class="inline-block text-2xl m-2"><a class="no-underline hover:text-gray-600 text-grey" target="_blank" href="https://www.instagram.com/igem.erlangen"><i class="fab fa-instagram"></i></a></li>
            
            
            <li class="inline-block text-2xl m-2"><a class="no-underline hover:text-gray-600 text-grey" target="_blank" href="https://www.youtube.com/channel/UCxT11JmhRH62aBn-HZxGhQA"><i class="fab fa-youtube"></i></a></li>
            
            </ul>
        </div>
        

    </div>

</footer>

</body>
</html>